{
 "cells": [
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Data set web-search\n",
    "\n",
    "> Can we run through the papers in a corpus and characterize the existance of datasets described in the paper."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Basics"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Python Imports\n",
    "\n",
    "Setting python imports, environment variables, and other crucial set up parameters here.  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from alhazen.aliases import *\n",
    "from alhazen.core import lookup_chat_models\n",
    "from alhazen.agent import AlhazenAgent\n",
    "from alhazen.schema_sqla import *\n",
    "from alhazen.core import lookup_chat_models\n",
    "from alhazen.tools.basic import AddCollectionFromEPMCTool, DeleteCollectionTool\n",
    "from alhazen.tools.paperqa_emulation_tool import PaperQAEmulationTool\n",
    "from alhazen.tools.metadata_extraction_tool import * \n",
    "from alhazen.tools.protocol_extraction_tool import *\n",
    "from alhazen.tools.tiab_classifier_tool import *\n",
    "from alhazen.tools.tiab_extraction_tool import *\n",
    "from alhazen.tools.tiab_mapping_tool import *\n",
    "from alhazen.toolkit import *\n",
    "from alhazen.utils.jats_text_extractor import NxmlDoc\n",
    "\n",
    "from alhazen.utils.ceifns_db import Ceifns_LiteratureDb, create_ceifns_database, drop_ceifns_database, backup_ceifns_database\n",
    "\n",
    "from alhazen.utils.searchEngineUtils import *\n",
    "\n",
    "\n",
    "from langchain.callbacks.tracers import ConsoleCallbackHandler\n",
    "from langchain.docstore.document import Document\n",
    "from langchain.text_splitter import CharacterTextSplitter\n",
    "from langchain.vectorstores.pgvector import PGVector\n",
    "from langchain_community.chat_models.ollama import ChatOllama\n",
    "from langchain_google_vertexai import ChatVertexAI\n",
    "from langchain_openai import ChatOpenAI\n",
    "\n",
    "from bs4 import BeautifulSoup,Tag,Comment,NavigableString\n",
    "from databricks import sql\n",
    "from datetime import datetime\n",
    "from importlib_resources import files\n",
    "import os\n",
    "import pandas as pd\n",
    "from pathlib import Path\n",
    "import re\n",
    "import requests\n",
    "\n",
    "from sqlalchemy import text, create_engine, exists, func, or_, and_, not_, desc, asc\n",
    "from sqlalchemy.orm import sessionmaker, aliased\n",
    "\n",
    "from time import time,sleep\n",
    "from tqdm import tqdm\n",
    "from urllib.request import urlopen\n",
    "from urllib.parse import quote_plus, quote, unquote\n",
    "from urllib.error import URLError, HTTPError\n",
    "import uuid\n",
    "import yaml"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Environment Variables\n",
    "\n",
    "Remember to set environmental variables for this code:\n",
    "\n",
    "* `ALHAZEN_DB_NAME` - the name of the PostGresQL database you are storing information into\n",
    "* `LOCAL_FILE_PATH` - the location on disk where you save temporary files, downloaded models or other data.   "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "if os.environ.get('LOCAL_FILE_PATH') is None: \n",
    "    raise Exception('Where are you storing your local literature database?')\n",
    "if os.path.exists(os.environ['LOCAL_FILE_PATH']) is False:\n",
    "    os.makedirs(os.environ['LOCAL_FILE_PATH'])    \n",
    "\n",
    "loc = os.environ['LOCAL_FILE_PATH']\n",
    "db_name = 'data_downloader'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Setup utils, agents, and tools "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/gully.burns/miniconda3/envs/alhazen/lib/python3.11/site-packages/langchain_core/_api/deprecation.py:119: LangChainDeprecationWarning: The class `ChatOpenAI` was deprecated in LangChain 0.0.10 and will be removed in 0.2.0. An updated version of the class exists in the langchain-openai package and should be used instead. To use it run `pip install -U langchain-openai` and import as `from langchain_openai import ChatOpenAI`.\n",
      "  warn_deprecated(\n",
      "/Users/gully.burns/miniconda3/envs/alhazen/lib/python3.11/site-packages/langchain_groq/chat_models.py:145: UserWarning: WARNING! stop is not default parameter.\n",
      "                    stop was transferred to model_kwargs.\n",
      "                    Please confirm that stop is what you intended.\n",
      "  warnings.warn(\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "dict_keys(['ollama_llama3', 'ollama_mixtral', 'gemini1.0', 'databricks_dbrx', 'databricks_mixtral', 'databricks_llama3', 'groq_mixtral', 'groq_llama3', 'gpt4_1106', 'gpt35'])\n"
     ]
    }
   ],
   "source": [
    "ldb = Ceifns_LiteratureDb(loc=loc, name=db_name)\n",
    "llms_lookup = lookup_chat_models()\n",
    "print(llms_lookup.keys())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AGENT TOOLS\n",
      "\tAddCollectionFromEPMCTool\n",
      "\tAddAuthorsToCollectionTool\n",
      "\tDescribeCollectionCompositionTool\n",
      "\tDeleteCollectionTool\n",
      "\tRetrieveFullTextTool\n",
      "\tRetrieveFullTextToolForACollection\n",
      "\tMetadataExtraction_EverythingEverywhere_Tool\n",
      "\tMetadataExtraction_MethodsSectionOnly_Tool\n",
      "\tSimpleExtractionWithRAGTool\n",
      "\tPaperQAEmulationTool\n",
      "\tProcotolEntitiesExtractionTool\n",
      "\tCheckExpressionTool\n",
      "\tTitleAbstractClassifier_OneDocAtATime_Tool\n",
      "\tTitleAbstractDiscourseMappingTool\n",
      "\tTitleAbstractExtraction_OneDocAtATime_Tool\n"
     ]
    }
   ],
   "source": [
    "llm = llms_lookup.get('gpt4_1106')\n",
    "\n",
    "cb = AlhazenAgent(llm, llm, db_name=db_name)\n",
    "print('AGENT TOOLS')\n",
    "for t in cb.tk.get_tools():\n",
    "    print('\\t'+type(t).__name__)\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Set Evaluation Dataset\n",
    "\n",
    "These are cases directly taken from `*.yaml` files that  "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Identify cases from the CZI CryoET Portal.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['10.1038/s41586-022-05255-2',\n",
       " '10.1038/s41594-022-00861-0',\n",
       " '10.1038/s41586-020-2665-2',\n",
       " '10.1038/s41586-022-04971-z',\n",
       " '10.1101/2023.04.28.538734',\n",
       " '10.1126/science.abm6704',\n",
       " '10.1101/2022.04.12.488077',\n",
       " '10.1083/jcb.202204093',\n",
       " '10.1038/s41592-020-01054-7',\n",
       " '10.1101/2022.01.23.477440']"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dois = {10000: ['10.1101/2022.04.12.488077'], \n",
    "        10001: ['10.1101/2022.04.12.488077'], \n",
    "        10003: ['10.1038/s41586-022-05255-2', '10.1038/s41592-020-01054-7'], \n",
    "        10004: ['10.1101/2023.04.28.538734'], \n",
    "        10005: ['10.1038/s41594-022-00861-0'], \n",
    "        10006: ['10.1038/s41586-020-2665-2'], \n",
    "        10007: [], \n",
    "        10008: ['10.1038/s41586-022-04971-z'], \n",
    "        10009: ['10.1126/science.abm6704'], \n",
    "        10010: ['10.1083/jcb.202204093', '10.1101/2022.01.23.477440']}\n",
    "dois_flattened = [doi for doi_list in dois.values() for doi in doi_list]\n",
    "dois_flattened = list(set(dois_flattened))\n",
    "dois_flattened"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Building the database\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Scripts to Build / Delete the database\n",
    "\n",
    "If you need to restore a deleted database from backup, use the following shell commands:\n",
    "\n",
    "```\n",
    "$ createdb em_tech\n",
    "$ psql -d em_tech -f /local/file/path/em_tech/backup<date_time>.sql\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "drop_ceifns_database(db_name)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "loc = os.environ['LOCAL_FILE_PATH']\n",
    "current_date_time = datetime.now()\n",
    "formatted_date_time = f'{current_date_time:%Y-%m-%d-%H-%M-%S}'\n",
    "backup_path = loc+'/'+db_name+'/backup'+formatted_date_time+'.sql'\n",
    "backup_ceifns_database(db_name, backup_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 309/309 [00:00<00:00, 3216.28it/s]\n"
     ]
    }
   ],
   "source": [
    "create_ceifns_database(db_name)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Build CEIFNS database from queries"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Add a collection based on EMPIAR papers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://www.ebi.ac.uk/europepmc/webservices/rest/search?format=JSON&pageSize=1000&synonym=TRUE&resultType=core&query=doi:\"10.1101/2022.01.23.477440\" OR doi:\"10.1038/s41592-020-01054-7\" OR doi:\"10.1038/s41586-020-2665-2\" OR doi:\"10.1101/2022.04.12.488077\" OR doi:\"10.1126/science.abm6704\" OR doi:\"10.1038/s41586-022-05255-2\" OR doi:\"10.1038/s41586-022-04971-z\" OR doi:\"10.1083/jcb.202204093\" OR doi:\"10.1038/s41594-022-00861-0\" OR doi:\"10.1101/2023.04.28.538734\", 10 European PMC PAPERS FOUND\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 1/1 [00:01<00:00,  1.22s/it]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " Returning 10\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 10/10 [00:00<00:00, 223.20it/s]\n",
      "100%|██████████| 9/9 [00:00<00:00, 664.94it/s]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Indexing 9 documents\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/gully.burns/miniconda3/envs/alhazen/lib/python3.11/site-packages/langchain_community/vectorstores/pgvector.py:293: LangChainPendingDeprecationWarning: Please use JSONB instead of JSON for metadata. This change will allow for more efficient querying that involves filtering based on metadata.Please note that filtering operators have been changed when using JSOB metadata to be prefixed with a $ sign to avoid name collisions with columns. If you're using an existing database, you will need to create adb migration for your metadata column to be JSONB and update your queries to use the new operators. \n",
      "  warn_deprecated(\n"
     ]
    }
   ],
   "source": [
    "addEMPCCollection_tool = [t for t in cb.tk.get_tools() if isinstance(t, AddCollectionFromEPMCTool)][0]\n",
    "step = 20\n",
    "for start_i in range(0, len(dois_flattened), step):\n",
    "    query = ' OR '.join(['doi:\"'+dois_flattened[i]+'\"' for i in range(start_i, start_i+step) if i<len(dois_flattened)])\n",
    "    addEMPCCollection_tool.run({'id': '1', 'name':'CryoET Examples', 'query':query, 'full_text':True})"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Read papers and extract information about datasets referred to in the paper"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_prompt = \"\"\"\n",
    "    You are helpful assistant, helping a scientist to extract information from scientific literature.\n",
    "\"\"\"\n",
    "prompt = \"\"\"\n",
    "    Read the following Markdown from scientific publication (delimited with triple backticks).\n",
    "\n",
    "    Markdown Code:- ```{markdown_code}```\n",
    "\n",
    "    From this code, identify any text paragraphs that refer to data sets with URLS links specified by '(URL)[TEXT]' Markdown fragments. \n",
    "\n",
    "    You will generate output as a JSON object with \"TEXT\", and \"LINKS\" as the keys. \n",
    "\n",
    "    The value of \"TEXT\" will be a list of paragraphs from the Markdown code that describe links to external biological datasets.\n",
    "    The value of \"LINKS\" will be a list of dicts where each entry has two attributes: URL and TEXT extracted from each URL link.\n",
    "    \n",
    "    Do not provide additional explanation or context for the answer.\n",
    "   \n",
    "    Do not include any other response other than a JSON object.\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://doi.org/10.1038/s41586-022-05255-2\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence] Entering Chain run with input:\n",
      "\u001b[0m{\n",
      "  \"markdown_code\": \"\\nVisualizing translation dynamics at atomic detail inside a bacterial cell | Nature\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nYour privacy, your choice\\n-------------------------\\n\\n\\n\\n\\n\\nWe use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.\\n\\nBy accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.\\n\\nSee our [privacy policy](https://www.nature.com/info/privacy) for more information on the use of your personal data.\\n\\nManage preferences for further information and to change your choices.\\n\\n\\n\\n\\n\\nAccept all cookies\\n\\n\\n\\n\\n\\n\\n[Skip to main content](#content)\\n\\n\\nThank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain\\n the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in\\n Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles\\n and JavaScript.\\n\\n\\n\\n\\n\\n\\n\\n\\nAdvertisement\\n\\n\\n\\n\\n[![Advertisement](//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature.com/article&sz=728x90&c=-918719175&t=pos%3Dtop%26type%3Darticle%26artid%3Ds41586-022-05255-2%26doi%3D10.1038/s41586-022-05255-2%26techmeta%3D101,28%26subjmeta%3D1258,1260,1789,2536,326,337,41,45,535,574,631%26kwrd%3DBacterial+structural+biology,Cryoelectron+tomography,Ribosome)](//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature.com/article&sz=728x90&c=-918719175&t=pos%3Dtop%26type%3Darticle%26artid%3Ds41586-022-05255-2%26doi%3D10.1038/s41586-022-05255-2%26techmeta%3D101,28%26subjmeta%3D1258,1260,1789,2536,326,337,41,45,535,574,631%26kwrd%3DBacterial+structural+biology,Cryoelectron+tomography,Ribosome)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n[![Nature](https://media.springernature.com/full/nature-cms/uploads/product/nature/header-86f1267ea01eccd46b530284be10585e.svg)](/)\\n\\n* [View all journals](https://www.nature.com/siteindex)\\n* [Search](javascript:;)\\n\\nSearch\\n------\\n\\n\\n\\nSearch articles by subject, keyword or author\\n\\n\\n\\n\\n\\n\\nShow results from\\n\\nAll journals\\nThis journal\\n\\n\\n\\nSearch\\n\\n\\n\\n\\n\\n[Advanced search](/search/advanced) \\n\\n### Quick links\\n\\n\\n\\n\\t+ [Explore articles by subject](/subjects)\\n\\t+ [Find a job](/naturecareers)\\n\\t+ [Guide to authors](/authors/index.html)\\n\\t+ [Editorial policies](/authors/editorial_policies/)\\n* [Log in](https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41586-022-05255-2)\\n\\n\\n\\n\\n\\n\\n\\n\\n* [Explore content](javascript:;)\\n\\nExplore content\\n---------------\\n\\n\\n\\n\\t+ [Research articles](/nature/research-articles)\\n\\t+ [News](/news)\\n\\t+ [Opinion](/opinion)\\n\\t+ [Research Analysis](/research-analysis)\\n\\t+ [Careers](/careers)\\n\\t+ [Books & Culture](/books-culture)\\n\\t+ [Podcasts](/nature/podcasts)\\n\\t+ [Videos](/nature/videos)\\n\\t+ [Current issue](/nature/current-issue)\\n\\t+ [Browse issues](/nature/browse-issues)\\n\\t+ [Collections](/nature/collections)\\n\\t+ [Subjects](/nature/browse-subjects)\\n\\n\\t+ [Follow us on Facebook](https://www.facebook.com/Nature)\\n\\t+ [Follow us on Twitter](https://twitter.com/nature)\\n\\t+ [Sign up for alerts](https://www.nature.com/my-account/alerts/subscribe-journal?list-id=1)\\n\\t+ [RSS feed](https://www.nature.com/nature.rss)\\n* [About the journal](javascript:;)\\n\\nAbout the journal\\n-----------------\\n\\n\\n\\n\\t+ [Journal Staff](/nature/journal-staff)\\n\\t+ [About the Editors](/nature/editors)\\n\\t+ [Journal Information](/nature/journal-information)\\n\\t+ [Our publishing models](/nature/our-publishing-models)\\n\\t+ [Editorial Values Statement](/nature/editorial-values-statement)\\n\\t+ [Journal Metrics](/nature/journal-impact)\\n\\t+ [Awards](/nature/awards)\\n\\t+ [Contact](/nature/contact)\\n\\t+ [Editorial policies](/nature/editorial-policies)\\n\\t+ [History of Nature](/nature/history-of-nature)\\n\\t+ [Send a news tip](/nature/send-a-news-tip)\\n* [Publish with us](javascript:;)\\n\\nPublish with us\\n---------------\\n\\n\\n\\n\\t+ [For Authors](/nature/for-authors)\\n\\t+ [For Referees](/nature/for-referees)\\n\\t+ [Language editing services](https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022)\\n\\t+ [Submit manuscript](https://mts-nature.nature.com/)\\n\\n\\n* [Sign up for alerts](https://idp.nature.com/auth/personal/springernature?redirect_uri=https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D1%26journal-link%3Dhttps%253A%252F%252Fwww.nature.com%252Fnature%252F)\\n* [RSS feed](https://www.nature.com/nature.rss)\\n\\n\\n\\n\\n\\n\\n\\n\\n1. [nature](/)\\n2. [articles](/nature/articles?type=article)\\n3. article\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n Visualizing translation dynamics at atomic detail inside a bacterial cell\\n \\n\\n[Download PDF](/articles/s41586-022-05255-2.pdf)\\n\\n\\n\\n\\n\\n\\n\\n[Download PDF](/articles/s41586-022-05255-2.pdf)\\n\\n\\n\\n\\n\\n* Article\\n* [Open access](https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research)\\n* Published: 28 September 2022\\n\\n\\nVisualizing translation dynamics at atomic detail inside a bacterial cell\\n=========================================================================\\n\\n\\n* [Liang Xue](#auth-Liang-Xue-Aff1-Aff2) \\n [ORCID: orcid.org/0000-0003-4368-2526](http://orcid.org/0000-0003-4368-2526)[1](#Aff1),[2](#Aff2),\\n* [Swantje Lenz](#auth-Swantje-Lenz-Aff3) \\n [ORCID: orcid.org/0000-0002-8839-5371](http://orcid.org/0000-0002-8839-5371)[3](#Aff3),\\n* [Maria Zimmermann-Kogadeeva](#auth-Maria-Zimmermann_Kogadeeva-Aff1) \\n [ORCID: orcid.org/0000-0001-6021-1246](http://orcid.org/0000-0001-6021-1246)[1](#Aff1),\\n* [Dimitry Tegunov](#auth-Dimitry-Tegunov-Aff4)[4](#Aff4),\\n* [Patrick Cramer](#auth-Patrick-Cramer-Aff4) \\n [ORCID: orcid.org/0000-0001-5454-7755](http://orcid.org/0000-0001-5454-7755)[4](#Aff4),\\n* [Peer Bork](#auth-Peer-Bork-Aff1-Aff5-Aff6) \\n [ORCID: orcid.org/0000-0002-2627-833X](http://orcid.org/0000-0002-2627-833X)[1](#Aff1),[5](#Aff5),[6](#Aff6),\\n* [Juri Rappsilber](#auth-Juri-Rappsilber-Aff3-Aff7) \\n [ORCID: orcid.org/0000-0001-5999-1310](http://orcid.org/0000-0001-5999-1310)[3](#Aff3),[7](#Aff7) &\\n* …\\n* [Julia Mahamid](#auth-Julia-Mahamid-Aff1) \\n [ORCID: orcid.org/0000-0001-6968-041X](http://orcid.org/0000-0001-6968-041X)[1](#Aff1)\\n\\nShow authors\\n\\n[*Nature*](/)\\n**volume 610**, pages 205–211 (2022)[Cite this article](#citeas)\\n\\n\\n\\n\\n* 34k Accesses\\n* 42 Citations\\n* 268 Altmetric\\n* [Metrics details](/articles/s41586-022-05255-2/metrics)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nA [Publisher Correction](https://doi.org/10.1038/s41586-022-05455-w) to this article was published on 10 November 2022\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nThis article has been [updated](#change-history)\\n\\n\\n\\n\\n\\nAbstract\\n--------\\n\\nTranslation is the fundamental process of protein synthesis and is catalysed by the ribosome in all living cells[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"). Here we use advances in cryo-electron tomography and sub-tomogram analysis[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"),[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\") to visualize the structural dynamics of translation inside the bacterium *Mycoplasma pneumoniae*. To interpret the functional states in detail, we first obtain a high-resolution in-cell average map of all translating ribosomes and build an atomic model for the *M.* *pneumoniae* ribosome that reveals distinct extensions of ribosomal proteins. Classification then resolves 13 ribosome states that differ in their conformation and composition. These recapitulate major states that were previously resolved in vitro, and reflect intermediates during active translation. On the basis of these states, we animate translation elongation inside native cells and show how antibiotics reshape the cellular translation landscapes. During translation elongation, ribosomes often assemble in defined three-dimensional arrangements to form polysomes[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"). By mapping the intracellular organization of translating ribosomes, we show that their association into polysomes involves a local coordination mechanism that is mediated by the ribosomal protein L9. We propose that an extended conformation of L9 within polysomes mitigates collisions to facilitate translation fidelity. Our work thus demonstrates the feasibility of visualizing molecular processes at atomic detail inside cells.\\n\\n\\n\\n\\n\\n### Similar content being viewed by others\\n\\n\\n\\n\\n\\n![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-022-34997-w/MediaObjects/41467_2022_34997_Fig1_HTML.png)\\n\\n### [Structures of the eukaryotic ribosome and its translational states in situ](https://www.nature.com/articles/s41467-022-34997-w?fromPaywallRec=false)\\n\\n\\n\\nArticle\\nOpen access\\n02 December 2022\\n\\n\\n\\n\\n\\n\\n![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-019-1249-5/MediaObjects/41586_2019_1249_Fig1_HTML.png)\\n\\n### [Late steps in bacterial translation initiation visualized using time-resolved cryo-EM](https://www.nature.com/articles/s41586-019-1249-5?fromPaywallRec=false)\\n\\n\\n\\nArticle\\n20 May 2019\\n\\n\\n\\n\\n\\n\\n![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs12276-023-00933-1/MediaObjects/12276_2023_933_Fig1_HTML.png)\\n\\n### [Single-molecule visualization of mRNA circularization during translation](https://www.nature.com/articles/s12276-023-00933-1?fromPaywallRec=false)\\n\\n\\n\\nArticle\\nOpen access\\n31 January 2023\\n\\n\\n\\n\\n\\n\\n\\n\\nMain\\n----\\n\\nTranslation of genetic information through messenger RNAs (mRNAs) into proteins is performed by the ribosome, one of the primordial macromolecular machines in cells[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"). The ribosome consists of a small and a large subunit (30S and 50S in prokaryotes), which form the aminoacyl (A), peptidyl (P) and exit (E) transfer RNA (tRNA) binding sites at their interface. The translation process can be divided into four phases: initiation, elongation, termination and recycling[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"). During the elongation phase, the ribosome undergoes a fundamental cycle to add one amino acid to the nascent peptide chain, which can be subdivided into three steps: decoding, peptidyl transfer and translocation. These steps involve structural changes in the ribosome that include subunit rotations, elongation factor association and tRNA accommodation[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"). Many intermediates during the elongation cycle have been identified on the basis of structures derived by cryo-electron microscopy (cryo-EM) and computational image classification[6](#ref-CR6 \\\"Frank, J. & Agrawal, R. K. A ratchet-like inter-subunit reorganization of the ribosome during translocation. Nature 406, 318–322 (2000).\\\"),[7](#ref-CR7 \\\"Villa, E. et al. Ribosome-induced changes in elongation factor Tu conformation control GTP hydrolysis. Proc. Natl Acad. Sci. USA 106, 1063–1068 (2009).\\\"),[8](#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[9](#ref-CR9 \\\"Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. Nature 468, 713–716 (2010).\\\"),[10](#ref-CR10 \\\"Ramrath, D. J. et al. Visualization of two transfer RNAs trapped in transit during elongation factor G-mediated translocation. Proc. Natl Acad. Sci. USA 110, 20964–20969 (2013).\\\"),[11](#ref-CR11 \\\"Brilot, A. F., Korostelev, A. A., Ermolenko, D. N. & Grigorieff, N. Structure of the ribosome with elongation factor G trapped in the pretranslocation state. Proc. Natl Acad. Sci. USA 110, 20994–20999 (2013).\\\"),[12](#ref-CR12 \\\"Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. Cell 161, 845–857 (2015).\\\"),[13](#ref-CR13 \\\"Loveland, A. B., Demo, G. & Korostelev, A. A. Cryo-EM of elongating ribosome with EF-Tu*GTP elucidates tRNA proofreading. Nature 584, 640–645 (2020).\\\"),[14](#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[15](#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\"),[16](/articles/s41586-022-05255-2#ref-CR16 \\\"Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. Nat. Commun. 12, 7236 (2021).\\\"). Most available structures, however, are of ribosomes that were isolated from model bacteria such as *Escherichia coli* and *Thermus thermophilus*, and were often trapped in specific states by antibiotics, GTP analogues or mutations[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"),[17](/articles/s41586-022-05255-2#ref-CR17 \\\"Brown, A. & Shao, S. Ribosomes and cryo-EM: a duet. Curr. Opin. Struct. Biol. 52, 1–7 (2018).\\\"). Thus, a detailed structural description of the translation process within the native cellular context is lacking. Although actively translating ribosomes have been visualized inside cells by cryo-electron tomography (cryo-ET), the maps generated in previous studies were limited to resolutions on the nanometre scale[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"),[18](#ref-CR18 \\\"Kuhner, S. et al. Proteome organization in a genome-reduced bacterium. Science 326, 1235–1240 (2009).\\\"),[19](#ref-CR19 \\\"Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. Mol. Cell 39, 560–569 (2010).\\\"),[20](#ref-CR20 \\\"Mahamid, J. et al. Visualizing the molecular sociology at the HeLa cell nuclear periphery. Science 351, 969–972 (2016).\\\"),[21](#ref-CR21 \\\"Schaffer, M. et al. A cryo-FIB lift-out technique enables molecular-resolution cryo-ET within native Caenorhabditis elegans tissue. Nat. Methods 16, 757–762 (2019).\\\"),[22](/articles/s41586-022-05255-2#ref-CR22 \\\"Waltz, F. et al. How to build a ribosome from RNA fragments in Chlamydomonas mitochondria. Nat. Commun. 12, 7176 (2021).\\\"). We have recently developed image-processing algorithms for cryo-ET that make it possible to resolve stalled ribosomes to residue level inside the genome-reduced bacterium *M.* *pneumoniae*[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\"). Here, we use these technical advances to perform large-scale structure classification and spatial analysis of ribosomes, to visualize parts of the translation process in great detail inside *M.* *pneumoniae* as a prokaryotic minimal cell model[18](/articles/s41586-022-05255-2#ref-CR18 \\\"Kuhner, S. et al. Proteome organization in a genome-reduced bacterium. Science 326, 1235–1240 (2009).\\\").\\n\\nIn-cell structure of the *M. pneumoniae* ribosome\\n-------------------------------------------------\\n\\nTo investigate the structural details of the translation machinery, we first obtained 3.5-Å consensus maps by averaging all ribosomes detected in cryo-electron tomograms of intact *M.* *pneumoniae* cells (Fig. [1a,b](/articles/s41586-022-05255-2#Fig1), Extended Data Fig. [1](/articles/s41586-022-05255-2#Fig5), Supplementary Table [1](/articles/s41586-022-05255-2#MOESM1), [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1) and [Methods](/articles/s41586-022-05255-2#Sec7)). Focused refinements on the 30S and 50S subunits improved the map quality and revealed well-resolved ribosomal RNA (rRNA) bases and ribosomal protein amino acid side chains (Fig. [1b](/articles/s41586-022-05255-2#Fig1) and Extended Data Fig. [2a–c](/articles/s41586-022-05255-2#Fig6)). The high-resolution in-cell consensus maps allowed us to build de novo an atomic model for the *M.* *pneumoniae* ribosome (Extended Data Fig. [2d–g](/articles/s41586-022-05255-2#Fig6) and [Methods](/articles/s41586-022-05255-2#Sec7)). The structure shows high similarity to other bacterial ribosomes, but also reveals several new features (Fig. [1c](/articles/s41586-022-05255-2#Fig1) and Extended Data Fig. [2d–g](/articles/s41586-022-05255-2#Fig6)). Specifically, 11 of the 52 ribosomal proteins in *M.* *pneumoniae* have extended sequences compared to *E.* *coli* (Extended Data Fig. [3a](/articles/s41586-022-05255-2#Fig7) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). Most of these extensions are predicted to be disordered, but those of ribosomal proteins S6, L22 and L29 form secondary structures and were built in the model (Fig. [1c](/articles/s41586-022-05255-2#Fig1) and Extended Data Figs. [2d–e](/articles/s41586-022-05255-2#Fig6) and [3a](/articles/s41586-022-05255-2#Fig7)). We found such extensions to be common throughout the bacterial kingdom (Extended Data Fig. [3b](/articles/s41586-022-05255-2#Fig7), [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1) and Supplementary Table [2](/articles/s41586-022-05255-2#MOESM1)), possibly representing ribosome diversity in adaptation to different environments and lifestyles[23](/articles/s41586-022-05255-2#ref-CR23 \\\"Melnikov, S., Manakongtreecheep, K. & Soll, D. Revising the structural diversity of ribosomal proteins across the three domains of life. Mol. Biol. Evol. 35, 1588–1598 (2018).\\\"). Although the functions of these extensions remain largely elusive, their disruption affects cellular fitness or survival in *M.*  *pneumoniae*[24](/articles/s41586-022-05255-2#ref-CR24 \\\"Miravet-Verde, S., Burgos, R., Delgado, J., Lluch-Senar, M. & Serrano, L. FASTQINS and ANUBIS: two bioinformatic tools to explore facts and artifacts in transposon sequencing and essentiality studies. Nucleic Acids Res. 48, e102 (2020).\\\") ([Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). Thus, the high-resolution ribosome map and the atomic model derived from intact *M.* *pneumoniae* provide the basis to investigate in detail the conformational and compositional changes of ribosomes during translation inside cells.\\n\\n\\n\\n**Fig. 1: Ribosome structure in *M.* *pneumoniae* cells.**\\n\\n[![figure 1](//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_Fig1_HTML.png)](/articles/s41586-022-05255-2/figures/1)**a**, A denoised tomographic slice of a *M.* *pneumoniae* cell. AO, attachment organelle; PM, plasma membrane. Examples of ribosomes are circled. The representative tomogram (selected from 15) was acquired with a Volta phase plate for better visualization of the cellular morphology. Similar imaging conditions, excluding the use of a phase plate, were used for the acquisition of all tomograms for subsequent analysis. Scale bar, 100 nm. **b**, A 3.5-Å in-cell ribosome map (left) shows well-resolved rRNA bases and ribosomal protein side chains (right). **c**, An atomic model of the *M.* *pneumoniae* ribosome shows structural similarity to other bacterial ribosomes. Eleven ribosomal proteins (dark green) have sequence extensions (magenta).\\n\\n[Full size image](/articles/s41586-022-05255-2/figures/1)Structural dynamics of translation in cells\\n-------------------------------------------\\n\\nTo analyse the structural changes that are associated with the translation process inside cells, we performed computational classification of 101,696 ribosomes from 356 tomograms of native cells, and obtained 13 distinct ribosome classes (Extended Data Fig. [4](/articles/s41586-022-05255-2#Fig8), Supplementary Table [3](/articles/s41586-022-05255-2#MOESM1) and [Methods](/articles/s41586-022-05255-2#Sec7)). Ten classes determined at resolutions ranging from 4 to 10 Å were assigned to the translation elongation phase on the basis of elongation factor and tRNAs binding to the ribosome (Fig. [2a](/articles/s41586-022-05255-2#Fig2) and Extended Data Figs. [4](/articles/s41586-022-05255-2#Fig8)–[7](/articles/s41586-022-05255-2#Fig11)). The remaining three classes represent 70S with a single P/E-site tRNA, 50S in complex with the ribosome recycling factor, and free 50S subunits (Extended Data Figs. [4](/articles/s41586-022-05255-2#Fig8)–[7](/articles/s41586-022-05255-2#Fig11) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). The ten classes within the elongation phase account for 70% of the detected ribosomes, consistent with the expectation that most ribosomes inside living cells are engaged in the elongation phase, which lasts considerably longer than the initiation, termination and recycling phases[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"),[25](/articles/s41586-022-05255-2#ref-CR25 \\\"Borg, A., Pavlov, M. & Ehrenberg, M. Complete kinetic mechanism for recycling of the bacterial ribosome. RNA 22, 10–21 (2016).\\\").\\n\\n\\n\\n**Fig. 2: Ribosome classification reconstructs the translation elongation cycle.**\\n\\n[![figure 2](//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_Fig2_HTML.png)](/articles/s41586-022-05255-2/figures/2)**a**, Ten 70S structures determined inside *M.* *pneumoniae* cells represent translation elongation intermediate states, which are characterized by the binding of different elongation factors (EFs) and tRNAs. The frequencies of occurrence for different intermediates are calculated from the classified particle numbers (Extended Data Fig. [4](/articles/s41586-022-05255-2#Fig8)). **b**, Densities for mRNA, tRNAs and the nascent peptide chain are well-resolved in the most-populated 'A, P' state. **c**, Trajectories of the elongation factors EF-Tu and EF-G and the A-, P- and E-site tRNAs along translation elongation. **d**, Major conformational changes of the ribosome along the elongation trajectory: 30S body rotation, 30S head swivel and L1 stalk opening.\\n\\n[Full size image](/articles/s41586-022-05255-2/figures/2)The identified elongation classes can be ordered to reconstruct the translation elongation cycle[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"). By flexible fitting of our *M.* *pneumoniae* ribosome atomic model into the classified maps ([Methods](/articles/s41586-022-05255-2#Sec7) and Supplementary Table [3](/articles/s41586-022-05255-2#MOESM1)), we obtained pseudo-atomic models that delineate conformational and compositional changes of the ribosome complex, including rotations of 30S body and head, coordination of elongation factors, movement of tRNAs through the A–P–E sites and dynamics of the L1 stalk (Fig. [2](/articles/s41586-022-05255-2#Fig2), Extended Data Fig. [7a–j](/articles/s41586-022-05255-2#Fig11) and Supplementary Videos [1](/articles/s41586-022-05255-2#MOESM3) and [2](/articles/s41586-022-05255-2#MOESM4)). The number of ribosomes within each of the classes provides the relative abundance of translation elongation intermediates, which together reflect a steady state distribution determined by their relative rates of formation and depletion inside the cells (Fig. [2a](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8)). The classic, non-rotated 'A, P' state was the most populated and determined at the highest resolution, showing clear density for mRNA, tRNAs and the nascent peptide (Fig. [2b](/articles/s41586-022-05255-2#Fig2)). In the following states, we observed the oscillation of A- and P-site tRNAs into the hybrid A/P and P/E state, coupled with 30S subunit rotations and L1 stalk movement (Fig. [2a,c,d](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [7a–j](/articles/s41586-022-05255-2#Fig11)), consistent with in vitro studies[6](/articles/s41586-022-05255-2#ref-CR6 \\\"Frank, J. & Agrawal, R. K. A ratchet-like inter-subunit reorganization of the ribosome during translocation. Nature 406, 318–322 (2000).\\\"),[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[9](/articles/s41586-022-05255-2#ref-CR9 \\\"Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. Nature 468, 713–716 (2010).\\\"),[12](/articles/s41586-022-05255-2#ref-CR12 \\\"Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. Cell 161, 845–857 (2015).\\\"),[26](#ref-CR26 \\\"Munro, J. B., Altman, R. B., O’Connor, N. & Blanchard, S. C. Identification of two distinct hybrid state intermediates on the ribosome. Mol. Cell 25, 505–517 (2007).\\\"),[27](#ref-CR27 \\\"Belardinelli, R. et al. Choreography of molecular movements during ribosome progression along mRNA. Nat. Struct. Mol. Biol. 23, 342–348 (2016).\\\"),[28](#ref-CR28 \\\"Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. Nat. Struct. Mol. Biol. 23, 333–341 (2016).\\\"),[29](#ref-CR29 \\\"Fei, J., Kosuri, P., MacDougall, D. D. & Gonzalez, R. L. Jr Coupling of ribosomal L1 stalk and tRNA dynamics during translation elongation. Mol. Cell 30, 348–359 (2008).\\\"),[30](/articles/s41586-022-05255-2#ref-CR30 \\\"Zhou, J., Lancaster, L., Donohue, J. P. & Noller, H. F. How the ribosome hands the A-site tRNA to the P site during EF-G-catalyzed translocation. Science 345, 1188–1191 (2014).\\\"). Along this trajectory, we also determined a partial hybrid 'A\\\\*, P/E' state with rotated 30S and deacyl-tRNA in the hybrid P/E site, but with only marginally relocated peptidyl-tRNA in the A site (Fig. [2a,d](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [8a](/articles/s41586-022-05255-2#Fig12)), similar to a previously reported pre-translocational H2\\\\* state[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[26](/articles/s41586-022-05255-2#ref-CR26 \\\"Munro, J. B., Altman, R. B., O’Connor, N. & Blanchard, S. C. Identification of two distinct hybrid state intermediates on the ribosome. Mol. Cell 25, 505–517 (2007).\\\"). When viewed in the context of this trajectory, the high enrichment of the 'A, P' state could suggest that inter-subunit rotation represents a rate-limiting step for translation elongation in *M.* *pneumoniae*. However, in view of the entirety of the elongation cycle, processive inter-subunit rotation leading to efficient translocation requires the binding of the elongation factor EF-G. Notably, EF-G was found to bind to the ribosome in either the partial hybrid or the full hybrid states (Extended Data Fig. [8b–m](/articles/s41586-022-05255-2#Fig12)). EF-G in the partial hybrid state (class 6e) is less extended and its domain IV does not overlap with the A site compared to the full hybrid state (Extended Data Fig. [8b–h](/articles/s41586-022-05255-2#Fig12)). This EF-G-bound partial hybrid state therefore resembles early translocation intermediates before phosphate release[9](/articles/s41586-022-05255-2#ref-CR9 \\\"Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. Nature 468, 713–716 (2010).\\\"),[14](#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[15](#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\"),[16](/articles/s41586-022-05255-2#ref-CR16 \\\"Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. Nat. Commun. 12, 7236 (2021).\\\"). In the following full hybrid state (class 7), domain IV of EF-G extends about 20 Å towards the A site, owing to both the rotation of entire EF-G and its inter-domain rearrangement (Fig. [2c](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [8f](/articles/s41586-022-05255-2#Fig12)). Class 8 next shows a reverse rotation of the 30S body, the largest magnitude for 30S head swivel, the fully extended EF-G and the chimeric 'ap/P, pe/E' tRNAs (Fig. [2c,d](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [8g, j](/articles/s41586-022-05255-2#Fig12)), indicating that it is a late translocation intermediate[9](/articles/s41586-022-05255-2#ref-CR9 \\\"Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. Nature 468, 713–716 (2010).\\\"),[10](/articles/s41586-022-05255-2#ref-CR10 \\\"Ramrath, D. J. et al. Visualization of two transfer RNAs trapped in transit during elongation factor G-mediated translocation. Proc. Natl Acad. Sci. USA 110, 20964–20969 (2013).\\\"),[14](#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[15](#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\"),[16](/articles/s41586-022-05255-2#ref-CR16 \\\"Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. Nat. Commun. 12, 7236 (2021).\\\"),[28](/articles/s41586-022-05255-2#ref-CR28 \\\"Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. Nat. Struct. Mol. Biol. 23, 333–341 (2016).\\\"). We thus demonstrate the existence of continual intermediates from early to late translocation stages during active translation elongation inside cells. Two EF-Tu-associated structures were also determined, with and without E-site tRNA (Fig. [2a](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [7c,d](/articles/s41586-022-05255-2#Fig11)). This suggests that the binding of EF-Tu•tRNA to the ribosome and the disassociation of E-site tRNA are independent of each other. The 'P, E' class has a relatively low abundance, which suggests that the E-site tRNA is not stable and tends to disassociate quickly after translocation. This is in agreement with previous single-molecule fluorescence studies showing rapid release of the E-site tRNA[31](/articles/s41586-022-05255-2#ref-CR31 \\\"Uemura, S. et al. Real-time tRNA transit on single translating ribosomes at codon resolution. Nature 464, 1012–U1073 (2010).\\\"),[32](/articles/s41586-022-05255-2#ref-CR32 \\\"Chen, C. et al. Single-molecule fluorescence measurements of ribosomal translocation dynamics. Mol. Cell 42, 367–377 (2011).\\\"), and helps to resolve a long-standing controversy over its disassociation time point[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[12](/articles/s41586-022-05255-2#ref-CR12 \\\"Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. Cell 161, 845–857 (2015).\\\"),[33](/articles/s41586-022-05255-2#ref-CR33 \\\"Wilson, D. N. & Nierhaus, K. H. The E-site story: the importance of maintaining two tRNAs on the ribosome during protein synthesis. Cell. Mol. Life Sci. 63, 2725–2737 (2006).\\\"). In summary, these results recapitulate major steps in translation elongation that have been defined by controlled in vitro studies and reconstruct the structural dynamics of the elongation cycles inside cells, highlighting the possible various paths of the reaction catalysed by the ribosome complex. Furthermore, with these results, a probability map for the occurrence of intermediate states can be derived to illustrate a putative energy landscape of translation elongation in cells. Energy landscapes of translation are suggested to be rugged[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[12](/articles/s41586-022-05255-2#ref-CR12 \\\"Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. Cell 161, 845–857 (2015).\\\"),[34](/articles/s41586-022-05255-2#ref-CR34 \\\"Munro, J. B., Sanbonmatsu, K. Y., Spahn, C. M. & Blanchard, S. C. Navigating the ribosome’s metastable energy landscape. Trends Biochem. Sci 34, 390–400 (2009).\\\"),[35](/articles/s41586-022-05255-2#ref-CR35 \\\"Bock, L. V. et al. Energy barriers and driving forces in tRNA translocation through the ribosome. Nat. Struct. Mol. Biol. 20, 1390–1396 (2013).\\\"), depending on factors such as Mg2+ concentrations[36](/articles/s41586-022-05255-2#ref-CR36 \\\"Julian, P. et al. Structure of ratcheted ribosomes with tRNAs in hybrid states. Proc. Natl Acad. Sci. USA 105, 16924–16927 (2008).\\\"),[37](/articles/s41586-022-05255-2#ref-CR37 \\\"Agirrezabala, X. et al. Visualization of the hybrid state of tRNA binding promoted by spontaneous ratcheting of the ribosome. Mol. Cell 32, 190–197 (2008).\\\"), the presence of EF-G[34](/articles/s41586-022-05255-2#ref-CR34 \\\"Munro, J. B., Sanbonmatsu, K. Y., Spahn, C. M. & Blanchard, S. C. Navigating the ribosome’s metastable energy landscape. Trends Biochem. Sci 34, 390–400 (2009).\\\"),[38](/articles/s41586-022-05255-2#ref-CR38 \\\"Rodnina, M. V., Peske, F., Peng, B. Z., Belardinelli, R. & Wintermeyer, W. Converting GTP hydrolysis into motion: versatile translational elongation factor G. Biol. Chem. 401, 131–142 (2019).\\\") and temperature[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"). In *M.* *pneumoniae*, the intracellular concentrations of ribosomes, translation elongation factors and substrates can be up to ten times lower than those in *E.* *coli* ([Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). It is therefore possible that the energy landscapes of translation vary for different organisms, cell types and conditions. Applying the analysis introduced here to other cell types and conditions can deepen our understanding of the translation process within the native cellular context.\\n\\nAntibiotics alter translation landscapes\\n----------------------------------------\\n\\nThe ribosome is one of the most important targets for antibiotics, many of which are known to stall translation by stabilizing certain intermediates in the process[39](/articles/s41586-022-05255-2#ref-CR39 \\\"Lin, J., Zhou, D., Steitz, T. A., Polikanov, Y. S. & Gagnon, M. G. Ribosome-targeting antibiotics: modes of action, mechanisms of resistance, and implications for drug design. Annu. Rev. Biochem. 87, 451–478 (2018).\\\"). To investigate the effects of antibiotics on the cellular translation machinery, we analysed ribosomes in cells that were treated for a short time (15–20 min; [Methods](/articles/s41586-022-05255-2#Sec7)) with two representative ribosome-targeting antibiotics: chloramphenicol (Cm), which binds to the peptidyl transfer centre in the 50S subunit and inhibits peptide bond formation[40](/articles/s41586-022-05255-2#ref-CR40 \\\"Bulkley, D., Innis, C. A., Blaha, G. & Steitz, T. A. Revisiting the structures of several antibiotics bound to the bacterial ribosome. Proc. Natl Acad. Sci. USA 107, 17158–17163 (2010).\\\"),[41](/articles/s41586-022-05255-2#ref-CR41 \\\"Svetlov, M. S. et al. High-resolution crystal structures of ribosome-bound chloramphenicol and erythromycin provide the ultimate basis for their competition. RNA 25, 600–606 (2019).\\\"); and spectinomycin (Spc), which binds to the 30S neck and blocks translocation[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[28](/articles/s41586-022-05255-2#ref-CR28 \\\"Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. Nat. Struct. Mol. Biol. 23, 333–341 (2016).\\\"),[42](/articles/s41586-022-05255-2#ref-CR42 \\\"Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. ACS Chem. Biol. 2, 545–552 (2007).\\\"). Overall, our sub-tomogram analysis of antibiotic-treated cells results in 17 ribosome maps, and shows that translation landscapes are markedly reshaped by different antibiotics (Fig. [3a](/articles/s41586-022-05255-2#Fig3), Extended Data Figs. [9](/articles/s41586-022-05255-2#Fig13)–[11](/articles/s41586-022-05255-2#Fig15), Supplementary Tables [4](/articles/s41586-022-05255-2#MOESM1)–[6](/articles/s41586-022-05255-2#MOESM1) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)).\\n\\n\\n\\n**Fig. 3: Antibiotics induce distinct translation elongation landscapes in cells.**\\n\\n[![figure 3](//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_Fig3_HTML.png)](/articles/s41586-022-05255-2/figures/3)**a**, Distribution of translation elongation intermediates in native untreated cells, and in cells treated with three different antibiotics. Bar and whiskers indicate mean and s.d. for each class across all cells in the different treatment groups: untreated (*n* = 356 cells); +Cm, chloramphenicol-treated (*n* = 65 cells); +Spc, spectinomycin-treated (*n* = 70 cells); and +PUM, pseudouridimycin-treated (*n* = 86 cells). **b**, Ribosomes in Spc-treated cells are largely stalled in the 'EF-G, A/PSpc, P/E' state. **c**, The Spc molecule (magenta) is well-resolved and built in the 'EF-G, A/PSpc, P/E' ribosome model. It is surrounded by several 16S rRNA bases and loop 2 of ribosomal protein S5 near the 30S neck. **d**, The major state in Spc-treated cells is similar to the 'EF-G, A/PPUM, P/E' class in PUM-treated cells (light blue) and the 'EF-G, A/P, P/E' class in untreated cells (light grey), differing only in the position of the A/P-site tRNA on the 50S side. **e**, Single-cell clustering analysis on the basis of the translation elongation states of 577 individual cells under native and different antibiotic treatment conditions.\\n\\n[Full size image](/articles/s41586-022-05255-2/figures/3)In Cm-treated cells, 72% of 70S ribosomes are trapped in the 'A, P' state (class 4), owing to the inhibition of peptidyl transfer[39](/articles/s41586-022-05255-2#ref-CR39 \\\"Lin, J., Zhou, D., Steitz, T. A., Polikanov, Y. S. & Gagnon, M. G. Ribosome-targeting antibiotics: modes of action, mechanisms of resistance, and implications for drug design. Annu. Rev. Biochem. 87, 451–478 (2018).\\\") (Fig. [3a–d](/articles/s41586-022-05255-2#Fig3), Extended Data Fig. [9](/articles/s41586-022-05255-2#Fig13) and Supplementary Table [4](/articles/s41586-022-05255-2#MOESM1)). The Cm molecule was resolved in its canonical binding site[40](/articles/s41586-022-05255-2#ref-CR40 \\\"Bulkley, D., Innis, C. A., Blaha, G. & Steitz, T. A. Revisiting the structures of several antibiotics bound to the bacterial ribosome. Proc. Natl Acad. Sci. USA 107, 17158–17163 (2010).\\\"),[41](/articles/s41586-022-05255-2#ref-CR41 \\\"Svetlov, M. S. et al. High-resolution crystal structures of ribosome-bound chloramphenicol and erythromycin provide the ultimate basis for their competition. RNA 25, 600–606 (2019).\\\") (Extended Data Fig. [9d,g](/articles/s41586-022-05255-2#Fig13)). Of note, 28% of ribosomes were found in several other elongation states, either before or following the major 'A, P' state (Fig. [3a](/articles/s41586-022-05255-2#Fig3) and Extended Data Fig. [9](/articles/s41586-022-05255-2#Fig13)). The existence of an EF-Tu•tRNA-bound state (class 3) is reminiscent of the effects of other antibiotics that inhibit peptidyl transfer in a similar manner[43](#ref-CR43 \\\"Polikanov, Y. S. et al. Distinct tRNA accommodation intermediates observed on the ribosome with the antibiotics hygromycin A and A201A. Mol. Cell 58, 832–844 (2015).\\\"),[44](#ref-CR44 \\\"Morse, J. C. et al. Elongation factor-Tu can repetitively engage aminoacyl-tRNA within the ribosome during the proofreading stage of tRNA selection. Proc. Natl Acad. Sci. USA 117, 3610–3620 (2020).\\\"),[45](/articles/s41586-022-05255-2#ref-CR45 \\\"Choi, J. et al. Dynamics of the context-specific translation arrest by chloramphenicol and linezolid. Nat. Chem. Biol. 16, 310–317 (2020).\\\").\\n\\nSpc is suggested to inhibit translocation by blocking head swivel of the 30S subunit[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[28](/articles/s41586-022-05255-2#ref-CR28 \\\"Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. Nat. Struct. Mol. Biol. 23, 333–341 (2016).\\\"),[42](/articles/s41586-022-05255-2#ref-CR42 \\\"Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. ACS Chem. Biol. 2, 545–552 (2007).\\\"). We found that 67.5% of 70S ribosomes were stalled in the 'EF-G, A/PSpc, P/E' state in Spc-treated cells, a state similar to the pre-translocational 'EF-G, A/P, P/E' state (class 7) in untreated cells (Fig. [3a–d](/articles/s41586-022-05255-2#Fig3), Extended Data Fig. [10](/articles/s41586-022-05255-2#Fig14) and Supplementary Table [5](/articles/s41586-022-05255-2#MOESM1)). The Spc molecule density is clearly visible near helix 34 within the 30S neck region (Fig. [3b,c](/articles/s41586-022-05255-2#Fig3) and Extended Data Fig. [10d,g](/articles/s41586-022-05255-2#Fig14)), consistent with its reported binding site[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[42](/articles/s41586-022-05255-2#ref-CR42 \\\"Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. ACS Chem. Biol. 2, 545–552 (2007).\\\"). These results show that Spc inhibits translocation by impeding the dynamics of the 30S subunit. Similar to Cm, ribosomes in three additional elongation states could be detected with lower frequencies (Fig. [3a](/articles/s41586-022-05255-2#Fig3) and Extended Data Fig. [10e,f](/articles/s41586-022-05255-2#Fig14)).\\n\\nPerturbation of other molecular pathways that are functionally coupled to translation in bacterial cells also affects the translation landscape; RNA polymerase stalled by pseudouridimycin (PUM) can physically block mRNA translocation in the ribosome that collides with it during transcription–translation coupling[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). Consistently, 59.5% of 70S ribosomes in PUM-treated cells were found in the 'EF-G, A/PPUM, P/E' state (Fig. [3a](/articles/s41586-022-05255-2#Fig3), Extended Data Fig. [11](/articles/s41586-022-05255-2#Fig15) and Supplementary Table [6](/articles/s41586-022-05255-2#MOESM1)), which resembles the pre-translocational 'EF-G, A/P, P/E' state in untreated cells and the stalled 'EF-G, A/PSpc, P/E' state in Spc-treated cells (Fig. [3d](/articles/s41586-022-05255-2#Fig3)). Our finding that physically obstructing mRNA translocation by a PUM-stalled RNA polymerase and chemically impeding 30S head dynamics by Spc lead to similar structures further confirms that mRNA translocation and 30S rotations are directly coupled.\\n\\nThe observation that treatment with antibiotics resulted in minor states that are not expected from their specific binding prompted us to investigate possible cell-to-cell variability in response to the antibiotics. To this end, we performed clustering analysis on the basis of translation elongation profiles in 577 tomograms of single cells under native and antibiotic treatment conditions. The analysis resulted in four major clusters in accordance with the four treatment groups, demonstrating small cell-to-cell variability within each cluster (Fig. [3e](/articles/s41586-022-05255-2#Fig3)). Thus, our results show that the translation landscapes in cells are globally reshaped by small molecules specifically binding to ribosomes, as well as to other targets ([Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1) and Supplementary Table [8](/articles/s41586-022-05255-2#MOESM1)). The presence of minor elongation states under antibiotic treatment is reminiscent of previous studies that have shown ongoing slow translation in antibiotic-treated cells and context-dependent inhibition[46](#ref-CR46 \\\"Marks, J. et al. Context-specific inhibition of translation by ribosomal antibiotics targeting the peptidyl transferase center. Proc. Natl Acad. Sci. USA 113, 12150–12155 (2016).\\\"),[47](#ref-CR47 \\\"Vazquez-Laslop, N. & Mankin, A. S. Context-specific action of ribosomal antibiotics. Annu. Rev. Microbiol. 72, 185–207 (2018).\\\"),[48](/articles/s41586-022-05255-2#ref-CR48 \\\"Aguirre Rivera, J. et al. Real-time measurements of aminoglycoside effects on protein synthesis in live cells. Proc. Natl Acad. Sci. USA 118, e2013315118 (2021).\\\"). For example, Cm inhibition is affected by specific residues of the nascent peptide[46](/articles/s41586-022-05255-2#ref-CR46 \\\"Marks, J. et al. Context-specific inhibition of translation by ribosomal antibiotics targeting the peptidyl transferase center. Proc. Natl Acad. Sci. USA 113, 12150–12155 (2016).\\\"),[47](/articles/s41586-022-05255-2#ref-CR47 \\\"Vazquez-Laslop, N. & Mankin, A. S. Context-specific action of ribosomal antibiotics. Annu. Rev. Microbiol. 72, 185–207 (2018).\\\"). Most antibiotics, including Cm and Spc, are known to inhibit cell growth but do not immediately kill the cell[49](/articles/s41586-022-05255-2#ref-CR49 \\\"Baquero, F. & Levin, B. R. Proximate and ultimate causes of the bactericidal action of antibiotics. Nat. Rev. Microbiol. 19, 123–132 (2021).\\\"). It is possible that the reshaped translation landscapes by antibiotics lead to an imbalance in protein synthesis, which in the long run has detrimental consequences for the cell.\\n\\nSpatial and functional organization of translation\\n--------------------------------------------------\\n\\nFinally, we investigated the spatial organization of active translation in native *M.* *pneumoniae* cells (Fig. [4a](/articles/s41586-022-05255-2#Fig4) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). It is known that ribosomes translating on the same mRNA can assemble closely in space to form polysomes[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"),[19](/articles/s41586-022-05255-2#ref-CR19 \\\"Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. Mol. Cell 39, 560–569 (2010).\\\"). We defined polysomes using a distance cut-off of 7 nm from the mRNA exit site to the mRNA entry site between neighbouring ribosomes (Extended Data Fig. [12a–e](/articles/s41586-022-05255-2#Fig16), [Methods](/articles/s41586-022-05255-2#Sec7) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). The detected polysomes account for 26.2% of all 70S ribosomes, and two arrangement patterns between neighbouring ribosomes can be defined (Fig. [4b–e](/articles/s41586-022-05255-2#Fig4) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)): the so-called 'top-top' (t-t; 78.5%, mRNA exit-to-entry distance 4.2 ± 1.4 nm) and 'top-back' (t-b; 21.5%, 5.4 ± 1.5 nm) configurations[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"). We also observed various topologies for long polysomes, ranging from loose assembly to tight packing with helix-like configurations[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"),[19](/articles/s41586-022-05255-2#ref-CR19 \\\"Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. Mol. Cell 39, 560–569 (2010).\\\"),[50](/articles/s41586-022-05255-2#ref-CR50 \\\"Myasnikov, A. G. et al. The molecular structure of the left-handed supra-molecular helix of eukaryotic polyribosomes. Nat. Commun. 5, 5294 (2014).\\\") (Fig. [4c,d](/articles/s41586-022-05255-2#Fig4)).\\n\\n\\n\\n**Fig. 4: Spatial and functional organization of ribosomes in native cells.**\\n\\n[![figure 4](//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_Fig4_HTML.png)](/articles/s41586-022-05255-2/figures/4)**a**, Three-dimensional map of ribosomes in a representative native untreated cell (selected from 356). Top, *x*–*y* view; bottom, orthogonal view. The 70S intermediates within the translation elongation cycle (classes 1–8, as in Fig. [2a](/articles/s41586-022-05255-2#Fig2)), and additional classes (classes 11–13 and 50–51 as detailed in Extended Data Fig. [4](/articles/s41586-022-05255-2#Fig8)) are coloured as indicated in the colour scheme (inset). 50S: light grey. **b**, 'top-back' (t-b) and 'top-top' (t-t) assembly patterns of adjacent ribosomes in polysomes. **c**,**d**, Representative long polysomes of loose (**c**) and tight (**d**) topologies, with the corresponding putative mRNA paths and nascent chain vectors shown underneath (not drawn to scale). **e**, Distribution of polysome lengths. **f**, Distribution of elongation states in polysomes compared to all ribosomes and mono-ribosomes. Bars and whiskers are mean and s.d. across 356 tomograms of untreated cells (*n* = 356 cells). Highlighted are states for which the fraction in polysomes differs by more than 50% compared to all ribosomes or mono-ribosomes. Asterisks indicate false discovery rate (FDR)-adjusted *P* value (*P*FDR) < 0.01 (two-sided Wilcoxon rank sum test). *P*FDR values for class 5: 6.44 × 10−23 (polysome versus all) and 2.76 × 10−39 (polysome versus monosome); class 6a: 3.06 × 10−28 and 1.44 × 10−34, respectively. **g**, Occurrence frequencies of elongation state pairs of adjacent ribosomes in polysomes normalized to the theoretical probability of random pairs. States that require elongation factor binding to proceed are 1, 2a, 5 and 6a (in bold). **h**, Map of a di-ribosome within polysomes shows the intervening mRNA density (inset: blue arrowhead) and the extended L9 of the preceding ribosome (i). The C-terminal domain of extended L9 can interfere with elongation factor (EF) binding to the following ribosome (i+1).\\n\\n[Full size image](/articles/s41586-022-05255-2/figures/4)Whether translation elongation is synchronized or coordinated within polysomes is a long-standing question[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"),[19](/articles/s41586-022-05255-2#ref-CR19 \\\"Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. Mol. Cell 39, 560–569 (2010).\\\"),[50](/articles/s41586-022-05255-2#ref-CR50 \\\"Myasnikov, A. G. et al. The molecular structure of the left-handed supra-molecular helix of eukaryotic polyribosomes. Nat. Commun. 5, 5294 (2014).\\\"). To address this, we first examined whether the distribution of elongation states differs between the total ribosome population and polysomes. We found that although most states occurred equally frequently in both populations, the fractions of two states before EF-G binding (class 5, 'A\\\\*, P/E' and class 6a, 'A/P, P/E') are more frequent in polysomes (Fig. [4f](/articles/s41586-022-05255-2#Fig4)). We next calculated the frequencies of state pairs between two adjacent ribosomes (preceding versus following) and compared them to theoretical pair frequencies calculated from the bulk distribution (Extended Data Fig. [13a–c](/articles/s41586-022-05255-2#Fig17)). This comparison revealed that the occurrence of states among preceding and following ribosomes is not symmetric: following ribosomes more frequently populate states that require elongation factor binding to proceed to the next state (classes 1, 2a, 5 and 6a; Fig. [4g](/articles/s41586-022-05255-2#Fig4)). We statistically validated such differences with a permutation test, and further show that the asymmetry between preceding and following ribosomes increases as the distance threshold used to define polysomes decreases (Extended Data Fig. [13d–h](/articles/s41586-022-05255-2#Fig17), [Methods](/articles/s41586-022-05255-2#Sec7) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). This suggests that local coordination of translation elongation between adjacent ribosomes within the polysome is likely to be achieved by obstructing elongation factor binding that is required for the following ribosome to proceed to the next elongation state.\\n\\nTo investigate whether local coordination arises from structures specific to ribosomes engaged in polysomes, we performed structural classification of ribosome pairs to better resolve the ribosome–ribosome interface (Fig. [4h](/articles/s41586-022-05255-2#Fig4) and Extended Data Fig. [12j,k](/articles/s41586-022-05255-2#Fig16)). We found that within the interface of tightly packed polysomes, the ribosomal protein L9 of the preceding ribosome adopts an extended conformation and its C-terminal domain protrudes into the elongation-factor-binding site of the following ribosome (Fig. [4h](/articles/s41586-022-05255-2#Fig4) and Extended Data Fig. [12j–m](/articles/s41586-022-05255-2#Fig16)). Independent focused classification on L9 of all ribosomes showed that it mainly (68.9%) adopts a flat conformation on the ribosome surface, whereas it is extended in 20.2% of 70S ribosomes (Extended Data Fig. [12n](/articles/s41586-022-05255-2#Fig16)). Ribosomes with the extended L9 largely overlap with the ribosomes detected as polysomes, especially those with a tighter 't-t' arrangement (Extended Data Fig. [12n–p](/articles/s41586-022-05255-2#Fig16)). These results suggest that L9 tends to have a flat conformation in single ribosomes and to adopt the extended conformation within tightly assembled polysomes. This clarifies previous observations of L9 being extended in X-ray crystallography structures in which the crystal packing recapitulates configurations of compacted polysomes, but being found in the flat conformation in cryo-EM structures of isolated ribosomes[51](/articles/s41586-022-05255-2#ref-CR51 \\\"Dunkle, J. A. & Cate, J. H. D. in Ribosomes (eds Rodnina, M. V., Wintermeyer, W. & Green, R.) Ch. 6 (2011).\\\"). Although L9 has been reported to be non-essential, its mutations can cause increased frameshifting and ribosome compaction by one codon[52](/articles/s41586-022-05255-2#ref-CR52 \\\"Herr, A. J., Nelson, C. C., Wills, N. M., Gesteland, R. F. & Atkins, J. F. Analysis of the roles of tRNA structure, ribosomal protein L9, and the bacteriophage T4 gene 60 bypassing signals during ribosome slippage on mRNA. J. Mol. Biol. 309, 1029–1048 (2001).\\\"),[53](/articles/s41586-022-05255-2#ref-CR53 \\\"Smith, A. M., Costello, M. S., Kettring, A. H., Wingo, R. J. & Moore, S. D. Ribosome collisions alter frameshifting at translational reprogramming motifs in bacterial mRNAs. Proc. Natl Acad. Sci. USA 116, 21769–21779 (2019).\\\"). We therefore propose that within tightly assembled polysomes, L9 of one ribosome can adopt an extended conformation that sterically interferes with elongation factor binding to the following ribosome. This local coordination mechanism can buffer adjacent ribosomes and help to maintain translation fidelity by avoiding direct collision within polysomes during active translation elongation.\\n\\nConclusions\\n-----------\\n\\nOur study demonstrates the potential of cryo-ET in shedding light on dynamic processes that are performed by macromolecular machines in living cells at a high level of detail. The analysis captures the structural and functional diversity of actively translating ribosomes inside a genome-reduced bacterium. The translation elongation cycle retrieved from the cellular cryo-ET data recapitulates and complements current mechanistic models of translation that are derived from controlled in vitro studies. The quantitative structural profiling of the cellular translation machinery presented here improves our understanding of protein biogenesis by providing distribution probabilities of translation intermediates. It also reveals how the translation machinery that functions as an interconnected system responds to different antibiotic perturbations on the single-cell level. Our investigation of polysomes further illustrates the advantage of in-cell structural biology, which can relate functional states of a molecular machine to its molecular sociology and cellular context, leading to the discovery of an elongation coordination mechanism mediated by L9. The approaches developed here establish a framework to analyse structural dynamics of cellular processes in the future and will contribute to the construction of functional cell models at atomic detail.\\n\\nMethods\\n-------\\n\\n### Cryo-ET sample preparation and data collection\\n\\nThe *M.* *pneumoniae* cultivation, sample preparation and data collection were described previously[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). The three datasets of native untreated, Cm-treated and PUM-treated cells were re-processed in Warp and M 1.0.7 (the alpha versions that were officially released as v.1.0.9.)[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\"),[54](/articles/s41586-022-05255-2#ref-CR54 \\\"Tegunov, D. & Cramer, P. Real-time cryo-electron microscopy data preprocessing with Warp. Nat. Methods 16, 1146–1152 (2019).\\\"). A small dataset of 15 tomograms acquired with Volta phase plate was processed with the denoising network in Warp 1.0.9 for visualization purposes only (as shown in Fig. [1a](/articles/s41586-022-05255-2#Fig1)).\\n\\nA dataset of spectinomycin-treated cells was collected following the same procedure as before[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). In brief, spectinomycin (Sigma-Aldrich) at a final concentration of 0.4 mg ml−1 was added into the culture medium, 15–20 min before plunge-freezing. Tilt-series collection with the dose-symmetric scheme[55](/articles/s41586-022-05255-2#ref-CR55 \\\"Hagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. J. Struct. Biol. 197, 191–198 (2017).\\\") was performed on a Titan Krios transmission electron microscope equipped with a K3 camera (Gatan) using SerialEM 3.8 (ref. [56](/articles/s41586-022-05255-2#ref-CR56 \\\"Schorb, M., Haberbosch, I., Hagen, W. J. H., Schwab, Y. & Mastronarde, D. N. Software tools for automated transmission electron microscopy. Nat. Methods 16, 471–477 (2019).\\\")) with the following parameters: magnification 81,000×, pixel size 1.053 Å, tilt range −60° to 60° with 3° increment, total dose 120–140 e− Å−2.\\n\\nIn total, 356 native untreated, 65 Cm-treated, 86 PUM-treated and 70 Spc-treated cellular tomograms were analysed in this work. Details of cryo-EM data collection, refinement and validation statistics are provided in Supplementary Tables [1](/articles/s41586-022-05255-2#MOESM1) and [3](/articles/s41586-022-05255-2#MOESM1)–[6](/articles/s41586-022-05255-2#MOESM1).\\n\\n### Image processing, ribosome template matching and map refinement\\n\\nPre-processing (motion correction, CTF estimation, dose filtering and tilt-series sorting) was performed in Warp 1.0.9 (ref.[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\")). For the untreated, Cm-treated and PUM-treated datasets, the ribosome coordinates were adopted from previous particle picking[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). For the Spc-treated dataset, template matching was performed in PyTom[57](/articles/s41586-022-05255-2#ref-CR57 \\\"Hrabe, T. et al. PyTom: a python-based toolbox for localization of macromolecules in cryo-electron tomograms and subtomogram analysis. J. Struct. Biol. 178, 177–188 (2012).\\\"), followed by computational classification in RELION 3.0 (refs.[58](/articles/s41586-022-05255-2#ref-CR58 \\\"Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. Nat. Protoc. 11, 2054–2065 (2016).\\\"),[59](/articles/s41586-022-05255-2#ref-CR59 \\\"Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7, e42166 (2018).\\\") to exclude false positives, without manual cleaning. In total, 109,990 untreated, 21,299 Cm-treated, 23,014 PUM-treated and 13,418 Spc-treated ribosome sub-tomograms were reconstructed in Warp.\\n\\nThree-dimensional (3D) refinement and classification were performed in RELION 3.0 (refs. [58](/articles/s41586-022-05255-2#ref-CR58 \\\"Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. Nat. Protoc. 11, 2054–2065 (2016).\\\"),[59](/articles/s41586-022-05255-2#ref-CR59 \\\"Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7, e42166 (2018).\\\")). We then used the software M (v.1.0.9) to perform multi-particle refinement of the tilt-series and refine the average map[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\"). Refinement of both geometric (image and volume deformation) and CTF parameters was done for five rounds in a sequential manner. After M refinement on the 70S, we performed focused refinement on the 30S and 50S subunits separately to improve the local map quality. Fourier shell correlation (FSC) calculation between independently refined random half subsets, local resolution estimation and post-processing were done in M and RELION.\\n\\n### Atomic model building in high-resolution ribosome maps\\n\\nAtomic models for the 30S and 50S subunits were built de novo using maps from focused refinement of Cm-treated ribosomes reported in our previous work[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\"), which were deposited in the Electron Microscopy Data Bank (EMDB) under the accession codes EMD-11998 and EMD-11999. Homology models for *M.* *pneumoniae* ribosomal proteins were generated using the SWISS-MODEL[60](/articles/s41586-022-05255-2#ref-CR60 \\\"Waterhouse, A. et al. SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic Acids Res. 46, W296–W303 (2018).\\\") online server (<https://swissmodel.expasy.org/>; accessed April 2020), with a *Bacillus subtilis* ribosome (Protein Data Bank (PDB) 3J9W) as the template, except for the ribosomal proteins L9 (PDB 1DIV and 4V63), L10 (PDB 1ZAV) and S21 (PDB 5MMJ). rRNA sequences (REFSEQ NC\\\\_000912) were aligned to the *E.* *coli* rRNA (PDB 4YBB) using the SINA online server[61](/articles/s41586-022-05255-2#ref-CR61 \\\"Pruesse, E., Peplies, J. & Glockner, F. O. SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes. Bioinformatics 28, 1823–1829 (2012).\\\"). ModeRNA[62](/articles/s41586-022-05255-2#ref-CR62 \\\"Rother, M., Rother, K., Puton, T. & Bujnicki, J. M. ModeRNA: a tool for comparative modeling of RNA 3D structure. Nucleic Acids Res. 39, 4007–4022 (2011).\\\") was used to build homology models for rRNAs on the basis of the alignment, with PDB 4YBB as the template. Homology models were rigid-body-fitted into the cryo-ET densities using Chimera[63](/articles/s41586-022-05255-2#ref-CR63 \\\"Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605–1612 (2004).\\\"), followed by iterative refinement using PHENIX real-space refinement[64](/articles/s41586-022-05255-2#ref-CR64 \\\"Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. Acta Crystallogr D 74, 531–544 (2018).\\\") and manual adjustment in Coot[65](/articles/s41586-022-05255-2#ref-CR65 \\\"Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Acta Crystallogr D 66, 486–501 (2010).\\\"). The ribosomal proteins L9, L10 and L11 were only fitted as rigid bodies into the map owing to the less-resolved local density. The flat conformation of L9 that resides on the ribosome surface was found to be predominant according to the focused classification (see Extended Data Fig. [12n](/articles/s41586-022-05255-2#Fig16)) and was thus built in the model. Sequence extensions for ribosomal proteins S6, L22 and L29 were built de novo. Models were validated using MolProbity[66](/articles/s41586-022-05255-2#ref-CR66 \\\"Williams, C. J. et al. MolProbity: more and better reference data for improved all-atom structure validation. Protein Sci. 27, 293–315 (2018).\\\"). FSC curves between the model and the map were also calculated for validation[67](/articles/s41586-022-05255-2#ref-CR67 \\\"Chen, S. et al. High-resolution noise substitution to measure overfitting and validate resolution in 3D structure determination by single particle electron cryomicroscopy. Ultramicroscopy 135, 24–35 (2013).\\\").\\n\\n### Bioinformatic analysis of ribosomal proteins\\n\\nBioinformatic sequence analysis of ribosomal proteins in *M.* *pneumoniae* and the comparison with *E.* *coli* homologues were performed as follows, with the different steps described in detail below: (i) protein sequences and RefSeq genome annotation for the *M.* *pneumoniae* strain M129 (ATCC 29342) were downloaded from NCBI; (ii) protein sequences were also annotated with eggNOG-mapper to obtain COG (Clusters of Orthologous Groups) IDs[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[68](/articles/s41586-022-05255-2#ref-CR68 \\\"Tatusov, R. L., Galperin, M. Y., Natale, D. A. & Koonin, E. V. The COG database: a tool for genome-scale analysis of protein functions and evolution. Nucleic Acids Res. 28, 33–36 (2000).\\\"); (iii) for the annotated ribosomal proteins, the corresponding COG multiple sequence alignments from representative bacterial species were downloaded from the eggNOG database[69](/articles/s41586-022-05255-2#ref-CR69 \\\"Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Nucleic Acids Res. 47, D309–D314 (2019).\\\"); (iv) as *M.* *pneumoniae* M129 is not among the representative species, its protein sequences were added to the multiple sequence alignments with MAFFT software[70](/articles/s41586-022-05255-2#ref-CR70 \\\"Katoh, K., Rozewicki, J. & Yamada, K. D. MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization. Brief. Bioinform. 20, 1160–1166 (2019).\\\"); (v) for each COG multiple sequence alignment, the number of amino acids in every representative species (including *M.* *pneumoniae* M129) present at positions before the N terminus and after the C terminus positions of *E.* *coli* K-12 substr. MG1655 were calculated; (vi) the presence of N- or C-terminus extensions longer than 20 amino acids was illustrated in iTOL using NCBI taxonomy tree as the basis[71](/articles/s41586-022-05255-2#ref-CR71 \\\"Letunic, I. & Bork, P. Interactive Tree Of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. Nucleic Acids Res. 49, W293–W296 (2021).\\\"); (vii) for all COGs corresponding to proteins with N- or C-terminus extensions in *M.* *pneumoniae* that are longer than 20 amino acids, protein disorder and secondary structure were analysed for all representative strains. The presence of cross-links or transposon insertions was also analysed.\\n\\nRefSeq genome annotation and protein sequences for the *M.* *pneumoniae* strain M129 (ATCC 29342) were downloaded from NCBI for the *M.* *pneumoniae* M129 GCF\\\\_000027345.1 assembly from <https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/027/345/GCF_000027345.1_ASM2734v1/> (accessed 21 April 2021). The file GCF\\\\_000027345.1\\\\_ASM2734v1\\\\_genomic.gff was used to identify ribosomal proteins annotated by RefSeq. The file GCF\\\\_000027345.1\\\\_ASM2734v1\\\\_protein.faa with protein sequences in FASTA format was used as input to the online tool eggNOG-mapper v2, which uses precomputed eggNOG v5.0 clusters and phylogenies for fast orthology assignment[69](/articles/s41586-022-05255-2#ref-CR69 \\\"Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Nucleic Acids Res. 47, D309–D314 (2019).\\\"),[72](/articles/s41586-022-05255-2#ref-CR72 \\\"Huerta-Cepas, J. et al. Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper. Mol. Biol. Evol. 34, 2115–2122 (2017).\\\"). The annotation tables by RefSeq and eggNOG-mapper were merged by protein ID. In total, 52 ribosomal proteins were annotated and mapped to 51 unique COGs[68](/articles/s41586-022-05255-2#ref-CR68 \\\"Tatusov, R. L., Galperin, M. Y., Natale, D. A. & Koonin, E. V. The COG database: a tool for genome-scale analysis of protein functions and evolution. Nucleic Acids Res. 28, 33–36 (2000).\\\"). Two proteins, WP\\\\_010874426.1 and WP\\\\_010874827.1, were mapped to the same COG0267 corresponding to the ribosomal protein L33. Multiple sequence alignments for the annotated proteins from representative bacterial species were downloaded from the eggNOG v5.0 database[69](/articles/s41586-022-05255-2#ref-CR69 \\\"Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Nucleic Acids Res. 47, D309–D314 (2019).\\\"). For each of the 51 COG IDs mapping to ribosomal proteins in *M.* *pneumoniae*, the trimmed alignments were downloaded from <http://eggnogapi5.embl.de/nog_data/text/trimmed_alg/COG_ID> ('COG\\\\_ID' in the url should be changed to the corresponding COG ID). As *M.* *pneumoniae* M129 is not among the imported representative species, its sequences were added to the multiple sequence alignments. The sequence of each of the proteins was saved in a separate fasta file, and the MAFFT software v.7.475 was used in the add mode for each alignment as follows: mafft --reorder --add protein.fasta --auto trimmed\\\\_alg\\\\_COG.fa > output\\\\_file (ref. [70](/articles/s41586-022-05255-2#ref-CR70 \\\"Katoh, K., Rozewicki, J. & Yamada, K. D. MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization. Brief. Bioinform. 20, 1160–1166 (2019).\\\")). For COG0627, the protein WP\\\\_010874827.1 from *M.* *pneumoniae* was used.\\n\\nFor each of the ribosomal COGs, the amino acid positions of the protein from the *E.* *coli* strain K-12 substr. MG1655 was taken as a reference, and for each protein from representative species (including *M.* *pneumoniae* M129), the number of amino acids before the N-terminus and after the C-terminus locations in *E.* *coli* was calculated. In total, 11 ribosomal proteins that had N- or C-terminus extensions of more than 20 amino acids in *M. pneumoniae* M129 were considered to have extensions (2 proteins with N-terminus extensions and 9 proteins with C-terminus extensions). Ribosomal protein S3 (COG0092), which has a 17-amino-acid extension at the C terminus, was also retained for further analysis. NCBI bacterial taxonomy file new\\\\_taxdump.zip was downloaded from <https://ftp.ncbi.nlm.nih.gov/pub/taxonomy/new_taxdump/> (version on 24 April 2021). The file names.dmp was used to map NCBI IDs to taxonomy names. Tree was reconstructed from the file taxidlineage.dmp in Python v.3.7.7 with ETE3 Toolkit v.3.1.2 (ref. [73](/articles/s41586-022-05255-2#ref-CR73 \\\"Huerta-Cepas, J., Serra, F. & Bork, P. ETE 3: reconstruction, analysis, and visualization of phylogenomic data. Mol. Biol. Evol. 33, 1635–1638 (2016).\\\")). EggNOG protein IDs containing NCBI species IDs were directly mapped to the NCBI tree nodes. The presence of ribosomal protein extensions was saved as tables and converted to iTOL format with table2itol utility (<https://github.com/mgoeker/table2itol>) in R v.3.6.1. The tree was visualized with iTOL v.6 (ref. [71](/articles/s41586-022-05255-2#ref-CR71 \\\"Letunic, I. & Bork, P. Interactive Tree Of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. Nucleic Acids Res. 49, W293–W296 (2021).\\\")).\\n\\nSequences for the 11 ribosomal proteins annotated with extensions in *M.* *pneumoniae*, and their orthologues in other species, were further analysed in terms of protein disorder and secondary structures. For disorder prediction, all protein sequences from the multiple sequence alignment files were saved in FASTA format without gaps. The IUPred2A tool in Python v.3.7.7 was run with the input parameter 'long' for each protein sequence[74](/articles/s41586-022-05255-2#ref-CR74 \\\"Meszaros, B., Erdos, G. & Dosztanyi, Z. IUPred2A: context-dependent prediction of protein disorder as a function of redox state and protein binding. Nucleic Acids Res. 46, W329–W337 (2018).\\\"). The number of disordered amino acids and disorder length in extended regions were calculated on the basis of the position of the extended region relative to *E.* *coli*, and an IUPred2A score of 0.5 was used as the disorder threshold. The JPred prediction tool[75](/articles/s41586-022-05255-2#ref-CR75 \\\"Cole, C., Barber, J. D. & Barton, G. J. The Jpred 3 secondary structure prediction server. Nucleic Acids Res. 36, W197–W201 (2008).\\\") was used to predict the secondary structure of each protein using the jpredapi utility and JPred-big-batch-submission utility for a large number of submissions (<https://github.com/fabianegli/JPred-big-batch-submission>). The number of helices in extended regions was calculated on the basis of the position of the extended region relative to *E.* *coli* protein.\\n\\nFor the 11 ribosomal proteins with extensions in *M.* *pneumoniae*, the positions of cross-links linking to the same protein or a different protein were mapped according to the amino acid cross-link positions reported previously[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). To analyse transposon insertions in *M.* *pneumoniae* ribosomal proteins with extensions, the supplementary materials from a previous publication[24](/articles/s41586-022-05255-2#ref-CR24 \\\"Miravet-Verde, S., Burgos, R., Delgado, J., Lluch-Senar, M. & Serrano, L. FASTQINS and ANUBIS: two bioinformatic tools to explore facts and artifacts in transposon sequencing and essentiality studies. Nucleic Acids Res. 48, e102 (2020).\\\") were used. All the files in the 'SupplementaryData1\\\\_fastqins\\\\_processed.zip folder' (files with extension '.qins' but not '\\\\_fw.qins' or '\\\\_rv.qins') were concatenated to obtain a list of nucleotide coordinates of transposon insertions in the sequence of *M.* *pneumoniae* M129 NC\\\\_000912.1. The transposon insertion locations in the 11 ribosomal proteins were selected on the basis of their genomic locations as per RefSeq annotations (GCF\\\\_000027345.1\\\\_ASM2734v1\\\\_genomic.gff). Nucleotide locations were converted to amino acid locations by calculating the difference between each transposon location and the start coordinate of the corresponding gene and dividing the number of nucleotides by three. All data were mapped to the protein sequences to derive Extended Data Fig. [3](/articles/s41586-022-05255-2#Fig7).\\n\\n### Sub-tomogram classification of the translation states of ribosomes\\n\\nMaximum-likelihood 3D classification[76](/articles/s41586-022-05255-2#ref-CR76 \\\"Scheres, S. H. Processing of structurally heterogeneous cryo-EM data in RELION. Methods Enzymol. 579, 125–157 (2016).\\\") was performed in RELION 3.0 (refs. [58](/articles/s41586-022-05255-2#ref-CR58 \\\"Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. Nat. Protoc. 11, 2054–2065 (2016).\\\"),[59](/articles/s41586-022-05255-2#ref-CR59 \\\"Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7, e42166 (2018).\\\")) with the re-extracted ribosome sub-tomograms after M refinement. A hierarchical and exhaustive classification strategy ([Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)) with at least three tiers was used to handle the heterogeneity in the native untreated dataset, which is described here in detail and illustrated in Extended Data Figs. [4](/articles/s41586-022-05255-2#Fig8) and [5](/articles/s41586-022-05255-2#Fig9). A similar procedure was applied for all antibiotics-treated datasets, and is detailed in Extended Data Figs. [9](/articles/s41586-022-05255-2#Fig13)–[11](/articles/s41586-022-05255-2#Fig15).\\n\\nIn the first tier, 109,990 sub-tomograms were classified into 70S and free 50S with a global spherical mask (320-Å diameter). The 24,157 free 50S were further classified into two classes, with a local spherical mask focusing on the ribosome recycling factor binding site. Before extensive 70S classification, structural heterogeneity was evaluated by visual inspection, multibody refinement and test classification runs in RELION. Classification set-ups were extensively tested, including different masks (global 70S mask, local 30S mask, spherical tRNA path mask, solvent tRNA path mask, spherical EF mask, solvent EF mask), initial references (ribosome average, features-less sphere or none), angular search options (global, local or without alignment) and RELION optimization parameters (class numbers 2–16, T values 2–10, iterations 25–40, limit resolution E-step 5–10 Å). To avoid bias, we mostly used a sphere or other featureless shapes with soft edges as the mask. Local spherical masks covering factors (elongation factors and/or tRNAs) generally provided more consistent and stable classification results than those obtained when using larger masks for the untreated data. The class number was made higher than the number of distinct structures that could be retained in one classification job and similar resulting classes were grouped.\\n\\nIn the second tier, the 77,539 identified 70S ribosomes were further classified with a local mask (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8), mask I) focusing on the tRNA path region, roughly covering the A, P and E sites. This resulted in four major classes with different tRNA occupancies: 'P, E', 'a, P/E', 'P' and 'A, P'. In the following rounds, we could further classify the 'a, P/E' class into 'A\\\\*, P/E' and 'A/P, P/E'. In addition, a 70S class with only one hybrid 'P/E' tRNA was classified. Two of the resulting classes were not interpretable: 2,150 particles with dim 30S density that were poorly resolved and 1,484 particles with density near the P site that does not resemble a tRNA.\\n\\nIn the third tier, focused classification with a local mask (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8), mask II) around the elongation factor and A/T tRNA binding sites was carried out. For the previous class with 'P, E' tRNAs, further classification resulted in 1,803 particles without additional density, 3,324 particles with EF-G (updated as 'EF-G, ap/P, pe/E') and 4,634 particles with EF-Tu•tRNA. For the classes with partial and full hybrid tRNAs, sub-classes with EF-G were obtained. For the class with only 'P' tRNA, 12,464 particles with additional EF-Tu•tRNA were classified.\\n\\nFor each classification step, at least three parallel RELION classification jobs with the same or slightly different parameters (either 30S mask or spherical EF/tRNA mask, local angular search range, class number, T value) were carried out for comparison. The classification job with the most stable result was selected and used for sorting sub-tomograms (Extended Data Fig. [5a–g](/articles/s41586-022-05255-2#Fig9)). After sorting, subsequent validation classification runs were performed for each sorted class to test whether new structures emerge or whether particles were 'wrongly' classified. Misclassified particles were relocated to the corresponding class and the validation runs were repeated until convergence. This approach was performed for all classification steps. For each of the final classes, refinement and post-processing were done in RELION. Further classification performed with the new refinement results as inputs did not generate any new classes.\\n\\n### Model building and comparison of the ribosome translation states\\n\\nTo build models for the ribosome classes, the 30S and 50S models built as described above were used as starting models for flexible fitting. Homology models of EF-G and EF-Tu were generated by SWISS-MODEL (<https://swissmodel.expasy.org/>; accessed April 2020) with PDB structures 4V7D and 4V5L as the template, respectively. For tRNA homology models, the tRNA in an *E.* *coli* ribosome structure (PDB 4V7C) was used as the template and mutated to the sequence of *M.* *pneumoniae* Phe-tRNA (REFSEQ NC\\\\_000912). The mRNA and nascent peptide were built using PDB 3J9W as the template. The homology model of the ribosome recycling factor was built using PDB 1EH1 as the template. For each class, the starting models were first rigid-body-fitted into the density using Chimera and then flexible fitting was done using Namdinator[77](#ref-CR77 \\\"Kidmose, R. T. et al. Namdinator—automatic molecular dynamics flexible fitting of structural models into cryo-EM and crystallography experimental maps. IUCrJ 6, 526–531 (2019).\\\"),[78](#ref-CR78 \\\"Trabuco, L. G., Villa, E., Mitra, K., Frank, J. & Schulten, K. Flexible fitting of atomic structures into electron microscopy maps using molecular dynamics. Structure 16, 673–683 (2008).\\\"),[79](/articles/s41586-022-05255-2#ref-CR79 \\\"Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. J. Chem. Phys. 153, 044130 (2020).\\\"). Validation was performed as described above.\\n\\nFor measuring 30S body rotations, structures of all classes were first aligned to the 50S subunit and then the rotation angles were estimated with a pivot point near nucleotide 11 of the 16S rRNA. With the perspective from the solvent side of the 30S subunit, positive numbers equal counter-clockwise rotations and negative numbers equal clockwise rotations. For measuring 30S head rotations, all class structures were first aligned to the 30S body and then the rotation angles were determined with the axis near the 30S neck. For both rotations, the angles in the 'A, P' class were defined as 0°. To describe the L1 dynamics, the distance between the mass centre of the L1 stalk (near nucleotide 2,181 of the 23S rRNA) and a fixed point near the centre of the classical E site (determined on the basis of the 'P, E' class) was measured after aligning all classes on the 50S subunit (excluding the L1 stalk).\\n\\n### Spatial analysis of ribosomes and polysomes\\n\\nSpatial mapping of ribosomes within cellular tomograms was achieved by projecting back the ribosome structures into the tomograms, with coordinates determined by template matching and shifts and rotations determined by RELION refinement. The projection was performed using the TOM toolbox[80](/articles/s41586-022-05255-2#ref-CR80 \\\"Nickell, S. et al. TOM software toolbox: acquisition and analysis for electron tomography. J. Struct. Biol. 149, 227–234 (2005).\\\") after Euler angle format conversion, at four times binning (voxel size 6.8 Å). To calculate the ribosome concentration, we first estimated the cellular volume covered in the tomogram and then divided the total number of detected ribosomes by the volume.\\n\\nDetection of the polysomes was based on both position and orientation information (Extended Data Fig. [12c](/articles/s41586-022-05255-2#Fig16)), using a custom script in MATLAB 2016b. It is noted that the annotated polysomes only refer to those assembled closely in space and thus can be detected on the basis of their spatial proximity. The positions of mRNA entry and exit sites for all 70S ribosomes were calculated on the basis of the rotations and shifts determined during RELION refinement. The distances from the mRNA exit site of one ribosome (defined as the preceding ribosome i) to the mRNA entry sites of all neighbouring ribosomes (as potential following ribosomes i+1) were calculated. A distance threshold of 7 nm was used to define whether two ribosomes belong to the same polysome. The calculation and polysome definition were done for all 70S ribosomes. A unique identifier was assigned to each polysome, as well as the sequential number for all ribosomes within the polysome.\\n\\nThe distribution of relative positions of adjacent ribosomes within the polysome—that is, the position of the following ribosome (i+1) relative to the preceding ribosome (i)—was analysed after normalizing the relative position vectors with rotations of the preceding ribosome determined from RELION refinement. The relative rotation of the following ribosome to the preceding ribosome was represented as three Euler angles (*ψ*, *θ*, *φ* in XYZ system). Using the *k*-means clustering function in MATLAB 2016b, we determined two major arrangement configurations for adjacent ribosomes within the polysome; that is, how the following ribosome rotates relative to the preceding ribosome (as shown in Extended Data Fig. [12f,g](/articles/s41586-022-05255-2#Fig16)). These two configurations are identical to the previously reported 'top-top' (t-t) and 'top-back' (t-b) configurations[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"), and the naming was adopted.\\n\\nTo further refine the ribosome–ribosome interface in polysomes, RELION classification was performed with sub-tomograms extracted with a large box size that can accommodate two ribosomes. After refinement on the preceding ribosome (i), classification without refinement was performed with a local mask focusing on the following ribosome (i+1). Only ribosome pairs within tightly packed polysomes (5,083 pairs from 5 di-ribosome classes; all with a 't-t' arrangement and extended L9 in between) resulted in average densities with both ribosomes well resolved. Models for the resulting classes were generated with rigid-body-fitting of the above described  ribosome models and the L9 homology model with the extended conformation (PDB 4V63).\\n\\n### Statistical analysis of translation elongation states in polysomes\\n\\nTo compare the experimental elongation state frequencies in polysomes with theoretical frequencies, the distributions of frequencies of each elongation state calculated across 356 tomograms of the untreated cells were compared between polysomes and all ribosomes, and between polysomes and mono-ribosomes by calculating the fold change between distribution medians. Statistical significance was assessed with a two-sided Wilcoxon–Mann–Whitney test using the ranksum function in MATLAB 2019b. The theoretical frequency of each ribosome pair was calculated as the product of the overall frequencies of the ribosome classes for the preceding ribosome (i) and the following ribosome (i+1). Experimental polysome pair frequencies were calculated by summarizing the numbers of all ribosome pairs engaged in polysomes, and dividing these numbers by the total number of pairs. Experimental and theoretical pair frequencies were compared by calculating the fold change per pair.\\n\\nPermutation analysis was performed to test the significance of differences between occurrence frequencies of the experimental pairs compared to the theoretical pairs. The ribosome pair sequences in polysomes were represented as a matrix with single polysomes as rows and ribosome positions as columns, in which each matrix cell contains the ribosome class representing its elongation state. Columns with positions beyond each ribosome’s length were assigned to NaN (not-a-number). All polysomes across all tomograms were combined in one matrix (8,641 polysomes in total). For permutation analysis, all elements of the polysome matrix were randomly shuffled for 10,000 times with the randperm function in MATLAB 2019b. For each row, the elements were sorted so that the ribosome classes are in the front columns followed by NaNs occurring in the same row after shuffling. Rows with fewer than two ribosomes in the sequence were deleted. Shuffled ribosome pair frequencies were calculated in the same way as experimental pair frequencies. The permutation *P* value for each ribosome pair was calculated as the minimum between the number of permutations in which the pair frequency was less or equal to the experimentally observed frequency divided by the total number of permutations, and the one minus this value. Permutation *P* values were adjusted for multiple hypotheses testing with the Benjamini–Hochberg procedure using the mafdr function in MATLAB 2019b with parameters ('bhfdr', 'true').\\n\\nFor polysome distance threshold analysis, matrices were created from the full ribosome dataset by varying the distance threshold to the nearest neighbour in the range from 3 to 10 nm. The distances for polysome definition were calculated between the mRNA exit site of one ribosome and the mRNA entry site of another ribosome as described above. The fractions of elongation states in ribosome pairs were calculated as for the original ribosome set (with the threshold of 7 nm). For each distance threshold and each ribosome class, the ratio between the number of pairs in which the preceding ribosome has this class and the number of pairs in which the following ribosome has this class was calculated.\\n\\n### Single-cell clustering analysis\\n\\nThe distributions of 70S ribosome classes identified in the translation elongation phase for all four datasets (356 untreated cells, 65 Cm-treated cells, 70 Spc-treated cells and 86 PUM-treated cells) were used for clustering analysis. As each tomogram covers most of one cell, the sub-tomograms in each class could be further separated by which cell they belong to. Classes in the antibiotic-treated cells were assigned with the class identifiers according to the closest classes in the untreated dataset. The percentages of different classes within each cell were calculated and these numbers were used as inputs for clustering analysis. Hierarchical clustering analysis was done using the clustergram function in MATLAB 2016b.\\n\\nStructure visualization, preparation for figures and videos were done in Chimera[63](/articles/s41586-022-05255-2#ref-CR63 \\\"Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605–1612 (2004).\\\") and ChimeraX[81](/articles/s41586-022-05255-2#ref-CR81 \\\"Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. Protein Sci. 30, 70–82 (2021).\\\").\\n\\n### Reporting summary\\n\\nFurther information on research design is available in the [Nature Research Reporting Summary](/articles/s41586-022-05255-2#MOESM2) linked to this article.\\n\\n\\n\\n\\nData availability\\n-----------------\\n\\n\\nMaps have been deposited in the EMDB under accession codes [13234](http://www.ebi.ac.uk/pdbe/entry/EMD-13234), [13272](http://www.ebi.ac.uk/pdbe/entry/EMD-13272), [13273](http://www.ebi.ac.uk/pdbe/entry/EMD-13273), [13274](http://www.ebi.ac.uk/pdbe/entry/EMD-13274), [13275](http://www.ebi.ac.uk/pdbe/entry/EMD-13275), [13276](http://www.ebi.ac.uk/pdbe/entry/EMD-13276), [13277](http://www.ebi.ac.uk/pdbe/entry/EMD-13277), [13278](http://www.ebi.ac.uk/pdbe/entry/EMD-13278), [13279](http://www.ebi.ac.uk/pdbe/entry/EMD-13279), [13280](http://www.ebi.ac.uk/pdbe/entry/EMD-13280), [13281](http://www.ebi.ac.uk/pdbe/entry/EMD-13281), [13282](http://www.ebi.ac.uk/pdbe/entry/EMD-13282), [13283](http://www.ebi.ac.uk/pdbe/entry/EMD-13283), [13284](http://www.ebi.ac.uk/pdbe/entry/EMD-13284), [13285](http://www.ebi.ac.uk/pdbe/entry/EMD-13285), [13286](http://www.ebi.ac.uk/pdbe/entry/EMD-13286), [13410](http://www.ebi.ac.uk/pdbe/entry/EMD-13410), [13411](http://www.ebi.ac.uk/pdbe/entry/EMD-13411), [13412](http://www.ebi.ac.uk/pdbe/entry/EMD-13412), [13413](http://www.ebi.ac.uk/pdbe/entry/EMD-13413), [13414](http://www.ebi.ac.uk/pdbe/entry/EMD-13414), [13431](http://www.ebi.ac.uk/pdbe/entry/EMD-13431), [13432](http://www.ebi.ac.uk/pdbe/entry/EMD-13432), [13433](http://www.ebi.ac.uk/pdbe/entry/EMD-13433), [13434](http://www.ebi.ac.uk/pdbe/entry/EMD-13434), [13435](http://www.ebi.ac.uk/pdbe/entry/EMD-13435), [13436](http://www.ebi.ac.uk/pdbe/entry/EMD-13436), [13445](http://www.ebi.ac.uk/pdbe/entry/EMD-13445), [13446](http://www.ebi.ac.uk/pdbe/entry/EMD-13446), [13447](http://www.ebi.ac.uk/pdbe/entry/EMD-13447), [13448](http://www.ebi.ac.uk/pdbe/entry/EMD-13448), [13449](http://www.ebi.ac.uk/pdbe/entry/EMD-13449), [13450](http://www.ebi.ac.uk/pdbe/entry/EMD-13450), [13451](http://www.ebi.ac.uk/pdbe/entry/EMD-13451), [13452](http://www.ebi.ac.uk/pdbe/entry/EMD-13452), [13287](http://www.ebi.ac.uk/pdbe/entry/EMD-13287), [13288](http://www.ebi.ac.uk/pdbe/entry/EMD-13288) and [13289](http://www.ebi.ac.uk/pdbe/entry/EMD-13289). Models have been deposited in the PDB under accession codes: [7OOC](https://doi.org/10.2210/pdb7OOC/pdb), [7OOD](http://doi.org/10.2210/pdb7OOD/pdb), [7P6Z](http://doi.org/10.2210/pdb7P6Z/pdb), [7PAH](http://doi.org/10.2210/pdb7PAH/pdb), [7PAI](http://doi.org/10.2210/pdb7PAI/pdb), [7PAJ](http://doi.org/10.2210/pdb7PAJ/pdb), [7PAK](http://doi.org/10.2210/pdb7PAK/pdb), [7PAL](http://doi.org/10.2210/pdb7PAL/pdb), [7PAM](http://doi.org/10.2210/pdb7PAM/pdb), [7PAN](http://doi.org/10.2210/pdb7PAN/pdb), [7PAO](http://doi.org/10.2210/pdb7PAO/pdb), [7PAQ](http://doi.org/10.2210/pdb7PAQ/pdb), [7PAR](http://doi.org/10.2210/pdb7PAR/pdb), [7PAS](http://doi.org/10.2210/pdb7PAS/pdb), [7PAT](http://doi.org/10.2210/pdb7PAT/pdb), [7PAU](http://doi.org/10.2210/pdb7PAU/pdb), [7PH9](http://doi.org/10.2210/pdb7PH9/pdb), [7PHA](http://doi.org/10.2210/pdb7PHA/pdb), [7PHB](http://doi.org/10.2210/pdb7PHB/pdb), [7PHC](http://doi.org/10.2210/pdb7PHC/pdb), [7PI8](http://doi.org/10.2210/pdb7PI8/pdb), [7PI9](http://doi.org/10.2210/pdb7PI9/pdb), [7PIA](http://doi.org/10.2210/pdb7PIA/pdb), [7PIB](http://doi.org/10.2210/pdb7PIB/pdb), [7PIC](http://doi.org/10.2210/pdb7PIC/pdb), [7PIO](http://doi.org/10.2210/pdb7PIO/pdb), [7PIP](http://doi.org/10.2210/pdb7PIP/pdb), [7PIQ](http://doi.org/10.2210/pdb7PIQ/pdb), [7PIR](http://doi.org/10.2210/pdb7PIR/pdb), [7PIS](http://doi.org/10.2210/pdb7PIS/pdb) and [7PIT](http://doi.org/10.2210/pdb7PIT/pdb). Detailed information for all maps and models generated in this work is provided in Supplementary Tables [1](/articles/s41586-022-05255-2#MOESM1) and [3](/articles/s41586-022-05255-2#MOESM1)–[6](/articles/s41586-022-05255-2#MOESM1). Maps and atomic models used from previous studies were obtained from the EMDB ([11998](http://www.ebi.ac.uk/pdbe/entry/EMD-11998) and [11999](http://www.ebi.ac.uk/pdbe/entry/EMD-11999)) and the PDB ([3J9W](http://doi.org/10.2210/pdb3J9W/pdb), [1DIV](http://doi.org/10.2210/pdb1DIV/pdb), [4V63](https://doi.org/10.2210/pdb4V63/pdb), [1ZAV](http://doi.org/10.2210/pdb1ZAV/pdb), [5MMJ](http://doi.org/10.2210/pdb5MMJ/pdb), [4YBB](http://doi.org/10.2210/pdb4YBB/pdb), [4V7C](http://doi.org/10.2210/pdb4V7C/pdb), [4V7D](http://doi.org/10.2210/pdb4V7D/pdb), [4V5L](http://doi.org/10.2210/pdb4V5L/pdb) and [1EH1](http://doi.org/10.2210/pdb1EH1/pdb)).\\n\\n\\nCode availability\\n-----------------\\n\\n\\nThe code and associated data for bioinformatics analysis of ribosomal protein extensions (Extended Data Fig. [3](/articles/s41586-022-05255-2#Fig7)) and statistical analysis of polysome sequences (Fig. [4](/articles/s41586-022-05255-2#Fig4) and Extended Data Fig. [13](/articles/s41586-022-05255-2#Fig17)) are deposited at GitHub (<https://github.com/mszimmermann/mycoplasma_ribosome>). The MATLAB script for polysome annotation (Fig. [4](/articles/s41586-022-05255-2#Fig4) and Extended Data Fig. [12](/articles/s41586-022-05255-2#Fig16)) can be found at <https://github.com/xueliang4906/polysome_detect>.\\n\\n\\nChange history\\n--------------\\n\\n* ### 10 November 2022\\n\\nA Correction to this paper has been published: <https://doi.org/10.1038/s41586-022-05455-w>\\nReferences\\n----------\\n\\n1. Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. *Annu. Rev. Biochem.* **82**, 203–236 (2013).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-biochem-113009-092313) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFOqsrzJ) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23746255) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20the%20translational%20elongation%20cycle&journal=Annu.%20Rev.%20Biochem.&doi=10.1146%2Fannurev-biochem-113009-092313&volume=82&pages=203-236&publication_year=2013&author=Voorhees%2CRM&author=Ramakrishnan%2CV)\\n2. O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. *Science* **369**, 554–557 (2020).\\n\\n[Article](https://doi.org/10.1126%2Fscience.abb3758) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020Sci...369..554O) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFWgtr7N) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32732422) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115962) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=In-cell%20architecture%20of%20an%20actively%20transcribing%E2%80%93translating%20expressome&journal=Science&doi=10.1126%2Fscience.abb3758&volume=369&pages=554-557&publication_year=2020&author=O%27Reilly%2CFJ)\\n3. Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. *Nat. Methods* **18**, 186–193 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-020-01054-7) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXjt1SmsL8%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33542511) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7611018) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-particle%20cryo-EM%20refinement%20with%20M%20visualizes%20ribosome%E2%80%93antibiotic%20complex%20at%203.5%20%C3%85%20in%20cells&journal=Nat.%20Methods&doi=10.1038%2Fs41592-020-01054-7&volume=18&pages=186-193&publication_year=2021&author=Tegunov%2CD&author=Xue%2CL&author=Dienemann%2CC&author=Cramer%2CP&author=Mahamid%2CJ)\\n4. Brandt, F. et al. The native 3D organization of bacterial polysomes. *Cell* **136**, 261–271 (2009).\\n\\n[Article](https://doi.org/10.1016%2Fj.cell.2008.11.016) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1MXhs1Kiurk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19167328) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20native%203D%20organization%20of%20bacterial%20polysomes&journal=Cell&doi=10.1016%2Fj.cell.2008.11.016&volume=136&pages=261-271&publication_year=2009&author=Brandt%2CF)\\n5. Rodnina, M. V. Translation in prokaryotes. *Cold Spring Harb. Perspect. Biol.* **10**, a032664 (2018).\\n\\n[Article](https://doi.org/10.1101%2Fcshperspect.a032664) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29661790) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6120702) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Translation%20in%20prokaryotes&journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&doi=10.1101%2Fcshperspect.a032664&volume=10&publication_year=2018&author=Rodnina%2CMV)\\n6. Frank, J. & Agrawal, R. K. A ratchet-like inter-subunit reorganization of the ribosome during translocation. *Nature* **406**, 318–322 (2000).\\n\\n[Article](https://doi.org/10.1038%2F35018597) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2000Natur.406..318F) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD3cXlsFKnsrk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10917535) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20ratchet-like%20inter-subunit%20reorganization%20of%20the%20ribosome%20during%20translocation&journal=Nature&doi=10.1038%2F35018597&volume=406&pages=318-322&publication_year=2000&author=Frank%2CJ&author=Agrawal%2CRK)\\n7. Villa, E. et al. Ribosome-induced changes in elongation factor Tu conformation control GTP hydrolysis. *Proc. Natl Acad. Sci. USA* **106**, 1063–1068 (2009).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.0811370106) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009PNAS..106.1063V) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1MXhsFyjtbk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19122150) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2613361) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome-induced%20changes%20in%20elongation%20factor%20Tu%20conformation%20control%20GTP%20hydrolysis&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.0811370106&volume=106&pages=1063-1068&publication_year=2009&author=Villa%2CE)\\n8. Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. *Nature* **466**, 329–333 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature09206) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.466..329F) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXosl2rsr0%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20631791) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome%20dynamics%20and%20tRNA%20movement%20by%20time-resolved%20electron%20cryomicroscopy&journal=Nature&doi=10.1038%2Fnature09206&volume=466&pages=329-333&publication_year=2010&author=Fischer%2CN&author=Konevega%2CAL&author=Wintermeyer%2CW&author=Rodnina%2CMV&author=Stark%2CH)\\n9. Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. *Nature* **468**, 713–716 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature09547) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.468..713R) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsFSku7jF) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21124459) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3272701) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Head%20swivel%20on%20the%20ribosome%20facilitates%20translocation%20by%20means%20of%20intra-subunit%20tRNA%20hybrid%20sites&journal=Nature&doi=10.1038%2Fnature09547&volume=468&pages=713-716&publication_year=2010&author=Ratje%2CAH)\\n10. Ramrath, D. J. et al. Visualization of two transfer RNAs trapped in transit during elongation factor G-mediated translocation. *Proc. Natl Acad. Sci. USA* **110**, 20964–20969 (2013).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1320387110) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2013PNAS..11020964R) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXnsF2jug%3D%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24324168) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876219) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualization%20of%20two%20transfer%20RNAs%20trapped%20in%20transit%20during%20elongation%20factor%20G-mediated%20translocation&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1320387110&volume=110&pages=20964-20969&publication_year=2013&author=Ramrath%2CDJ)\\n11. Brilot, A. F., Korostelev, A. A., Ermolenko, D. N. & Grigorieff, N. Structure of the ribosome with elongation factor G trapped in the pretranslocation state. *Proc. Natl Acad. Sci. USA* **110**, 20994–20999 (2013).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1311423110) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2013PNAS..11020994B) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXnsF2iuw%3D%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24324137) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876248) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structure%20of%20the%20ribosome%20with%20elongation%20factor%20G%20trapped%20in%20the%20pretranslocation%20state&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1311423110&volume=110&pages=20994-20999&publication_year=2013&author=Brilot%2CAF&author=Korostelev%2CAA&author=Ermolenko%2CDN&author=Grigorieff%2CN)\\n12. Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. *Cell* **161**, 845–857 (2015).\\n\\n[Article](https://doi.org/10.1016%2Fj.cell.2015.03.052) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2MXotVKlur0%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25957688) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4432480) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20snapshots%20of%20actively%20translating%20human%20ribosomes&journal=Cell&doi=10.1016%2Fj.cell.2015.03.052&volume=161&pages=845-857&publication_year=2015&author=Behrmann%2CE)\\n13. Loveland, A. B., Demo, G. & Korostelev, A. A. Cryo-EM of elongating ribosome with EF-Tu\\\\*GTP elucidates tRNA proofreading. *Nature* **584**, 640–645 (2020).\\n\\n[Article](https://doi.org/10.1038%2Fs41586-020-2447-x) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020Natur.584..640L) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlShtL%2FE) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32612237) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7483604) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cryo-EM%20of%20elongating%20ribosome%20with%20EF-Tu%2AGTP%20elucidates%20tRNA%20proofreading&journal=Nature&doi=10.1038%2Fs41586-020-2447-x&volume=584&pages=640-645&publication_year=2020&author=Loveland%2CAB&author=Demo%2CG&author=Korostelev%2CAA)\\n14. Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. *Nature* **595**, 741–745 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41586-021-03713-x) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Natur.595..741R) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFSnsbnM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34234344) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8318882) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20early%20translocation%20events%20on%20the%20ribosome&journal=Nature&doi=10.1038%2Fs41586-021-03713-x&volume=595&pages=741-745&publication_year=2021&author=Rundlet%2CEJ)\\n15. Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. *Nat. Commun.* **12**, 5933 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-26133-x) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.5933P) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1aqsL7M) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34635670) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8505512) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20mechanism%20of%20GTPase-powered%20ribosome%E2%80%93tRNA%20movement&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-26133-x&volume=12&publication_year=2021&author=Petrychenko%2CV)\\n16. Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. *Nat. Commun.* **12**, 7236 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-27415-0) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.7236C) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXislejtbjN) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34903725) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8668904) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Time-resolved%20cryo-EM%20visualizes%20ribosomal%20translocation%20with%20EF-G%20and%20GTP&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-27415-0&volume=12&publication_year=2021&author=Carbone%2CCE)\\n17. Brown, A. & Shao, S. Ribosomes and cryo-EM: a duet. *Curr. Opin. Struct. Biol.* **52**, 1–7 (2018).\\n\\n[Article](https://doi.org/10.1016%2Fj.sbi.2018.07.001) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlSjs77J) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30015201) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosomes%20and%20cryo-EM%3A%20a%20duet&journal=Curr.%20Opin.%20Struct.%20Biol.&doi=10.1016%2Fj.sbi.2018.07.001&volume=52&pages=1-7&publication_year=2018&author=Brown%2CA&author=Shao%2CS)\\n18. Kuhner, S. et al. Proteome organization in a genome-reduced bacterium. *Science* **326**, 1235–1240 (2009).\\n\\n[Article](https://doi.org/10.1126%2Fscience.1176343) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009Sci...326.1235K) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19965468) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Proteome%20organization%20in%20a%20genome-reduced%20bacterium&journal=Science&doi=10.1126%2Fscience.1176343&volume=326&pages=1235-1240&publication_year=2009&author=Kuhner%2CS)\\n19. Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. *Mol. Cell* **39**, 560–569 (2010).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2010.08.003) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVyrt7jF) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20797628) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20three-dimensional%20organization%20of%20polyribosomes%20in%20intact%20human%20cells&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2010.08.003&volume=39&pages=560-569&publication_year=2010&author=Brandt%2CF&author=Carlson%2CLA&author=Hartl%2CFU&author=Baumeister%2CW&author=Grunewald%2CK)\\n20. Mahamid, J. et al. Visualizing the molecular sociology at the HeLa cell nuclear periphery. *Science* **351**, 969–972 (2016).\\n\\n[Article](https://doi.org/10.1126%2Fscience.aad8857) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016Sci...351..969M) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XivFOnt7c%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26917770) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualizing%20the%20molecular%20sociology%20at%20the%20HeLa%20cell%20nuclear%20periphery&journal=Science&doi=10.1126%2Fscience.aad8857&volume=351&pages=969-972&publication_year=2016&author=Mahamid%2CJ)\\n21. Schaffer, M. et al. A cryo-FIB lift-out technique enables molecular-resolution cryo-ET within native *Caenorhabditis elegans* tissue. *Nat. Methods* **16**, 757–762 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0497-5) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVOku7zL) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31363205) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20cryo-FIB%20lift-out%20technique%20enables%20molecular-resolution%20cryo-ET%20within%20native%20Caenorhabditis%20elegans%20tissue&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0497-5&volume=16&pages=757-762&publication_year=2019&author=Schaffer%2CM)\\n22. Waltz, F. et al. How to build a ribosome from RNA fragments in *Chlamydomonas* mitochondria. *Nat. Commun.* **12**, 7176 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-27200-z) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.7176W) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXislWktLrO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34887394) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8660880) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=How%20to%20build%20a%20ribosome%20from%20RNA%20fragments%20in%20Chlamydomonas%20mitochondria&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-27200-z&volume=12&publication_year=2021&author=Waltz%2CF)\\n23. Melnikov, S., Manakongtreecheep, K. & Soll, D. Revising the structural diversity of ribosomal proteins across the three domains of life. *Mol. Biol. Evol.* **35**, 1588–1598 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsy021) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtF2rur7L) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29529322) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5995209) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Revising%20the%20structural%20diversity%20of%20ribosomal%20proteins%20across%20the%20three%20domains%20of%20life&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsy021&volume=35&pages=1588-1598&publication_year=2018&author=Melnikov%2CS&author=Manakongtreecheep%2CK&author=Soll%2CD)\\n24. Miravet-Verde, S., Burgos, R., Delgado, J., Lluch-Senar, M. & Serrano, L. FASTQINS and ANUBIS: two bioinformatic tools to explore facts and artifacts in transposon sequencing and essentiality studies. *Nucleic Acids Res.* **48**, e102 (2020).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkaa679) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXkt1GntbY%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32813015) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7515713) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FASTQINS%20and%20ANUBIS%3A%20two%20bioinformatic%20tools%20to%20explore%20facts%20and%20artifacts%20in%20transposon%20sequencing%20and%20essentiality%20studies&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkaa679&volume=48&publication_year=2020&author=Miravet-Verde%2CS&author=Burgos%2CR&author=Delgado%2CJ&author=Lluch-Senar%2CM&author=Serrano%2CL)\\n25. Borg, A., Pavlov, M. & Ehrenberg, M. Complete kinetic mechanism for recycling of the bacterial ribosome. *RNA* **22**, 10–21 (2016).\\n\\n[Article](https://doi.org/10.1261%2Frna.053157.115) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVKlur3M) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26527791) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4691825) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Complete%20kinetic%20mechanism%20for%20recycling%20of%20the%20bacterial%20ribosome&journal=RNA&doi=10.1261%2Frna.053157.115&volume=22&pages=10-21&publication_year=2016&author=Borg%2CA&author=Pavlov%2CM&author=Ehrenberg%2CM)\\n26. Munro, J. B., Altman, R. B., O’Connor, N. & Blanchard, S. C. Identification of two distinct hybrid state intermediates on the ribosome. *Mol. Cell* **25**, 505–517 (2007).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2007.01.022) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2sXjtlWkur8%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17317624) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2244649) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20two%20distinct%20hybrid%20state%20intermediates%20on%20the%20ribosome&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2007.01.022&volume=25&pages=505-517&publication_year=2007&author=Munro%2CJB&author=Altman%2CRB&author=O%E2%80%99Connor%2CN&author=Blanchard%2CSC)\\n27. Belardinelli, R. et al. Choreography of molecular movements during ribosome progression along mRNA. *Nat. Struct. Mol. Biol.* **23**, 342–348 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.3193) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XktFeqtrk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26999556) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Choreography%20of%20molecular%20movements%20during%20ribosome%20progression%20along%20mRNA&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.3193&volume=23&pages=342-348&publication_year=2016&author=Belardinelli%2CR)\\n28. Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. *Nat. Struct. Mol. Biol.* **23**, 333–341 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.3177) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XivFKhurY%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26926435) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4821728) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiperspective%20smFRET%20reveals%20rate-determining%20late%20intermediates%20of%20ribosomal%20translocation&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.3177&volume=23&pages=333-341&publication_year=2016&author=Wasserman%2CMR&author=Alejo%2CJL&author=Altman%2CRB&author=Blanchard%2CSC)\\n29. Fei, J., Kosuri, P., MacDougall, D. D. & Gonzalez, R. L. Jr Coupling of ribosomal L1 stalk and tRNA dynamics during translation elongation. *Mol. Cell* **30**, 348–359 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2008.03.012) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXmtFaktLg%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18471980) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Coupling%20of%20ribosomal%20L1%20stalk%20and%20tRNA%20dynamics%20during%20translation%20elongation&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2008.03.012&volume=30&pages=348-359&publication_year=2008&author=Fei%2CJ&author=Kosuri%2CP&author=MacDougall%2CDD&author=Gonzalez%2CRL)\\n30. Zhou, J., Lancaster, L., Donohue, J. P. & Noller, H. F. How the ribosome hands the A-site tRNA to the P site during EF-G-catalyzed translocation. *Science* **345**, 1188–1191 (2014).\\n\\n[Article](https://doi.org/10.1126%2Fscience.1255030) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2014Sci...345.1188Z) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsVGhtb%2FO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25190797) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4242719) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=How%20the%20ribosome%20hands%20the%20A-site%20tRNA%20to%20the%20P%20site%20during%20EF-G-catalyzed%20translocation&journal=Science&doi=10.1126%2Fscience.1255030&volume=345&pages=1188-1191&publication_year=2014&author=Zhou%2CJ&author=Lancaster%2CL&author=Donohue%2CJP&author=Noller%2CHF)\\n31. Uemura, S. et al. Real-time tRNA transit on single translating ribosomes at codon resolution. *Nature* **464**, 1012–U1073 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature08925) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.464.1012U) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXkslart7c%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20393556) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4466108) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20tRNA%20transit%20on%20single%20translating%20ribosomes%20at%20codon%20resolution&journal=Nature&doi=10.1038%2Fnature08925&volume=464&pages=1012-U1073&publication_year=2010&author=Uemura%2CS)\\n32. Chen, C. et al. Single-molecule fluorescence measurements of ribosomal translocation dynamics. *Mol. Cell* **42**, 367–377 (2011).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2011.03.024) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3MXlvVyiu7w%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21549313) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3090999) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-molecule%20fluorescence%20measurements%20of%20ribosomal%20translocation%20dynamics&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2011.03.024&volume=42&pages=367-377&publication_year=2011&author=Chen%2CC)\\n33. Wilson, D. N. & Nierhaus, K. H. The E-site story: the importance of maintaining two tRNAs on the ribosome during protein synthesis. *Cell. Mol. Life Sci.* **63**, 2725–2737 (2006).\\n\\n[Article](https://link.springer.com/doi/10.1007/s00018-006-6125-4) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFSrt7w%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17013564) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20E-site%20story%3A%20the%20importance%20of%20maintaining%20two%20tRNAs%20on%20the%20ribosome%20during%20protein%20synthesis&journal=Cell.%20Mol.%20Life%20Sci.&doi=10.1007%2Fs00018-006-6125-4&volume=63&pages=2725-2737&publication_year=2006&author=Wilson%2CDN&author=Nierhaus%2CKH)\\n34. Munro, J. B., Sanbonmatsu, K. Y., Spahn, C. M. & Blanchard, S. C. Navigating the ribosome’s metastable energy landscape. *Trends Biochem. Sci* **34**, 390–400 (2009).\\n\\n[Article](https://doi.org/10.1016%2Fj.tibs.2009.04.004) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVSntrnM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19647434) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2914510) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Navigating%20the%20ribosome%E2%80%99s%20metastable%20energy%20landscape&journal=Trends%20Biochem.%20Sci&doi=10.1016%2Fj.tibs.2009.04.004&volume=34&pages=390-400&publication_year=2009&author=Munro%2CJB&author=Sanbonmatsu%2CKY&author=Spahn%2CCM&author=Blanchard%2CSC)\\n35. Bock, L. V. et al. Energy barriers and driving forces in tRNA translocation through the ribosome. *Nat. Struct. Mol. Biol.* **20**, 1390–1396 (2013).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.2690) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslWisbvK) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24186064) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Energy%20barriers%20and%20driving%20forces%20in%20tRNA%20translocation%20through%20the%20ribosome&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.2690&volume=20&pages=1390-1396&publication_year=2013&author=Bock%2CLV)\\n36. Julian, P. et al. Structure of ratcheted ribosomes with tRNAs in hybrid states. *Proc. Natl Acad. Sci. USA* **105**, 16924–16927 (2008).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.0809587105) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2008PNAS..10516924J) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlKqt7%2FM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18971332) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2579354) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structure%20of%20ratcheted%20ribosomes%20with%20tRNAs%20in%20hybrid%20states&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.0809587105&volume=105&pages=16924-16927&publication_year=2008&author=Julian%2CP)\\n37. Agirrezabala, X. et al. Visualization of the hybrid state of tRNA binding promoted by spontaneous ratcheting of the ribosome. *Mol. Cell* **32**, 190–197 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2008.10.001) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlehsb3L) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18951087) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614368) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualization%20of%20the%20hybrid%20state%20of%20tRNA%20binding%20promoted%20by%20spontaneous%20ratcheting%20of%20the%20ribosome&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2008.10.001&volume=32&pages=190-197&publication_year=2008&author=Agirrezabala%2CX)\\n38. Rodnina, M. V., Peske, F., Peng, B. Z., Belardinelli, R. & Wintermeyer, W. Converting GTP hydrolysis into motion: versatile translational elongation factor G. *Biol. Chem.* **401**, 131–142 (2019).\\n\\n[Article](https://doi.org/10.1515%2Fhsz-2019-0313) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31600135) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Converting%20GTP%20hydrolysis%20into%20motion%3A%20versatile%20translational%20elongation%20factor%20G&journal=Biol.%20Chem.&doi=10.1515%2Fhsz-2019-0313&volume=401&pages=131-142&publication_year=2019&author=Rodnina%2CMV&author=Peske%2CF&author=Peng%2CBZ&author=Belardinelli%2CR&author=Wintermeyer%2CW)\\n39. Lin, J., Zhou, D., Steitz, T. A., Polikanov, Y. S. & Gagnon, M. G. Ribosome-targeting antibiotics: modes of action, mechanisms of resistance, and implications for drug design. *Annu. Rev. Biochem.* **87**, 451–478 (2018).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-biochem-062917-011942) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXlslSrurw%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29570352) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9176271) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome-targeting%20antibiotics%3A%20modes%20of%20action%2C%20mechanisms%20of%20resistance%2C%20and%20implications%20for%20drug%20design&journal=Annu.%20Rev.%20Biochem.&doi=10.1146%2Fannurev-biochem-062917-011942&volume=87&pages=451-478&publication_year=2018&author=Lin%2CJ&author=Zhou%2CD&author=Steitz%2CTA&author=Polikanov%2CYS&author=Gagnon%2CMG)\\n40. Bulkley, D., Innis, C. A., Blaha, G. & Steitz, T. A. Revisiting the structures of several antibiotics bound to the bacterial ribosome. *Proc. Natl Acad. Sci. USA* **107**, 17158–17163 (2010).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1008685107) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010PNAS..10717158B) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlSjurrM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20876130) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2951403) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Revisiting%20the%20structures%20of%20several%20antibiotics%20bound%20to%20the%20bacterial%20ribosome&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1008685107&volume=107&pages=17158-17163&publication_year=2010&author=Bulkley%2CD&author=Innis%2CCA&author=Blaha%2CG&author=Steitz%2CTA)\\n41. Svetlov, M. S. et al. High-resolution crystal structures of ribosome-bound chloramphenicol and erythromycin provide the ultimate basis for their competition. *RNA* **25**, 600–606 (2019).\\n\\n[Article](https://doi.org/10.1261%2Frna.069260.118) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1Ons77I) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30733327) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6467010) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-resolution%20crystal%20structures%20of%20ribosome-bound%20chloramphenicol%20and%20erythromycin%20provide%20the%20ultimate%20basis%20for%20their%20competition&journal=RNA&doi=10.1261%2Frna.069260.118&volume=25&pages=600-606&publication_year=2019&author=Svetlov%2CMS)\\n42. Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. *ACS Chem. Biol.* **2**, 545–552 (2007).\\n\\n[Article](https://doi.org/10.1021%2Fcb700100n) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2sXpvVOhtrc%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17696316) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4624401) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20steric%20block%20in%20translation%20caused%20by%20the%20antibiotic%20spectinomycin&journal=ACS%20Chem.%20Biol.&doi=10.1021%2Fcb700100n&volume=2&pages=545-552&publication_year=2007&author=Borovinskaya%2CMA&author=Shoji%2CS&author=Holton%2CJM&author=Fredrick%2CK&author=Cate%2CJHD)\\n43. Polikanov, Y. S. et al. Distinct tRNA accommodation intermediates observed on the ribosome with the antibiotics hygromycin A and A201A. *Mol. Cell* **58**, 832–844 (2015).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2015.04.014) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2MXpt1Skurg%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26028538) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4458074) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20tRNA%20accommodation%20intermediates%20observed%20on%20the%20ribosome%20with%20the%20antibiotics%20hygromycin%20A%20and%20A201A&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2015.04.014&volume=58&pages=832-844&publication_year=2015&author=Polikanov%2CYS)\\n44. Morse, J. C. et al. Elongation factor-Tu can repetitively engage aminoacyl-tRNA within the ribosome during the proofreading stage of tRNA selection. *Proc. Natl Acad. Sci. USA* **117**, 3610–3620 (2020).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1904469117) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020PNAS..117.3610M) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXjsFyhu7Y%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32024753) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7035488) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Elongation%20factor-Tu%20can%20repetitively%20engage%20aminoacyl-tRNA%20within%20the%20ribosome%20during%20the%20proofreading%20stage%20of%20tRNA%20selection&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1904469117&volume=117&pages=3610-3620&publication_year=2020&author=Morse%2CJC)\\n45. Choi, J. et al. Dynamics of the context-specific translation arrest by chloramphenicol and linezolid. *Nat. Chem. Biol.* **16**, 310–317 (2020).\\n\\n[Article](https://doi.org/10.1038%2Fs41589-019-0423-2) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVSktb%2FM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31844301) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dynamics%20of%20the%20context-specific%20translation%20arrest%20by%20chloramphenicol%20and%20linezolid&journal=Nat.%20Chem.%20Biol.&doi=10.1038%2Fs41589-019-0423-2&volume=16&pages=310-317&publication_year=2020&author=Choi%2CJ)\\n46. Marks, J. et al. Context-specific inhibition of translation by ribosomal antibiotics targeting the peptidyl transferase center. *Proc. Natl Acad. Sci. USA* **113**, 12150–12155 (2016).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1613055113) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016PNAS..11312150M) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1elur3N) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27791002) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5086994) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Context-specific%20inhibition%20of%20translation%20by%20ribosomal%20antibiotics%20targeting%20the%20peptidyl%20transferase%20center&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1613055113&volume=113&pages=12150-12155&publication_year=2016&author=Marks%2CJ)\\n47. Vazquez-Laslop, N. & Mankin, A. S. Context-specific action of ribosomal antibiotics. *Annu. Rev. Microbiol.* **72**, 185–207 (2018).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-micro-090817-062329) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFCjs77O) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29906204) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8742604) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Context-specific%20action%20of%20ribosomal%20antibiotics&journal=Annu.%20Rev.%20Microbiol.&doi=10.1146%2Fannurev-micro-090817-062329&volume=72&pages=185-207&publication_year=2018&author=Vazquez-Laslop%2CN&author=Mankin%2CAS)\\n48. Aguirre Rivera, J. et al. Real-time measurements of aminoglycoside effects on protein synthesis in live cells. *Proc. Natl Acad. Sci. USA* **118**, e2013315118 (2021).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.2013315118) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33619089) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7936356) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20measurements%20of%20aminoglycoside%20effects%20on%20protein%20synthesis%20in%20live%20cells&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.2013315118&volume=118&publication_year=2021&author=Aguirre%20Rivera%2CJ)\\n49. Baquero, F. & Levin, B. R. Proximate and ultimate causes of the bactericidal action of antibiotics. *Nat. Rev. Microbiol.* **19**, 123–132 (2021).\\n50. Myasnikov, A. G. et al. The molecular structure of the left-handed supra-molecular helix of eukaryotic polyribosomes. *Nat. Commun.* **5**, 5294 (2014).\\n\\n[Article](https://doi.org/10.1038%2Fncomms6294) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2014NatCo...5.5294M) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFantrfM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25376914) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20molecular%20structure%20of%20the%20left-handed%20supra-molecular%20helix%20of%20eukaryotic%20polyribosomes&journal=Nat.%20Commun.&doi=10.1038%2Fncomms6294&volume=5&publication_year=2014&author=Myasnikov%2CAG)\\n51. Dunkle, J. A. & Cate, J. H. D. in *Ribosomes* (eds Rodnina, M. V., Wintermeyer, W. & Green, R.) Ch. 6 (2011).\\n52. Herr, A. J., Nelson, C. C., Wills, N. M., Gesteland, R. F. & Atkins, J. F. Analysis of the roles of tRNA structure, ribosomal protein L9, and the bacteriophage T4 gene 60 bypassing signals during ribosome slippage on mRNA. *J. Mol. Biol.* **309**, 1029–1048 (2001).\\n\\n[Article](https://doi.org/10.1006%2Fjmbi.2001.4717) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD3MXkslGgurY%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11399077) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20the%20roles%20of%20tRNA%20structure%2C%20ribosomal%20protein%20L9%2C%20and%20the%20bacteriophage%20T4%20gene%2060%20bypassing%20signals%20during%20ribosome%20slippage%20on%20mRNA&journal=J.%20Mol.%20Biol.&doi=10.1006%2Fjmbi.2001.4717&volume=309&pages=1029-1048&publication_year=2001&author=Herr%2CAJ&author=Nelson%2CCC&author=Wills%2CNM&author=Gesteland%2CRF&author=Atkins%2CJF)\\n53. Smith, A. M., Costello, M. S., Kettring, A. H., Wingo, R. J. & Moore, S. D. Ribosome collisions alter frameshifting at translational reprogramming motifs in bacterial mRNAs. *Proc. Natl Acad. Sci. USA* **116**, 21769–21779 (2019).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1910613116) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019PNAS..11621769S) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXitValtbjM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31591196) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6815119) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome%20collisions%20alter%20frameshifting%20at%20translational%20reprogramming%20motifs%20in%20bacterial%20mRNAs&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1910613116&volume=116&pages=21769-21779&publication_year=2019&author=Smith%2CAM&author=Costello%2CMS&author=Kettring%2CAH&author=Wingo%2CRJ&author=Moore%2CSD)\\n54. Tegunov, D. & Cramer, P. Real-time cryo-electron microscopy data preprocessing with Warp. *Nat. Methods* **16**, 1146–1152 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0580-y) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFejs77E) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31591575) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6858868) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20cryo-electron%20microscopy%20data%20preprocessing%20with%20Warp&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0580-y&volume=16&pages=1146-1152&publication_year=2019&author=Tegunov%2CD&author=Cramer%2CP)\\n55. Hagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. *J. Struct. Biol.* **197**, 191–198 (2017).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2016.06.007) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27313000) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5287356) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Implementation%20of%20a%20cryo-electron%20tomography%20tilt-scheme%20optimized%20for%20high%20resolution%20subtomogram%20averaging&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2016.06.007&volume=197&pages=191-198&publication_year=2017&author=Hagen%2CWJH&author=Wan%2CW&author=Briggs%2CJAG)\\n56. Schorb, M., Haberbosch, I., Hagen, W. J. H., Schwab, Y. & Mastronarde, D. N. Software tools for automated transmission electron microscopy. *Nat. Methods* **16**, 471–477 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0396-9) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVSqs7zL) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31086343) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7000238) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Software%20tools%20for%20automated%20transmission%20electron%20microscopy&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0396-9&volume=16&pages=471-477&publication_year=2019&author=Schorb%2CM&author=Haberbosch%2CI&author=Hagen%2CWJH&author=Schwab%2CY&author=Mastronarde%2CDN)\\n57. Hrabe, T. et al. PyTom: a python-based toolbox for localization of macromolecules in cryo-electron tomograms and subtomogram analysis. *J. Struct. Biol.* **178**, 177–188 (2012).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2011.12.003) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC38Xnt1ehtL4%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22193517) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PyTom%3A%20a%20python-based%20toolbox%20for%20localization%20of%20macromolecules%20in%20cryo-electron%20tomograms%20and%20subtomogram%20analysis&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2011.12.003&volume=178&pages=177-188&publication_year=2012&author=Hrabe%2CT)\\n58. Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. *Nat. Protoc.* **11**, 2054–2065 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnprot.2016.124) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XhsF2mtLrL) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27685097) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5215819) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Resolving%20macromolecular%20structures%20from%20electron%20cryo-tomography%20data%20using%20subtomogram%20averaging%20in%20RELION&journal=Nat.%20Protoc.&doi=10.1038%2Fnprot.2016.124&volume=11&pages=2054-2065&publication_year=2016&author=Bharat%2CTA&author=Scheres%2CSH)\\n59. Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. *eLife* **7**, e42166 (2018).\\n\\n[Article](https://doi.org/10.7554%2FeLife.42166) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30412051) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6250425) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=New%20tools%20for%20automated%20high-resolution%20cryo-EM%20structure%20determination%20in%20RELION-3&journal=eLife&doi=10.7554%2FeLife.42166&volume=7&publication_year=2018&author=Zivanov%2CJ)\\n60. Waterhouse, A. et al. SWISS-MODEL: homology modelling of protein structures and complexes. *Nucleic Acids Res.* **46**, W296–W303 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky427) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyru70%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29788355) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030848) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SWISS-MODEL%3A%20homology%20modelling%20of%20protein%20structures%20and%20complexes&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky427&volume=46&pages=W296-W303&publication_year=2018&author=Waterhouse%2CA)\\n61. Pruesse, E., Peplies, J. & Glockner, F. O. SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes. *Bioinformatics* **28**, 1823–1829 (2012).\\n\\n[Article](https://doi.org/10.1093%2Fbioinformatics%2Fbts252) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVSgsbfO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22556368) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3389763) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SINA%3A%20accurate%20high-throughput%20multiple%20sequence%20alignment%20of%20ribosomal%20RNA%20genes&journal=Bioinformatics&doi=10.1093%2Fbioinformatics%2Fbts252&volume=28&pages=1823-1829&publication_year=2012&author=Pruesse%2CE&author=Peplies%2CJ&author=Glockner%2CFO)\\n62. Rother, M., Rother, K., Puton, T. & Bujnicki, J. M. ModeRNA: a tool for comparative modeling of RNA 3D structure. *Nucleic Acids Res.* **39**, 4007–4022 (2011).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkq1320) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3MXmvValsL4%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21300639) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3105415) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ModeRNA%3A%20a%20tool%20for%20comparative%20modeling%20of%20RNA%203D%20structure&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkq1320&volume=39&pages=4007-4022&publication_year=2011&author=Rother%2CM&author=Rother%2CK&author=Puton%2CT&author=Bujnicki%2CJM)\\n63. Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. *J. Comput. Chem.* **25**, 1605–1612 (2004).\\n\\n[Article](https://doi.org/10.1002%2Fjcc.20084) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2cXmvVOhsbs%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15264254) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCSF%20Chimera%E2%80%94a%20visualization%20system%20for%20exploratory%20research%20and%20analysis&journal=J.%20Comput.%20Chem.&doi=10.1002%2Fjcc.20084&volume=25&pages=1605-1612&publication_year=2004&author=Pettersen%2CEF)\\n64. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. *Acta Crystallogr D* **74**, 531–544 (2018).\\n\\n[Article](https://doi.org/10.1107%2FS2059798318006551) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2ltbzP) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-space%20refinement%20in%20PHENIX%20for%20cryo-EM%20and%20crystallography&journal=Acta%20Crystallogr%20D&doi=10.1107%2FS2059798318006551&volume=74&pages=531-544&publication_year=2018&author=Afonine%2CPV)\\n65. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. *Acta Crystallogr D* **66**, 486–501 (2010).\\n\\n[Article](https://doi.org/10.1107%2FS0907444910007493) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXksFKisb8%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20383002) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2852313) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Features%20and%20development%20of%20Coot&journal=Acta%20Crystallogr%20D&doi=10.1107%2FS0907444910007493&volume=66&pages=486-501&publication_year=2010&author=Emsley%2CP&author=Lohkamp%2CB&author=Scott%2CWG&author=Cowtan%2CK)\\n66. Williams, C. J. et al. MolProbity: more and better reference data for improved all-atom structure validation. *Protein Sci.* **27**, 293–315 (2018).\\n\\n[Article](https://doi.org/10.1002%2Fpro.3330) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvVKgsrnO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29067766) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MolProbity%3A%20more%20and%20better%20reference%20data%20for%20improved%20all-atom%20structure%20validation&journal=Protein%20Sci.&doi=10.1002%2Fpro.3330&volume=27&pages=293-315&publication_year=2018&author=Williams%2CCJ)\\n67. Chen, S. et al. High-resolution noise substitution to measure overfitting and validate resolution in 3D structure determination by single particle electron cryomicroscopy. *Ultramicroscopy* **135**, 24–35 (2013).\\n\\n[Article](https://doi.org/10.1016%2Fj.ultramic.2013.06.004) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslWhtLzO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23872039) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3834153) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-resolution%20noise%20substitution%20to%20measure%20overfitting%20and%20validate%20resolution%20in%203D%20structure%20determination%20by%20single%20particle%20electron%20cryomicroscopy&journal=Ultramicroscopy&doi=10.1016%2Fj.ultramic.2013.06.004&volume=135&pages=24-35&publication_year=2013&author=Chen%2CS)\\n68. Tatusov, R. L., Galperin, M. Y., Natale, D. A. & Koonin, E. V. The COG database: a tool for genome-scale analysis of protein functions and evolution. *Nucleic Acids Res.* **28**, 33–36 (2000).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2F28.1.33) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD3cXhvVGqu7w%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10592175) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC102395) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20COG%20database%3A%20a%20tool%20for%20genome-scale%20analysis%20of%20protein%20functions%20and%20evolution&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2F28.1.33&volume=28&pages=33-36&publication_year=2000&author=Tatusov%2CRL&author=Galperin%2CMY&author=Natale%2CDA&author=Koonin%2CEV)\\n69. Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. *Nucleic Acids Res.* **47**, D309–D314 (2019).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky1085) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Gqtr%2FL) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30418610) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=eggNOG%205.0%3A%20a%20hierarchical%2C%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky1085&volume=47&pages=D309-D314&publication_year=2019&author=Huerta-Cepas%2CJ)\\n70. Katoh, K., Rozewicki, J. & Yamada, K. D. MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization. *Brief. Bioinform.* **20**, 1160–1166 (2019).\\n\\n[Article](https://doi.org/10.1093%2Fbib%2Fbbx108) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1Kqtb3K) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28968734) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MAFFT%20online%20service%3A%20multiple%20sequence%20alignment%2C%20interactive%20sequence%20choice%20and%20visualization&journal=Brief.%20Bioinform.&doi=10.1093%2Fbib%2Fbbx108&volume=20&pages=1160-1166&publication_year=2019&author=Katoh%2CK&author=Rozewicki%2CJ&author=Yamada%2CKD)\\n71. Letunic, I. & Bork, P. Interactive Tree Of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. *Nucleic Acids Res.* **49**, W293–W296 (2021).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkab301) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvV2isLvM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33885785) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8265157) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interactive%20Tree%20Of%20Life%20%28iTOL%29%20v5%3A%20an%20online%20tool%20for%20phylogenetic%20tree%20display%20and%20annotation&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkab301&volume=49&pages=W293-W296&publication_year=2021&author=Letunic%2CI&author=Bork%2CP)\\n72. Huerta-Cepas, J. et al. Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper. *Mol. Biol. Evol.* **34**, 2115–2122 (2017).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsx148) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFaru7vF) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28460117) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850834) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20genome-wide%20functional%20annotation%20through%20orthology%20assignment%20by%20eggNOG-Mapper&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsx148&volume=34&pages=2115-2122&publication_year=2017&author=Huerta-Cepas%2CJ)\\n73. Huerta-Cepas, J., Serra, F. & Bork, P. ETE 3: reconstruction, analysis, and visualization of phylogenomic data. *Mol. Biol. Evol.* **33**, 1635–1638 (2016).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsw046) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFeisrrN) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26921390) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4868116) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ETE%203%3A%20reconstruction%2C%20analysis%2C%20and%20visualization%20of%20phylogenomic%20data&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsw046&volume=33&pages=1635-1638&publication_year=2016&author=Huerta-Cepas%2CJ&author=Serra%2CF&author=Bork%2CP)\\n74. Meszaros, B., Erdos, G. & Dosztanyi, Z. IUPred2A: context-dependent prediction of protein disorder as a function of redox state and protein binding. *Nucleic Acids Res.* **46**, W329–W337 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky384) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyrt7o%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29860432) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030935) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=IUPred2A%3A%20context-dependent%20prediction%20of%20protein%20disorder%20as%20a%20function%20of%20redox%20state%20and%20protein%20binding&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky384&volume=46&pages=W329-W337&publication_year=2018&author=Meszaros%2CB&author=Erdos%2CG&author=Dosztanyi%2CZ)\\n75. Cole, C., Barber, J. D. & Barton, G. J. The Jpred 3 secondary structure prediction server. *Nucleic Acids Res.* **36**, W197–W201 (2008).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkn238) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXptlKkt7w%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18463136) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2447793) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Jpred%203%20secondary%20structure%20prediction%20server&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkn238&volume=36&pages=W197-W201&publication_year=2008&author=Cole%2CC&author=Barber%2CJD&author=Barton%2CGJ)\\n76. Scheres, S. H. Processing of structurally heterogeneous cryo-EM data in RELION. *Methods Enzymol.* **579**, 125–157 (2016).\\n\\n[Article](https://doi.org/10.1016%2Fbs.mie.2016.04.012) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXmtlKnurk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27572726) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Processing%20of%20structurally%20heterogeneous%20cryo-EM%20data%20in%20RELION&journal=Methods%20Enzymol.&doi=10.1016%2Fbs.mie.2016.04.012&volume=579&pages=125-157&publication_year=2016&author=Scheres%2CSH)\\n77. Kidmose, R. T. et al. Namdinator—automatic molecular dynamics flexible fitting of structural models into cryo-EM and crystallography experimental maps. *IUCrJ* **6**, 526–531 (2019).\\n\\n[Article](https://doi.org/10.1107%2FS2052252519007619) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlWit7%2FO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31316797) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6608625) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Namdinator%E2%80%94automatic%20molecular%20dynamics%20flexible%20fitting%20of%20structural%20models%20into%20cryo-EM%20and%20crystallography%20experimental%20maps&journal=IUCrJ&doi=10.1107%2FS2052252519007619&volume=6&pages=526-531&publication_year=2019&author=Kidmose%2CRT)\\n78. Trabuco, L. G., Villa, E., Mitra, K., Frank, J. & Schulten, K. Flexible fitting of atomic structures into electron microscopy maps using molecular dynamics. *Structure* **16**, 673–683 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.str.2008.03.005) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXlslGrtro%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18462672) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2430731) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Flexible%20fitting%20of%20atomic%20structures%20into%20electron%20microscopy%20maps%20using%20molecular%20dynamics&journal=Structure&doi=10.1016%2Fj.str.2008.03.005&volume=16&pages=673-683&publication_year=2008&author=Trabuco%2CLG&author=Villa%2CE&author=Mitra%2CK&author=Frank%2CJ&author=Schulten%2CK)\\n79. Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. *J. Chem. Phys.* **153**, 044130 (2020).\\n\\n[Article](https://doi.org/10.1063%2F5.0014475) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFajsL3J) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32752662) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7395834) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scalable%20molecular%20dynamics%20on%20CPU%20and%20GPU%20architectures%20with%20NAMD&journal=J.%20Chem.%20Phys.&doi=10.1063%2F5.0014475&volume=153&publication_year=2020&author=Phillips%2CJC)\\n80. Nickell, S. et al. TOM software toolbox: acquisition and analysis for electron tomography. *J. Struct. Biol.* **149**, 227–234 (2005).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2004.10.006) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15721576) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TOM%20software%20toolbox%3A%20acquisition%20and%20analysis%20for%20electron%20tomography&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2004.10.006&volume=149&pages=227-234&publication_year=2005&author=Nickell%2CS)\\n81. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. *Protein Sci.* **30**, 70–82 (2021).\\n\\n[Article](https://doi.org/10.1002%2Fpro.3943) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFamtrzJ) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32881101) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCSF%20ChimeraX%3A%20structure%20visualization%20for%20researchers%2C%20educators%2C%20and%20developers&journal=Protein%20Sci.&doi=10.1002%2Fpro.3943&volume=30&pages=70-82&publication_year=2021&author=Pettersen%2CEF)\\n\\n[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41586-022-05255-2?format=refman&flavour=references)\\n\\nAcknowledgements\\n----------------\\n\\nWe thank I. Zagoriy, L. Adam, F. Weis, W. Hagen, T. Hoffmann and EMBL IT for technical support; L. Serrano for providing the *M.* *pneumoniae* transposon library and proteomics data; M. Kuhn, F. Marotta and J. C. Somody for discussions on bioinformatic and statistical data analysis; and C. Spahn, A. Jakobi and S. Pfeffer for discussions and critical reading of the manuscript. J.R. acknowledges the Deutsche Forschungsgemeinschaft (project no. 426290502) and the Wellcome Trust Senior Research Fellowship (103139). J.M. acknowledges the EMBL and the European Research Council starting grant (3DCellPhase-, grant no. 760067).\\n\\nAuthor information\\n------------------\\n\\n### Authors and Affiliations\\n\\n1. Structural and Computational Biology Unit, European Molecular Biology Laboratory (EMBL), Heidelberg, Germany\\n\\nLiang Xue, Maria Zimmermann-Kogadeeva, Peer Bork & Julia Mahamid\\n2. Collaboration for joint PhD degree between EMBL and Heidelberg University, Faculty of Biosciences, Heidelberg, Germany\\n\\nLiang Xue\\n3. Chair of Bioanalytics, Technische Universität Berlin, Berlin, Germany\\n\\nSwantje Lenz & Juri Rappsilber\\n4. Department of Molecular Biology, Max-Planck-Institute for Biophysical Chemistry, Göttingen, Germany\\n\\nDimitry Tegunov & Patrick Cramer\\n5. Yonsei Frontier Lab, Yonsei University, Seoul, South Korea\\n\\nPeer Bork\\n6. Department of Bioinformatics, Biocenter, University of Würzburg, Würzburg, Germany\\n\\nPeer Bork\\n7. Wellcome Centre for Cell Biology, University of Edinburgh, Edinburgh, UK\\n\\nJuri Rappsilber\\n\\nAuthors1. Liang Xue[View author publications](/search?author=Liang%20Xue)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Liang%20Xue) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Liang%20Xue%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n2. Swantje Lenz[View author publications](/search?author=Swantje%20Lenz)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Swantje%20Lenz) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Swantje%20Lenz%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n3. Maria Zimmermann-Kogadeeva[View author publications](/search?author=Maria%20Zimmermann-Kogadeeva)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Maria%20Zimmermann-Kogadeeva) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Maria%20Zimmermann-Kogadeeva%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n4. Dimitry Tegunov[View author publications](/search?author=Dimitry%20Tegunov)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Dimitry%20Tegunov) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Dimitry%20Tegunov%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n5. Patrick Cramer[View author publications](/search?author=Patrick%20Cramer)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Patrick%20Cramer) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Patrick%20Cramer%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n6. Peer Bork[View author publications](/search?author=Peer%20Bork)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Peer%20Bork) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Peer%20Bork%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n7. Juri Rappsilber[View author publications](/search?author=Juri%20Rappsilber)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Juri%20Rappsilber) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Juri%20Rappsilber%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n8. Julia Mahamid[View author publications](/search?author=Julia%20Mahamid)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Julia%20Mahamid) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Julia%20Mahamid%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n### Contributions\\n\\nL.X. and J.M. designed the study. L.X. collected and processed the cryo-ET data and performed structural analysis. L.X. and S.L. built the atomic models with support from J.R. M.Z.-K. performed the bioinformatic and statistical analysis with support from P.B. D.T. and P.C. provided software support and advice on data interpretation. L.X. and J.M. wrote the manuscript with input from all authors.\\n\\n### Corresponding author\\n\\nCorrespondence to\\n [Julia Mahamid](mailto:julia.mahamid@embl.de).\\n\\nEthics declarations\\n-------------------\\n\\n\\n### Competing interests\\n\\n\\nThe authors declare no competing interests.\\n\\n\\nPeer review\\n-----------\\n\\n\\n### Peer review information\\n\\n\\n*Nature* thanks Elizabeth Villa and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.\\n\\n\\nAdditional information\\n----------------------\\n\\n**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\\n\\nExtended data figures and tables\\n--------------------------------\\n\\n### [Extended Data Fig. 1 *M. pneumoniae* in-cell ribosome maps.](/articles/s41586-022-05255-2/figures/5)\\n\\n**a**, 70S ribosome map determined from 77,539 sub-tomograms from 356 untreated *M. pneumoniae* cells. **b**, Map coloured by local resolutions. 50S has the highest local resolutions as it dominates the overall alignment. The relatively lower local resolutions of the 30S indicate their high flexibility during active translation. **c**, Fourier shell correlation (FSC) curves for global 70S and focused 30S refinement, and the reported resolution value at FSC = 0.143. The Nyquist limit for the data is 3.4 Å. **d**, 70S ribosome map determined from 18,987 sub-tomograms from 65 Cm-treated cells. **e**, Map coloured by local resolutions. **f**, FSC curves of the Cm-treated 70S ribosome, and of focused refinements on 30S and 50S respectively. **g**-**h**, Focused refined 30S and 50S maps from the Cm-treated dataset. **i**-**j**, The corresponding local-resolution maps. Atomic models for 30S and 50S were first built based on maps in **g** and **h**. **k**, Model-to-map FSC curves for 30S and 50S, respectively.\\n\\n### [Extended Data Fig. 2 Structural features of the *M. pneumoniae* ribosome.](/articles/s41586-022-05255-2/figures/6)\\n\\n**a**,**b**, Examples of regions of the atomic model fitted into the density map for rRNAs (**a**) and proteins (**b**). **c**, Densities corresponding to ions are also observed. **d**, Extensions of ribosomal proteins S6, L22 and L29 form secondary structures (between arrowheads) that were clearly resolved in the map. **e**, The 70S model showing ribosomal proteins with sequence extensions (dark green). The extensions built in the model are highlighted in magenta. In addition to the helix-forming sequences of S6, L22 and L29, some loops can be traced, especially the long C-terminal loop of S6. **f**, Whole-cell cross-linking mass spectrometry data confirms the model built for the long loop extension of S6. **g**, The cross-linking network of S6 identified in the previous work[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"), with the sequence range (1–167) resolved in the map coloured in green.\\n\\n### [Extended Data Fig. 3 Ribosomal protein extensions.](/articles/s41586-022-05255-2/figures/7)\\n\\n**a**, Eleven ribosomal proteins in *M. pneumoniae* have sequence extensions compared to *E. coli*. Along the sequences, cross-links identified by in-cell cross-linking mass spectrometry, secondary structure prediction, disorder prediction, and sequence essentiality prediction based on a transposon mutation library are displayed (quantified in Supplementary Table [2](/articles/s41586-022-05255-2#MOESM1)). **b**, Extensions in the eleven ribosomal proteins are found throughout bacteria, but are not specific to any sub-groups.\\n\\n### [Extended Data Fig. 4 Classification and refinement of ribosomes in native untreated cells.](/articles/s41586-022-05255-2/figures/8)\\n\\n**a**, A diagram of the image-processing workflow. The classification process can be grouped into three tiers: global classification, focused classification on the tRNA path region (mask I), and focused classification on elongation factor and A/T tRNA binding region (mask II). For each class, a unique number identifier and class name are assigned for tracking. The particle number and the global map resolution at FSC = 0.143 are provided for each class. **b**-**d**, FSC curves for all refined maps.\\n\\n### [Extended Data Fig. 5 Validation of ribosome detection and classification.](/articles/s41586-022-05255-2/figures/9)\\n\\n**a**, Mask I for focused classification on tRNA path region. **b**, A representative RELION classification job with mask I. Each line indicates the change in particle numbers in one class over 25 iterations. Classes that show the same structure were grouped according to the tRNA occupancy ('a, P/E', 'P, E', 'P', 'A, P'). 'a, P/E' contains heterogeneous density around the A site. **c**, Results of the classification job as shown in **b**, and of three additional parallel jobs. **d**, Results of following classification jobs that further classify the 'a, P/E' class into 'A\\\\*, P/E' and 'A/P, P/E'. **e**, Mask II for focused classification on elongation factor (EF) and A/T tRNA sites. **f**, Changes in particle numbers over iterations in a representative classification job with the mask II. **g**, Results of parallel focused classification jobs with mask II. **h**, Distribution of the ribosome classes against template matching cross-correlation scores used for ribosome localization. For each of the 356 tomograms of untreated cells, the 400 highest scoring hits were extracted and ranked. Obvious false positives were manually excluded first. Additional false positives were identified during RELION classification. 70S classes that are structurally similar were grouped in the plot for better visualization. **i**, Same as **h**, but only showing the 70S classes in the elongation phase. The proportions of different 70S classes remain stable across the top 400 hits, demonstrating that the classification results are not biased by the ribosome picking.\\n\\n### [Extended Data Fig. 6 Local-resolution maps for ribosome classes in untreated cells.](/articles/s41586-022-05255-2/figures/10)\\n\\nMaps of the 15 classes determined in the dataset of untreated cells (Extended Data Fig. [4](/articles/s41586-022-05255-2#Fig8)), coloured by local resolution calculated in RELION.\\n\\n### [Extended Data Fig. 7 Model building for ribosome classes in untreated cells.](/articles/s41586-022-05255-2/figures/11)\\n\\n**a**-**j**, Models of the ten ribosome classes in the elongation phase constructed by flexible fitting. **k**, The model for the ribosome with hybrid P/E-site tRNA. **l**, Free 50S in complex with ribosome recycling factor (RRF).\\n\\n### [Extended Data Fig. 8 Early-to-late translocation intermediates during elongation.](/articles/s41586-022-05255-2/figures/12)\\n\\n**a**, Early translocation intermediates prior to EF-G binding. Only tRNAs in the aligned class models are displayed. **b**, Early-to-late translocation intermediates in the presence of EF-G. Continuous structural changes are observed from class 6e to 7 to 8, including a roughly 20 Å movement of EF-G’s domain IV toward the A site (black arrow), an overall rotation of the entire EF-G (orange arrows) and inter-domain conformational changes. **c**-**e**, Densities and models for EF-G and tRNAs in classes 6e, 7 and 8. In class 6e, EF-G’s domain IV is less resolved in the map (red circle). It may contain a mixture of intermediate states with domain IV moving toward the A site. The fitted model represents the average position. Further classification could not address the high flexibility. **f**, From class 6e to 7, EF-G undergoes a small overall rotation (orange arrows) and an inter-domain rearrangement. Inset (rotated view, aligned on EF-G domains I-III) shows rotation of domain IV relative to other domains. Movement of EF-G’s domain IV towards the A site results from both overall rotation on the ribosome and inter-domain rearrangement. **g**, From class 7 to 8, EF-G rotates as one body without significant inter-domain rearrangement (inset, rotated view, aligned on EF-G domains I-III). **h**, Class 6e shows structural similarity to a reported early translocation state 'H1-EF-G–GDP-pi' (PDB 7PJV) determined by Petrychenko *et al*. 2021[15](/articles/s41586-022-05255-2#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\"). **i**, Class 7 resembles 'PRE–EF-G–GDP–Pi' (PDB 7SSL) reported by Carbone *et al*. 2021[16](/articles/s41586-022-05255-2#ref-CR16 \\\"Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. Nat. Commun. 12, 7236 (2021).\\\"), and 'INT1' (PDB 7N2V, not shown) by Rundlet *et al*. 2021[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"). **j**, Class 8 resembles the late translocation intermediates reported by Rundlet *et al*. 2021[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\") ('INT2', PDB 7N2C) and Petrychenko *et al*. 2021[15](/articles/s41586-022-05255-2#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\") ('CHI1-EF-G-GDP', PDB 7PJY). **k**-**m**, Minor inter-domain conformational difference found between EF-Gs in our models and those in the reported in vitro structures.\\n\\n### [Extended Data Fig. 9 Classification, refinement and modelling of ribosomes in Cm-treated cells.](/articles/s41586-022-05255-2/figures/13)\\n\\n**a**, Image-processing workflow for the Cm-treated dataset. The sub-tomogram classification and refinement are similar to those developed for the untreated dataset (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8)). Heterogeneity may exist in the minor classes (2a, 3, 5), but the low particle numbers hindered further classification. **b**, FSC curves for all classes calculated following RELION refinements. **c**, Local-resolution maps. **d**, Density of the Cm molecule is resolved in the major 'A, P' class, but not in the other three minor classes owing to relatively low resolutions of these maps. **e**, Models built for the four classes, fitted into their corresponding densities. **f**, Elongation factors and tRNAs in the four classes. **g**, In the predominant 'A, P' class, mRNA, A- and P-site tRNA, the nascent peptide chain, and the Cm drug are well-resolved. The nascent chain has strong continuous density linked to the P-site tRNA, resulting from the inhibition of peptidyl transfer by the Cm molecule.\\n\\n### [Extended Data Fig. 10 Classification, refinement and modelling of ribosomes in Spc-treated cells.](/articles/s41586-022-05255-2/figures/14)\\n\\n**a**, Image-processing workflow for the Spc-treated dataset. The sub-tomogram classification and refinement are similar to those developed for the untreated dataset (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8)), except for exclusion of the manual inspection step after template matching. **b**, FSC curves for all classes calculated following refinements. **c**, Local-resolution maps for Spc-treated 70S classes. **d**, Density corresponding to the Spc molecule (magenta) is clearly resolved in the major 'EF-G, A/PSpc, P/E' class. Densities from the untreated and PUM-treated data, where Spc is not present, are shown for comparison. **e**, Models built for the five Spc-treated 70S classes. **f**, Elongation factors and tRNAs in the five classes. **g**, The model of 'EF-G, A/PSpc, P/E' shows Spc binds to the 30S neck region, confirming its role in inhibiting 30S head dynamics and mRNA translocation.\\n\\n### [Extended Data Fig. 11 Classification, refinement and modelling of ribosomes in PUM-treated cells.](/articles/s41586-022-05255-2/figures/15)\\n\\n**a**, Image-processing workflow for the PUM-treated dataset. The sub-tomogram classification and refinement are similar to those developed for the untreated dataset (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8)). **b**, Distribution of translation states for ribosomes collided with a PUM-stalled RNA polymerase (purple; stalled expressome) and for the remaining ribosomes (green) in the PUM-treated cells. **c**, FSC curves for all classes calculated after refinements. **d**-**f**, Local-resolution maps. **g**, Models built for PUM-treated 70S classes. **h**, Elongation factors and tRNAs in the six 70S ribosome classes.\\n\\n### [Extended Data Fig. 12 Spatial analysis of ribosomes and polysomes in native untreated cells.](/articles/s41586-022-05255-2/figures/16)\\n\\n**a**, 70S ribosomes (grey) and detected polysomes (light blue) in a representative tomogram. **b**, Distribution of neighbouring ribosomes within a 50 nm centre-to-centre distance. Rotation normalized. **c**, Illustration of the polysome detection approach based on the distance from the mRNA exit site of one ribosome to the mRNA entry site of the next. **d**, Histogram of distances from the mRNA exit site of one ribosome to the mRNA entry sites of all neighbouring ribosomes. **e**, Percentages of ribosomes detected as polysomes using different distance thresholds (d). Mean and s.d. are across 356 tomograms (n = 356 cells). A 7 nm threshold was selected. **f**, In polysomes, the following ribosome (i) adopts various orientations relative to the preceding ribosome (i+1), mainly with regard to rotation around a plane perpendicular to the preceding ribosome’s mRNA exit site. **g**, Rotations of the following ribosome relative to the preceding ribosome in polysomes. The two clusters correspond to the previously defined 't-t' and 't-b' configurations. **h**-**i**, Positions of the following ribosomes (coloured according to different rotations) relative to the preceding ribosome (triangle indicates the mRNA exit site). **j**-**k**, Two representative di-ribosome pairs in polysomes with extended L9. **l**-**m**, Clash between the C-terminal domain of the extended L9 and the superposed EF-G and EF-Tu, respectively. **n**, *De novo* focused classification on the L9 region using a spherical mask (light green) of 70S ribosomes resulted in three classes: no L9 resolved (I), flat L9 (II) and extended L9 (III). The additional density connecting to the extended L9 originates from the neighbouring ribosome. **o**, Overlap between polysomes independently defined based on spatial analysis, RELION classification for polysomes and classified ribosomes with the extended L9. **p**, The extended L9 is more frequent in the 't-t' ribosome pairs, especially in compacted ones.\\n\\n### [Extended Data Fig. 13 Correlation of translation elongation states within polysomes in native untreated cells.](/articles/s41586-022-05255-2/figures/17)\\n\\n**a**, Occurrence frequencies of elongation state pairs of two adjacent ribosomes (preceding ribosome i versus following ribosome i+1) within polysomes calculated from the experimental data. **b**, Theoretical frequencies of elongation state pairs if there is no cross-influence within the polysome and ribosome state pairs form randomly. **c**, Frequencies of elongation state pairs calculated after random shuffling of the experimental data. **d**, Schematic representation of polysome shuffling analysis and permutation *p*-value calculation. Detailed procedure can be found in the Materials and Methods. **e**, Comparison of the experimental and shuffled pair fractions for all major pairs. **f**, Fold changes between the experimental and shuffled pair fractions from the shuffling experiments. Ribosomes of states that need elongation factor binding to proceed (states 1 and 2a for EF-Tu, and states 5 and 6a for EF-G) are more frequently engaged as the following ribosomes. Permutation *p*-values were adjusted for multiple hypotheses testing with the Benjamini–Hochberg procedure, and are provided in Supplementary Table [7](/articles/s41586-022-05255-2#MOESM1). **g**, Difference between experimental and theoretical pair frequencies when using different distance thresholds for polysome definition. **h**, Ratios of the number of ribosomes in each state being the preceding one against the following one in polysome pairs (across all polysomes), calculated using different distance thresholds to define polysomes. Symmetric engagement as the preceding and following ribosome results in a ratio of 1.\\n\\nSupplementary information\\n-------------------------\\n\\n### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_MOESM1_ESM.pdf)\\n\\nThis file contains Supplementary Discussion; Supplementary Tables 1–8; legends for Supplementary Videos 1-2 and Supplementary References.\\n\\n### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_MOESM2_ESM.pdf)\\n\\n### [Supplementary Video 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_MOESM3_ESM.mp4)\\n\\n### [Supplementary Video 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_MOESM4_ESM.mp4)\\n\\nRights and permissions\\n----------------------\\n\\n\\n**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <http://creativecommons.org/licenses/by/4.0/>.\\n\\n\\n[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Visualizing%20translation%20dynamics%20at%20atomic%20detail%20inside%20a%20bacterial%20cell&author=Liang%20Xue%20et%20al&contentID=10.1038%2Fs41586-022-05255-2&copyright=The%20Author%28s%29&publication=0028-0836&publicationDate=2022-09-28&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)\\n\\nAbout this article\\n------------------\\n\\n[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41586-022-05255-2)### Cite this article\\n\\nXue, L., Lenz, S., Zimmermann-Kogadeeva, M. *et al.* Visualizing translation dynamics at atomic detail inside a bacterial cell.\\n *Nature* **610**, 205–211 (2022). https://doi.org/10.1038/s41586-022-05255-2\\n\\n[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41586-022-05255-2?format=refman&flavour=citation)\\n\\n* Received: 18 December 2021\\n* Accepted: 19 August 2022\\n* Published: 28 September 2022\\n* Issue Date: 06 October 2022\\n* DOI: https://doi.org/10.1038/s41586-022-05255-2\\n\\n### Share this article\\n\\nAnyone you share the following link with will be able to read this content:\\n\\nGet shareable linkSorry, a shareable link is not currently available for this article.\\n\\nCopy to clipboard\\n Provided by the Springer Nature SharedIt content-sharing initiative\\n \\n\\n\\n### Subjects\\n\\n\\n* [Bacterial structural biology](/subjects/bacterial-structural-biology)\\n* [Cryoelectron tomography](/subjects/cryoelectron-tomography)\\n* [Ribosome](/subjects/ribosome)\\n\\n\\n\\n\\n\\n\\nThis article is cited by\\n------------------------\\n\\n\\n\\n* ### [Learning structural heterogeneity from cryo-electron sub-tomograms with tomoDRGN](https://doi.org/10.1038/s41592-024-02210-z)\\n\\n\\n\\t+ Barrett M. Powell\\n\\t+ Joseph H. Davis*Nature Methods* (2024)\\n* ### [Transient disome complex formation in native polysomes during ongoing protein synthesis captured by cryo-EM](https://doi.org/10.1038/s41467-024-46092-3)\\n\\n\\n\\t+ Timo Flügel\\n\\t+ Magdalena Schacherl\\n\\t+ Christian M. T. Spahn*Nature Communications* (2024)\\n* ### [Patchy and widespread distribution of bacterial translation arrest peptides associated with the protein localization machinery](https://doi.org/10.1038/s41467-024-46993-3)\\n\\n\\n\\t+ Keigo Fujiwara\\n\\t+ Naoko Tsuji\\n\\t+ Shinobu Chiba*Nature Communications* (2024)\\n* ### [Visualization of translation and protein biogenesis at the ER membrane](https://doi.org/10.1038/s41586-022-05638-5)\\n\\n\\n\\t+ Max Gemmer\\n\\t+ Marten L. Chaillet\\n\\t+ Friedrich Förster*Nature* (2023)\\n* ### [The translating bacterial ribosome at 1.55 Å resolution generated by cryo-EM imaging services](https://doi.org/10.1038/s41467-023-36742-3)\\n\\n\\n\\t+ Simon A. Fromm\\n\\t+ Kate M. O’Connor\\n\\t+ Simone Mattei*Nature Communications* (2023)\\n\\n\\n\\n\\n\\nComments\\n--------\\n\\nBy submitting a comment you agree to abide by our [Terms](/info/tandc.html) and [Community Guidelines](/info/community-guidelines.html). If you find something abusive or that does not comply with our terms or guidelines please flag it as inappropriate.\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n[Download PDF](/articles/s41586-022-05255-2.pdf)\\n\\n\\n\\n\\nAssociated content\\n------------------\\n\\n\\n\\n\\n\\n\\n### [Atomic ribosome activity](https://www.nature.com/articles/s41594-022-00878-5)\\n\\n\\n* Jake Rogers\\n\\n\\n\\nNature Structural & Molecular Biology\\nResearch Highlight\\n09 Nov 2022\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n* Sections\\n* Figures\\n* References\\n\\n* [Abstract](#Abs1)\\n* [Main](#Sec1)\\n* [In-cell structure of the *M. pneumoniae* ribosome](#Sec2)\\n* [Structural dynamics of translation in cells](#Sec3)\\n* [Antibiotics alter translation landscapes](#Sec4)\\n* [Spatial and functional organization of translation](#Sec5)\\n* [Conclusions](#Sec6)\\n* [Methods](#Sec7)\\n* [Data availability](#data-availability)\\n* [Code availability](#code-availability)\\n* [Change history](#change-history)\\n* [References](#Bib1)\\n* [Acknowledgements](#Ack1)\\n* [Author information](#author-information)\\n* [Ethics declarations](#ethics)\\n* [Peer review](#peer-review)\\n* [Additional information](#additional-information)\\n* [Extended data figures and tables](#Sec19)\\n* [Supplementary information](#Sec20)\\n* [Rights and permissions](#rightslink)\\n* [About this article](#article-info)\\n* [This article is cited by](#further-reading)\\n* [Comments](#article-comments)\\n\\n\\n\\n\\nAdvertisement\\n\\n\\n\\n\\n[![Advertisement](//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature.com/article&sz=300x250&c=-563873083&t=pos%3Dright%26type%3Darticle%26artid%3Ds41586-022-05255-2%26doi%3D10.1038/s41586-022-05255-2%26techmeta%3D101,28%26subjmeta%3D1258,1260,1789,2536,326,337,41,45,535,574,631%26kwrd%3DBacterial+structural+biology,Cryoelectron+tomography,Ribosome)](//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature.com/article&sz=300x250&c=-563873083&t=pos%3Dright%26type%3Darticle%26artid%3Ds41586-022-05255-2%26doi%3D10.1038/s41586-022-05255-2%26techmeta%3D101,28%26subjmeta%3D1258,1260,1789,2536,326,337,41,45,535,574,631%26kwrd%3DBacterial+structural+biology,Cryoelectron+tomography,Ribosome)\\n\\n\\n\\n\\n\\n\\n* **Fig. 1: Ribosome structure in *M.* *pneumoniae* cells.**\\n\\n![figure 1]()[View in article](#Fig1)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/1)\\n* **Fig. 2: Ribosome classification reconstructs the translation elongation cycle.**\\n\\n![figure 2]()[View in article](#Fig2)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/2)\\n* **Fig. 3: Antibiotics induce distinct translation elongation landscapes in cells.**\\n\\n![figure 3]()[View in article](#Fig3)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/3)\\n* **Fig. 4: Spatial and functional organization of ribosomes in native cells.**\\n\\n![figure 4]()[View in article](#Fig4)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/4)\\n* **Extended Data Fig. 1 *M. pneumoniae* in-cell ribosome maps.**\\n\\n![extended data figure 5]()[View in article](#Fig5)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/5)\\n* **Extended Data Fig. 2 Structural features of the *M. pneumoniae* ribosome.**\\n\\n![extended data figure 6]()[View in article](#Fig6)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/6)\\n* **Extended Data Fig. 3 Ribosomal protein extensions.**\\n\\n![extended data figure 7]()[View in article](#Fig7)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/7)\\n* **Extended Data Fig. 4 Classification and refinement of ribosomes in native untreated cells.**\\n\\n![extended data figure 8]()[View in article](#Fig8)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/8)\\n* **Extended Data Fig. 5 Validation of ribosome detection and classification.**\\n\\n![extended data figure 9]()[View in article](#Fig9)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/9)\\n* **Extended Data Fig. 6 Local-resolution maps for ribosome classes in untreated cells.**\\n\\n![extended data figure 10]()[View in article](#Fig10)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/10)\\n* **Extended Data Fig. 7 Model building for ribosome classes in untreated cells.**\\n\\n![extended data figure 11]()[View in article](#Fig11)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/11)\\n* **Extended Data Fig. 8 Early-to-late translocation intermediates during elongation.**\\n\\n![extended data figure 12]()[View in article](#Fig12)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/12)\\n* **Extended Data Fig. 9 Classification, refinement and modelling of ribosomes in Cm-treated cells.**\\n\\n![extended data figure 13]()[View in article](#Fig13)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/13)\\n* **Extended Data Fig. 10 Classification, refinement and modelling of ribosomes in Spc-treated cells.**\\n\\n![extended data figure 14]()[View in article](#Fig14)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/14)\\n* **Extended Data Fig. 11 Classification, refinement and modelling of ribosomes in PUM-treated cells.**\\n\\n![extended data figure 15]()[View in article](#Fig15)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/15)\\n* **Extended Data Fig. 12 Spatial analysis of ribosomes and polysomes in native untreated cells.**\\n\\n![extended data figure 16]()[View in article](#Fig16)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/16)\\n* **Extended Data Fig. 13 Correlation of translation elongation states within polysomes in native untreated cells.**\\n\\n![extended data figure 17]()[View in article](#Fig17)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/17)\\n\\n1. Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. *Annu. Rev. Biochem.* **82**, 203–236 (2013).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-biochem-113009-092313) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFOqsrzJ) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23746255) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20the%20translational%20elongation%20cycle&journal=Annu.%20Rev.%20Biochem.&doi=10.1146%2Fannurev-biochem-113009-092313&volume=82&pages=203-236&publication_year=2013&author=Voorhees%2CRM&author=Ramakrishnan%2CV)\\n2. O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. *Science* **369**, 554–557 (2020).\\n\\n[Article](https://doi.org/10.1126%2Fscience.abb3758) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020Sci...369..554O) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFWgtr7N) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32732422) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115962) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=In-cell%20architecture%20of%20an%20actively%20transcribing%E2%80%93translating%20expressome&journal=Science&doi=10.1126%2Fscience.abb3758&volume=369&pages=554-557&publication_year=2020&author=O%27Reilly%2CFJ)\\n3. Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. *Nat. Methods* **18**, 186–193 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-020-01054-7) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjt1SmsL8%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33542511) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7611018) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-particle%20cryo-EM%20refinement%20with%20M%20visualizes%20ribosome%E2%80%93antibiotic%20complex%20at%203.5%20%C3%85%20in%20cells&journal=Nat.%20Methods&doi=10.1038%2Fs41592-020-01054-7&volume=18&pages=186-193&publication_year=2021&author=Tegunov%2CD&author=Xue%2CL&author=Dienemann%2CC&author=Cramer%2CP&author=Mahamid%2CJ)\\n4. Brandt, F. et al. The native 3D organization of bacterial polysomes. *Cell* **136**, 261–271 (2009).\\n\\n[Article](https://doi.org/10.1016%2Fj.cell.2008.11.016) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhs1Kiurk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19167328) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20native%203D%20organization%20of%20bacterial%20polysomes&journal=Cell&doi=10.1016%2Fj.cell.2008.11.016&volume=136&pages=261-271&publication_year=2009&author=Brandt%2CF)\\n5. Rodnina, M. V. Translation in prokaryotes. *Cold Spring Harb. Perspect. Biol.* **10**, a032664 (2018).\\n\\n[Article](https://doi.org/10.1101%2Fcshperspect.a032664) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29661790) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6120702) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Translation%20in%20prokaryotes&journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&doi=10.1101%2Fcshperspect.a032664&volume=10&publication_year=2018&author=Rodnina%2CMV)\\n6. Frank, J. & Agrawal, R. K. A ratchet-like inter-subunit reorganization of the ribosome during translocation. *Nature* **406**, 318–322 (2000).\\n\\n[Article](https://doi.org/10.1038%2F35018597) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2000Natur.406..318F) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXlsFKnsrk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10917535) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20ratchet-like%20inter-subunit%20reorganization%20of%20the%20ribosome%20during%20translocation&journal=Nature&doi=10.1038%2F35018597&volume=406&pages=318-322&publication_year=2000&author=Frank%2CJ&author=Agrawal%2CRK)\\n7. Villa, E. et al. Ribosome-induced changes in elongation factor Tu conformation control GTP hydrolysis. *Proc. Natl Acad. Sci. USA* **106**, 1063–1068 (2009).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.0811370106) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009PNAS..106.1063V) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhsFyjtbk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19122150) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2613361) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome-induced%20changes%20in%20elongation%20factor%20Tu%20conformation%20control%20GTP%20hydrolysis&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.0811370106&volume=106&pages=1063-1068&publication_year=2009&author=Villa%2CE)\\n8. Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. *Nature* **466**, 329–333 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature09206) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.466..329F) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXosl2rsr0%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20631791) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome%20dynamics%20and%20tRNA%20movement%20by%20time-resolved%20electron%20cryomicroscopy&journal=Nature&doi=10.1038%2Fnature09206&volume=466&pages=329-333&publication_year=2010&author=Fischer%2CN&author=Konevega%2CAL&author=Wintermeyer%2CW&author=Rodnina%2CMV&author=Stark%2CH)\\n9. Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. *Nature* **468**, 713–716 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature09547) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.468..713R) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsFSku7jF) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21124459) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3272701) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Head%20swivel%20on%20the%20ribosome%20facilitates%20translocation%20by%20means%20of%20intra-subunit%20tRNA%20hybrid%20sites&journal=Nature&doi=10.1038%2Fnature09547&volume=468&pages=713-716&publication_year=2010&author=Ratje%2CAH)\\n10. Ramrath, D. J. et al. Visualization of two transfer RNAs trapped in transit during elongation factor G-mediated translocation. *Proc. Natl Acad. Sci. USA* **110**, 20964–20969 (2013).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1320387110) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2013PNAS..11020964R) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXnsF2jug%3D%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24324168) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876219) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualization%20of%20two%20transfer%20RNAs%20trapped%20in%20transit%20during%20elongation%20factor%20G-mediated%20translocation&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1320387110&volume=110&pages=20964-20969&publication_year=2013&author=Ramrath%2CDJ)\\n11. Brilot, A. F., Korostelev, A. A., Ermolenko, D. N. & Grigorieff, N. Structure of the ribosome with elongation factor G trapped in the pretranslocation state. *Proc. Natl Acad. Sci. USA* **110**, 20994–20999 (2013).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1311423110) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2013PNAS..11020994B) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXnsF2iuw%3D%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24324137) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876248) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structure%20of%20the%20ribosome%20with%20elongation%20factor%20G%20trapped%20in%20the%20pretranslocation%20state&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1311423110&volume=110&pages=20994-20999&publication_year=2013&author=Brilot%2CAF&author=Korostelev%2CAA&author=Ermolenko%2CDN&author=Grigorieff%2CN)\\n12. Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. *Cell* **161**, 845–857 (2015).\\n\\n[Article](https://doi.org/10.1016%2Fj.cell.2015.03.052) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXotVKlur0%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25957688) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4432480) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20snapshots%20of%20actively%20translating%20human%20ribosomes&journal=Cell&doi=10.1016%2Fj.cell.2015.03.052&volume=161&pages=845-857&publication_year=2015&author=Behrmann%2CE)\\n13. Loveland, A. B., Demo, G. & Korostelev, A. A. Cryo-EM of elongating ribosome with EF-Tu\\\\*GTP elucidates tRNA proofreading. *Nature* **584**, 640–645 (2020).\\n\\n[Article](https://doi.org/10.1038%2Fs41586-020-2447-x) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020Natur.584..640L) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlShtL%2FE) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32612237) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7483604) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cryo-EM%20of%20elongating%20ribosome%20with%20EF-Tu%2AGTP%20elucidates%20tRNA%20proofreading&journal=Nature&doi=10.1038%2Fs41586-020-2447-x&volume=584&pages=640-645&publication_year=2020&author=Loveland%2CAB&author=Demo%2CG&author=Korostelev%2CAA)\\n14. Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. *Nature* **595**, 741–745 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41586-021-03713-x) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Natur.595..741R) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFSnsbnM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34234344) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8318882) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20early%20translocation%20events%20on%20the%20ribosome&journal=Nature&doi=10.1038%2Fs41586-021-03713-x&volume=595&pages=741-745&publication_year=2021&author=Rundlet%2CEJ)\\n15. Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. *Nat. Commun.* **12**, 5933 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-26133-x) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.5933P) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1aqsL7M) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34635670) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8505512) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20mechanism%20of%20GTPase-powered%20ribosome%E2%80%93tRNA%20movement&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-26133-x&volume=12&publication_year=2021&author=Petrychenko%2CV)\\n16. Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. *Nat. Commun.* **12**, 7236 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-27415-0) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.7236C) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislejtbjN) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34903725) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8668904) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Time-resolved%20cryo-EM%20visualizes%20ribosomal%20translocation%20with%20EF-G%20and%20GTP&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-27415-0&volume=12&publication_year=2021&author=Carbone%2CCE)\\n17. Brown, A. & Shao, S. Ribosomes and cryo-EM: a duet. *Curr. Opin. Struct. Biol.* **52**, 1–7 (2018).\\n\\n[Article](https://doi.org/10.1016%2Fj.sbi.2018.07.001) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlSjs77J) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30015201) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosomes%20and%20cryo-EM%3A%20a%20duet&journal=Curr.%20Opin.%20Struct.%20Biol.&doi=10.1016%2Fj.sbi.2018.07.001&volume=52&pages=1-7&publication_year=2018&author=Brown%2CA&author=Shao%2CS)\\n18. Kuhner, S. et al. Proteome organization in a genome-reduced bacterium. *Science* **326**, 1235–1240 (2009).\\n\\n[Article](https://doi.org/10.1126%2Fscience.1176343) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009Sci...326.1235K) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19965468) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Proteome%20organization%20in%20a%20genome-reduced%20bacterium&journal=Science&doi=10.1126%2Fscience.1176343&volume=326&pages=1235-1240&publication_year=2009&author=Kuhner%2CS)\\n19. Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. *Mol. Cell* **39**, 560–569 (2010).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2010.08.003) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVyrt7jF) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20797628) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20three-dimensional%20organization%20of%20polyribosomes%20in%20intact%20human%20cells&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2010.08.003&volume=39&pages=560-569&publication_year=2010&author=Brandt%2CF&author=Carlson%2CLA&author=Hartl%2CFU&author=Baumeister%2CW&author=Grunewald%2CK)\\n20. Mahamid, J. et al. Visualizing the molecular sociology at the HeLa cell nuclear periphery. *Science* **351**, 969–972 (2016).\\n\\n[Article](https://doi.org/10.1126%2Fscience.aad8857) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016Sci...351..969M) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XivFOnt7c%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26917770) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualizing%20the%20molecular%20sociology%20at%20the%20HeLa%20cell%20nuclear%20periphery&journal=Science&doi=10.1126%2Fscience.aad8857&volume=351&pages=969-972&publication_year=2016&author=Mahamid%2CJ)\\n21. Schaffer, M. et al. A cryo-FIB lift-out technique enables molecular-resolution cryo-ET within native *Caenorhabditis elegans* tissue. *Nat. Methods* **16**, 757–762 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0497-5) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVOku7zL) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31363205) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20cryo-FIB%20lift-out%20technique%20enables%20molecular-resolution%20cryo-ET%20within%20native%20Caenorhabditis%20elegans%20tissue&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0497-5&volume=16&pages=757-762&publication_year=2019&author=Schaffer%2CM)\\n22. Waltz, F. et al. How to build a ribosome from RNA fragments in *Chlamydomonas* mitochondria. *Nat. Commun.* **12**, 7176 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-27200-z) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.7176W) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislWktLrO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34887394) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8660880) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=How%20to%20build%20a%20ribosome%20from%20RNA%20fragments%20in%20Chlamydomonas%20mitochondria&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-27200-z&volume=12&publication_year=2021&author=Waltz%2CF)\\n23. Melnikov, S., Manakongtreecheep, K. & Soll, D. Revising the structural diversity of ribosomal proteins across the three domains of life. *Mol. Biol. Evol.* **35**, 1588–1598 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsy021) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtF2rur7L) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29529322) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5995209) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Revising%20the%20structural%20diversity%20of%20ribosomal%20proteins%20across%20the%20three%20domains%20of%20life&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsy021&volume=35&pages=1588-1598&publication_year=2018&author=Melnikov%2CS&author=Manakongtreecheep%2CK&author=Soll%2CD)\\n24. Miravet-Verde, S., Burgos, R., Delgado, J., Lluch-Senar, M. & Serrano, L. FASTQINS and ANUBIS: two bioinformatic tools to explore facts and artifacts in transposon sequencing and essentiality studies. *Nucleic Acids Res.* **48**, e102 (2020).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkaa679) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXkt1GntbY%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32813015) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7515713) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FASTQINS%20and%20ANUBIS%3A%20two%20bioinformatic%20tools%20to%20explore%20facts%20and%20artifacts%20in%20transposon%20sequencing%20and%20essentiality%20studies&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkaa679&volume=48&publication_year=2020&author=Miravet-Verde%2CS&author=Burgos%2CR&author=Delgado%2CJ&author=Lluch-Senar%2CM&author=Serrano%2CL)\\n25. Borg, A., Pavlov, M. & Ehrenberg, M. Complete kinetic mechanism for recycling of the bacterial ribosome. *RNA* **22**, 10–21 (2016).\\n\\n[Article](https://doi.org/10.1261%2Frna.053157.115) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVKlur3M) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26527791) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4691825) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Complete%20kinetic%20mechanism%20for%20recycling%20of%20the%20bacterial%20ribosome&journal=RNA&doi=10.1261%2Frna.053157.115&volume=22&pages=10-21&publication_year=2016&author=Borg%2CA&author=Pavlov%2CM&author=Ehrenberg%2CM)\\n26. Munro, J. B., Altman, R. B., O’Connor, N. & Blanchard, S. C. Identification of two distinct hybrid state intermediates on the ribosome. *Mol. Cell* **25**, 505–517 (2007).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2007.01.022) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXjtlWkur8%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17317624) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2244649) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20two%20distinct%20hybrid%20state%20intermediates%20on%20the%20ribosome&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2007.01.022&volume=25&pages=505-517&publication_year=2007&author=Munro%2CJB&author=Altman%2CRB&author=O%E2%80%99Connor%2CN&author=Blanchard%2CSC)\\n27. Belardinelli, R. et al. Choreography of molecular movements during ribosome progression along mRNA. *Nat. Struct. Mol. Biol.* **23**, 342–348 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.3193) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XktFeqtrk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26999556) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Choreography%20of%20molecular%20movements%20during%20ribosome%20progression%20along%20mRNA&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.3193&volume=23&pages=342-348&publication_year=2016&author=Belardinelli%2CR)\\n28. Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. *Nat. Struct. Mol. Biol.* **23**, 333–341 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.3177) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XivFKhurY%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26926435) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4821728) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiperspective%20smFRET%20reveals%20rate-determining%20late%20intermediates%20of%20ribosomal%20translocation&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.3177&volume=23&pages=333-341&publication_year=2016&author=Wasserman%2CMR&author=Alejo%2CJL&author=Altman%2CRB&author=Blanchard%2CSC)\\n29. Fei, J., Kosuri, P., MacDougall, D. D. & Gonzalez, R. L. Jr Coupling of ribosomal L1 stalk and tRNA dynamics during translation elongation. *Mol. Cell* **30**, 348–359 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2008.03.012) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXmtFaktLg%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18471980) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Coupling%20of%20ribosomal%20L1%20stalk%20and%20tRNA%20dynamics%20during%20translation%20elongation&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2008.03.012&volume=30&pages=348-359&publication_year=2008&author=Fei%2CJ&author=Kosuri%2CP&author=MacDougall%2CDD&author=Gonzalez%2CRL)\\n30. Zhou, J., Lancaster, L., Donohue, J. P. & Noller, H. F. How the ribosome hands the A-site tRNA to the P site during EF-G-catalyzed translocation. *Science* **345**, 1188–1191 (2014).\\n\\n[Article](https://doi.org/10.1126%2Fscience.1255030) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2014Sci...345.1188Z) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsVGhtb%2FO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25190797) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4242719) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=How%20the%20ribosome%20hands%20the%20A-site%20tRNA%20to%20the%20P%20site%20during%20EF-G-catalyzed%20translocation&journal=Science&doi=10.1126%2Fscience.1255030&volume=345&pages=1188-1191&publication_year=2014&author=Zhou%2CJ&author=Lancaster%2CL&author=Donohue%2CJP&author=Noller%2CHF)\\n31. Uemura, S. et al. Real-time tRNA transit on single translating ribosomes at codon resolution. *Nature* **464**, 1012–U1073 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature08925) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.464.1012U) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXkslart7c%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20393556) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4466108) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20tRNA%20transit%20on%20single%20translating%20ribosomes%20at%20codon%20resolution&journal=Nature&doi=10.1038%2Fnature08925&volume=464&pages=1012-U1073&publication_year=2010&author=Uemura%2CS)\\n32. Chen, C. et al. Single-molecule fluorescence measurements of ribosomal translocation dynamics. *Mol. Cell* **42**, 367–377 (2011).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2011.03.024) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXlvVyiu7w%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21549313) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3090999) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-molecule%20fluorescence%20measurements%20of%20ribosomal%20translocation%20dynamics&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2011.03.024&volume=42&pages=367-377&publication_year=2011&author=Chen%2CC)\\n33. Wilson, D. N. & Nierhaus, K. H. The E-site story: the importance of maintaining two tRNAs on the ribosome during protein synthesis. *Cell. Mol. Life Sci.* **63**, 2725–2737 (2006).\\n\\n[Article](https://link.springer.com/doi/10.1007/s00018-006-6125-4) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFSrt7w%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17013564) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20E-site%20story%3A%20the%20importance%20of%20maintaining%20two%20tRNAs%20on%20the%20ribosome%20during%20protein%20synthesis&journal=Cell.%20Mol.%20Life%20Sci.&doi=10.1007%2Fs00018-006-6125-4&volume=63&pages=2725-2737&publication_year=2006&author=Wilson%2CDN&author=Nierhaus%2CKH)\\n34. Munro, J. B., Sanbonmatsu, K. Y., Spahn, C. M. & Blanchard, S. C. Navigating the ribosome’s metastable energy landscape. *Trends Biochem. Sci* **34**, 390–400 (2009).\\n\\n[Article](https://doi.org/10.1016%2Fj.tibs.2009.04.004) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVSntrnM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19647434) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2914510) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Navigating%20the%20ribosome%E2%80%99s%20metastable%20energy%20landscape&journal=Trends%20Biochem.%20Sci&doi=10.1016%2Fj.tibs.2009.04.004&volume=34&pages=390-400&publication_year=2009&author=Munro%2CJB&author=Sanbonmatsu%2CKY&author=Spahn%2CCM&author=Blanchard%2CSC)\\n35. Bock, L. V. et al. Energy barriers and driving forces in tRNA translocation through the ribosome. *Nat. Struct. Mol. Biol.* **20**, 1390–1396 (2013).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.2690) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslWisbvK) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24186064) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Energy%20barriers%20and%20driving%20forces%20in%20tRNA%20translocation%20through%20the%20ribosome&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.2690&volume=20&pages=1390-1396&publication_year=2013&author=Bock%2CLV)\\n36. Julian, P. et al. Structure of ratcheted ribosomes with tRNAs in hybrid states. *Proc. Natl Acad. Sci. USA* **105**, 16924–16927 (2008).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.0809587105) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2008PNAS..10516924J) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlKqt7%2FM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18971332) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2579354) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structure%20of%20ratcheted%20ribosomes%20with%20tRNAs%20in%20hybrid%20states&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.0809587105&volume=105&pages=16924-16927&publication_year=2008&author=Julian%2CP)\\n37. Agirrezabala, X. et al. Visualization of the hybrid state of tRNA binding promoted by spontaneous ratcheting of the ribosome. *Mol. Cell* **32**, 190–197 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2008.10.001) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlehsb3L) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18951087) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614368) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualization%20of%20the%20hybrid%20state%20of%20tRNA%20binding%20promoted%20by%20spontaneous%20ratcheting%20of%20the%20ribosome&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2008.10.001&volume=32&pages=190-197&publication_year=2008&author=Agirrezabala%2CX)\\n38. Rodnina, M. V., Peske, F., Peng, B. Z., Belardinelli, R. & Wintermeyer, W. Converting GTP hydrolysis into motion: versatile translational elongation factor G. *Biol. Chem.* **401**, 131–142 (2019).\\n\\n[Article](https://doi.org/10.1515%2Fhsz-2019-0313) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31600135) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Converting%20GTP%20hydrolysis%20into%20motion%3A%20versatile%20translational%20elongation%20factor%20G&journal=Biol.%20Chem.&doi=10.1515%2Fhsz-2019-0313&volume=401&pages=131-142&publication_year=2019&author=Rodnina%2CMV&author=Peske%2CF&author=Peng%2CBZ&author=Belardinelli%2CR&author=Wintermeyer%2CW)\\n39. Lin, J., Zhou, D., Steitz, T. A., Polikanov, Y. S. & Gagnon, M. G. Ribosome-targeting antibiotics: modes of action, mechanisms of resistance, and implications for drug design. *Annu. Rev. Biochem.* **87**, 451–478 (2018).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-biochem-062917-011942) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXlslSrurw%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29570352) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9176271) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome-targeting%20antibiotics%3A%20modes%20of%20action%2C%20mechanisms%20of%20resistance%2C%20and%20implications%20for%20drug%20design&journal=Annu.%20Rev.%20Biochem.&doi=10.1146%2Fannurev-biochem-062917-011942&volume=87&pages=451-478&publication_year=2018&author=Lin%2CJ&author=Zhou%2CD&author=Steitz%2CTA&author=Polikanov%2CYS&author=Gagnon%2CMG)\\n40. Bulkley, D., Innis, C. A., Blaha, G. & Steitz, T. A. Revisiting the structures of several antibiotics bound to the bacterial ribosome. *Proc. Natl Acad. Sci. USA* **107**, 17158–17163 (2010).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1008685107) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010PNAS..10717158B) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlSjurrM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20876130) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2951403) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Revisiting%20the%20structures%20of%20several%20antibiotics%20bound%20to%20the%20bacterial%20ribosome&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1008685107&volume=107&pages=17158-17163&publication_year=2010&author=Bulkley%2CD&author=Innis%2CCA&author=Blaha%2CG&author=Steitz%2CTA)\\n41. Svetlov, M. S. et al. High-resolution crystal structures of ribosome-bound chloramphenicol and erythromycin provide the ultimate basis for their competition. *RNA* **25**, 600–606 (2019).\\n\\n[Article](https://doi.org/10.1261%2Frna.069260.118) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1Ons77I) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30733327) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6467010) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-resolution%20crystal%20structures%20of%20ribosome-bound%20chloramphenicol%20and%20erythromycin%20provide%20the%20ultimate%20basis%20for%20their%20competition&journal=RNA&doi=10.1261%2Frna.069260.118&volume=25&pages=600-606&publication_year=2019&author=Svetlov%2CMS)\\n42. Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. *ACS Chem. Biol.* **2**, 545–552 (2007).\\n\\n[Article](https://doi.org/10.1021%2Fcb700100n) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXpvVOhtrc%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17696316) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4624401) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20steric%20block%20in%20translation%20caused%20by%20the%20antibiotic%20spectinomycin&journal=ACS%20Chem.%20Biol.&doi=10.1021%2Fcb700100n&volume=2&pages=545-552&publication_year=2007&author=Borovinskaya%2CMA&author=Shoji%2CS&author=Holton%2CJM&author=Fredrick%2CK&author=Cate%2CJHD)\\n43. Polikanov, Y. S. et al. Distinct tRNA accommodation intermediates observed on the ribosome with the antibiotics hygromycin A and A201A. *Mol. Cell* **58**, 832–844 (2015).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2015.04.014) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXpt1Skurg%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26028538) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4458074) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20tRNA%20accommodation%20intermediates%20observed%20on%20the%20ribosome%20with%20the%20antibiotics%20hygromycin%20A%20and%20A201A&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2015.04.014&volume=58&pages=832-844&publication_year=2015&author=Polikanov%2CYS)\\n44. Morse, J. C. et al. Elongation factor-Tu can repetitively engage aminoacyl-tRNA within the ribosome during the proofreading stage of tRNA selection. *Proc. Natl Acad. Sci. USA* **117**, 3610–3620 (2020).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1904469117) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020PNAS..117.3610M) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjsFyhu7Y%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32024753) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7035488) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Elongation%20factor-Tu%20can%20repetitively%20engage%20aminoacyl-tRNA%20within%20the%20ribosome%20during%20the%20proofreading%20stage%20of%20tRNA%20selection&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1904469117&volume=117&pages=3610-3620&publication_year=2020&author=Morse%2CJC)\\n45. Choi, J. et al. Dynamics of the context-specific translation arrest by chloramphenicol and linezolid. *Nat. Chem. Biol.* **16**, 310–317 (2020).\\n\\n[Article](https://doi.org/10.1038%2Fs41589-019-0423-2) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVSktb%2FM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31844301) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dynamics%20of%20the%20context-specific%20translation%20arrest%20by%20chloramphenicol%20and%20linezolid&journal=Nat.%20Chem.%20Biol.&doi=10.1038%2Fs41589-019-0423-2&volume=16&pages=310-317&publication_year=2020&author=Choi%2CJ)\\n46. Marks, J. et al. Context-specific inhibition of translation by ribosomal antibiotics targeting the peptidyl transferase center. *Proc. Natl Acad. Sci. USA* **113**, 12150–12155 (2016).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1613055113) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016PNAS..11312150M) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1elur3N) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27791002) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5086994) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Context-specific%20inhibition%20of%20translation%20by%20ribosomal%20antibiotics%20targeting%20the%20peptidyl%20transferase%20center&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1613055113&volume=113&pages=12150-12155&publication_year=2016&author=Marks%2CJ)\\n47. Vazquez-Laslop, N. & Mankin, A. S. Context-specific action of ribosomal antibiotics. *Annu. Rev. Microbiol.* **72**, 185–207 (2018).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-micro-090817-062329) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFCjs77O) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29906204) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8742604) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Context-specific%20action%20of%20ribosomal%20antibiotics&journal=Annu.%20Rev.%20Microbiol.&doi=10.1146%2Fannurev-micro-090817-062329&volume=72&pages=185-207&publication_year=2018&author=Vazquez-Laslop%2CN&author=Mankin%2CAS)\\n48. Aguirre Rivera, J. et al. Real-time measurements of aminoglycoside effects on protein synthesis in live cells. *Proc. Natl Acad. Sci. USA* **118**, e2013315118 (2021).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.2013315118) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33619089) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7936356) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20measurements%20of%20aminoglycoside%20effects%20on%20protein%20synthesis%20in%20live%20cells&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.2013315118&volume=118&publication_year=2021&author=Aguirre%20Rivera%2CJ)\\n49. Baquero, F. & Levin, B. R. Proximate and ultimate causes of the bactericidal action of antibiotics. *Nat. Rev. Microbiol.* **19**, 123–132 (2021).\\n50. Myasnikov, A. G. et al. The molecular structure of the left-handed supra-molecular helix of eukaryotic polyribosomes. *Nat. Commun.* **5**, 5294 (2014).\\n\\n[Article](https://doi.org/10.1038%2Fncomms6294) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2014NatCo...5.5294M) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFantrfM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25376914) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20molecular%20structure%20of%20the%20left-handed%20supra-molecular%20helix%20of%20eukaryotic%20polyribosomes&journal=Nat.%20Commun.&doi=10.1038%2Fncomms6294&volume=5&publication_year=2014&author=Myasnikov%2CAG)\\n51. Dunkle, J. A. & Cate, J. H. D. in *Ribosomes* (eds Rodnina, M. V., Wintermeyer, W. & Green, R.) Ch. 6 (2011).\\n52. Herr, A. J., Nelson, C. C., Wills, N. M., Gesteland, R. F. & Atkins, J. F. Analysis of the roles of tRNA structure, ribosomal protein L9, and the bacteriophage T4 gene 60 bypassing signals during ribosome slippage on mRNA. *J. Mol. Biol.* **309**, 1029–1048 (2001).\\n\\n[Article](https://doi.org/10.1006%2Fjmbi.2001.4717) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXkslGgurY%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11399077) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20the%20roles%20of%20tRNA%20structure%2C%20ribosomal%20protein%20L9%2C%20and%20the%20bacteriophage%20T4%20gene%2060%20bypassing%20signals%20during%20ribosome%20slippage%20on%20mRNA&journal=J.%20Mol.%20Biol.&doi=10.1006%2Fjmbi.2001.4717&volume=309&pages=1029-1048&publication_year=2001&author=Herr%2CAJ&author=Nelson%2CCC&author=Wills%2CNM&author=Gesteland%2CRF&author=Atkins%2CJF)\\n53. Smith, A. M., Costello, M. S., Kettring, A. H., Wingo, R. J. & Moore, S. D. Ribosome collisions alter frameshifting at translational reprogramming motifs in bacterial mRNAs. *Proc. Natl Acad. Sci. USA* **116**, 21769–21779 (2019).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1910613116) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019PNAS..11621769S) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitValtbjM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31591196) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6815119) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome%20collisions%20alter%20frameshifting%20at%20translational%20reprogramming%20motifs%20in%20bacterial%20mRNAs&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1910613116&volume=116&pages=21769-21779&publication_year=2019&author=Smith%2CAM&author=Costello%2CMS&author=Kettring%2CAH&author=Wingo%2CRJ&author=Moore%2CSD)\\n54. Tegunov, D. & Cramer, P. Real-time cryo-electron microscopy data preprocessing with Warp. *Nat. Methods* **16**, 1146–1152 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0580-y) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFejs77E) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31591575) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6858868) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20cryo-electron%20microscopy%20data%20preprocessing%20with%20Warp&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0580-y&volume=16&pages=1146-1152&publication_year=2019&author=Tegunov%2CD&author=Cramer%2CP)\\n55. Hagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. *J. Struct. Biol.* **197**, 191–198 (2017).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2016.06.007) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27313000) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5287356) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Implementation%20of%20a%20cryo-electron%20tomography%20tilt-scheme%20optimized%20for%20high%20resolution%20subtomogram%20averaging&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2016.06.007&volume=197&pages=191-198&publication_year=2017&author=Hagen%2CWJH&author=Wan%2CW&author=Briggs%2CJAG)\\n56. Schorb, M., Haberbosch, I., Hagen, W. J. H., Schwab, Y. & Mastronarde, D. N. Software tools for automated transmission electron microscopy. *Nat. Methods* **16**, 471–477 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0396-9) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVSqs7zL) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31086343) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7000238) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Software%20tools%20for%20automated%20transmission%20electron%20microscopy&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0396-9&volume=16&pages=471-477&publication_year=2019&author=Schorb%2CM&author=Haberbosch%2CI&author=Hagen%2CWJH&author=Schwab%2CY&author=Mastronarde%2CDN)\\n57. Hrabe, T. et al. PyTom: a python-based toolbox for localization of macromolecules in cryo-electron tomograms and subtomogram analysis. *J. Struct. Biol.* **178**, 177–188 (2012).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2011.12.003) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xnt1ehtL4%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22193517) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PyTom%3A%20a%20python-based%20toolbox%20for%20localization%20of%20macromolecules%20in%20cryo-electron%20tomograms%20and%20subtomogram%20analysis&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2011.12.003&volume=178&pages=177-188&publication_year=2012&author=Hrabe%2CT)\\n58. Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. *Nat. Protoc.* **11**, 2054–2065 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnprot.2016.124) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsF2mtLrL) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27685097) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5215819) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Resolving%20macromolecular%20structures%20from%20electron%20cryo-tomography%20data%20using%20subtomogram%20averaging%20in%20RELION&journal=Nat.%20Protoc.&doi=10.1038%2Fnprot.2016.124&volume=11&pages=2054-2065&publication_year=2016&author=Bharat%2CTA&author=Scheres%2CSH)\\n59. Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. *eLife* **7**, e42166 (2018).\\n\\n[Article](https://doi.org/10.7554%2FeLife.42166) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30412051) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6250425) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=New%20tools%20for%20automated%20high-resolution%20cryo-EM%20structure%20determination%20in%20RELION-3&journal=eLife&doi=10.7554%2FeLife.42166&volume=7&publication_year=2018&author=Zivanov%2CJ)\\n60. Waterhouse, A. et al. SWISS-MODEL: homology modelling of protein structures and complexes. *Nucleic Acids Res.* **46**, W296–W303 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky427) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyru70%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29788355) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030848) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SWISS-MODEL%3A%20homology%20modelling%20of%20protein%20structures%20and%20complexes&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky427&volume=46&pages=W296-W303&publication_year=2018&author=Waterhouse%2CA)\\n61. Pruesse, E., Peplies, J. & Glockner, F. O. SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes. *Bioinformatics* **28**, 1823–1829 (2012).\\n\\n[Article](https://doi.org/10.1093%2Fbioinformatics%2Fbts252) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVSgsbfO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22556368) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3389763) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SINA%3A%20accurate%20high-throughput%20multiple%20sequence%20alignment%20of%20ribosomal%20RNA%20genes&journal=Bioinformatics&doi=10.1093%2Fbioinformatics%2Fbts252&volume=28&pages=1823-1829&publication_year=2012&author=Pruesse%2CE&author=Peplies%2CJ&author=Glockner%2CFO)\\n62. Rother, M., Rother, K., Puton, T. & Bujnicki, J. M. ModeRNA: a tool for comparative modeling of RNA 3D structure. *Nucleic Acids Res.* **39**, 4007–4022 (2011).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkq1320) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmvValsL4%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21300639) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3105415) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ModeRNA%3A%20a%20tool%20for%20comparative%20modeling%20of%20RNA%203D%20structure&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkq1320&volume=39&pages=4007-4022&publication_year=2011&author=Rother%2CM&author=Rother%2CK&author=Puton%2CT&author=Bujnicki%2CJM)\\n63. Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. *J. Comput. Chem.* **25**, 1605–1612 (2004).\\n\\n[Article](https://doi.org/10.1002%2Fjcc.20084) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXmvVOhsbs%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15264254) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCSF%20Chimera%E2%80%94a%20visualization%20system%20for%20exploratory%20research%20and%20analysis&journal=J.%20Comput.%20Chem.&doi=10.1002%2Fjcc.20084&volume=25&pages=1605-1612&publication_year=2004&author=Pettersen%2CEF)\\n64. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. *Acta Crystallogr D* **74**, 531–544 (2018).\\n\\n[Article](https://doi.org/10.1107%2FS2059798318006551) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2ltbzP) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-space%20refinement%20in%20PHENIX%20for%20cryo-EM%20and%20crystallography&journal=Acta%20Crystallogr%20D&doi=10.1107%2FS2059798318006551&volume=74&pages=531-544&publication_year=2018&author=Afonine%2CPV)\\n65. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. *Acta Crystallogr D* **66**, 486–501 (2010).\\n\\n[Article](https://doi.org/10.1107%2FS0907444910007493) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXksFKisb8%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20383002) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2852313) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Features%20and%20development%20of%20Coot&journal=Acta%20Crystallogr%20D&doi=10.1107%2FS0907444910007493&volume=66&pages=486-501&publication_year=2010&author=Emsley%2CP&author=Lohkamp%2CB&author=Scott%2CWG&author=Cowtan%2CK)\\n66. Williams, C. J. et al. MolProbity: more and better reference data for improved all-atom structure validation. *Protein Sci.* **27**, 293–315 (2018).\\n\\n[Article](https://doi.org/10.1002%2Fpro.3330) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvVKgsrnO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29067766) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MolProbity%3A%20more%20and%20better%20reference%20data%20for%20improved%20all-atom%20structure%20validation&journal=Protein%20Sci.&doi=10.1002%2Fpro.3330&volume=27&pages=293-315&publication_year=2018&author=Williams%2CCJ)\\n67. Chen, S. et al. High-resolution noise substitution to measure overfitting and validate resolution in 3D structure determination by single particle electron cryomicroscopy. *Ultramicroscopy* **135**, 24–35 (2013).\\n\\n[Article](https://doi.org/10.1016%2Fj.ultramic.2013.06.004) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslWhtLzO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23872039) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3834153) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-resolution%20noise%20substitution%20to%20measure%20overfitting%20and%20validate%20resolution%20in%203D%20structure%20determination%20by%20single%20particle%20electron%20cryomicroscopy&journal=Ultramicroscopy&doi=10.1016%2Fj.ultramic.2013.06.004&volume=135&pages=24-35&publication_year=2013&author=Chen%2CS)\\n68. Tatusov, R. L., Galperin, M. Y., Natale, D. A. & Koonin, E. V. The COG database: a tool for genome-scale analysis of protein functions and evolution. *Nucleic Acids Res.* **28**, 33–36 (2000).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2F28.1.33) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXhvVGqu7w%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10592175) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC102395) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20COG%20database%3A%20a%20tool%20for%20genome-scale%20analysis%20of%20protein%20functions%20and%20evolution&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2F28.1.33&volume=28&pages=33-36&publication_year=2000&author=Tatusov%2CRL&author=Galperin%2CMY&author=Natale%2CDA&author=Koonin%2CEV)\\n69. Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. *Nucleic Acids Res.* **47**, D309–D314 (2019).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky1085) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Gqtr%2FL) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30418610) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=eggNOG%205.0%3A%20a%20hierarchical%2C%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky1085&volume=47&pages=D309-D314&publication_year=2019&author=Huerta-Cepas%2CJ)\\n70. Katoh, K., Rozewicki, J. & Yamada, K. D. MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization. *Brief. Bioinform.* **20**, 1160–1166 (2019).\\n\\n[Article](https://doi.org/10.1093%2Fbib%2Fbbx108) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1Kqtb3K) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28968734) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MAFFT%20online%20service%3A%20multiple%20sequence%20alignment%2C%20interactive%20sequence%20choice%20and%20visualization&journal=Brief.%20Bioinform.&doi=10.1093%2Fbib%2Fbbx108&volume=20&pages=1160-1166&publication_year=2019&author=Katoh%2CK&author=Rozewicki%2CJ&author=Yamada%2CKD)\\n71. Letunic, I. & Bork, P. Interactive Tree Of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. *Nucleic Acids Res.* **49**, W293–W296 (2021).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkab301) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvV2isLvM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33885785) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8265157) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interactive%20Tree%20Of%20Life%20%28iTOL%29%20v5%3A%20an%20online%20tool%20for%20phylogenetic%20tree%20display%20and%20annotation&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkab301&volume=49&pages=W293-W296&publication_year=2021&author=Letunic%2CI&author=Bork%2CP)\\n72. Huerta-Cepas, J. et al. Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper. *Mol. Biol. Evol.* **34**, 2115–2122 (2017).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsx148) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFaru7vF) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28460117) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850834) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20genome-wide%20functional%20annotation%20through%20orthology%20assignment%20by%20eggNOG-Mapper&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsx148&volume=34&pages=2115-2122&publication_year=2017&author=Huerta-Cepas%2CJ)\\n73. Huerta-Cepas, J., Serra, F. & Bork, P. ETE 3: reconstruction, analysis, and visualization of phylogenomic data. *Mol. Biol. Evol.* **33**, 1635–1638 (2016).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsw046) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFeisrrN) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26921390) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4868116) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ETE%203%3A%20reconstruction%2C%20analysis%2C%20and%20visualization%20of%20phylogenomic%20data&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsw046&volume=33&pages=1635-1638&publication_year=2016&author=Huerta-Cepas%2CJ&author=Serra%2CF&author=Bork%2CP)\\n74. Meszaros, B., Erdos, G. & Dosztanyi, Z. IUPred2A: context-dependent prediction of protein disorder as a function of redox state and protein binding. *Nucleic Acids Res.* **46**, W329–W337 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky384) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyrt7o%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29860432) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030935) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=IUPred2A%3A%20context-dependent%20prediction%20of%20protein%20disorder%20as%20a%20function%20of%20redox%20state%20and%20protein%20binding&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky384&volume=46&pages=W329-W337&publication_year=2018&author=Meszaros%2CB&author=Erdos%2CG&author=Dosztanyi%2CZ)\\n75. Cole, C., Barber, J. D. & Barton, G. J. The Jpred 3 secondary structure prediction server. *Nucleic Acids Res.* **36**, W197–W201 (2008).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkn238) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXptlKkt7w%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18463136) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2447793) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Jpred%203%20secondary%20structure%20prediction%20server&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkn238&volume=36&pages=W197-W201&publication_year=2008&author=Cole%2CC&author=Barber%2CJD&author=Barton%2CGJ)\\n76. Scheres, S. H. Processing of structurally heterogeneous cryo-EM data in RELION. *Methods Enzymol.* **579**, 125–157 (2016).\\n\\n[Article](https://doi.org/10.1016%2Fbs.mie.2016.04.012) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmtlKnurk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27572726) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Processing%20of%20structurally%20heterogeneous%20cryo-EM%20data%20in%20RELION&journal=Methods%20Enzymol.&doi=10.1016%2Fbs.mie.2016.04.012&volume=579&pages=125-157&publication_year=2016&author=Scheres%2CSH)\\n77. Kidmose, R. T. et al. Namdinator—automatic molecular dynamics flexible fitting of structural models into cryo-EM and crystallography experimental maps. *IUCrJ* **6**, 526–531 (2019).\\n\\n[Article](https://doi.org/10.1107%2FS2052252519007619) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlWit7%2FO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31316797) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6608625) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Namdinator%E2%80%94automatic%20molecular%20dynamics%20flexible%20fitting%20of%20structural%20models%20into%20cryo-EM%20and%20crystallography%20experimental%20maps&journal=IUCrJ&doi=10.1107%2FS2052252519007619&volume=6&pages=526-531&publication_year=2019&author=Kidmose%2CRT)\\n78. Trabuco, L. G., Villa, E., Mitra, K., Frank, J. & Schulten, K. Flexible fitting of atomic structures into electron microscopy maps using molecular dynamics. *Structure* **16**, 673–683 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.str.2008.03.005) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXlslGrtro%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18462672) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2430731) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Flexible%20fitting%20of%20atomic%20structures%20into%20electron%20microscopy%20maps%20using%20molecular%20dynamics&journal=Structure&doi=10.1016%2Fj.str.2008.03.005&volume=16&pages=673-683&publication_year=2008&author=Trabuco%2CLG&author=Villa%2CE&author=Mitra%2CK&author=Frank%2CJ&author=Schulten%2CK)\\n79. Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. *J. Chem. Phys.* **153**, 044130 (2020).\\n\\n[Article](https://doi.org/10.1063%2F5.0014475) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFajsL3J) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32752662) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7395834) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scalable%20molecular%20dynamics%20on%20CPU%20and%20GPU%20architectures%20with%20NAMD&journal=J.%20Chem.%20Phys.&doi=10.1063%2F5.0014475&volume=153&publication_year=2020&author=Phillips%2CJC)\\n80. Nickell, S. et al. TOM software toolbox: acquisition and analysis for electron tomography. *J. Struct. Biol.* **149**, 227–234 (2005).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2004.10.006) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15721576) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TOM%20software%20toolbox%3A%20acquisition%20and%20analysis%20for%20electron%20tomography&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2004.10.006&volume=149&pages=227-234&publication_year=2005&author=Nickell%2CS)\\n81. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. *Protein Sci.* **30**, 70–82 (2021).\\n\\n[Article](https://doi.org/10.1002%2Fpro.3943) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFamtrzJ) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32881101) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCSF%20ChimeraX%3A%20structure%20visualization%20for%20researchers%2C%20educators%2C%20and%20developers&journal=Protein%20Sci.&doi=10.1002%2Fpro.3943&volume=30&pages=70-82&publication_year=2021&author=Pettersen%2CEF)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n Nature (*Nature*)\\n \\n\\nISSN 1476-4687 (online)\\n \\n\\nISSN 0028-0836 (print)\\n \\n\\n\\n\\n\\n\\n\\n\\n\\nnature.com sitemap\\n------------------\\n\\n\\n\\n\\n\\n### About Nature Portfolio\\n\\n\\n* [About us](https://www.nature.com/npg_/company_info/index.html)\\n* [Press releases](https://www.nature.com/npg_/press_room/press_releases.html)\\n* [Press office](https://press.nature.com/)\\n* [Contact us](https://support.nature.com/support/home)\\n\\n\\n\\n\\n### Discover content\\n\\n\\n* [Journals A-Z](https://www.nature.com/siteindex)\\n* [Articles by subject](https://www.nature.com/subjects)\\n* [protocols.io](https://www.protocols.io/)\\n* [Nature Index](https://www.natureindex.com/)\\n\\n\\n\\n\\n### Publishing policies\\n\\n\\n* [Nature portfolio policies](https://www.nature.com/authors/editorial_policies)\\n* [Open access](https://www.nature.com/nature-research/open-access)\\n\\n\\n\\n\\n### Author & Researcher services\\n\\n\\n* [Reprints & permissions](https://www.nature.com/reprints)\\n* [Research data](https://www.springernature.com/gp/authors/research-data)\\n* [Language editing](https://authorservices.springernature.com/language-editing/)\\n* [Scientific editing](https://authorservices.springernature.com/scientific-editing/)\\n* [Nature Masterclasses](https://masterclasses.nature.com/)\\n* [Research Solutions](https://solutions.springernature.com/)\\n\\n\\n\\n\\n### Libraries & institutions\\n\\n\\n* [Librarian service & tools](https://www.springernature.com/gp/librarians/tools-services)\\n* [Librarian portal](https://www.springernature.com/gp/librarians/manage-your-account/librarianportal)\\n* [Open research](https://www.nature.com/openresearch/about-open-access/information-for-institutions)\\n* [Recommend to library](https://www.springernature.com/gp/librarians/recommend-to-your-library)\\n\\n\\n\\n\\n### Advertising & partnerships\\n\\n\\n* [Advertising](https://partnerships.nature.com/product/digital-advertising/)\\n* [Partnerships & Services](https://partnerships.nature.com/)\\n* [Media kits](https://partnerships.nature.com/media-kits/)\\n* [Branded\\n content](https://partnerships.nature.com/product/branded-content-native-advertising/)\\n\\n\\n\\n\\n### Professional development\\n\\n\\n* [Nature Careers](https://www.nature.com/naturecareers/)\\n* [Nature \\n Conferences](https://conferences.nature.com)\\n\\n\\n\\n\\n### Regional websites\\n\\n\\n* [Nature Africa](https://www.nature.com/natafrica)\\n* [Nature China](http://www.naturechina.com)\\n* [Nature India](https://www.nature.com/nindia)\\n* [Nature Italy](https://www.nature.com/natitaly)\\n* [Nature Japan](https://www.natureasia.com/ja-jp)\\n* [Nature Middle East](https://www.nature.com/nmiddleeast)\\n\\n\\n\\n\\n\\n\\n* [Privacy\\n Policy](https://www.nature.com/info/privacy)\\n* [Use\\n of cookies](https://www.nature.com/info/cookies)\\n* Your privacy choices/Manage cookies\\n* [Legal\\n notice](https://www.nature.com/info/legal-notice)\\n* [Accessibility\\n statement](https://www.nature.com/info/accessibility-statement)\\n* [Terms & Conditions](https://www.nature.com/info/terms-and-conditions)\\n* [Your US state privacy rights](https://www.springernature.com/ccpa)\\n\\n\\n\\n\\n\\n[![Springer Nature](/static/images/logos/sn-logo-white-ea63208b81.svg)](https://www.springernature.com/)\\n© 2024 Springer Nature Limited\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nClose banner\\n\\n\\n\\n\\n\\n\\nClose\\n\\n\\n\\n\\n\\n![Nature Briefing](/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg)\\nSign up for the *Nature Briefing* newsletter — what matters in science, free to your inbox daily.\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nEmail address\\n\\n\\n\\nSign up\\n\\n\\n\\nI agree my information will be processed in accordance with the *Nature* and Springer Nature Limited [Privacy Policy](https://www.nature.com/info/privacy).\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nClose banner\\n\\n\\n\\n\\n\\n\\nClose\\n\\n\\n\\nGet the most important science stories of the day, free in your inbox.\\n[Sign up for Nature Briefing](https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner) \\n\\n\\n\\n\\n![](https://verify.nature.com/verify/nature.png)\\n\\n\\n![](/e8zlnwlx/article/s41586-022-05255-2)\\n\\n\"\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:prompt:ChatPromptTemplate] Entering Prompt run with input:\n",
      "\u001b[0m{\n",
      "  \"markdown_code\": \"\\nVisualizing translation dynamics at atomic detail inside a bacterial cell | Nature\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nYour privacy, your choice\\n-------------------------\\n\\n\\n\\n\\n\\nWe use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.\\n\\nBy accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.\\n\\nSee our [privacy policy](https://www.nature.com/info/privacy) for more information on the use of your personal data.\\n\\nManage preferences for further information and to change your choices.\\n\\n\\n\\n\\n\\nAccept all cookies\\n\\n\\n\\n\\n\\n\\n[Skip to main content](#content)\\n\\n\\nThank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain\\n the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in\\n Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles\\n and JavaScript.\\n\\n\\n\\n\\n\\n\\n\\n\\nAdvertisement\\n\\n\\n\\n\\n[![Advertisement](//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature.com/article&sz=728x90&c=-918719175&t=pos%3Dtop%26type%3Darticle%26artid%3Ds41586-022-05255-2%26doi%3D10.1038/s41586-022-05255-2%26techmeta%3D101,28%26subjmeta%3D1258,1260,1789,2536,326,337,41,45,535,574,631%26kwrd%3DBacterial+structural+biology,Cryoelectron+tomography,Ribosome)](//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature.com/article&sz=728x90&c=-918719175&t=pos%3Dtop%26type%3Darticle%26artid%3Ds41586-022-05255-2%26doi%3D10.1038/s41586-022-05255-2%26techmeta%3D101,28%26subjmeta%3D1258,1260,1789,2536,326,337,41,45,535,574,631%26kwrd%3DBacterial+structural+biology,Cryoelectron+tomography,Ribosome)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n[![Nature](https://media.springernature.com/full/nature-cms/uploads/product/nature/header-86f1267ea01eccd46b530284be10585e.svg)](/)\\n\\n* [View all journals](https://www.nature.com/siteindex)\\n* [Search](javascript:;)\\n\\nSearch\\n------\\n\\n\\n\\nSearch articles by subject, keyword or author\\n\\n\\n\\n\\n\\n\\nShow results from\\n\\nAll journals\\nThis journal\\n\\n\\n\\nSearch\\n\\n\\n\\n\\n\\n[Advanced search](/search/advanced) \\n\\n### Quick links\\n\\n\\n\\n\\t+ [Explore articles by subject](/subjects)\\n\\t+ [Find a job](/naturecareers)\\n\\t+ [Guide to authors](/authors/index.html)\\n\\t+ [Editorial policies](/authors/editorial_policies/)\\n* [Log in](https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41586-022-05255-2)\\n\\n\\n\\n\\n\\n\\n\\n\\n* [Explore content](javascript:;)\\n\\nExplore content\\n---------------\\n\\n\\n\\n\\t+ [Research articles](/nature/research-articles)\\n\\t+ [News](/news)\\n\\t+ [Opinion](/opinion)\\n\\t+ [Research Analysis](/research-analysis)\\n\\t+ [Careers](/careers)\\n\\t+ [Books & Culture](/books-culture)\\n\\t+ [Podcasts](/nature/podcasts)\\n\\t+ [Videos](/nature/videos)\\n\\t+ [Current issue](/nature/current-issue)\\n\\t+ [Browse issues](/nature/browse-issues)\\n\\t+ [Collections](/nature/collections)\\n\\t+ [Subjects](/nature/browse-subjects)\\n\\n\\t+ [Follow us on Facebook](https://www.facebook.com/Nature)\\n\\t+ [Follow us on Twitter](https://twitter.com/nature)\\n\\t+ [Sign up for alerts](https://www.nature.com/my-account/alerts/subscribe-journal?list-id=1)\\n\\t+ [RSS feed](https://www.nature.com/nature.rss)\\n* [About the journal](javascript:;)\\n\\nAbout the journal\\n-----------------\\n\\n\\n\\n\\t+ [Journal Staff](/nature/journal-staff)\\n\\t+ [About the Editors](/nature/editors)\\n\\t+ [Journal Information](/nature/journal-information)\\n\\t+ [Our publishing models](/nature/our-publishing-models)\\n\\t+ [Editorial Values Statement](/nature/editorial-values-statement)\\n\\t+ [Journal Metrics](/nature/journal-impact)\\n\\t+ [Awards](/nature/awards)\\n\\t+ [Contact](/nature/contact)\\n\\t+ [Editorial policies](/nature/editorial-policies)\\n\\t+ [History of Nature](/nature/history-of-nature)\\n\\t+ [Send a news tip](/nature/send-a-news-tip)\\n* [Publish with us](javascript:;)\\n\\nPublish with us\\n---------------\\n\\n\\n\\n\\t+ [For Authors](/nature/for-authors)\\n\\t+ [For Referees](/nature/for-referees)\\n\\t+ [Language editing services](https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022)\\n\\t+ [Submit manuscript](https://mts-nature.nature.com/)\\n\\n\\n* [Sign up for alerts](https://idp.nature.com/auth/personal/springernature?redirect_uri=https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D1%26journal-link%3Dhttps%253A%252F%252Fwww.nature.com%252Fnature%252F)\\n* [RSS feed](https://www.nature.com/nature.rss)\\n\\n\\n\\n\\n\\n\\n\\n\\n1. [nature](/)\\n2. [articles](/nature/articles?type=article)\\n3. article\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n Visualizing translation dynamics at atomic detail inside a bacterial cell\\n \\n\\n[Download PDF](/articles/s41586-022-05255-2.pdf)\\n\\n\\n\\n\\n\\n\\n\\n[Download PDF](/articles/s41586-022-05255-2.pdf)\\n\\n\\n\\n\\n\\n* Article\\n* [Open access](https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research)\\n* Published: 28 September 2022\\n\\n\\nVisualizing translation dynamics at atomic detail inside a bacterial cell\\n=========================================================================\\n\\n\\n* [Liang Xue](#auth-Liang-Xue-Aff1-Aff2) \\n [ORCID: orcid.org/0000-0003-4368-2526](http://orcid.org/0000-0003-4368-2526)[1](#Aff1),[2](#Aff2),\\n* [Swantje Lenz](#auth-Swantje-Lenz-Aff3) \\n [ORCID: orcid.org/0000-0002-8839-5371](http://orcid.org/0000-0002-8839-5371)[3](#Aff3),\\n* [Maria Zimmermann-Kogadeeva](#auth-Maria-Zimmermann_Kogadeeva-Aff1) \\n [ORCID: orcid.org/0000-0001-6021-1246](http://orcid.org/0000-0001-6021-1246)[1](#Aff1),\\n* [Dimitry Tegunov](#auth-Dimitry-Tegunov-Aff4)[4](#Aff4),\\n* [Patrick Cramer](#auth-Patrick-Cramer-Aff4) \\n [ORCID: orcid.org/0000-0001-5454-7755](http://orcid.org/0000-0001-5454-7755)[4](#Aff4),\\n* [Peer Bork](#auth-Peer-Bork-Aff1-Aff5-Aff6) \\n [ORCID: orcid.org/0000-0002-2627-833X](http://orcid.org/0000-0002-2627-833X)[1](#Aff1),[5](#Aff5),[6](#Aff6),\\n* [Juri Rappsilber](#auth-Juri-Rappsilber-Aff3-Aff7) \\n [ORCID: orcid.org/0000-0001-5999-1310](http://orcid.org/0000-0001-5999-1310)[3](#Aff3),[7](#Aff7) &\\n* …\\n* [Julia Mahamid](#auth-Julia-Mahamid-Aff1) \\n [ORCID: orcid.org/0000-0001-6968-041X](http://orcid.org/0000-0001-6968-041X)[1](#Aff1)\\n\\nShow authors\\n\\n[*Nature*](/)\\n**volume 610**, pages 205–211 (2022)[Cite this article](#citeas)\\n\\n\\n\\n\\n* 34k Accesses\\n* 42 Citations\\n* 268 Altmetric\\n* [Metrics details](/articles/s41586-022-05255-2/metrics)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nA [Publisher Correction](https://doi.org/10.1038/s41586-022-05455-w) to this article was published on 10 November 2022\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nThis article has been [updated](#change-history)\\n\\n\\n\\n\\n\\nAbstract\\n--------\\n\\nTranslation is the fundamental process of protein synthesis and is catalysed by the ribosome in all living cells[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"). Here we use advances in cryo-electron tomography and sub-tomogram analysis[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"),[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\") to visualize the structural dynamics of translation inside the bacterium *Mycoplasma pneumoniae*. To interpret the functional states in detail, we first obtain a high-resolution in-cell average map of all translating ribosomes and build an atomic model for the *M.* *pneumoniae* ribosome that reveals distinct extensions of ribosomal proteins. Classification then resolves 13 ribosome states that differ in their conformation and composition. These recapitulate major states that were previously resolved in vitro, and reflect intermediates during active translation. On the basis of these states, we animate translation elongation inside native cells and show how antibiotics reshape the cellular translation landscapes. During translation elongation, ribosomes often assemble in defined three-dimensional arrangements to form polysomes[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"). By mapping the intracellular organization of translating ribosomes, we show that their association into polysomes involves a local coordination mechanism that is mediated by the ribosomal protein L9. We propose that an extended conformation of L9 within polysomes mitigates collisions to facilitate translation fidelity. Our work thus demonstrates the feasibility of visualizing molecular processes at atomic detail inside cells.\\n\\n\\n\\n\\n\\n### Similar content being viewed by others\\n\\n\\n\\n\\n\\n![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-022-34997-w/MediaObjects/41467_2022_34997_Fig1_HTML.png)\\n\\n### [Structures of the eukaryotic ribosome and its translational states in situ](https://www.nature.com/articles/s41467-022-34997-w?fromPaywallRec=false)\\n\\n\\n\\nArticle\\nOpen access\\n02 December 2022\\n\\n\\n\\n\\n\\n\\n![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-019-1249-5/MediaObjects/41586_2019_1249_Fig1_HTML.png)\\n\\n### [Late steps in bacterial translation initiation visualized using time-resolved cryo-EM](https://www.nature.com/articles/s41586-019-1249-5?fromPaywallRec=false)\\n\\n\\n\\nArticle\\n20 May 2019\\n\\n\\n\\n\\n\\n\\n![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs12276-023-00933-1/MediaObjects/12276_2023_933_Fig1_HTML.png)\\n\\n### [Single-molecule visualization of mRNA circularization during translation](https://www.nature.com/articles/s12276-023-00933-1?fromPaywallRec=false)\\n\\n\\n\\nArticle\\nOpen access\\n31 January 2023\\n\\n\\n\\n\\n\\n\\n\\n\\nMain\\n----\\n\\nTranslation of genetic information through messenger RNAs (mRNAs) into proteins is performed by the ribosome, one of the primordial macromolecular machines in cells[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"). The ribosome consists of a small and a large subunit (30S and 50S in prokaryotes), which form the aminoacyl (A), peptidyl (P) and exit (E) transfer RNA (tRNA) binding sites at their interface. The translation process can be divided into four phases: initiation, elongation, termination and recycling[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"). During the elongation phase, the ribosome undergoes a fundamental cycle to add one amino acid to the nascent peptide chain, which can be subdivided into three steps: decoding, peptidyl transfer and translocation. These steps involve structural changes in the ribosome that include subunit rotations, elongation factor association and tRNA accommodation[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"). Many intermediates during the elongation cycle have been identified on the basis of structures derived by cryo-electron microscopy (cryo-EM) and computational image classification[6](#ref-CR6 \\\"Frank, J. & Agrawal, R. K. A ratchet-like inter-subunit reorganization of the ribosome during translocation. Nature 406, 318–322 (2000).\\\"),[7](#ref-CR7 \\\"Villa, E. et al. Ribosome-induced changes in elongation factor Tu conformation control GTP hydrolysis. Proc. Natl Acad. Sci. USA 106, 1063–1068 (2009).\\\"),[8](#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[9](#ref-CR9 \\\"Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. Nature 468, 713–716 (2010).\\\"),[10](#ref-CR10 \\\"Ramrath, D. J. et al. Visualization of two transfer RNAs trapped in transit during elongation factor G-mediated translocation. Proc. Natl Acad. Sci. USA 110, 20964–20969 (2013).\\\"),[11](#ref-CR11 \\\"Brilot, A. F., Korostelev, A. A., Ermolenko, D. N. & Grigorieff, N. Structure of the ribosome with elongation factor G trapped in the pretranslocation state. Proc. Natl Acad. Sci. USA 110, 20994–20999 (2013).\\\"),[12](#ref-CR12 \\\"Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. Cell 161, 845–857 (2015).\\\"),[13](#ref-CR13 \\\"Loveland, A. B., Demo, G. & Korostelev, A. A. Cryo-EM of elongating ribosome with EF-Tu*GTP elucidates tRNA proofreading. Nature 584, 640–645 (2020).\\\"),[14](#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[15](#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\"),[16](/articles/s41586-022-05255-2#ref-CR16 \\\"Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. Nat. Commun. 12, 7236 (2021).\\\"). Most available structures, however, are of ribosomes that were isolated from model bacteria such as *Escherichia coli* and *Thermus thermophilus*, and were often trapped in specific states by antibiotics, GTP analogues or mutations[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"),[17](/articles/s41586-022-05255-2#ref-CR17 \\\"Brown, A. & Shao, S. Ribosomes and cryo-EM: a duet. Curr. Opin. Struct. Biol. 52, 1–7 (2018).\\\"). Thus, a detailed structural description of the translation process within the native cellular context is lacking. Although actively translating ribosomes have been visualized inside cells by cryo-electron tomography (cryo-ET), the maps generated in previous studies were limited to resolutions on the nanometre scale[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"),[18](#ref-CR18 \\\"Kuhner, S. et al. Proteome organization in a genome-reduced bacterium. Science 326, 1235–1240 (2009).\\\"),[19](#ref-CR19 \\\"Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. Mol. Cell 39, 560–569 (2010).\\\"),[20](#ref-CR20 \\\"Mahamid, J. et al. Visualizing the molecular sociology at the HeLa cell nuclear periphery. Science 351, 969–972 (2016).\\\"),[21](#ref-CR21 \\\"Schaffer, M. et al. A cryo-FIB lift-out technique enables molecular-resolution cryo-ET within native Caenorhabditis elegans tissue. Nat. Methods 16, 757–762 (2019).\\\"),[22](/articles/s41586-022-05255-2#ref-CR22 \\\"Waltz, F. et al. How to build a ribosome from RNA fragments in Chlamydomonas mitochondria. Nat. Commun. 12, 7176 (2021).\\\"). We have recently developed image-processing algorithms for cryo-ET that make it possible to resolve stalled ribosomes to residue level inside the genome-reduced bacterium *M.* *pneumoniae*[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\"). Here, we use these technical advances to perform large-scale structure classification and spatial analysis of ribosomes, to visualize parts of the translation process in great detail inside *M.* *pneumoniae* as a prokaryotic minimal cell model[18](/articles/s41586-022-05255-2#ref-CR18 \\\"Kuhner, S. et al. Proteome organization in a genome-reduced bacterium. Science 326, 1235–1240 (2009).\\\").\\n\\nIn-cell structure of the *M. pneumoniae* ribosome\\n-------------------------------------------------\\n\\nTo investigate the structural details of the translation machinery, we first obtained 3.5-Å consensus maps by averaging all ribosomes detected in cryo-electron tomograms of intact *M.* *pneumoniae* cells (Fig. [1a,b](/articles/s41586-022-05255-2#Fig1), Extended Data Fig. [1](/articles/s41586-022-05255-2#Fig5), Supplementary Table [1](/articles/s41586-022-05255-2#MOESM1), [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1) and [Methods](/articles/s41586-022-05255-2#Sec7)). Focused refinements on the 30S and 50S subunits improved the map quality and revealed well-resolved ribosomal RNA (rRNA) bases and ribosomal protein amino acid side chains (Fig. [1b](/articles/s41586-022-05255-2#Fig1) and Extended Data Fig. [2a–c](/articles/s41586-022-05255-2#Fig6)). The high-resolution in-cell consensus maps allowed us to build de novo an atomic model for the *M.* *pneumoniae* ribosome (Extended Data Fig. [2d–g](/articles/s41586-022-05255-2#Fig6) and [Methods](/articles/s41586-022-05255-2#Sec7)). The structure shows high similarity to other bacterial ribosomes, but also reveals several new features (Fig. [1c](/articles/s41586-022-05255-2#Fig1) and Extended Data Fig. [2d–g](/articles/s41586-022-05255-2#Fig6)). Specifically, 11 of the 52 ribosomal proteins in *M.* *pneumoniae* have extended sequences compared to *E.* *coli* (Extended Data Fig. [3a](/articles/s41586-022-05255-2#Fig7) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). Most of these extensions are predicted to be disordered, but those of ribosomal proteins S6, L22 and L29 form secondary structures and were built in the model (Fig. [1c](/articles/s41586-022-05255-2#Fig1) and Extended Data Figs. [2d–e](/articles/s41586-022-05255-2#Fig6) and [3a](/articles/s41586-022-05255-2#Fig7)). We found such extensions to be common throughout the bacterial kingdom (Extended Data Fig. [3b](/articles/s41586-022-05255-2#Fig7), [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1) and Supplementary Table [2](/articles/s41586-022-05255-2#MOESM1)), possibly representing ribosome diversity in adaptation to different environments and lifestyles[23](/articles/s41586-022-05255-2#ref-CR23 \\\"Melnikov, S., Manakongtreecheep, K. & Soll, D. Revising the structural diversity of ribosomal proteins across the three domains of life. Mol. Biol. Evol. 35, 1588–1598 (2018).\\\"). Although the functions of these extensions remain largely elusive, their disruption affects cellular fitness or survival in *M.*  *pneumoniae*[24](/articles/s41586-022-05255-2#ref-CR24 \\\"Miravet-Verde, S., Burgos, R., Delgado, J., Lluch-Senar, M. & Serrano, L. FASTQINS and ANUBIS: two bioinformatic tools to explore facts and artifacts in transposon sequencing and essentiality studies. Nucleic Acids Res. 48, e102 (2020).\\\") ([Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). Thus, the high-resolution ribosome map and the atomic model derived from intact *M.* *pneumoniae* provide the basis to investigate in detail the conformational and compositional changes of ribosomes during translation inside cells.\\n\\n\\n\\n**Fig. 1: Ribosome structure in *M.* *pneumoniae* cells.**\\n\\n[![figure 1](//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_Fig1_HTML.png)](/articles/s41586-022-05255-2/figures/1)**a**, A denoised tomographic slice of a *M.* *pneumoniae* cell. AO, attachment organelle; PM, plasma membrane. Examples of ribosomes are circled. The representative tomogram (selected from 15) was acquired with a Volta phase plate for better visualization of the cellular morphology. Similar imaging conditions, excluding the use of a phase plate, were used for the acquisition of all tomograms for subsequent analysis. Scale bar, 100 nm. **b**, A 3.5-Å in-cell ribosome map (left) shows well-resolved rRNA bases and ribosomal protein side chains (right). **c**, An atomic model of the *M.* *pneumoniae* ribosome shows structural similarity to other bacterial ribosomes. Eleven ribosomal proteins (dark green) have sequence extensions (magenta).\\n\\n[Full size image](/articles/s41586-022-05255-2/figures/1)Structural dynamics of translation in cells\\n-------------------------------------------\\n\\nTo analyse the structural changes that are associated with the translation process inside cells, we performed computational classification of 101,696 ribosomes from 356 tomograms of native cells, and obtained 13 distinct ribosome classes (Extended Data Fig. [4](/articles/s41586-022-05255-2#Fig8), Supplementary Table [3](/articles/s41586-022-05255-2#MOESM1) and [Methods](/articles/s41586-022-05255-2#Sec7)). Ten classes determined at resolutions ranging from 4 to 10 Å were assigned to the translation elongation phase on the basis of elongation factor and tRNAs binding to the ribosome (Fig. [2a](/articles/s41586-022-05255-2#Fig2) and Extended Data Figs. [4](/articles/s41586-022-05255-2#Fig8)–[7](/articles/s41586-022-05255-2#Fig11)). The remaining three classes represent 70S with a single P/E-site tRNA, 50S in complex with the ribosome recycling factor, and free 50S subunits (Extended Data Figs. [4](/articles/s41586-022-05255-2#Fig8)–[7](/articles/s41586-022-05255-2#Fig11) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). The ten classes within the elongation phase account for 70% of the detected ribosomes, consistent with the expectation that most ribosomes inside living cells are engaged in the elongation phase, which lasts considerably longer than the initiation, termination and recycling phases[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"),[25](/articles/s41586-022-05255-2#ref-CR25 \\\"Borg, A., Pavlov, M. & Ehrenberg, M. Complete kinetic mechanism for recycling of the bacterial ribosome. RNA 22, 10–21 (2016).\\\").\\n\\n\\n\\n**Fig. 2: Ribosome classification reconstructs the translation elongation cycle.**\\n\\n[![figure 2](//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_Fig2_HTML.png)](/articles/s41586-022-05255-2/figures/2)**a**, Ten 70S structures determined inside *M.* *pneumoniae* cells represent translation elongation intermediate states, which are characterized by the binding of different elongation factors (EFs) and tRNAs. The frequencies of occurrence for different intermediates are calculated from the classified particle numbers (Extended Data Fig. [4](/articles/s41586-022-05255-2#Fig8)). **b**, Densities for mRNA, tRNAs and the nascent peptide chain are well-resolved in the most-populated 'A, P' state. **c**, Trajectories of the elongation factors EF-Tu and EF-G and the A-, P- and E-site tRNAs along translation elongation. **d**, Major conformational changes of the ribosome along the elongation trajectory: 30S body rotation, 30S head swivel and L1 stalk opening.\\n\\n[Full size image](/articles/s41586-022-05255-2/figures/2)The identified elongation classes can be ordered to reconstruct the translation elongation cycle[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"). By flexible fitting of our *M.* *pneumoniae* ribosome atomic model into the classified maps ([Methods](/articles/s41586-022-05255-2#Sec7) and Supplementary Table [3](/articles/s41586-022-05255-2#MOESM1)), we obtained pseudo-atomic models that delineate conformational and compositional changes of the ribosome complex, including rotations of 30S body and head, coordination of elongation factors, movement of tRNAs through the A–P–E sites and dynamics of the L1 stalk (Fig. [2](/articles/s41586-022-05255-2#Fig2), Extended Data Fig. [7a–j](/articles/s41586-022-05255-2#Fig11) and Supplementary Videos [1](/articles/s41586-022-05255-2#MOESM3) and [2](/articles/s41586-022-05255-2#MOESM4)). The number of ribosomes within each of the classes provides the relative abundance of translation elongation intermediates, which together reflect a steady state distribution determined by their relative rates of formation and depletion inside the cells (Fig. [2a](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8)). The classic, non-rotated 'A, P' state was the most populated and determined at the highest resolution, showing clear density for mRNA, tRNAs and the nascent peptide (Fig. [2b](/articles/s41586-022-05255-2#Fig2)). In the following states, we observed the oscillation of A- and P-site tRNAs into the hybrid A/P and P/E state, coupled with 30S subunit rotations and L1 stalk movement (Fig. [2a,c,d](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [7a–j](/articles/s41586-022-05255-2#Fig11)), consistent with in vitro studies[6](/articles/s41586-022-05255-2#ref-CR6 \\\"Frank, J. & Agrawal, R. K. A ratchet-like inter-subunit reorganization of the ribosome during translocation. Nature 406, 318–322 (2000).\\\"),[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[9](/articles/s41586-022-05255-2#ref-CR9 \\\"Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. Nature 468, 713–716 (2010).\\\"),[12](/articles/s41586-022-05255-2#ref-CR12 \\\"Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. Cell 161, 845–857 (2015).\\\"),[26](#ref-CR26 \\\"Munro, J. B., Altman, R. B., O’Connor, N. & Blanchard, S. C. Identification of two distinct hybrid state intermediates on the ribosome. Mol. Cell 25, 505–517 (2007).\\\"),[27](#ref-CR27 \\\"Belardinelli, R. et al. Choreography of molecular movements during ribosome progression along mRNA. Nat. Struct. Mol. Biol. 23, 342–348 (2016).\\\"),[28](#ref-CR28 \\\"Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. Nat. Struct. Mol. Biol. 23, 333–341 (2016).\\\"),[29](#ref-CR29 \\\"Fei, J., Kosuri, P., MacDougall, D. D. & Gonzalez, R. L. Jr Coupling of ribosomal L1 stalk and tRNA dynamics during translation elongation. Mol. Cell 30, 348–359 (2008).\\\"),[30](/articles/s41586-022-05255-2#ref-CR30 \\\"Zhou, J., Lancaster, L., Donohue, J. P. & Noller, H. F. How the ribosome hands the A-site tRNA to the P site during EF-G-catalyzed translocation. Science 345, 1188–1191 (2014).\\\"). Along this trajectory, we also determined a partial hybrid 'A\\\\*, P/E' state with rotated 30S and deacyl-tRNA in the hybrid P/E site, but with only marginally relocated peptidyl-tRNA in the A site (Fig. [2a,d](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [8a](/articles/s41586-022-05255-2#Fig12)), similar to a previously reported pre-translocational H2\\\\* state[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[26](/articles/s41586-022-05255-2#ref-CR26 \\\"Munro, J. B., Altman, R. B., O’Connor, N. & Blanchard, S. C. Identification of two distinct hybrid state intermediates on the ribosome. Mol. Cell 25, 505–517 (2007).\\\"). When viewed in the context of this trajectory, the high enrichment of the 'A, P' state could suggest that inter-subunit rotation represents a rate-limiting step for translation elongation in *M.* *pneumoniae*. However, in view of the entirety of the elongation cycle, processive inter-subunit rotation leading to efficient translocation requires the binding of the elongation factor EF-G. Notably, EF-G was found to bind to the ribosome in either the partial hybrid or the full hybrid states (Extended Data Fig. [8b–m](/articles/s41586-022-05255-2#Fig12)). EF-G in the partial hybrid state (class 6e) is less extended and its domain IV does not overlap with the A site compared to the full hybrid state (Extended Data Fig. [8b–h](/articles/s41586-022-05255-2#Fig12)). This EF-G-bound partial hybrid state therefore resembles early translocation intermediates before phosphate release[9](/articles/s41586-022-05255-2#ref-CR9 \\\"Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. Nature 468, 713–716 (2010).\\\"),[14](#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[15](#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\"),[16](/articles/s41586-022-05255-2#ref-CR16 \\\"Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. Nat. Commun. 12, 7236 (2021).\\\"). In the following full hybrid state (class 7), domain IV of EF-G extends about 20 Å towards the A site, owing to both the rotation of entire EF-G and its inter-domain rearrangement (Fig. [2c](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [8f](/articles/s41586-022-05255-2#Fig12)). Class 8 next shows a reverse rotation of the 30S body, the largest magnitude for 30S head swivel, the fully extended EF-G and the chimeric 'ap/P, pe/E' tRNAs (Fig. [2c,d](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [8g, j](/articles/s41586-022-05255-2#Fig12)), indicating that it is a late translocation intermediate[9](/articles/s41586-022-05255-2#ref-CR9 \\\"Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. Nature 468, 713–716 (2010).\\\"),[10](/articles/s41586-022-05255-2#ref-CR10 \\\"Ramrath, D. J. et al. Visualization of two transfer RNAs trapped in transit during elongation factor G-mediated translocation. Proc. Natl Acad. Sci. USA 110, 20964–20969 (2013).\\\"),[14](#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[15](#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\"),[16](/articles/s41586-022-05255-2#ref-CR16 \\\"Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. Nat. Commun. 12, 7236 (2021).\\\"),[28](/articles/s41586-022-05255-2#ref-CR28 \\\"Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. Nat. Struct. Mol. Biol. 23, 333–341 (2016).\\\"). We thus demonstrate the existence of continual intermediates from early to late translocation stages during active translation elongation inside cells. Two EF-Tu-associated structures were also determined, with and without E-site tRNA (Fig. [2a](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [7c,d](/articles/s41586-022-05255-2#Fig11)). This suggests that the binding of EF-Tu•tRNA to the ribosome and the disassociation of E-site tRNA are independent of each other. The 'P, E' class has a relatively low abundance, which suggests that the E-site tRNA is not stable and tends to disassociate quickly after translocation. This is in agreement with previous single-molecule fluorescence studies showing rapid release of the E-site tRNA[31](/articles/s41586-022-05255-2#ref-CR31 \\\"Uemura, S. et al. Real-time tRNA transit on single translating ribosomes at codon resolution. Nature 464, 1012–U1073 (2010).\\\"),[32](/articles/s41586-022-05255-2#ref-CR32 \\\"Chen, C. et al. Single-molecule fluorescence measurements of ribosomal translocation dynamics. Mol. Cell 42, 367–377 (2011).\\\"), and helps to resolve a long-standing controversy over its disassociation time point[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[12](/articles/s41586-022-05255-2#ref-CR12 \\\"Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. Cell 161, 845–857 (2015).\\\"),[33](/articles/s41586-022-05255-2#ref-CR33 \\\"Wilson, D. N. & Nierhaus, K. H. The E-site story: the importance of maintaining two tRNAs on the ribosome during protein synthesis. Cell. Mol. Life Sci. 63, 2725–2737 (2006).\\\"). In summary, these results recapitulate major steps in translation elongation that have been defined by controlled in vitro studies and reconstruct the structural dynamics of the elongation cycles inside cells, highlighting the possible various paths of the reaction catalysed by the ribosome complex. Furthermore, with these results, a probability map for the occurrence of intermediate states can be derived to illustrate a putative energy landscape of translation elongation in cells. Energy landscapes of translation are suggested to be rugged[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[12](/articles/s41586-022-05255-2#ref-CR12 \\\"Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. Cell 161, 845–857 (2015).\\\"),[34](/articles/s41586-022-05255-2#ref-CR34 \\\"Munro, J. B., Sanbonmatsu, K. Y., Spahn, C. M. & Blanchard, S. C. Navigating the ribosome’s metastable energy landscape. Trends Biochem. Sci 34, 390–400 (2009).\\\"),[35](/articles/s41586-022-05255-2#ref-CR35 \\\"Bock, L. V. et al. Energy barriers and driving forces in tRNA translocation through the ribosome. Nat. Struct. Mol. Biol. 20, 1390–1396 (2013).\\\"), depending on factors such as Mg2+ concentrations[36](/articles/s41586-022-05255-2#ref-CR36 \\\"Julian, P. et al. Structure of ratcheted ribosomes with tRNAs in hybrid states. Proc. Natl Acad. Sci. USA 105, 16924–16927 (2008).\\\"),[37](/articles/s41586-022-05255-2#ref-CR37 \\\"Agirrezabala, X. et al. Visualization of the hybrid state of tRNA binding promoted by spontaneous ratcheting of the ribosome. Mol. Cell 32, 190–197 (2008).\\\"), the presence of EF-G[34](/articles/s41586-022-05255-2#ref-CR34 \\\"Munro, J. B., Sanbonmatsu, K. Y., Spahn, C. M. & Blanchard, S. C. Navigating the ribosome’s metastable energy landscape. Trends Biochem. Sci 34, 390–400 (2009).\\\"),[38](/articles/s41586-022-05255-2#ref-CR38 \\\"Rodnina, M. V., Peske, F., Peng, B. Z., Belardinelli, R. & Wintermeyer, W. Converting GTP hydrolysis into motion: versatile translational elongation factor G. Biol. Chem. 401, 131–142 (2019).\\\") and temperature[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"). In *M.* *pneumoniae*, the intracellular concentrations of ribosomes, translation elongation factors and substrates can be up to ten times lower than those in *E.* *coli* ([Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). It is therefore possible that the energy landscapes of translation vary for different organisms, cell types and conditions. Applying the analysis introduced here to other cell types and conditions can deepen our understanding of the translation process within the native cellular context.\\n\\nAntibiotics alter translation landscapes\\n----------------------------------------\\n\\nThe ribosome is one of the most important targets for antibiotics, many of which are known to stall translation by stabilizing certain intermediates in the process[39](/articles/s41586-022-05255-2#ref-CR39 \\\"Lin, J., Zhou, D., Steitz, T. A., Polikanov, Y. S. & Gagnon, M. G. Ribosome-targeting antibiotics: modes of action, mechanisms of resistance, and implications for drug design. Annu. Rev. Biochem. 87, 451–478 (2018).\\\"). To investigate the effects of antibiotics on the cellular translation machinery, we analysed ribosomes in cells that were treated for a short time (15–20 min; [Methods](/articles/s41586-022-05255-2#Sec7)) with two representative ribosome-targeting antibiotics: chloramphenicol (Cm), which binds to the peptidyl transfer centre in the 50S subunit and inhibits peptide bond formation[40](/articles/s41586-022-05255-2#ref-CR40 \\\"Bulkley, D., Innis, C. A., Blaha, G. & Steitz, T. A. Revisiting the structures of several antibiotics bound to the bacterial ribosome. Proc. Natl Acad. Sci. USA 107, 17158–17163 (2010).\\\"),[41](/articles/s41586-022-05255-2#ref-CR41 \\\"Svetlov, M. S. et al. High-resolution crystal structures of ribosome-bound chloramphenicol and erythromycin provide the ultimate basis for their competition. RNA 25, 600–606 (2019).\\\"); and spectinomycin (Spc), which binds to the 30S neck and blocks translocation[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[28](/articles/s41586-022-05255-2#ref-CR28 \\\"Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. Nat. Struct. Mol. Biol. 23, 333–341 (2016).\\\"),[42](/articles/s41586-022-05255-2#ref-CR42 \\\"Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. ACS Chem. Biol. 2, 545–552 (2007).\\\"). Overall, our sub-tomogram analysis of antibiotic-treated cells results in 17 ribosome maps, and shows that translation landscapes are markedly reshaped by different antibiotics (Fig. [3a](/articles/s41586-022-05255-2#Fig3), Extended Data Figs. [9](/articles/s41586-022-05255-2#Fig13)–[11](/articles/s41586-022-05255-2#Fig15), Supplementary Tables [4](/articles/s41586-022-05255-2#MOESM1)–[6](/articles/s41586-022-05255-2#MOESM1) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)).\\n\\n\\n\\n**Fig. 3: Antibiotics induce distinct translation elongation landscapes in cells.**\\n\\n[![figure 3](//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_Fig3_HTML.png)](/articles/s41586-022-05255-2/figures/3)**a**, Distribution of translation elongation intermediates in native untreated cells, and in cells treated with three different antibiotics. Bar and whiskers indicate mean and s.d. for each class across all cells in the different treatment groups: untreated (*n* = 356 cells); +Cm, chloramphenicol-treated (*n* = 65 cells); +Spc, spectinomycin-treated (*n* = 70 cells); and +PUM, pseudouridimycin-treated (*n* = 86 cells). **b**, Ribosomes in Spc-treated cells are largely stalled in the 'EF-G, A/PSpc, P/E' state. **c**, The Spc molecule (magenta) is well-resolved and built in the 'EF-G, A/PSpc, P/E' ribosome model. It is surrounded by several 16S rRNA bases and loop 2 of ribosomal protein S5 near the 30S neck. **d**, The major state in Spc-treated cells is similar to the 'EF-G, A/PPUM, P/E' class in PUM-treated cells (light blue) and the 'EF-G, A/P, P/E' class in untreated cells (light grey), differing only in the position of the A/P-site tRNA on the 50S side. **e**, Single-cell clustering analysis on the basis of the translation elongation states of 577 individual cells under native and different antibiotic treatment conditions.\\n\\n[Full size image](/articles/s41586-022-05255-2/figures/3)In Cm-treated cells, 72% of 70S ribosomes are trapped in the 'A, P' state (class 4), owing to the inhibition of peptidyl transfer[39](/articles/s41586-022-05255-2#ref-CR39 \\\"Lin, J., Zhou, D., Steitz, T. A., Polikanov, Y. S. & Gagnon, M. G. Ribosome-targeting antibiotics: modes of action, mechanisms of resistance, and implications for drug design. Annu. Rev. Biochem. 87, 451–478 (2018).\\\") (Fig. [3a–d](/articles/s41586-022-05255-2#Fig3), Extended Data Fig. [9](/articles/s41586-022-05255-2#Fig13) and Supplementary Table [4](/articles/s41586-022-05255-2#MOESM1)). The Cm molecule was resolved in its canonical binding site[40](/articles/s41586-022-05255-2#ref-CR40 \\\"Bulkley, D., Innis, C. A., Blaha, G. & Steitz, T. A. Revisiting the structures of several antibiotics bound to the bacterial ribosome. Proc. Natl Acad. Sci. USA 107, 17158–17163 (2010).\\\"),[41](/articles/s41586-022-05255-2#ref-CR41 \\\"Svetlov, M. S. et al. High-resolution crystal structures of ribosome-bound chloramphenicol and erythromycin provide the ultimate basis for their competition. RNA 25, 600–606 (2019).\\\") (Extended Data Fig. [9d,g](/articles/s41586-022-05255-2#Fig13)). Of note, 28% of ribosomes were found in several other elongation states, either before or following the major 'A, P' state (Fig. [3a](/articles/s41586-022-05255-2#Fig3) and Extended Data Fig. [9](/articles/s41586-022-05255-2#Fig13)). The existence of an EF-Tu•tRNA-bound state (class 3) is reminiscent of the effects of other antibiotics that inhibit peptidyl transfer in a similar manner[43](#ref-CR43 \\\"Polikanov, Y. S. et al. Distinct tRNA accommodation intermediates observed on the ribosome with the antibiotics hygromycin A and A201A. Mol. Cell 58, 832–844 (2015).\\\"),[44](#ref-CR44 \\\"Morse, J. C. et al. Elongation factor-Tu can repetitively engage aminoacyl-tRNA within the ribosome during the proofreading stage of tRNA selection. Proc. Natl Acad. Sci. USA 117, 3610–3620 (2020).\\\"),[45](/articles/s41586-022-05255-2#ref-CR45 \\\"Choi, J. et al. Dynamics of the context-specific translation arrest by chloramphenicol and linezolid. Nat. Chem. Biol. 16, 310–317 (2020).\\\").\\n\\nSpc is suggested to inhibit translocation by blocking head swivel of the 30S subunit[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[28](/articles/s41586-022-05255-2#ref-CR28 \\\"Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. Nat. Struct. Mol. Biol. 23, 333–341 (2016).\\\"),[42](/articles/s41586-022-05255-2#ref-CR42 \\\"Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. ACS Chem. Biol. 2, 545–552 (2007).\\\"). We found that 67.5% of 70S ribosomes were stalled in the 'EF-G, A/PSpc, P/E' state in Spc-treated cells, a state similar to the pre-translocational 'EF-G, A/P, P/E' state (class 7) in untreated cells (Fig. [3a–d](/articles/s41586-022-05255-2#Fig3), Extended Data Fig. [10](/articles/s41586-022-05255-2#Fig14) and Supplementary Table [5](/articles/s41586-022-05255-2#MOESM1)). The Spc molecule density is clearly visible near helix 34 within the 30S neck region (Fig. [3b,c](/articles/s41586-022-05255-2#Fig3) and Extended Data Fig. [10d,g](/articles/s41586-022-05255-2#Fig14)), consistent with its reported binding site[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[42](/articles/s41586-022-05255-2#ref-CR42 \\\"Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. ACS Chem. Biol. 2, 545–552 (2007).\\\"). These results show that Spc inhibits translocation by impeding the dynamics of the 30S subunit. Similar to Cm, ribosomes in three additional elongation states could be detected with lower frequencies (Fig. [3a](/articles/s41586-022-05255-2#Fig3) and Extended Data Fig. [10e,f](/articles/s41586-022-05255-2#Fig14)).\\n\\nPerturbation of other molecular pathways that are functionally coupled to translation in bacterial cells also affects the translation landscape; RNA polymerase stalled by pseudouridimycin (PUM) can physically block mRNA translocation in the ribosome that collides with it during transcription–translation coupling[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). Consistently, 59.5% of 70S ribosomes in PUM-treated cells were found in the 'EF-G, A/PPUM, P/E' state (Fig. [3a](/articles/s41586-022-05255-2#Fig3), Extended Data Fig. [11](/articles/s41586-022-05255-2#Fig15) and Supplementary Table [6](/articles/s41586-022-05255-2#MOESM1)), which resembles the pre-translocational 'EF-G, A/P, P/E' state in untreated cells and the stalled 'EF-G, A/PSpc, P/E' state in Spc-treated cells (Fig. [3d](/articles/s41586-022-05255-2#Fig3)). Our finding that physically obstructing mRNA translocation by a PUM-stalled RNA polymerase and chemically impeding 30S head dynamics by Spc lead to similar structures further confirms that mRNA translocation and 30S rotations are directly coupled.\\n\\nThe observation that treatment with antibiotics resulted in minor states that are not expected from their specific binding prompted us to investigate possible cell-to-cell variability in response to the antibiotics. To this end, we performed clustering analysis on the basis of translation elongation profiles in 577 tomograms of single cells under native and antibiotic treatment conditions. The analysis resulted in four major clusters in accordance with the four treatment groups, demonstrating small cell-to-cell variability within each cluster (Fig. [3e](/articles/s41586-022-05255-2#Fig3)). Thus, our results show that the translation landscapes in cells are globally reshaped by small molecules specifically binding to ribosomes, as well as to other targets ([Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1) and Supplementary Table [8](/articles/s41586-022-05255-2#MOESM1)). The presence of minor elongation states under antibiotic treatment is reminiscent of previous studies that have shown ongoing slow translation in antibiotic-treated cells and context-dependent inhibition[46](#ref-CR46 \\\"Marks, J. et al. Context-specific inhibition of translation by ribosomal antibiotics targeting the peptidyl transferase center. Proc. Natl Acad. Sci. USA 113, 12150–12155 (2016).\\\"),[47](#ref-CR47 \\\"Vazquez-Laslop, N. & Mankin, A. S. Context-specific action of ribosomal antibiotics. Annu. Rev. Microbiol. 72, 185–207 (2018).\\\"),[48](/articles/s41586-022-05255-2#ref-CR48 \\\"Aguirre Rivera, J. et al. Real-time measurements of aminoglycoside effects on protein synthesis in live cells. Proc. Natl Acad. Sci. USA 118, e2013315118 (2021).\\\"). For example, Cm inhibition is affected by specific residues of the nascent peptide[46](/articles/s41586-022-05255-2#ref-CR46 \\\"Marks, J. et al. Context-specific inhibition of translation by ribosomal antibiotics targeting the peptidyl transferase center. Proc. Natl Acad. Sci. USA 113, 12150–12155 (2016).\\\"),[47](/articles/s41586-022-05255-2#ref-CR47 \\\"Vazquez-Laslop, N. & Mankin, A. S. Context-specific action of ribosomal antibiotics. Annu. Rev. Microbiol. 72, 185–207 (2018).\\\"). Most antibiotics, including Cm and Spc, are known to inhibit cell growth but do not immediately kill the cell[49](/articles/s41586-022-05255-2#ref-CR49 \\\"Baquero, F. & Levin, B. R. Proximate and ultimate causes of the bactericidal action of antibiotics. Nat. Rev. Microbiol. 19, 123–132 (2021).\\\"). It is possible that the reshaped translation landscapes by antibiotics lead to an imbalance in protein synthesis, which in the long run has detrimental consequences for the cell.\\n\\nSpatial and functional organization of translation\\n--------------------------------------------------\\n\\nFinally, we investigated the spatial organization of active translation in native *M.* *pneumoniae* cells (Fig. [4a](/articles/s41586-022-05255-2#Fig4) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). It is known that ribosomes translating on the same mRNA can assemble closely in space to form polysomes[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"),[19](/articles/s41586-022-05255-2#ref-CR19 \\\"Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. Mol. Cell 39, 560–569 (2010).\\\"). We defined polysomes using a distance cut-off of 7 nm from the mRNA exit site to the mRNA entry site between neighbouring ribosomes (Extended Data Fig. [12a–e](/articles/s41586-022-05255-2#Fig16), [Methods](/articles/s41586-022-05255-2#Sec7) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). The detected polysomes account for 26.2% of all 70S ribosomes, and two arrangement patterns between neighbouring ribosomes can be defined (Fig. [4b–e](/articles/s41586-022-05255-2#Fig4) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)): the so-called 'top-top' (t-t; 78.5%, mRNA exit-to-entry distance 4.2 ± 1.4 nm) and 'top-back' (t-b; 21.5%, 5.4 ± 1.5 nm) configurations[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"). We also observed various topologies for long polysomes, ranging from loose assembly to tight packing with helix-like configurations[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"),[19](/articles/s41586-022-05255-2#ref-CR19 \\\"Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. Mol. Cell 39, 560–569 (2010).\\\"),[50](/articles/s41586-022-05255-2#ref-CR50 \\\"Myasnikov, A. G. et al. The molecular structure of the left-handed supra-molecular helix of eukaryotic polyribosomes. Nat. Commun. 5, 5294 (2014).\\\") (Fig. [4c,d](/articles/s41586-022-05255-2#Fig4)).\\n\\n\\n\\n**Fig. 4: Spatial and functional organization of ribosomes in native cells.**\\n\\n[![figure 4](//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_Fig4_HTML.png)](/articles/s41586-022-05255-2/figures/4)**a**, Three-dimensional map of ribosomes in a representative native untreated cell (selected from 356). Top, *x*–*y* view; bottom, orthogonal view. The 70S intermediates within the translation elongation cycle (classes 1–8, as in Fig. [2a](/articles/s41586-022-05255-2#Fig2)), and additional classes (classes 11–13 and 50–51 as detailed in Extended Data Fig. [4](/articles/s41586-022-05255-2#Fig8)) are coloured as indicated in the colour scheme (inset). 50S: light grey. **b**, 'top-back' (t-b) and 'top-top' (t-t) assembly patterns of adjacent ribosomes in polysomes. **c**,**d**, Representative long polysomes of loose (**c**) and tight (**d**) topologies, with the corresponding putative mRNA paths and nascent chain vectors shown underneath (not drawn to scale). **e**, Distribution of polysome lengths. **f**, Distribution of elongation states in polysomes compared to all ribosomes and mono-ribosomes. Bars and whiskers are mean and s.d. across 356 tomograms of untreated cells (*n* = 356 cells). Highlighted are states for which the fraction in polysomes differs by more than 50% compared to all ribosomes or mono-ribosomes. Asterisks indicate false discovery rate (FDR)-adjusted *P* value (*P*FDR) < 0.01 (two-sided Wilcoxon rank sum test). *P*FDR values for class 5: 6.44 × 10−23 (polysome versus all) and 2.76 × 10−39 (polysome versus monosome); class 6a: 3.06 × 10−28 and 1.44 × 10−34, respectively. **g**, Occurrence frequencies of elongation state pairs of adjacent ribosomes in polysomes normalized to the theoretical probability of random pairs. States that require elongation factor binding to proceed are 1, 2a, 5 and 6a (in bold). **h**, Map of a di-ribosome within polysomes shows the intervening mRNA density (inset: blue arrowhead) and the extended L9 of the preceding ribosome (i). The C-terminal domain of extended L9 can interfere with elongation factor (EF) binding to the following ribosome (i+1).\\n\\n[Full size image](/articles/s41586-022-05255-2/figures/4)Whether translation elongation is synchronized or coordinated within polysomes is a long-standing question[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"),[19](/articles/s41586-022-05255-2#ref-CR19 \\\"Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. Mol. Cell 39, 560–569 (2010).\\\"),[50](/articles/s41586-022-05255-2#ref-CR50 \\\"Myasnikov, A. G. et al. The molecular structure of the left-handed supra-molecular helix of eukaryotic polyribosomes. Nat. Commun. 5, 5294 (2014).\\\"). To address this, we first examined whether the distribution of elongation states differs between the total ribosome population and polysomes. We found that although most states occurred equally frequently in both populations, the fractions of two states before EF-G binding (class 5, 'A\\\\*, P/E' and class 6a, 'A/P, P/E') are more frequent in polysomes (Fig. [4f](/articles/s41586-022-05255-2#Fig4)). We next calculated the frequencies of state pairs between two adjacent ribosomes (preceding versus following) and compared them to theoretical pair frequencies calculated from the bulk distribution (Extended Data Fig. [13a–c](/articles/s41586-022-05255-2#Fig17)). This comparison revealed that the occurrence of states among preceding and following ribosomes is not symmetric: following ribosomes more frequently populate states that require elongation factor binding to proceed to the next state (classes 1, 2a, 5 and 6a; Fig. [4g](/articles/s41586-022-05255-2#Fig4)). We statistically validated such differences with a permutation test, and further show that the asymmetry between preceding and following ribosomes increases as the distance threshold used to define polysomes decreases (Extended Data Fig. [13d–h](/articles/s41586-022-05255-2#Fig17), [Methods](/articles/s41586-022-05255-2#Sec7) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). This suggests that local coordination of translation elongation between adjacent ribosomes within the polysome is likely to be achieved by obstructing elongation factor binding that is required for the following ribosome to proceed to the next elongation state.\\n\\nTo investigate whether local coordination arises from structures specific to ribosomes engaged in polysomes, we performed structural classification of ribosome pairs to better resolve the ribosome–ribosome interface (Fig. [4h](/articles/s41586-022-05255-2#Fig4) and Extended Data Fig. [12j,k](/articles/s41586-022-05255-2#Fig16)). We found that within the interface of tightly packed polysomes, the ribosomal protein L9 of the preceding ribosome adopts an extended conformation and its C-terminal domain protrudes into the elongation-factor-binding site of the following ribosome (Fig. [4h](/articles/s41586-022-05255-2#Fig4) and Extended Data Fig. [12j–m](/articles/s41586-022-05255-2#Fig16)). Independent focused classification on L9 of all ribosomes showed that it mainly (68.9%) adopts a flat conformation on the ribosome surface, whereas it is extended in 20.2% of 70S ribosomes (Extended Data Fig. [12n](/articles/s41586-022-05255-2#Fig16)). Ribosomes with the extended L9 largely overlap with the ribosomes detected as polysomes, especially those with a tighter 't-t' arrangement (Extended Data Fig. [12n–p](/articles/s41586-022-05255-2#Fig16)). These results suggest that L9 tends to have a flat conformation in single ribosomes and to adopt the extended conformation within tightly assembled polysomes. This clarifies previous observations of L9 being extended in X-ray crystallography structures in which the crystal packing recapitulates configurations of compacted polysomes, but being found in the flat conformation in cryo-EM structures of isolated ribosomes[51](/articles/s41586-022-05255-2#ref-CR51 \\\"Dunkle, J. A. & Cate, J. H. D. in Ribosomes (eds Rodnina, M. V., Wintermeyer, W. & Green, R.) Ch. 6 (2011).\\\"). Although L9 has been reported to be non-essential, its mutations can cause increased frameshifting and ribosome compaction by one codon[52](/articles/s41586-022-05255-2#ref-CR52 \\\"Herr, A. J., Nelson, C. C., Wills, N. M., Gesteland, R. F. & Atkins, J. F. Analysis of the roles of tRNA structure, ribosomal protein L9, and the bacteriophage T4 gene 60 bypassing signals during ribosome slippage on mRNA. J. Mol. Biol. 309, 1029–1048 (2001).\\\"),[53](/articles/s41586-022-05255-2#ref-CR53 \\\"Smith, A. M., Costello, M. S., Kettring, A. H., Wingo, R. J. & Moore, S. D. Ribosome collisions alter frameshifting at translational reprogramming motifs in bacterial mRNAs. Proc. Natl Acad. Sci. USA 116, 21769–21779 (2019).\\\"). We therefore propose that within tightly assembled polysomes, L9 of one ribosome can adopt an extended conformation that sterically interferes with elongation factor binding to the following ribosome. This local coordination mechanism can buffer adjacent ribosomes and help to maintain translation fidelity by avoiding direct collision within polysomes during active translation elongation.\\n\\nConclusions\\n-----------\\n\\nOur study demonstrates the potential of cryo-ET in shedding light on dynamic processes that are performed by macromolecular machines in living cells at a high level of detail. The analysis captures the structural and functional diversity of actively translating ribosomes inside a genome-reduced bacterium. The translation elongation cycle retrieved from the cellular cryo-ET data recapitulates and complements current mechanistic models of translation that are derived from controlled in vitro studies. The quantitative structural profiling of the cellular translation machinery presented here improves our understanding of protein biogenesis by providing distribution probabilities of translation intermediates. It also reveals how the translation machinery that functions as an interconnected system responds to different antibiotic perturbations on the single-cell level. Our investigation of polysomes further illustrates the advantage of in-cell structural biology, which can relate functional states of a molecular machine to its molecular sociology and cellular context, leading to the discovery of an elongation coordination mechanism mediated by L9. The approaches developed here establish a framework to analyse structural dynamics of cellular processes in the future and will contribute to the construction of functional cell models at atomic detail.\\n\\nMethods\\n-------\\n\\n### Cryo-ET sample preparation and data collection\\n\\nThe *M.* *pneumoniae* cultivation, sample preparation and data collection were described previously[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). The three datasets of native untreated, Cm-treated and PUM-treated cells were re-processed in Warp and M 1.0.7 (the alpha versions that were officially released as v.1.0.9.)[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\"),[54](/articles/s41586-022-05255-2#ref-CR54 \\\"Tegunov, D. & Cramer, P. Real-time cryo-electron microscopy data preprocessing with Warp. Nat. Methods 16, 1146–1152 (2019).\\\"). A small dataset of 15 tomograms acquired with Volta phase plate was processed with the denoising network in Warp 1.0.9 for visualization purposes only (as shown in Fig. [1a](/articles/s41586-022-05255-2#Fig1)).\\n\\nA dataset of spectinomycin-treated cells was collected following the same procedure as before[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). In brief, spectinomycin (Sigma-Aldrich) at a final concentration of 0.4 mg ml−1 was added into the culture medium, 15–20 min before plunge-freezing. Tilt-series collection with the dose-symmetric scheme[55](/articles/s41586-022-05255-2#ref-CR55 \\\"Hagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. J. Struct. Biol. 197, 191–198 (2017).\\\") was performed on a Titan Krios transmission electron microscope equipped with a K3 camera (Gatan) using SerialEM 3.8 (ref. [56](/articles/s41586-022-05255-2#ref-CR56 \\\"Schorb, M., Haberbosch, I., Hagen, W. J. H., Schwab, Y. & Mastronarde, D. N. Software tools for automated transmission electron microscopy. Nat. Methods 16, 471–477 (2019).\\\")) with the following parameters: magnification 81,000×, pixel size 1.053 Å, tilt range −60° to 60° with 3° increment, total dose 120–140 e− Å−2.\\n\\nIn total, 356 native untreated, 65 Cm-treated, 86 PUM-treated and 70 Spc-treated cellular tomograms were analysed in this work. Details of cryo-EM data collection, refinement and validation statistics are provided in Supplementary Tables [1](/articles/s41586-022-05255-2#MOESM1) and [3](/articles/s41586-022-05255-2#MOESM1)–[6](/articles/s41586-022-05255-2#MOESM1).\\n\\n### Image processing, ribosome template matching and map refinement\\n\\nPre-processing (motion correction, CTF estimation, dose filtering and tilt-series sorting) was performed in Warp 1.0.9 (ref.[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\")). For the untreated, Cm-treated and PUM-treated datasets, the ribosome coordinates were adopted from previous particle picking[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). For the Spc-treated dataset, template matching was performed in PyTom[57](/articles/s41586-022-05255-2#ref-CR57 \\\"Hrabe, T. et al. PyTom: a python-based toolbox for localization of macromolecules in cryo-electron tomograms and subtomogram analysis. J. Struct. Biol. 178, 177–188 (2012).\\\"), followed by computational classification in RELION 3.0 (refs.[58](/articles/s41586-022-05255-2#ref-CR58 \\\"Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. Nat. Protoc. 11, 2054–2065 (2016).\\\"),[59](/articles/s41586-022-05255-2#ref-CR59 \\\"Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7, e42166 (2018).\\\") to exclude false positives, without manual cleaning. In total, 109,990 untreated, 21,299 Cm-treated, 23,014 PUM-treated and 13,418 Spc-treated ribosome sub-tomograms were reconstructed in Warp.\\n\\nThree-dimensional (3D) refinement and classification were performed in RELION 3.0 (refs. [58](/articles/s41586-022-05255-2#ref-CR58 \\\"Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. Nat. Protoc. 11, 2054–2065 (2016).\\\"),[59](/articles/s41586-022-05255-2#ref-CR59 \\\"Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7, e42166 (2018).\\\")). We then used the software M (v.1.0.9) to perform multi-particle refinement of the tilt-series and refine the average map[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\"). Refinement of both geometric (image and volume deformation) and CTF parameters was done for five rounds in a sequential manner. After M refinement on the 70S, we performed focused refinement on the 30S and 50S subunits separately to improve the local map quality. Fourier shell correlation (FSC) calculation between independently refined random half subsets, local resolution estimation and post-processing were done in M and RELION.\\n\\n### Atomic model building in high-resolution ribosome maps\\n\\nAtomic models for the 30S and 50S subunits were built de novo using maps from focused refinement of Cm-treated ribosomes reported in our previous work[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\"), which were deposited in the Electron Microscopy Data Bank (EMDB) under the accession codes EMD-11998 and EMD-11999. Homology models for *M.* *pneumoniae* ribosomal proteins were generated using the SWISS-MODEL[60](/articles/s41586-022-05255-2#ref-CR60 \\\"Waterhouse, A. et al. SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic Acids Res. 46, W296–W303 (2018).\\\") online server (<https://swissmodel.expasy.org/>; accessed April 2020), with a *Bacillus subtilis* ribosome (Protein Data Bank (PDB) 3J9W) as the template, except for the ribosomal proteins L9 (PDB 1DIV and 4V63), L10 (PDB 1ZAV) and S21 (PDB 5MMJ). rRNA sequences (REFSEQ NC\\\\_000912) were aligned to the *E.* *coli* rRNA (PDB 4YBB) using the SINA online server[61](/articles/s41586-022-05255-2#ref-CR61 \\\"Pruesse, E., Peplies, J. & Glockner, F. O. SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes. Bioinformatics 28, 1823–1829 (2012).\\\"). ModeRNA[62](/articles/s41586-022-05255-2#ref-CR62 \\\"Rother, M., Rother, K., Puton, T. & Bujnicki, J. M. ModeRNA: a tool for comparative modeling of RNA 3D structure. Nucleic Acids Res. 39, 4007–4022 (2011).\\\") was used to build homology models for rRNAs on the basis of the alignment, with PDB 4YBB as the template. Homology models were rigid-body-fitted into the cryo-ET densities using Chimera[63](/articles/s41586-022-05255-2#ref-CR63 \\\"Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605–1612 (2004).\\\"), followed by iterative refinement using PHENIX real-space refinement[64](/articles/s41586-022-05255-2#ref-CR64 \\\"Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. Acta Crystallogr D 74, 531–544 (2018).\\\") and manual adjustment in Coot[65](/articles/s41586-022-05255-2#ref-CR65 \\\"Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Acta Crystallogr D 66, 486–501 (2010).\\\"). The ribosomal proteins L9, L10 and L11 were only fitted as rigid bodies into the map owing to the less-resolved local density. The flat conformation of L9 that resides on the ribosome surface was found to be predominant according to the focused classification (see Extended Data Fig. [12n](/articles/s41586-022-05255-2#Fig16)) and was thus built in the model. Sequence extensions for ribosomal proteins S6, L22 and L29 were built de novo. Models were validated using MolProbity[66](/articles/s41586-022-05255-2#ref-CR66 \\\"Williams, C. J. et al. MolProbity: more and better reference data for improved all-atom structure validation. Protein Sci. 27, 293–315 (2018).\\\"). FSC curves between the model and the map were also calculated for validation[67](/articles/s41586-022-05255-2#ref-CR67 \\\"Chen, S. et al. High-resolution noise substitution to measure overfitting and validate resolution in 3D structure determination by single particle electron cryomicroscopy. Ultramicroscopy 135, 24–35 (2013).\\\").\\n\\n### Bioinformatic analysis of ribosomal proteins\\n\\nBioinformatic sequence analysis of ribosomal proteins in *M.* *pneumoniae* and the comparison with *E.* *coli* homologues were performed as follows, with the different steps described in detail below: (i) protein sequences and RefSeq genome annotation for the *M.* *pneumoniae* strain M129 (ATCC 29342) were downloaded from NCBI; (ii) protein sequences were also annotated with eggNOG-mapper to obtain COG (Clusters of Orthologous Groups) IDs[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[68](/articles/s41586-022-05255-2#ref-CR68 \\\"Tatusov, R. L., Galperin, M. Y., Natale, D. A. & Koonin, E. V. The COG database: a tool for genome-scale analysis of protein functions and evolution. Nucleic Acids Res. 28, 33–36 (2000).\\\"); (iii) for the annotated ribosomal proteins, the corresponding COG multiple sequence alignments from representative bacterial species were downloaded from the eggNOG database[69](/articles/s41586-022-05255-2#ref-CR69 \\\"Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Nucleic Acids Res. 47, D309–D314 (2019).\\\"); (iv) as *M.* *pneumoniae* M129 is not among the representative species, its protein sequences were added to the multiple sequence alignments with MAFFT software[70](/articles/s41586-022-05255-2#ref-CR70 \\\"Katoh, K., Rozewicki, J. & Yamada, K. D. MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization. Brief. Bioinform. 20, 1160–1166 (2019).\\\"); (v) for each COG multiple sequence alignment, the number of amino acids in every representative species (including *M.* *pneumoniae* M129) present at positions before the N terminus and after the C terminus positions of *E.* *coli* K-12 substr. MG1655 were calculated; (vi) the presence of N- or C-terminus extensions longer than 20 amino acids was illustrated in iTOL using NCBI taxonomy tree as the basis[71](/articles/s41586-022-05255-2#ref-CR71 \\\"Letunic, I. & Bork, P. Interactive Tree Of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. Nucleic Acids Res. 49, W293–W296 (2021).\\\"); (vii) for all COGs corresponding to proteins with N- or C-terminus extensions in *M.* *pneumoniae* that are longer than 20 amino acids, protein disorder and secondary structure were analysed for all representative strains. The presence of cross-links or transposon insertions was also analysed.\\n\\nRefSeq genome annotation and protein sequences for the *M.* *pneumoniae* strain M129 (ATCC 29342) were downloaded from NCBI for the *M.* *pneumoniae* M129 GCF\\\\_000027345.1 assembly from <https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/027/345/GCF_000027345.1_ASM2734v1/> (accessed 21 April 2021). The file GCF\\\\_000027345.1\\\\_ASM2734v1\\\\_genomic.gff was used to identify ribosomal proteins annotated by RefSeq. The file GCF\\\\_000027345.1\\\\_ASM2734v1\\\\_protein.faa with protein sequences in FASTA format was used as input to the online tool eggNOG-mapper v2, which uses precomputed eggNOG v5.0 clusters and phylogenies for fast orthology assignment[69](/articles/s41586-022-05255-2#ref-CR69 \\\"Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Nucleic Acids Res. 47, D309–D314 (2019).\\\"),[72](/articles/s41586-022-05255-2#ref-CR72 \\\"Huerta-Cepas, J. et al. Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper. Mol. Biol. Evol. 34, 2115–2122 (2017).\\\"). The annotation tables by RefSeq and eggNOG-mapper were merged by protein ID. In total, 52 ribosomal proteins were annotated and mapped to 51 unique COGs[68](/articles/s41586-022-05255-2#ref-CR68 \\\"Tatusov, R. L., Galperin, M. Y., Natale, D. A. & Koonin, E. V. The COG database: a tool for genome-scale analysis of protein functions and evolution. Nucleic Acids Res. 28, 33–36 (2000).\\\"). Two proteins, WP\\\\_010874426.1 and WP\\\\_010874827.1, were mapped to the same COG0267 corresponding to the ribosomal protein L33. Multiple sequence alignments for the annotated proteins from representative bacterial species were downloaded from the eggNOG v5.0 database[69](/articles/s41586-022-05255-2#ref-CR69 \\\"Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Nucleic Acids Res. 47, D309–D314 (2019).\\\"). For each of the 51 COG IDs mapping to ribosomal proteins in *M.* *pneumoniae*, the trimmed alignments were downloaded from <http://eggnogapi5.embl.de/nog_data/text/trimmed_alg/COG_ID> ('COG\\\\_ID' in the url should be changed to the corresponding COG ID). As *M.* *pneumoniae* M129 is not among the imported representative species, its sequences were added to the multiple sequence alignments. The sequence of each of the proteins was saved in a separate fasta file, and the MAFFT software v.7.475 was used in the add mode for each alignment as follows: mafft --reorder --add protein.fasta --auto trimmed\\\\_alg\\\\_COG.fa > output\\\\_file (ref. [70](/articles/s41586-022-05255-2#ref-CR70 \\\"Katoh, K., Rozewicki, J. & Yamada, K. D. MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization. Brief. Bioinform. 20, 1160–1166 (2019).\\\")). For COG0627, the protein WP\\\\_010874827.1 from *M.* *pneumoniae* was used.\\n\\nFor each of the ribosomal COGs, the amino acid positions of the protein from the *E.* *coli* strain K-12 substr. MG1655 was taken as a reference, and for each protein from representative species (including *M.* *pneumoniae* M129), the number of amino acids before the N-terminus and after the C-terminus locations in *E.* *coli* was calculated. In total, 11 ribosomal proteins that had N- or C-terminus extensions of more than 20 amino acids in *M. pneumoniae* M129 were considered to have extensions (2 proteins with N-terminus extensions and 9 proteins with C-terminus extensions). Ribosomal protein S3 (COG0092), which has a 17-amino-acid extension at the C terminus, was also retained for further analysis. NCBI bacterial taxonomy file new\\\\_taxdump.zip was downloaded from <https://ftp.ncbi.nlm.nih.gov/pub/taxonomy/new_taxdump/> (version on 24 April 2021). The file names.dmp was used to map NCBI IDs to taxonomy names. Tree was reconstructed from the file taxidlineage.dmp in Python v.3.7.7 with ETE3 Toolkit v.3.1.2 (ref. [73](/articles/s41586-022-05255-2#ref-CR73 \\\"Huerta-Cepas, J., Serra, F. & Bork, P. ETE 3: reconstruction, analysis, and visualization of phylogenomic data. Mol. Biol. Evol. 33, 1635–1638 (2016).\\\")). EggNOG protein IDs containing NCBI species IDs were directly mapped to the NCBI tree nodes. The presence of ribosomal protein extensions was saved as tables and converted to iTOL format with table2itol utility (<https://github.com/mgoeker/table2itol>) in R v.3.6.1. The tree was visualized with iTOL v.6 (ref. [71](/articles/s41586-022-05255-2#ref-CR71 \\\"Letunic, I. & Bork, P. Interactive Tree Of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. Nucleic Acids Res. 49, W293–W296 (2021).\\\")).\\n\\nSequences for the 11 ribosomal proteins annotated with extensions in *M.* *pneumoniae*, and their orthologues in other species, were further analysed in terms of protein disorder and secondary structures. For disorder prediction, all protein sequences from the multiple sequence alignment files were saved in FASTA format without gaps. The IUPred2A tool in Python v.3.7.7 was run with the input parameter 'long' for each protein sequence[74](/articles/s41586-022-05255-2#ref-CR74 \\\"Meszaros, B., Erdos, G. & Dosztanyi, Z. IUPred2A: context-dependent prediction of protein disorder as a function of redox state and protein binding. Nucleic Acids Res. 46, W329–W337 (2018).\\\"). The number of disordered amino acids and disorder length in extended regions were calculated on the basis of the position of the extended region relative to *E.* *coli*, and an IUPred2A score of 0.5 was used as the disorder threshold. The JPred prediction tool[75](/articles/s41586-022-05255-2#ref-CR75 \\\"Cole, C., Barber, J. D. & Barton, G. J. The Jpred 3 secondary structure prediction server. Nucleic Acids Res. 36, W197–W201 (2008).\\\") was used to predict the secondary structure of each protein using the jpredapi utility and JPred-big-batch-submission utility for a large number of submissions (<https://github.com/fabianegli/JPred-big-batch-submission>). The number of helices in extended regions was calculated on the basis of the position of the extended region relative to *E.* *coli* protein.\\n\\nFor the 11 ribosomal proteins with extensions in *M.* *pneumoniae*, the positions of cross-links linking to the same protein or a different protein were mapped according to the amino acid cross-link positions reported previously[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). To analyse transposon insertions in *M.* *pneumoniae* ribosomal proteins with extensions, the supplementary materials from a previous publication[24](/articles/s41586-022-05255-2#ref-CR24 \\\"Miravet-Verde, S., Burgos, R., Delgado, J., Lluch-Senar, M. & Serrano, L. FASTQINS and ANUBIS: two bioinformatic tools to explore facts and artifacts in transposon sequencing and essentiality studies. Nucleic Acids Res. 48, e102 (2020).\\\") were used. All the files in the 'SupplementaryData1\\\\_fastqins\\\\_processed.zip folder' (files with extension '.qins' but not '\\\\_fw.qins' or '\\\\_rv.qins') were concatenated to obtain a list of nucleotide coordinates of transposon insertions in the sequence of *M.* *pneumoniae* M129 NC\\\\_000912.1. The transposon insertion locations in the 11 ribosomal proteins were selected on the basis of their genomic locations as per RefSeq annotations (GCF\\\\_000027345.1\\\\_ASM2734v1\\\\_genomic.gff). Nucleotide locations were converted to amino acid locations by calculating the difference between each transposon location and the start coordinate of the corresponding gene and dividing the number of nucleotides by three. All data were mapped to the protein sequences to derive Extended Data Fig. [3](/articles/s41586-022-05255-2#Fig7).\\n\\n### Sub-tomogram classification of the translation states of ribosomes\\n\\nMaximum-likelihood 3D classification[76](/articles/s41586-022-05255-2#ref-CR76 \\\"Scheres, S. H. Processing of structurally heterogeneous cryo-EM data in RELION. Methods Enzymol. 579, 125–157 (2016).\\\") was performed in RELION 3.0 (refs. [58](/articles/s41586-022-05255-2#ref-CR58 \\\"Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. Nat. Protoc. 11, 2054–2065 (2016).\\\"),[59](/articles/s41586-022-05255-2#ref-CR59 \\\"Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7, e42166 (2018).\\\")) with the re-extracted ribosome sub-tomograms after M refinement. A hierarchical and exhaustive classification strategy ([Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)) with at least three tiers was used to handle the heterogeneity in the native untreated dataset, which is described here in detail and illustrated in Extended Data Figs. [4](/articles/s41586-022-05255-2#Fig8) and [5](/articles/s41586-022-05255-2#Fig9). A similar procedure was applied for all antibiotics-treated datasets, and is detailed in Extended Data Figs. [9](/articles/s41586-022-05255-2#Fig13)–[11](/articles/s41586-022-05255-2#Fig15).\\n\\nIn the first tier, 109,990 sub-tomograms were classified into 70S and free 50S with a global spherical mask (320-Å diameter). The 24,157 free 50S were further classified into two classes, with a local spherical mask focusing on the ribosome recycling factor binding site. Before extensive 70S classification, structural heterogeneity was evaluated by visual inspection, multibody refinement and test classification runs in RELION. Classification set-ups were extensively tested, including different masks (global 70S mask, local 30S mask, spherical tRNA path mask, solvent tRNA path mask, spherical EF mask, solvent EF mask), initial references (ribosome average, features-less sphere or none), angular search options (global, local or without alignment) and RELION optimization parameters (class numbers 2–16, T values 2–10, iterations 25–40, limit resolution E-step 5–10 Å). To avoid bias, we mostly used a sphere or other featureless shapes with soft edges as the mask. Local spherical masks covering factors (elongation factors and/or tRNAs) generally provided more consistent and stable classification results than those obtained when using larger masks for the untreated data. The class number was made higher than the number of distinct structures that could be retained in one classification job and similar resulting classes were grouped.\\n\\nIn the second tier, the 77,539 identified 70S ribosomes were further classified with a local mask (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8), mask I) focusing on the tRNA path region, roughly covering the A, P and E sites. This resulted in four major classes with different tRNA occupancies: 'P, E', 'a, P/E', 'P' and 'A, P'. In the following rounds, we could further classify the 'a, P/E' class into 'A\\\\*, P/E' and 'A/P, P/E'. In addition, a 70S class with only one hybrid 'P/E' tRNA was classified. Two of the resulting classes were not interpretable: 2,150 particles with dim 30S density that were poorly resolved and 1,484 particles with density near the P site that does not resemble a tRNA.\\n\\nIn the third tier, focused classification with a local mask (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8), mask II) around the elongation factor and A/T tRNA binding sites was carried out. For the previous class with 'P, E' tRNAs, further classification resulted in 1,803 particles without additional density, 3,324 particles with EF-G (updated as 'EF-G, ap/P, pe/E') and 4,634 particles with EF-Tu•tRNA. For the classes with partial and full hybrid tRNAs, sub-classes with EF-G were obtained. For the class with only 'P' tRNA, 12,464 particles with additional EF-Tu•tRNA were classified.\\n\\nFor each classification step, at least three parallel RELION classification jobs with the same or slightly different parameters (either 30S mask or spherical EF/tRNA mask, local angular search range, class number, T value) were carried out for comparison. The classification job with the most stable result was selected and used for sorting sub-tomograms (Extended Data Fig. [5a–g](/articles/s41586-022-05255-2#Fig9)). After sorting, subsequent validation classification runs were performed for each sorted class to test whether new structures emerge or whether particles were 'wrongly' classified. Misclassified particles were relocated to the corresponding class and the validation runs were repeated until convergence. This approach was performed for all classification steps. For each of the final classes, refinement and post-processing were done in RELION. Further classification performed with the new refinement results as inputs did not generate any new classes.\\n\\n### Model building and comparison of the ribosome translation states\\n\\nTo build models for the ribosome classes, the 30S and 50S models built as described above were used as starting models for flexible fitting. Homology models of EF-G and EF-Tu were generated by SWISS-MODEL (<https://swissmodel.expasy.org/>; accessed April 2020) with PDB structures 4V7D and 4V5L as the template, respectively. For tRNA homology models, the tRNA in an *E.* *coli* ribosome structure (PDB 4V7C) was used as the template and mutated to the sequence of *M.* *pneumoniae* Phe-tRNA (REFSEQ NC\\\\_000912). The mRNA and nascent peptide were built using PDB 3J9W as the template. The homology model of the ribosome recycling factor was built using PDB 1EH1 as the template. For each class, the starting models were first rigid-body-fitted into the density using Chimera and then flexible fitting was done using Namdinator[77](#ref-CR77 \\\"Kidmose, R. T. et al. Namdinator—automatic molecular dynamics flexible fitting of structural models into cryo-EM and crystallography experimental maps. IUCrJ 6, 526–531 (2019).\\\"),[78](#ref-CR78 \\\"Trabuco, L. G., Villa, E., Mitra, K., Frank, J. & Schulten, K. Flexible fitting of atomic structures into electron microscopy maps using molecular dynamics. Structure 16, 673–683 (2008).\\\"),[79](/articles/s41586-022-05255-2#ref-CR79 \\\"Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. J. Chem. Phys. 153, 044130 (2020).\\\"). Validation was performed as described above.\\n\\nFor measuring 30S body rotations, structures of all classes were first aligned to the 50S subunit and then the rotation angles were estimated with a pivot point near nucleotide 11 of the 16S rRNA. With the perspective from the solvent side of the 30S subunit, positive numbers equal counter-clockwise rotations and negative numbers equal clockwise rotations. For measuring 30S head rotations, all class structures were first aligned to the 30S body and then the rotation angles were determined with the axis near the 30S neck. For both rotations, the angles in the 'A, P' class were defined as 0°. To describe the L1 dynamics, the distance between the mass centre of the L1 stalk (near nucleotide 2,181 of the 23S rRNA) and a fixed point near the centre of the classical E site (determined on the basis of the 'P, E' class) was measured after aligning all classes on the 50S subunit (excluding the L1 stalk).\\n\\n### Spatial analysis of ribosomes and polysomes\\n\\nSpatial mapping of ribosomes within cellular tomograms was achieved by projecting back the ribosome structures into the tomograms, with coordinates determined by template matching and shifts and rotations determined by RELION refinement. The projection was performed using the TOM toolbox[80](/articles/s41586-022-05255-2#ref-CR80 \\\"Nickell, S. et al. TOM software toolbox: acquisition and analysis for electron tomography. J. Struct. Biol. 149, 227–234 (2005).\\\") after Euler angle format conversion, at four times binning (voxel size 6.8 Å). To calculate the ribosome concentration, we first estimated the cellular volume covered in the tomogram and then divided the total number of detected ribosomes by the volume.\\n\\nDetection of the polysomes was based on both position and orientation information (Extended Data Fig. [12c](/articles/s41586-022-05255-2#Fig16)), using a custom script in MATLAB 2016b. It is noted that the annotated polysomes only refer to those assembled closely in space and thus can be detected on the basis of their spatial proximity. The positions of mRNA entry and exit sites for all 70S ribosomes were calculated on the basis of the rotations and shifts determined during RELION refinement. The distances from the mRNA exit site of one ribosome (defined as the preceding ribosome i) to the mRNA entry sites of all neighbouring ribosomes (as potential following ribosomes i+1) were calculated. A distance threshold of 7 nm was used to define whether two ribosomes belong to the same polysome. The calculation and polysome definition were done for all 70S ribosomes. A unique identifier was assigned to each polysome, as well as the sequential number for all ribosomes within the polysome.\\n\\nThe distribution of relative positions of adjacent ribosomes within the polysome—that is, the position of the following ribosome (i+1) relative to the preceding ribosome (i)—was analysed after normalizing the relative position vectors with rotations of the preceding ribosome determined from RELION refinement. The relative rotation of the following ribosome to the preceding ribosome was represented as three Euler angles (*ψ*, *θ*, *φ* in XYZ system). Using the *k*-means clustering function in MATLAB 2016b, we determined two major arrangement configurations for adjacent ribosomes within the polysome; that is, how the following ribosome rotates relative to the preceding ribosome (as shown in Extended Data Fig. [12f,g](/articles/s41586-022-05255-2#Fig16)). These two configurations are identical to the previously reported 'top-top' (t-t) and 'top-back' (t-b) configurations[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"), and the naming was adopted.\\n\\nTo further refine the ribosome–ribosome interface in polysomes, RELION classification was performed with sub-tomograms extracted with a large box size that can accommodate two ribosomes. After refinement on the preceding ribosome (i), classification without refinement was performed with a local mask focusing on the following ribosome (i+1). Only ribosome pairs within tightly packed polysomes (5,083 pairs from 5 di-ribosome classes; all with a 't-t' arrangement and extended L9 in between) resulted in average densities with both ribosomes well resolved. Models for the resulting classes were generated with rigid-body-fitting of the above described  ribosome models and the L9 homology model with the extended conformation (PDB 4V63).\\n\\n### Statistical analysis of translation elongation states in polysomes\\n\\nTo compare the experimental elongation state frequencies in polysomes with theoretical frequencies, the distributions of frequencies of each elongation state calculated across 356 tomograms of the untreated cells were compared between polysomes and all ribosomes, and between polysomes and mono-ribosomes by calculating the fold change between distribution medians. Statistical significance was assessed with a two-sided Wilcoxon–Mann–Whitney test using the ranksum function in MATLAB 2019b. The theoretical frequency of each ribosome pair was calculated as the product of the overall frequencies of the ribosome classes for the preceding ribosome (i) and the following ribosome (i+1). Experimental polysome pair frequencies were calculated by summarizing the numbers of all ribosome pairs engaged in polysomes, and dividing these numbers by the total number of pairs. Experimental and theoretical pair frequencies were compared by calculating the fold change per pair.\\n\\nPermutation analysis was performed to test the significance of differences between occurrence frequencies of the experimental pairs compared to the theoretical pairs. The ribosome pair sequences in polysomes were represented as a matrix with single polysomes as rows and ribosome positions as columns, in which each matrix cell contains the ribosome class representing its elongation state. Columns with positions beyond each ribosome’s length were assigned to NaN (not-a-number). All polysomes across all tomograms were combined in one matrix (8,641 polysomes in total). For permutation analysis, all elements of the polysome matrix were randomly shuffled for 10,000 times with the randperm function in MATLAB 2019b. For each row, the elements were sorted so that the ribosome classes are in the front columns followed by NaNs occurring in the same row after shuffling. Rows with fewer than two ribosomes in the sequence were deleted. Shuffled ribosome pair frequencies were calculated in the same way as experimental pair frequencies. The permutation *P* value for each ribosome pair was calculated as the minimum between the number of permutations in which the pair frequency was less or equal to the experimentally observed frequency divided by the total number of permutations, and the one minus this value. Permutation *P* values were adjusted for multiple hypotheses testing with the Benjamini–Hochberg procedure using the mafdr function in MATLAB 2019b with parameters ('bhfdr', 'true').\\n\\nFor polysome distance threshold analysis, matrices were created from the full ribosome dataset by varying the distance threshold to the nearest neighbour in the range from 3 to 10 nm. The distances for polysome definition were calculated between the mRNA exit site of one ribosome and the mRNA entry site of another ribosome as described above. The fractions of elongation states in ribosome pairs were calculated as for the original ribosome set (with the threshold of 7 nm). For each distance threshold and each ribosome class, the ratio between the number of pairs in which the preceding ribosome has this class and the number of pairs in which the following ribosome has this class was calculated.\\n\\n### Single-cell clustering analysis\\n\\nThe distributions of 70S ribosome classes identified in the translation elongation phase for all four datasets (356 untreated cells, 65 Cm-treated cells, 70 Spc-treated cells and 86 PUM-treated cells) were used for clustering analysis. As each tomogram covers most of one cell, the sub-tomograms in each class could be further separated by which cell they belong to. Classes in the antibiotic-treated cells were assigned with the class identifiers according to the closest classes in the untreated dataset. The percentages of different classes within each cell were calculated and these numbers were used as inputs for clustering analysis. Hierarchical clustering analysis was done using the clustergram function in MATLAB 2016b.\\n\\nStructure visualization, preparation for figures and videos were done in Chimera[63](/articles/s41586-022-05255-2#ref-CR63 \\\"Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605–1612 (2004).\\\") and ChimeraX[81](/articles/s41586-022-05255-2#ref-CR81 \\\"Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. Protein Sci. 30, 70–82 (2021).\\\").\\n\\n### Reporting summary\\n\\nFurther information on research design is available in the [Nature Research Reporting Summary](/articles/s41586-022-05255-2#MOESM2) linked to this article.\\n\\n\\n\\n\\nData availability\\n-----------------\\n\\n\\nMaps have been deposited in the EMDB under accession codes [13234](http://www.ebi.ac.uk/pdbe/entry/EMD-13234), [13272](http://www.ebi.ac.uk/pdbe/entry/EMD-13272), [13273](http://www.ebi.ac.uk/pdbe/entry/EMD-13273), [13274](http://www.ebi.ac.uk/pdbe/entry/EMD-13274), [13275](http://www.ebi.ac.uk/pdbe/entry/EMD-13275), [13276](http://www.ebi.ac.uk/pdbe/entry/EMD-13276), [13277](http://www.ebi.ac.uk/pdbe/entry/EMD-13277), [13278](http://www.ebi.ac.uk/pdbe/entry/EMD-13278), [13279](http://www.ebi.ac.uk/pdbe/entry/EMD-13279), [13280](http://www.ebi.ac.uk/pdbe/entry/EMD-13280), [13281](http://www.ebi.ac.uk/pdbe/entry/EMD-13281), [13282](http://www.ebi.ac.uk/pdbe/entry/EMD-13282), [13283](http://www.ebi.ac.uk/pdbe/entry/EMD-13283), [13284](http://www.ebi.ac.uk/pdbe/entry/EMD-13284), [13285](http://www.ebi.ac.uk/pdbe/entry/EMD-13285), [13286](http://www.ebi.ac.uk/pdbe/entry/EMD-13286), [13410](http://www.ebi.ac.uk/pdbe/entry/EMD-13410), [13411](http://www.ebi.ac.uk/pdbe/entry/EMD-13411), [13412](http://www.ebi.ac.uk/pdbe/entry/EMD-13412), [13413](http://www.ebi.ac.uk/pdbe/entry/EMD-13413), [13414](http://www.ebi.ac.uk/pdbe/entry/EMD-13414), [13431](http://www.ebi.ac.uk/pdbe/entry/EMD-13431), [13432](http://www.ebi.ac.uk/pdbe/entry/EMD-13432), [13433](http://www.ebi.ac.uk/pdbe/entry/EMD-13433), [13434](http://www.ebi.ac.uk/pdbe/entry/EMD-13434), [13435](http://www.ebi.ac.uk/pdbe/entry/EMD-13435), [13436](http://www.ebi.ac.uk/pdbe/entry/EMD-13436), [13445](http://www.ebi.ac.uk/pdbe/entry/EMD-13445), [13446](http://www.ebi.ac.uk/pdbe/entry/EMD-13446), [13447](http://www.ebi.ac.uk/pdbe/entry/EMD-13447), [13448](http://www.ebi.ac.uk/pdbe/entry/EMD-13448), [13449](http://www.ebi.ac.uk/pdbe/entry/EMD-13449), [13450](http://www.ebi.ac.uk/pdbe/entry/EMD-13450), [13451](http://www.ebi.ac.uk/pdbe/entry/EMD-13451), [13452](http://www.ebi.ac.uk/pdbe/entry/EMD-13452), [13287](http://www.ebi.ac.uk/pdbe/entry/EMD-13287), [13288](http://www.ebi.ac.uk/pdbe/entry/EMD-13288) and [13289](http://www.ebi.ac.uk/pdbe/entry/EMD-13289). Models have been deposited in the PDB under accession codes: [7OOC](https://doi.org/10.2210/pdb7OOC/pdb), [7OOD](http://doi.org/10.2210/pdb7OOD/pdb), [7P6Z](http://doi.org/10.2210/pdb7P6Z/pdb), [7PAH](http://doi.org/10.2210/pdb7PAH/pdb), [7PAI](http://doi.org/10.2210/pdb7PAI/pdb), [7PAJ](http://doi.org/10.2210/pdb7PAJ/pdb), [7PAK](http://doi.org/10.2210/pdb7PAK/pdb), [7PAL](http://doi.org/10.2210/pdb7PAL/pdb), [7PAM](http://doi.org/10.2210/pdb7PAM/pdb), [7PAN](http://doi.org/10.2210/pdb7PAN/pdb), [7PAO](http://doi.org/10.2210/pdb7PAO/pdb), [7PAQ](http://doi.org/10.2210/pdb7PAQ/pdb), [7PAR](http://doi.org/10.2210/pdb7PAR/pdb), [7PAS](http://doi.org/10.2210/pdb7PAS/pdb), [7PAT](http://doi.org/10.2210/pdb7PAT/pdb), [7PAU](http://doi.org/10.2210/pdb7PAU/pdb), [7PH9](http://doi.org/10.2210/pdb7PH9/pdb), [7PHA](http://doi.org/10.2210/pdb7PHA/pdb), [7PHB](http://doi.org/10.2210/pdb7PHB/pdb), [7PHC](http://doi.org/10.2210/pdb7PHC/pdb), [7PI8](http://doi.org/10.2210/pdb7PI8/pdb), [7PI9](http://doi.org/10.2210/pdb7PI9/pdb), [7PIA](http://doi.org/10.2210/pdb7PIA/pdb), [7PIB](http://doi.org/10.2210/pdb7PIB/pdb), [7PIC](http://doi.org/10.2210/pdb7PIC/pdb), [7PIO](http://doi.org/10.2210/pdb7PIO/pdb), [7PIP](http://doi.org/10.2210/pdb7PIP/pdb), [7PIQ](http://doi.org/10.2210/pdb7PIQ/pdb), [7PIR](http://doi.org/10.2210/pdb7PIR/pdb), [7PIS](http://doi.org/10.2210/pdb7PIS/pdb) and [7PIT](http://doi.org/10.2210/pdb7PIT/pdb). Detailed information for all maps and models generated in this work is provided in Supplementary Tables [1](/articles/s41586-022-05255-2#MOESM1) and [3](/articles/s41586-022-05255-2#MOESM1)–[6](/articles/s41586-022-05255-2#MOESM1). Maps and atomic models used from previous studies were obtained from the EMDB ([11998](http://www.ebi.ac.uk/pdbe/entry/EMD-11998) and [11999](http://www.ebi.ac.uk/pdbe/entry/EMD-11999)) and the PDB ([3J9W](http://doi.org/10.2210/pdb3J9W/pdb), [1DIV](http://doi.org/10.2210/pdb1DIV/pdb), [4V63](https://doi.org/10.2210/pdb4V63/pdb), [1ZAV](http://doi.org/10.2210/pdb1ZAV/pdb), [5MMJ](http://doi.org/10.2210/pdb5MMJ/pdb), [4YBB](http://doi.org/10.2210/pdb4YBB/pdb), [4V7C](http://doi.org/10.2210/pdb4V7C/pdb), [4V7D](http://doi.org/10.2210/pdb4V7D/pdb), [4V5L](http://doi.org/10.2210/pdb4V5L/pdb) and [1EH1](http://doi.org/10.2210/pdb1EH1/pdb)).\\n\\n\\nCode availability\\n-----------------\\n\\n\\nThe code and associated data for bioinformatics analysis of ribosomal protein extensions (Extended Data Fig. [3](/articles/s41586-022-05255-2#Fig7)) and statistical analysis of polysome sequences (Fig. [4](/articles/s41586-022-05255-2#Fig4) and Extended Data Fig. [13](/articles/s41586-022-05255-2#Fig17)) are deposited at GitHub (<https://github.com/mszimmermann/mycoplasma_ribosome>). The MATLAB script for polysome annotation (Fig. [4](/articles/s41586-022-05255-2#Fig4) and Extended Data Fig. [12](/articles/s41586-022-05255-2#Fig16)) can be found at <https://github.com/xueliang4906/polysome_detect>.\\n\\n\\nChange history\\n--------------\\n\\n* ### 10 November 2022\\n\\nA Correction to this paper has been published: <https://doi.org/10.1038/s41586-022-05455-w>\\nReferences\\n----------\\n\\n1. Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. *Annu. Rev. Biochem.* **82**, 203–236 (2013).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-biochem-113009-092313) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFOqsrzJ) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23746255) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20the%20translational%20elongation%20cycle&journal=Annu.%20Rev.%20Biochem.&doi=10.1146%2Fannurev-biochem-113009-092313&volume=82&pages=203-236&publication_year=2013&author=Voorhees%2CRM&author=Ramakrishnan%2CV)\\n2. O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. *Science* **369**, 554–557 (2020).\\n\\n[Article](https://doi.org/10.1126%2Fscience.abb3758) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020Sci...369..554O) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFWgtr7N) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32732422) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115962) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=In-cell%20architecture%20of%20an%20actively%20transcribing%E2%80%93translating%20expressome&journal=Science&doi=10.1126%2Fscience.abb3758&volume=369&pages=554-557&publication_year=2020&author=O%27Reilly%2CFJ)\\n3. Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. *Nat. Methods* **18**, 186–193 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-020-01054-7) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXjt1SmsL8%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33542511) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7611018) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-particle%20cryo-EM%20refinement%20with%20M%20visualizes%20ribosome%E2%80%93antibiotic%20complex%20at%203.5%20%C3%85%20in%20cells&journal=Nat.%20Methods&doi=10.1038%2Fs41592-020-01054-7&volume=18&pages=186-193&publication_year=2021&author=Tegunov%2CD&author=Xue%2CL&author=Dienemann%2CC&author=Cramer%2CP&author=Mahamid%2CJ)\\n4. Brandt, F. et al. The native 3D organization of bacterial polysomes. *Cell* **136**, 261–271 (2009).\\n\\n[Article](https://doi.org/10.1016%2Fj.cell.2008.11.016) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1MXhs1Kiurk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19167328) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20native%203D%20organization%20of%20bacterial%20polysomes&journal=Cell&doi=10.1016%2Fj.cell.2008.11.016&volume=136&pages=261-271&publication_year=2009&author=Brandt%2CF)\\n5. Rodnina, M. V. Translation in prokaryotes. *Cold Spring Harb. Perspect. Biol.* **10**, a032664 (2018).\\n\\n[Article](https://doi.org/10.1101%2Fcshperspect.a032664) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29661790) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6120702) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Translation%20in%20prokaryotes&journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&doi=10.1101%2Fcshperspect.a032664&volume=10&publication_year=2018&author=Rodnina%2CMV)\\n6. Frank, J. & Agrawal, R. K. A ratchet-like inter-subunit reorganization of the ribosome during translocation. *Nature* **406**, 318–322 (2000).\\n\\n[Article](https://doi.org/10.1038%2F35018597) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2000Natur.406..318F) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD3cXlsFKnsrk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10917535) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20ratchet-like%20inter-subunit%20reorganization%20of%20the%20ribosome%20during%20translocation&journal=Nature&doi=10.1038%2F35018597&volume=406&pages=318-322&publication_year=2000&author=Frank%2CJ&author=Agrawal%2CRK)\\n7. Villa, E. et al. Ribosome-induced changes in elongation factor Tu conformation control GTP hydrolysis. *Proc. Natl Acad. Sci. USA* **106**, 1063–1068 (2009).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.0811370106) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009PNAS..106.1063V) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1MXhsFyjtbk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19122150) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2613361) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome-induced%20changes%20in%20elongation%20factor%20Tu%20conformation%20control%20GTP%20hydrolysis&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.0811370106&volume=106&pages=1063-1068&publication_year=2009&author=Villa%2CE)\\n8. Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. *Nature* **466**, 329–333 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature09206) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.466..329F) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXosl2rsr0%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20631791) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome%20dynamics%20and%20tRNA%20movement%20by%20time-resolved%20electron%20cryomicroscopy&journal=Nature&doi=10.1038%2Fnature09206&volume=466&pages=329-333&publication_year=2010&author=Fischer%2CN&author=Konevega%2CAL&author=Wintermeyer%2CW&author=Rodnina%2CMV&author=Stark%2CH)\\n9. Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. *Nature* **468**, 713–716 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature09547) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.468..713R) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsFSku7jF) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21124459) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3272701) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Head%20swivel%20on%20the%20ribosome%20facilitates%20translocation%20by%20means%20of%20intra-subunit%20tRNA%20hybrid%20sites&journal=Nature&doi=10.1038%2Fnature09547&volume=468&pages=713-716&publication_year=2010&author=Ratje%2CAH)\\n10. Ramrath, D. J. et al. Visualization of two transfer RNAs trapped in transit during elongation factor G-mediated translocation. *Proc. Natl Acad. Sci. USA* **110**, 20964–20969 (2013).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1320387110) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2013PNAS..11020964R) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXnsF2jug%3D%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24324168) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876219) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualization%20of%20two%20transfer%20RNAs%20trapped%20in%20transit%20during%20elongation%20factor%20G-mediated%20translocation&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1320387110&volume=110&pages=20964-20969&publication_year=2013&author=Ramrath%2CDJ)\\n11. Brilot, A. F., Korostelev, A. A., Ermolenko, D. N. & Grigorieff, N. Structure of the ribosome with elongation factor G trapped in the pretranslocation state. *Proc. Natl Acad. Sci. USA* **110**, 20994–20999 (2013).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1311423110) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2013PNAS..11020994B) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXnsF2iuw%3D%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24324137) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876248) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structure%20of%20the%20ribosome%20with%20elongation%20factor%20G%20trapped%20in%20the%20pretranslocation%20state&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1311423110&volume=110&pages=20994-20999&publication_year=2013&author=Brilot%2CAF&author=Korostelev%2CAA&author=Ermolenko%2CDN&author=Grigorieff%2CN)\\n12. Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. *Cell* **161**, 845–857 (2015).\\n\\n[Article](https://doi.org/10.1016%2Fj.cell.2015.03.052) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2MXotVKlur0%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25957688) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4432480) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20snapshots%20of%20actively%20translating%20human%20ribosomes&journal=Cell&doi=10.1016%2Fj.cell.2015.03.052&volume=161&pages=845-857&publication_year=2015&author=Behrmann%2CE)\\n13. Loveland, A. B., Demo, G. & Korostelev, A. A. Cryo-EM of elongating ribosome with EF-Tu\\\\*GTP elucidates tRNA proofreading. *Nature* **584**, 640–645 (2020).\\n\\n[Article](https://doi.org/10.1038%2Fs41586-020-2447-x) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020Natur.584..640L) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlShtL%2FE) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32612237) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7483604) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cryo-EM%20of%20elongating%20ribosome%20with%20EF-Tu%2AGTP%20elucidates%20tRNA%20proofreading&journal=Nature&doi=10.1038%2Fs41586-020-2447-x&volume=584&pages=640-645&publication_year=2020&author=Loveland%2CAB&author=Demo%2CG&author=Korostelev%2CAA)\\n14. Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. *Nature* **595**, 741–745 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41586-021-03713-x) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Natur.595..741R) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFSnsbnM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34234344) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8318882) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20early%20translocation%20events%20on%20the%20ribosome&journal=Nature&doi=10.1038%2Fs41586-021-03713-x&volume=595&pages=741-745&publication_year=2021&author=Rundlet%2CEJ)\\n15. Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. *Nat. Commun.* **12**, 5933 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-26133-x) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.5933P) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1aqsL7M) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34635670) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8505512) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20mechanism%20of%20GTPase-powered%20ribosome%E2%80%93tRNA%20movement&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-26133-x&volume=12&publication_year=2021&author=Petrychenko%2CV)\\n16. Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. *Nat. Commun.* **12**, 7236 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-27415-0) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.7236C) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXislejtbjN) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34903725) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8668904) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Time-resolved%20cryo-EM%20visualizes%20ribosomal%20translocation%20with%20EF-G%20and%20GTP&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-27415-0&volume=12&publication_year=2021&author=Carbone%2CCE)\\n17. Brown, A. & Shao, S. Ribosomes and cryo-EM: a duet. *Curr. Opin. Struct. Biol.* **52**, 1–7 (2018).\\n\\n[Article](https://doi.org/10.1016%2Fj.sbi.2018.07.001) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlSjs77J) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30015201) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosomes%20and%20cryo-EM%3A%20a%20duet&journal=Curr.%20Opin.%20Struct.%20Biol.&doi=10.1016%2Fj.sbi.2018.07.001&volume=52&pages=1-7&publication_year=2018&author=Brown%2CA&author=Shao%2CS)\\n18. Kuhner, S. et al. Proteome organization in a genome-reduced bacterium. *Science* **326**, 1235–1240 (2009).\\n\\n[Article](https://doi.org/10.1126%2Fscience.1176343) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009Sci...326.1235K) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19965468) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Proteome%20organization%20in%20a%20genome-reduced%20bacterium&journal=Science&doi=10.1126%2Fscience.1176343&volume=326&pages=1235-1240&publication_year=2009&author=Kuhner%2CS)\\n19. Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. *Mol. Cell* **39**, 560–569 (2010).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2010.08.003) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVyrt7jF) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20797628) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20three-dimensional%20organization%20of%20polyribosomes%20in%20intact%20human%20cells&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2010.08.003&volume=39&pages=560-569&publication_year=2010&author=Brandt%2CF&author=Carlson%2CLA&author=Hartl%2CFU&author=Baumeister%2CW&author=Grunewald%2CK)\\n20. Mahamid, J. et al. Visualizing the molecular sociology at the HeLa cell nuclear periphery. *Science* **351**, 969–972 (2016).\\n\\n[Article](https://doi.org/10.1126%2Fscience.aad8857) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016Sci...351..969M) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XivFOnt7c%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26917770) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualizing%20the%20molecular%20sociology%20at%20the%20HeLa%20cell%20nuclear%20periphery&journal=Science&doi=10.1126%2Fscience.aad8857&volume=351&pages=969-972&publication_year=2016&author=Mahamid%2CJ)\\n21. Schaffer, M. et al. A cryo-FIB lift-out technique enables molecular-resolution cryo-ET within native *Caenorhabditis elegans* tissue. *Nat. Methods* **16**, 757–762 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0497-5) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVOku7zL) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31363205) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20cryo-FIB%20lift-out%20technique%20enables%20molecular-resolution%20cryo-ET%20within%20native%20Caenorhabditis%20elegans%20tissue&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0497-5&volume=16&pages=757-762&publication_year=2019&author=Schaffer%2CM)\\n22. Waltz, F. et al. How to build a ribosome from RNA fragments in *Chlamydomonas* mitochondria. *Nat. Commun.* **12**, 7176 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-27200-z) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.7176W) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXislWktLrO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34887394) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8660880) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=How%20to%20build%20a%20ribosome%20from%20RNA%20fragments%20in%20Chlamydomonas%20mitochondria&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-27200-z&volume=12&publication_year=2021&author=Waltz%2CF)\\n23. Melnikov, S., Manakongtreecheep, K. & Soll, D. Revising the structural diversity of ribosomal proteins across the three domains of life. *Mol. Biol. Evol.* **35**, 1588–1598 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsy021) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtF2rur7L) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29529322) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5995209) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Revising%20the%20structural%20diversity%20of%20ribosomal%20proteins%20across%20the%20three%20domains%20of%20life&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsy021&volume=35&pages=1588-1598&publication_year=2018&author=Melnikov%2CS&author=Manakongtreecheep%2CK&author=Soll%2CD)\\n24. Miravet-Verde, S., Burgos, R., Delgado, J., Lluch-Senar, M. & Serrano, L. FASTQINS and ANUBIS: two bioinformatic tools to explore facts and artifacts in transposon sequencing and essentiality studies. *Nucleic Acids Res.* **48**, e102 (2020).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkaa679) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXkt1GntbY%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32813015) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7515713) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FASTQINS%20and%20ANUBIS%3A%20two%20bioinformatic%20tools%20to%20explore%20facts%20and%20artifacts%20in%20transposon%20sequencing%20and%20essentiality%20studies&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkaa679&volume=48&publication_year=2020&author=Miravet-Verde%2CS&author=Burgos%2CR&author=Delgado%2CJ&author=Lluch-Senar%2CM&author=Serrano%2CL)\\n25. Borg, A., Pavlov, M. & Ehrenberg, M. Complete kinetic mechanism for recycling of the bacterial ribosome. *RNA* **22**, 10–21 (2016).\\n\\n[Article](https://doi.org/10.1261%2Frna.053157.115) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVKlur3M) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26527791) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4691825) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Complete%20kinetic%20mechanism%20for%20recycling%20of%20the%20bacterial%20ribosome&journal=RNA&doi=10.1261%2Frna.053157.115&volume=22&pages=10-21&publication_year=2016&author=Borg%2CA&author=Pavlov%2CM&author=Ehrenberg%2CM)\\n26. Munro, J. B., Altman, R. B., O’Connor, N. & Blanchard, S. C. Identification of two distinct hybrid state intermediates on the ribosome. *Mol. Cell* **25**, 505–517 (2007).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2007.01.022) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2sXjtlWkur8%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17317624) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2244649) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20two%20distinct%20hybrid%20state%20intermediates%20on%20the%20ribosome&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2007.01.022&volume=25&pages=505-517&publication_year=2007&author=Munro%2CJB&author=Altman%2CRB&author=O%E2%80%99Connor%2CN&author=Blanchard%2CSC)\\n27. Belardinelli, R. et al. Choreography of molecular movements during ribosome progression along mRNA. *Nat. Struct. Mol. Biol.* **23**, 342–348 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.3193) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XktFeqtrk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26999556) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Choreography%20of%20molecular%20movements%20during%20ribosome%20progression%20along%20mRNA&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.3193&volume=23&pages=342-348&publication_year=2016&author=Belardinelli%2CR)\\n28. Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. *Nat. Struct. Mol. Biol.* **23**, 333–341 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.3177) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XivFKhurY%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26926435) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4821728) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiperspective%20smFRET%20reveals%20rate-determining%20late%20intermediates%20of%20ribosomal%20translocation&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.3177&volume=23&pages=333-341&publication_year=2016&author=Wasserman%2CMR&author=Alejo%2CJL&author=Altman%2CRB&author=Blanchard%2CSC)\\n29. Fei, J., Kosuri, P., MacDougall, D. D. & Gonzalez, R. L. Jr Coupling of ribosomal L1 stalk and tRNA dynamics during translation elongation. *Mol. Cell* **30**, 348–359 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2008.03.012) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXmtFaktLg%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18471980) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Coupling%20of%20ribosomal%20L1%20stalk%20and%20tRNA%20dynamics%20during%20translation%20elongation&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2008.03.012&volume=30&pages=348-359&publication_year=2008&author=Fei%2CJ&author=Kosuri%2CP&author=MacDougall%2CDD&author=Gonzalez%2CRL)\\n30. Zhou, J., Lancaster, L., Donohue, J. P. & Noller, H. F. How the ribosome hands the A-site tRNA to the P site during EF-G-catalyzed translocation. *Science* **345**, 1188–1191 (2014).\\n\\n[Article](https://doi.org/10.1126%2Fscience.1255030) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2014Sci...345.1188Z) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsVGhtb%2FO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25190797) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4242719) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=How%20the%20ribosome%20hands%20the%20A-site%20tRNA%20to%20the%20P%20site%20during%20EF-G-catalyzed%20translocation&journal=Science&doi=10.1126%2Fscience.1255030&volume=345&pages=1188-1191&publication_year=2014&author=Zhou%2CJ&author=Lancaster%2CL&author=Donohue%2CJP&author=Noller%2CHF)\\n31. Uemura, S. et al. Real-time tRNA transit on single translating ribosomes at codon resolution. *Nature* **464**, 1012–U1073 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature08925) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.464.1012U) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXkslart7c%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20393556) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4466108) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20tRNA%20transit%20on%20single%20translating%20ribosomes%20at%20codon%20resolution&journal=Nature&doi=10.1038%2Fnature08925&volume=464&pages=1012-U1073&publication_year=2010&author=Uemura%2CS)\\n32. Chen, C. et al. Single-molecule fluorescence measurements of ribosomal translocation dynamics. *Mol. Cell* **42**, 367–377 (2011).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2011.03.024) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3MXlvVyiu7w%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21549313) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3090999) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-molecule%20fluorescence%20measurements%20of%20ribosomal%20translocation%20dynamics&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2011.03.024&volume=42&pages=367-377&publication_year=2011&author=Chen%2CC)\\n33. Wilson, D. N. & Nierhaus, K. H. The E-site story: the importance of maintaining two tRNAs on the ribosome during protein synthesis. *Cell. Mol. Life Sci.* **63**, 2725–2737 (2006).\\n\\n[Article](https://link.springer.com/doi/10.1007/s00018-006-6125-4) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFSrt7w%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17013564) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20E-site%20story%3A%20the%20importance%20of%20maintaining%20two%20tRNAs%20on%20the%20ribosome%20during%20protein%20synthesis&journal=Cell.%20Mol.%20Life%20Sci.&doi=10.1007%2Fs00018-006-6125-4&volume=63&pages=2725-2737&publication_year=2006&author=Wilson%2CDN&author=Nierhaus%2CKH)\\n34. Munro, J. B., Sanbonmatsu, K. Y., Spahn, C. M. & Blanchard, S. C. Navigating the ribosome’s metastable energy landscape. *Trends Biochem. Sci* **34**, 390–400 (2009).\\n\\n[Article](https://doi.org/10.1016%2Fj.tibs.2009.04.004) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVSntrnM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19647434) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2914510) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Navigating%20the%20ribosome%E2%80%99s%20metastable%20energy%20landscape&journal=Trends%20Biochem.%20Sci&doi=10.1016%2Fj.tibs.2009.04.004&volume=34&pages=390-400&publication_year=2009&author=Munro%2CJB&author=Sanbonmatsu%2CKY&author=Spahn%2CCM&author=Blanchard%2CSC)\\n35. Bock, L. V. et al. Energy barriers and driving forces in tRNA translocation through the ribosome. *Nat. Struct. Mol. Biol.* **20**, 1390–1396 (2013).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.2690) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslWisbvK) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24186064) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Energy%20barriers%20and%20driving%20forces%20in%20tRNA%20translocation%20through%20the%20ribosome&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.2690&volume=20&pages=1390-1396&publication_year=2013&author=Bock%2CLV)\\n36. Julian, P. et al. Structure of ratcheted ribosomes with tRNAs in hybrid states. *Proc. Natl Acad. Sci. USA* **105**, 16924–16927 (2008).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.0809587105) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2008PNAS..10516924J) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlKqt7%2FM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18971332) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2579354) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structure%20of%20ratcheted%20ribosomes%20with%20tRNAs%20in%20hybrid%20states&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.0809587105&volume=105&pages=16924-16927&publication_year=2008&author=Julian%2CP)\\n37. Agirrezabala, X. et al. Visualization of the hybrid state of tRNA binding promoted by spontaneous ratcheting of the ribosome. *Mol. Cell* **32**, 190–197 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2008.10.001) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlehsb3L) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18951087) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614368) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualization%20of%20the%20hybrid%20state%20of%20tRNA%20binding%20promoted%20by%20spontaneous%20ratcheting%20of%20the%20ribosome&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2008.10.001&volume=32&pages=190-197&publication_year=2008&author=Agirrezabala%2CX)\\n38. Rodnina, M. V., Peske, F., Peng, B. Z., Belardinelli, R. & Wintermeyer, W. Converting GTP hydrolysis into motion: versatile translational elongation factor G. *Biol. Chem.* **401**, 131–142 (2019).\\n\\n[Article](https://doi.org/10.1515%2Fhsz-2019-0313) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31600135) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Converting%20GTP%20hydrolysis%20into%20motion%3A%20versatile%20translational%20elongation%20factor%20G&journal=Biol.%20Chem.&doi=10.1515%2Fhsz-2019-0313&volume=401&pages=131-142&publication_year=2019&author=Rodnina%2CMV&author=Peske%2CF&author=Peng%2CBZ&author=Belardinelli%2CR&author=Wintermeyer%2CW)\\n39. Lin, J., Zhou, D., Steitz, T. A., Polikanov, Y. S. & Gagnon, M. G. Ribosome-targeting antibiotics: modes of action, mechanisms of resistance, and implications for drug design. *Annu. Rev. Biochem.* **87**, 451–478 (2018).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-biochem-062917-011942) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXlslSrurw%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29570352) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9176271) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome-targeting%20antibiotics%3A%20modes%20of%20action%2C%20mechanisms%20of%20resistance%2C%20and%20implications%20for%20drug%20design&journal=Annu.%20Rev.%20Biochem.&doi=10.1146%2Fannurev-biochem-062917-011942&volume=87&pages=451-478&publication_year=2018&author=Lin%2CJ&author=Zhou%2CD&author=Steitz%2CTA&author=Polikanov%2CYS&author=Gagnon%2CMG)\\n40. Bulkley, D., Innis, C. A., Blaha, G. & Steitz, T. A. Revisiting the structures of several antibiotics bound to the bacterial ribosome. *Proc. Natl Acad. Sci. USA* **107**, 17158–17163 (2010).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1008685107) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010PNAS..10717158B) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlSjurrM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20876130) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2951403) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Revisiting%20the%20structures%20of%20several%20antibiotics%20bound%20to%20the%20bacterial%20ribosome&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1008685107&volume=107&pages=17158-17163&publication_year=2010&author=Bulkley%2CD&author=Innis%2CCA&author=Blaha%2CG&author=Steitz%2CTA)\\n41. Svetlov, M. S. et al. High-resolution crystal structures of ribosome-bound chloramphenicol and erythromycin provide the ultimate basis for their competition. *RNA* **25**, 600–606 (2019).\\n\\n[Article](https://doi.org/10.1261%2Frna.069260.118) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1Ons77I) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30733327) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6467010) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-resolution%20crystal%20structures%20of%20ribosome-bound%20chloramphenicol%20and%20erythromycin%20provide%20the%20ultimate%20basis%20for%20their%20competition&journal=RNA&doi=10.1261%2Frna.069260.118&volume=25&pages=600-606&publication_year=2019&author=Svetlov%2CMS)\\n42. Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. *ACS Chem. Biol.* **2**, 545–552 (2007).\\n\\n[Article](https://doi.org/10.1021%2Fcb700100n) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2sXpvVOhtrc%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17696316) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4624401) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20steric%20block%20in%20translation%20caused%20by%20the%20antibiotic%20spectinomycin&journal=ACS%20Chem.%20Biol.&doi=10.1021%2Fcb700100n&volume=2&pages=545-552&publication_year=2007&author=Borovinskaya%2CMA&author=Shoji%2CS&author=Holton%2CJM&author=Fredrick%2CK&author=Cate%2CJHD)\\n43. Polikanov, Y. S. et al. Distinct tRNA accommodation intermediates observed on the ribosome with the antibiotics hygromycin A and A201A. *Mol. Cell* **58**, 832–844 (2015).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2015.04.014) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2MXpt1Skurg%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26028538) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4458074) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20tRNA%20accommodation%20intermediates%20observed%20on%20the%20ribosome%20with%20the%20antibiotics%20hygromycin%20A%20and%20A201A&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2015.04.014&volume=58&pages=832-844&publication_year=2015&author=Polikanov%2CYS)\\n44. Morse, J. C. et al. Elongation factor-Tu can repetitively engage aminoacyl-tRNA within the ribosome during the proofreading stage of tRNA selection. *Proc. Natl Acad. Sci. USA* **117**, 3610–3620 (2020).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1904469117) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020PNAS..117.3610M) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXjsFyhu7Y%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32024753) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7035488) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Elongation%20factor-Tu%20can%20repetitively%20engage%20aminoacyl-tRNA%20within%20the%20ribosome%20during%20the%20proofreading%20stage%20of%20tRNA%20selection&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1904469117&volume=117&pages=3610-3620&publication_year=2020&author=Morse%2CJC)\\n45. Choi, J. et al. Dynamics of the context-specific translation arrest by chloramphenicol and linezolid. *Nat. Chem. Biol.* **16**, 310–317 (2020).\\n\\n[Article](https://doi.org/10.1038%2Fs41589-019-0423-2) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVSktb%2FM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31844301) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dynamics%20of%20the%20context-specific%20translation%20arrest%20by%20chloramphenicol%20and%20linezolid&journal=Nat.%20Chem.%20Biol.&doi=10.1038%2Fs41589-019-0423-2&volume=16&pages=310-317&publication_year=2020&author=Choi%2CJ)\\n46. Marks, J. et al. Context-specific inhibition of translation by ribosomal antibiotics targeting the peptidyl transferase center. *Proc. Natl Acad. Sci. USA* **113**, 12150–12155 (2016).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1613055113) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016PNAS..11312150M) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1elur3N) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27791002) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5086994) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Context-specific%20inhibition%20of%20translation%20by%20ribosomal%20antibiotics%20targeting%20the%20peptidyl%20transferase%20center&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1613055113&volume=113&pages=12150-12155&publication_year=2016&author=Marks%2CJ)\\n47. Vazquez-Laslop, N. & Mankin, A. S. Context-specific action of ribosomal antibiotics. *Annu. Rev. Microbiol.* **72**, 185–207 (2018).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-micro-090817-062329) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFCjs77O) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29906204) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8742604) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Context-specific%20action%20of%20ribosomal%20antibiotics&journal=Annu.%20Rev.%20Microbiol.&doi=10.1146%2Fannurev-micro-090817-062329&volume=72&pages=185-207&publication_year=2018&author=Vazquez-Laslop%2CN&author=Mankin%2CAS)\\n48. Aguirre Rivera, J. et al. Real-time measurements of aminoglycoside effects on protein synthesis in live cells. *Proc. Natl Acad. Sci. USA* **118**, e2013315118 (2021).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.2013315118) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33619089) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7936356) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20measurements%20of%20aminoglycoside%20effects%20on%20protein%20synthesis%20in%20live%20cells&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.2013315118&volume=118&publication_year=2021&author=Aguirre%20Rivera%2CJ)\\n49. Baquero, F. & Levin, B. R. Proximate and ultimate causes of the bactericidal action of antibiotics. *Nat. Rev. Microbiol.* **19**, 123–132 (2021).\\n50. Myasnikov, A. G. et al. The molecular structure of the left-handed supra-molecular helix of eukaryotic polyribosomes. *Nat. Commun.* **5**, 5294 (2014).\\n\\n[Article](https://doi.org/10.1038%2Fncomms6294) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2014NatCo...5.5294M) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFantrfM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25376914) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20molecular%20structure%20of%20the%20left-handed%20supra-molecular%20helix%20of%20eukaryotic%20polyribosomes&journal=Nat.%20Commun.&doi=10.1038%2Fncomms6294&volume=5&publication_year=2014&author=Myasnikov%2CAG)\\n51. Dunkle, J. A. & Cate, J. H. D. in *Ribosomes* (eds Rodnina, M. V., Wintermeyer, W. & Green, R.) Ch. 6 (2011).\\n52. Herr, A. J., Nelson, C. C., Wills, N. M., Gesteland, R. F. & Atkins, J. F. Analysis of the roles of tRNA structure, ribosomal protein L9, and the bacteriophage T4 gene 60 bypassing signals during ribosome slippage on mRNA. *J. Mol. Biol.* **309**, 1029–1048 (2001).\\n\\n[Article](https://doi.org/10.1006%2Fjmbi.2001.4717) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD3MXkslGgurY%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11399077) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20the%20roles%20of%20tRNA%20structure%2C%20ribosomal%20protein%20L9%2C%20and%20the%20bacteriophage%20T4%20gene%2060%20bypassing%20signals%20during%20ribosome%20slippage%20on%20mRNA&journal=J.%20Mol.%20Biol.&doi=10.1006%2Fjmbi.2001.4717&volume=309&pages=1029-1048&publication_year=2001&author=Herr%2CAJ&author=Nelson%2CCC&author=Wills%2CNM&author=Gesteland%2CRF&author=Atkins%2CJF)\\n53. Smith, A. M., Costello, M. S., Kettring, A. H., Wingo, R. J. & Moore, S. D. Ribosome collisions alter frameshifting at translational reprogramming motifs in bacterial mRNAs. *Proc. Natl Acad. Sci. USA* **116**, 21769–21779 (2019).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1910613116) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019PNAS..11621769S) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXitValtbjM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31591196) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6815119) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome%20collisions%20alter%20frameshifting%20at%20translational%20reprogramming%20motifs%20in%20bacterial%20mRNAs&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1910613116&volume=116&pages=21769-21779&publication_year=2019&author=Smith%2CAM&author=Costello%2CMS&author=Kettring%2CAH&author=Wingo%2CRJ&author=Moore%2CSD)\\n54. Tegunov, D. & Cramer, P. Real-time cryo-electron microscopy data preprocessing with Warp. *Nat. Methods* **16**, 1146–1152 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0580-y) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFejs77E) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31591575) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6858868) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20cryo-electron%20microscopy%20data%20preprocessing%20with%20Warp&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0580-y&volume=16&pages=1146-1152&publication_year=2019&author=Tegunov%2CD&author=Cramer%2CP)\\n55. Hagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. *J. Struct. Biol.* **197**, 191–198 (2017).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2016.06.007) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27313000) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5287356) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Implementation%20of%20a%20cryo-electron%20tomography%20tilt-scheme%20optimized%20for%20high%20resolution%20subtomogram%20averaging&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2016.06.007&volume=197&pages=191-198&publication_year=2017&author=Hagen%2CWJH&author=Wan%2CW&author=Briggs%2CJAG)\\n56. Schorb, M., Haberbosch, I., Hagen, W. J. H., Schwab, Y. & Mastronarde, D. N. Software tools for automated transmission electron microscopy. *Nat. Methods* **16**, 471–477 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0396-9) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVSqs7zL) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31086343) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7000238) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Software%20tools%20for%20automated%20transmission%20electron%20microscopy&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0396-9&volume=16&pages=471-477&publication_year=2019&author=Schorb%2CM&author=Haberbosch%2CI&author=Hagen%2CWJH&author=Schwab%2CY&author=Mastronarde%2CDN)\\n57. Hrabe, T. et al. PyTom: a python-based toolbox for localization of macromolecules in cryo-electron tomograms and subtomogram analysis. *J. Struct. Biol.* **178**, 177–188 (2012).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2011.12.003) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC38Xnt1ehtL4%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22193517) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PyTom%3A%20a%20python-based%20toolbox%20for%20localization%20of%20macromolecules%20in%20cryo-electron%20tomograms%20and%20subtomogram%20analysis&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2011.12.003&volume=178&pages=177-188&publication_year=2012&author=Hrabe%2CT)\\n58. Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. *Nat. Protoc.* **11**, 2054–2065 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnprot.2016.124) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XhsF2mtLrL) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27685097) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5215819) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Resolving%20macromolecular%20structures%20from%20electron%20cryo-tomography%20data%20using%20subtomogram%20averaging%20in%20RELION&journal=Nat.%20Protoc.&doi=10.1038%2Fnprot.2016.124&volume=11&pages=2054-2065&publication_year=2016&author=Bharat%2CTA&author=Scheres%2CSH)\\n59. Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. *eLife* **7**, e42166 (2018).\\n\\n[Article](https://doi.org/10.7554%2FeLife.42166) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30412051) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6250425) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=New%20tools%20for%20automated%20high-resolution%20cryo-EM%20structure%20determination%20in%20RELION-3&journal=eLife&doi=10.7554%2FeLife.42166&volume=7&publication_year=2018&author=Zivanov%2CJ)\\n60. Waterhouse, A. et al. SWISS-MODEL: homology modelling of protein structures and complexes. *Nucleic Acids Res.* **46**, W296–W303 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky427) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyru70%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29788355) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030848) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SWISS-MODEL%3A%20homology%20modelling%20of%20protein%20structures%20and%20complexes&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky427&volume=46&pages=W296-W303&publication_year=2018&author=Waterhouse%2CA)\\n61. Pruesse, E., Peplies, J. & Glockner, F. O. SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes. *Bioinformatics* **28**, 1823–1829 (2012).\\n\\n[Article](https://doi.org/10.1093%2Fbioinformatics%2Fbts252) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVSgsbfO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22556368) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3389763) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SINA%3A%20accurate%20high-throughput%20multiple%20sequence%20alignment%20of%20ribosomal%20RNA%20genes&journal=Bioinformatics&doi=10.1093%2Fbioinformatics%2Fbts252&volume=28&pages=1823-1829&publication_year=2012&author=Pruesse%2CE&author=Peplies%2CJ&author=Glockner%2CFO)\\n62. Rother, M., Rother, K., Puton, T. & Bujnicki, J. M. ModeRNA: a tool for comparative modeling of RNA 3D structure. *Nucleic Acids Res.* **39**, 4007–4022 (2011).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkq1320) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3MXmvValsL4%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21300639) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3105415) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ModeRNA%3A%20a%20tool%20for%20comparative%20modeling%20of%20RNA%203D%20structure&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkq1320&volume=39&pages=4007-4022&publication_year=2011&author=Rother%2CM&author=Rother%2CK&author=Puton%2CT&author=Bujnicki%2CJM)\\n63. Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. *J. Comput. Chem.* **25**, 1605–1612 (2004).\\n\\n[Article](https://doi.org/10.1002%2Fjcc.20084) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2cXmvVOhsbs%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15264254) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCSF%20Chimera%E2%80%94a%20visualization%20system%20for%20exploratory%20research%20and%20analysis&journal=J.%20Comput.%20Chem.&doi=10.1002%2Fjcc.20084&volume=25&pages=1605-1612&publication_year=2004&author=Pettersen%2CEF)\\n64. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. *Acta Crystallogr D* **74**, 531–544 (2018).\\n\\n[Article](https://doi.org/10.1107%2FS2059798318006551) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2ltbzP) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-space%20refinement%20in%20PHENIX%20for%20cryo-EM%20and%20crystallography&journal=Acta%20Crystallogr%20D&doi=10.1107%2FS2059798318006551&volume=74&pages=531-544&publication_year=2018&author=Afonine%2CPV)\\n65. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. *Acta Crystallogr D* **66**, 486–501 (2010).\\n\\n[Article](https://doi.org/10.1107%2FS0907444910007493) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXksFKisb8%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20383002) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2852313) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Features%20and%20development%20of%20Coot&journal=Acta%20Crystallogr%20D&doi=10.1107%2FS0907444910007493&volume=66&pages=486-501&publication_year=2010&author=Emsley%2CP&author=Lohkamp%2CB&author=Scott%2CWG&author=Cowtan%2CK)\\n66. Williams, C. J. et al. MolProbity: more and better reference data for improved all-atom structure validation. *Protein Sci.* **27**, 293–315 (2018).\\n\\n[Article](https://doi.org/10.1002%2Fpro.3330) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvVKgsrnO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29067766) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MolProbity%3A%20more%20and%20better%20reference%20data%20for%20improved%20all-atom%20structure%20validation&journal=Protein%20Sci.&doi=10.1002%2Fpro.3330&volume=27&pages=293-315&publication_year=2018&author=Williams%2CCJ)\\n67. Chen, S. et al. High-resolution noise substitution to measure overfitting and validate resolution in 3D structure determination by single particle electron cryomicroscopy. *Ultramicroscopy* **135**, 24–35 (2013).\\n\\n[Article](https://doi.org/10.1016%2Fj.ultramic.2013.06.004) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslWhtLzO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23872039) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3834153) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-resolution%20noise%20substitution%20to%20measure%20overfitting%20and%20validate%20resolution%20in%203D%20structure%20determination%20by%20single%20particle%20electron%20cryomicroscopy&journal=Ultramicroscopy&doi=10.1016%2Fj.ultramic.2013.06.004&volume=135&pages=24-35&publication_year=2013&author=Chen%2CS)\\n68. Tatusov, R. L., Galperin, M. Y., Natale, D. A. & Koonin, E. V. The COG database: a tool for genome-scale analysis of protein functions and evolution. *Nucleic Acids Res.* **28**, 33–36 (2000).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2F28.1.33) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD3cXhvVGqu7w%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10592175) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC102395) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20COG%20database%3A%20a%20tool%20for%20genome-scale%20analysis%20of%20protein%20functions%20and%20evolution&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2F28.1.33&volume=28&pages=33-36&publication_year=2000&author=Tatusov%2CRL&author=Galperin%2CMY&author=Natale%2CDA&author=Koonin%2CEV)\\n69. Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. *Nucleic Acids Res.* **47**, D309–D314 (2019).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky1085) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Gqtr%2FL) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30418610) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=eggNOG%205.0%3A%20a%20hierarchical%2C%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky1085&volume=47&pages=D309-D314&publication_year=2019&author=Huerta-Cepas%2CJ)\\n70. Katoh, K., Rozewicki, J. & Yamada, K. D. MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization. *Brief. Bioinform.* **20**, 1160–1166 (2019).\\n\\n[Article](https://doi.org/10.1093%2Fbib%2Fbbx108) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1Kqtb3K) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28968734) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MAFFT%20online%20service%3A%20multiple%20sequence%20alignment%2C%20interactive%20sequence%20choice%20and%20visualization&journal=Brief.%20Bioinform.&doi=10.1093%2Fbib%2Fbbx108&volume=20&pages=1160-1166&publication_year=2019&author=Katoh%2CK&author=Rozewicki%2CJ&author=Yamada%2CKD)\\n71. Letunic, I. & Bork, P. Interactive Tree Of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. *Nucleic Acids Res.* **49**, W293–W296 (2021).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkab301) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvV2isLvM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33885785) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8265157) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interactive%20Tree%20Of%20Life%20%28iTOL%29%20v5%3A%20an%20online%20tool%20for%20phylogenetic%20tree%20display%20and%20annotation&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkab301&volume=49&pages=W293-W296&publication_year=2021&author=Letunic%2CI&author=Bork%2CP)\\n72. Huerta-Cepas, J. et al. Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper. *Mol. Biol. Evol.* **34**, 2115–2122 (2017).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsx148) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFaru7vF) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28460117) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850834) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20genome-wide%20functional%20annotation%20through%20orthology%20assignment%20by%20eggNOG-Mapper&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsx148&volume=34&pages=2115-2122&publication_year=2017&author=Huerta-Cepas%2CJ)\\n73. Huerta-Cepas, J., Serra, F. & Bork, P. ETE 3: reconstruction, analysis, and visualization of phylogenomic data. *Mol. Biol. Evol.* **33**, 1635–1638 (2016).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsw046) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFeisrrN) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26921390) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4868116) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ETE%203%3A%20reconstruction%2C%20analysis%2C%20and%20visualization%20of%20phylogenomic%20data&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsw046&volume=33&pages=1635-1638&publication_year=2016&author=Huerta-Cepas%2CJ&author=Serra%2CF&author=Bork%2CP)\\n74. Meszaros, B., Erdos, G. & Dosztanyi, Z. IUPred2A: context-dependent prediction of protein disorder as a function of redox state and protein binding. *Nucleic Acids Res.* **46**, W329–W337 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky384) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyrt7o%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29860432) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030935) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=IUPred2A%3A%20context-dependent%20prediction%20of%20protein%20disorder%20as%20a%20function%20of%20redox%20state%20and%20protein%20binding&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky384&volume=46&pages=W329-W337&publication_year=2018&author=Meszaros%2CB&author=Erdos%2CG&author=Dosztanyi%2CZ)\\n75. Cole, C., Barber, J. D. & Barton, G. J. The Jpred 3 secondary structure prediction server. *Nucleic Acids Res.* **36**, W197–W201 (2008).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkn238) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXptlKkt7w%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18463136) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2447793) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Jpred%203%20secondary%20structure%20prediction%20server&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkn238&volume=36&pages=W197-W201&publication_year=2008&author=Cole%2CC&author=Barber%2CJD&author=Barton%2CGJ)\\n76. Scheres, S. H. Processing of structurally heterogeneous cryo-EM data in RELION. *Methods Enzymol.* **579**, 125–157 (2016).\\n\\n[Article](https://doi.org/10.1016%2Fbs.mie.2016.04.012) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXmtlKnurk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27572726) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Processing%20of%20structurally%20heterogeneous%20cryo-EM%20data%20in%20RELION&journal=Methods%20Enzymol.&doi=10.1016%2Fbs.mie.2016.04.012&volume=579&pages=125-157&publication_year=2016&author=Scheres%2CSH)\\n77. Kidmose, R. T. et al. Namdinator—automatic molecular dynamics flexible fitting of structural models into cryo-EM and crystallography experimental maps. *IUCrJ* **6**, 526–531 (2019).\\n\\n[Article](https://doi.org/10.1107%2FS2052252519007619) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlWit7%2FO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31316797) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6608625) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Namdinator%E2%80%94automatic%20molecular%20dynamics%20flexible%20fitting%20of%20structural%20models%20into%20cryo-EM%20and%20crystallography%20experimental%20maps&journal=IUCrJ&doi=10.1107%2FS2052252519007619&volume=6&pages=526-531&publication_year=2019&author=Kidmose%2CRT)\\n78. Trabuco, L. G., Villa, E., Mitra, K., Frank, J. & Schulten, K. Flexible fitting of atomic structures into electron microscopy maps using molecular dynamics. *Structure* **16**, 673–683 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.str.2008.03.005) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXlslGrtro%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18462672) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2430731) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Flexible%20fitting%20of%20atomic%20structures%20into%20electron%20microscopy%20maps%20using%20molecular%20dynamics&journal=Structure&doi=10.1016%2Fj.str.2008.03.005&volume=16&pages=673-683&publication_year=2008&author=Trabuco%2CLG&author=Villa%2CE&author=Mitra%2CK&author=Frank%2CJ&author=Schulten%2CK)\\n79. Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. *J. Chem. Phys.* **153**, 044130 (2020).\\n\\n[Article](https://doi.org/10.1063%2F5.0014475) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFajsL3J) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32752662) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7395834) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scalable%20molecular%20dynamics%20on%20CPU%20and%20GPU%20architectures%20with%20NAMD&journal=J.%20Chem.%20Phys.&doi=10.1063%2F5.0014475&volume=153&publication_year=2020&author=Phillips%2CJC)\\n80. Nickell, S. et al. TOM software toolbox: acquisition and analysis for electron tomography. *J. Struct. Biol.* **149**, 227–234 (2005).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2004.10.006) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15721576) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TOM%20software%20toolbox%3A%20acquisition%20and%20analysis%20for%20electron%20tomography&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2004.10.006&volume=149&pages=227-234&publication_year=2005&author=Nickell%2CS)\\n81. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. *Protein Sci.* **30**, 70–82 (2021).\\n\\n[Article](https://doi.org/10.1002%2Fpro.3943) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFamtrzJ) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32881101) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCSF%20ChimeraX%3A%20structure%20visualization%20for%20researchers%2C%20educators%2C%20and%20developers&journal=Protein%20Sci.&doi=10.1002%2Fpro.3943&volume=30&pages=70-82&publication_year=2021&author=Pettersen%2CEF)\\n\\n[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41586-022-05255-2?format=refman&flavour=references)\\n\\nAcknowledgements\\n----------------\\n\\nWe thank I. Zagoriy, L. Adam, F. Weis, W. Hagen, T. Hoffmann and EMBL IT for technical support; L. Serrano for providing the *M.* *pneumoniae* transposon library and proteomics data; M. Kuhn, F. Marotta and J. C. Somody for discussions on bioinformatic and statistical data analysis; and C. Spahn, A. Jakobi and S. Pfeffer for discussions and critical reading of the manuscript. J.R. acknowledges the Deutsche Forschungsgemeinschaft (project no. 426290502) and the Wellcome Trust Senior Research Fellowship (103139). J.M. acknowledges the EMBL and the European Research Council starting grant (3DCellPhase-, grant no. 760067).\\n\\nAuthor information\\n------------------\\n\\n### Authors and Affiliations\\n\\n1. Structural and Computational Biology Unit, European Molecular Biology Laboratory (EMBL), Heidelberg, Germany\\n\\nLiang Xue, Maria Zimmermann-Kogadeeva, Peer Bork & Julia Mahamid\\n2. Collaboration for joint PhD degree between EMBL and Heidelberg University, Faculty of Biosciences, Heidelberg, Germany\\n\\nLiang Xue\\n3. Chair of Bioanalytics, Technische Universität Berlin, Berlin, Germany\\n\\nSwantje Lenz & Juri Rappsilber\\n4. Department of Molecular Biology, Max-Planck-Institute for Biophysical Chemistry, Göttingen, Germany\\n\\nDimitry Tegunov & Patrick Cramer\\n5. Yonsei Frontier Lab, Yonsei University, Seoul, South Korea\\n\\nPeer Bork\\n6. Department of Bioinformatics, Biocenter, University of Würzburg, Würzburg, Germany\\n\\nPeer Bork\\n7. Wellcome Centre for Cell Biology, University of Edinburgh, Edinburgh, UK\\n\\nJuri Rappsilber\\n\\nAuthors1. Liang Xue[View author publications](/search?author=Liang%20Xue)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Liang%20Xue) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Liang%20Xue%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n2. Swantje Lenz[View author publications](/search?author=Swantje%20Lenz)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Swantje%20Lenz) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Swantje%20Lenz%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n3. Maria Zimmermann-Kogadeeva[View author publications](/search?author=Maria%20Zimmermann-Kogadeeva)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Maria%20Zimmermann-Kogadeeva) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Maria%20Zimmermann-Kogadeeva%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n4. Dimitry Tegunov[View author publications](/search?author=Dimitry%20Tegunov)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Dimitry%20Tegunov) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Dimitry%20Tegunov%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n5. Patrick Cramer[View author publications](/search?author=Patrick%20Cramer)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Patrick%20Cramer) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Patrick%20Cramer%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n6. Peer Bork[View author publications](/search?author=Peer%20Bork)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Peer%20Bork) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Peer%20Bork%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n7. Juri Rappsilber[View author publications](/search?author=Juri%20Rappsilber)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Juri%20Rappsilber) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Juri%20Rappsilber%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n8. Julia Mahamid[View author publications](/search?author=Julia%20Mahamid)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Julia%20Mahamid) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Julia%20Mahamid%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n### Contributions\\n\\nL.X. and J.M. designed the study. L.X. collected and processed the cryo-ET data and performed structural analysis. L.X. and S.L. built the atomic models with support from J.R. M.Z.-K. performed the bioinformatic and statistical analysis with support from P.B. D.T. and P.C. provided software support and advice on data interpretation. L.X. and J.M. wrote the manuscript with input from all authors.\\n\\n### Corresponding author\\n\\nCorrespondence to\\n [Julia Mahamid](mailto:julia.mahamid@embl.de).\\n\\nEthics declarations\\n-------------------\\n\\n\\n### Competing interests\\n\\n\\nThe authors declare no competing interests.\\n\\n\\nPeer review\\n-----------\\n\\n\\n### Peer review information\\n\\n\\n*Nature* thanks Elizabeth Villa and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.\\n\\n\\nAdditional information\\n----------------------\\n\\n**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\\n\\nExtended data figures and tables\\n--------------------------------\\n\\n### [Extended Data Fig. 1 *M. pneumoniae* in-cell ribosome maps.](/articles/s41586-022-05255-2/figures/5)\\n\\n**a**, 70S ribosome map determined from 77,539 sub-tomograms from 356 untreated *M. pneumoniae* cells. **b**, Map coloured by local resolutions. 50S has the highest local resolutions as it dominates the overall alignment. The relatively lower local resolutions of the 30S indicate their high flexibility during active translation. **c**, Fourier shell correlation (FSC) curves for global 70S and focused 30S refinement, and the reported resolution value at FSC = 0.143. The Nyquist limit for the data is 3.4 Å. **d**, 70S ribosome map determined from 18,987 sub-tomograms from 65 Cm-treated cells. **e**, Map coloured by local resolutions. **f**, FSC curves of the Cm-treated 70S ribosome, and of focused refinements on 30S and 50S respectively. **g**-**h**, Focused refined 30S and 50S maps from the Cm-treated dataset. **i**-**j**, The corresponding local-resolution maps. Atomic models for 30S and 50S were first built based on maps in **g** and **h**. **k**, Model-to-map FSC curves for 30S and 50S, respectively.\\n\\n### [Extended Data Fig. 2 Structural features of the *M. pneumoniae* ribosome.](/articles/s41586-022-05255-2/figures/6)\\n\\n**a**,**b**, Examples of regions of the atomic model fitted into the density map for rRNAs (**a**) and proteins (**b**). **c**, Densities corresponding to ions are also observed. **d**, Extensions of ribosomal proteins S6, L22 and L29 form secondary structures (between arrowheads) that were clearly resolved in the map. **e**, The 70S model showing ribosomal proteins with sequence extensions (dark green). The extensions built in the model are highlighted in magenta. In addition to the helix-forming sequences of S6, L22 and L29, some loops can be traced, especially the long C-terminal loop of S6. **f**, Whole-cell cross-linking mass spectrometry data confirms the model built for the long loop extension of S6. **g**, The cross-linking network of S6 identified in the previous work[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"), with the sequence range (1–167) resolved in the map coloured in green.\\n\\n### [Extended Data Fig. 3 Ribosomal protein extensions.](/articles/s41586-022-05255-2/figures/7)\\n\\n**a**, Eleven ribosomal proteins in *M. pneumoniae* have sequence extensions compared to *E. coli*. Along the sequences, cross-links identified by in-cell cross-linking mass spectrometry, secondary structure prediction, disorder prediction, and sequence essentiality prediction based on a transposon mutation library are displayed (quantified in Supplementary Table [2](/articles/s41586-022-05255-2#MOESM1)). **b**, Extensions in the eleven ribosomal proteins are found throughout bacteria, but are not specific to any sub-groups.\\n\\n### [Extended Data Fig. 4 Classification and refinement of ribosomes in native untreated cells.](/articles/s41586-022-05255-2/figures/8)\\n\\n**a**, A diagram of the image-processing workflow. The classification process can be grouped into three tiers: global classification, focused classification on the tRNA path region (mask I), and focused classification on elongation factor and A/T tRNA binding region (mask II). For each class, a unique number identifier and class name are assigned for tracking. The particle number and the global map resolution at FSC = 0.143 are provided for each class. **b**-**d**, FSC curves for all refined maps.\\n\\n### [Extended Data Fig. 5 Validation of ribosome detection and classification.](/articles/s41586-022-05255-2/figures/9)\\n\\n**a**, Mask I for focused classification on tRNA path region. **b**, A representative RELION classification job with mask I. Each line indicates the change in particle numbers in one class over 25 iterations. Classes that show the same structure were grouped according to the tRNA occupancy ('a, P/E', 'P, E', 'P', 'A, P'). 'a, P/E' contains heterogeneous density around the A site. **c**, Results of the classification job as shown in **b**, and of three additional parallel jobs. **d**, Results of following classification jobs that further classify the 'a, P/E' class into 'A\\\\*, P/E' and 'A/P, P/E'. **e**, Mask II for focused classification on elongation factor (EF) and A/T tRNA sites. **f**, Changes in particle numbers over iterations in a representative classification job with the mask II. **g**, Results of parallel focused classification jobs with mask II. **h**, Distribution of the ribosome classes against template matching cross-correlation scores used for ribosome localization. For each of the 356 tomograms of untreated cells, the 400 highest scoring hits were extracted and ranked. Obvious false positives were manually excluded first. Additional false positives were identified during RELION classification. 70S classes that are structurally similar were grouped in the plot for better visualization. **i**, Same as **h**, but only showing the 70S classes in the elongation phase. The proportions of different 70S classes remain stable across the top 400 hits, demonstrating that the classification results are not biased by the ribosome picking.\\n\\n### [Extended Data Fig. 6 Local-resolution maps for ribosome classes in untreated cells.](/articles/s41586-022-05255-2/figures/10)\\n\\nMaps of the 15 classes determined in the dataset of untreated cells (Extended Data Fig. [4](/articles/s41586-022-05255-2#Fig8)), coloured by local resolution calculated in RELION.\\n\\n### [Extended Data Fig. 7 Model building for ribosome classes in untreated cells.](/articles/s41586-022-05255-2/figures/11)\\n\\n**a**-**j**, Models of the ten ribosome classes in the elongation phase constructed by flexible fitting. **k**, The model for the ribosome with hybrid P/E-site tRNA. **l**, Free 50S in complex with ribosome recycling factor (RRF).\\n\\n### [Extended Data Fig. 8 Early-to-late translocation intermediates during elongation.](/articles/s41586-022-05255-2/figures/12)\\n\\n**a**, Early translocation intermediates prior to EF-G binding. Only tRNAs in the aligned class models are displayed. **b**, Early-to-late translocation intermediates in the presence of EF-G. Continuous structural changes are observed from class 6e to 7 to 8, including a roughly 20 Å movement of EF-G’s domain IV toward the A site (black arrow), an overall rotation of the entire EF-G (orange arrows) and inter-domain conformational changes. **c**-**e**, Densities and models for EF-G and tRNAs in classes 6e, 7 and 8. In class 6e, EF-G’s domain IV is less resolved in the map (red circle). It may contain a mixture of intermediate states with domain IV moving toward the A site. The fitted model represents the average position. Further classification could not address the high flexibility. **f**, From class 6e to 7, EF-G undergoes a small overall rotation (orange arrows) and an inter-domain rearrangement. Inset (rotated view, aligned on EF-G domains I-III) shows rotation of domain IV relative to other domains. Movement of EF-G’s domain IV towards the A site results from both overall rotation on the ribosome and inter-domain rearrangement. **g**, From class 7 to 8, EF-G rotates as one body without significant inter-domain rearrangement (inset, rotated view, aligned on EF-G domains I-III). **h**, Class 6e shows structural similarity to a reported early translocation state 'H1-EF-G–GDP-pi' (PDB 7PJV) determined by Petrychenko *et al*. 2021[15](/articles/s41586-022-05255-2#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\"). **i**, Class 7 resembles 'PRE–EF-G–GDP–Pi' (PDB 7SSL) reported by Carbone *et al*. 2021[16](/articles/s41586-022-05255-2#ref-CR16 \\\"Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. Nat. Commun. 12, 7236 (2021).\\\"), and 'INT1' (PDB 7N2V, not shown) by Rundlet *et al*. 2021[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"). **j**, Class 8 resembles the late translocation intermediates reported by Rundlet *et al*. 2021[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\") ('INT2', PDB 7N2C) and Petrychenko *et al*. 2021[15](/articles/s41586-022-05255-2#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\") ('CHI1-EF-G-GDP', PDB 7PJY). **k**-**m**, Minor inter-domain conformational difference found between EF-Gs in our models and those in the reported in vitro structures.\\n\\n### [Extended Data Fig. 9 Classification, refinement and modelling of ribosomes in Cm-treated cells.](/articles/s41586-022-05255-2/figures/13)\\n\\n**a**, Image-processing workflow for the Cm-treated dataset. The sub-tomogram classification and refinement are similar to those developed for the untreated dataset (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8)). Heterogeneity may exist in the minor classes (2a, 3, 5), but the low particle numbers hindered further classification. **b**, FSC curves for all classes calculated following RELION refinements. **c**, Local-resolution maps. **d**, Density of the Cm molecule is resolved in the major 'A, P' class, but not in the other three minor classes owing to relatively low resolutions of these maps. **e**, Models built for the four classes, fitted into their corresponding densities. **f**, Elongation factors and tRNAs in the four classes. **g**, In the predominant 'A, P' class, mRNA, A- and P-site tRNA, the nascent peptide chain, and the Cm drug are well-resolved. The nascent chain has strong continuous density linked to the P-site tRNA, resulting from the inhibition of peptidyl transfer by the Cm molecule.\\n\\n### [Extended Data Fig. 10 Classification, refinement and modelling of ribosomes in Spc-treated cells.](/articles/s41586-022-05255-2/figures/14)\\n\\n**a**, Image-processing workflow for the Spc-treated dataset. The sub-tomogram classification and refinement are similar to those developed for the untreated dataset (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8)), except for exclusion of the manual inspection step after template matching. **b**, FSC curves for all classes calculated following refinements. **c**, Local-resolution maps for Spc-treated 70S classes. **d**, Density corresponding to the Spc molecule (magenta) is clearly resolved in the major 'EF-G, A/PSpc, P/E' class. Densities from the untreated and PUM-treated data, where Spc is not present, are shown for comparison. **e**, Models built for the five Spc-treated 70S classes. **f**, Elongation factors and tRNAs in the five classes. **g**, The model of 'EF-G, A/PSpc, P/E' shows Spc binds to the 30S neck region, confirming its role in inhibiting 30S head dynamics and mRNA translocation.\\n\\n### [Extended Data Fig. 11 Classification, refinement and modelling of ribosomes in PUM-treated cells.](/articles/s41586-022-05255-2/figures/15)\\n\\n**a**, Image-processing workflow for the PUM-treated dataset. The sub-tomogram classification and refinement are similar to those developed for the untreated dataset (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8)). **b**, Distribution of translation states for ribosomes collided with a PUM-stalled RNA polymerase (purple; stalled expressome) and for the remaining ribosomes (green) in the PUM-treated cells. **c**, FSC curves for all classes calculated after refinements. **d**-**f**, Local-resolution maps. **g**, Models built for PUM-treated 70S classes. **h**, Elongation factors and tRNAs in the six 70S ribosome classes.\\n\\n### [Extended Data Fig. 12 Spatial analysis of ribosomes and polysomes in native untreated cells.](/articles/s41586-022-05255-2/figures/16)\\n\\n**a**, 70S ribosomes (grey) and detected polysomes (light blue) in a representative tomogram. **b**, Distribution of neighbouring ribosomes within a 50 nm centre-to-centre distance. Rotation normalized. **c**, Illustration of the polysome detection approach based on the distance from the mRNA exit site of one ribosome to the mRNA entry site of the next. **d**, Histogram of distances from the mRNA exit site of one ribosome to the mRNA entry sites of all neighbouring ribosomes. **e**, Percentages of ribosomes detected as polysomes using different distance thresholds (d). Mean and s.d. are across 356 tomograms (n = 356 cells). A 7 nm threshold was selected. **f**, In polysomes, the following ribosome (i) adopts various orientations relative to the preceding ribosome (i+1), mainly with regard to rotation around a plane perpendicular to the preceding ribosome’s mRNA exit site. **g**, Rotations of the following ribosome relative to the preceding ribosome in polysomes. The two clusters correspond to the previously defined 't-t' and 't-b' configurations. **h**-**i**, Positions of the following ribosomes (coloured according to different rotations) relative to the preceding ribosome (triangle indicates the mRNA exit site). **j**-**k**, Two representative di-ribosome pairs in polysomes with extended L9. **l**-**m**, Clash between the C-terminal domain of the extended L9 and the superposed EF-G and EF-Tu, respectively. **n**, *De novo* focused classification on the L9 region using a spherical mask (light green) of 70S ribosomes resulted in three classes: no L9 resolved (I), flat L9 (II) and extended L9 (III). The additional density connecting to the extended L9 originates from the neighbouring ribosome. **o**, Overlap between polysomes independently defined based on spatial analysis, RELION classification for polysomes and classified ribosomes with the extended L9. **p**, The extended L9 is more frequent in the 't-t' ribosome pairs, especially in compacted ones.\\n\\n### [Extended Data Fig. 13 Correlation of translation elongation states within polysomes in native untreated cells.](/articles/s41586-022-05255-2/figures/17)\\n\\n**a**, Occurrence frequencies of elongation state pairs of two adjacent ribosomes (preceding ribosome i versus following ribosome i+1) within polysomes calculated from the experimental data. **b**, Theoretical frequencies of elongation state pairs if there is no cross-influence within the polysome and ribosome state pairs form randomly. **c**, Frequencies of elongation state pairs calculated after random shuffling of the experimental data. **d**, Schematic representation of polysome shuffling analysis and permutation *p*-value calculation. Detailed procedure can be found in the Materials and Methods. **e**, Comparison of the experimental and shuffled pair fractions for all major pairs. **f**, Fold changes between the experimental and shuffled pair fractions from the shuffling experiments. Ribosomes of states that need elongation factor binding to proceed (states 1 and 2a for EF-Tu, and states 5 and 6a for EF-G) are more frequently engaged as the following ribosomes. Permutation *p*-values were adjusted for multiple hypotheses testing with the Benjamini–Hochberg procedure, and are provided in Supplementary Table [7](/articles/s41586-022-05255-2#MOESM1). **g**, Difference between experimental and theoretical pair frequencies when using different distance thresholds for polysome definition. **h**, Ratios of the number of ribosomes in each state being the preceding one against the following one in polysome pairs (across all polysomes), calculated using different distance thresholds to define polysomes. Symmetric engagement as the preceding and following ribosome results in a ratio of 1.\\n\\nSupplementary information\\n-------------------------\\n\\n### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_MOESM1_ESM.pdf)\\n\\nThis file contains Supplementary Discussion; Supplementary Tables 1–8; legends for Supplementary Videos 1-2 and Supplementary References.\\n\\n### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_MOESM2_ESM.pdf)\\n\\n### [Supplementary Video 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_MOESM3_ESM.mp4)\\n\\n### [Supplementary Video 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_MOESM4_ESM.mp4)\\n\\nRights and permissions\\n----------------------\\n\\n\\n**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <http://creativecommons.org/licenses/by/4.0/>.\\n\\n\\n[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Visualizing%20translation%20dynamics%20at%20atomic%20detail%20inside%20a%20bacterial%20cell&author=Liang%20Xue%20et%20al&contentID=10.1038%2Fs41586-022-05255-2&copyright=The%20Author%28s%29&publication=0028-0836&publicationDate=2022-09-28&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)\\n\\nAbout this article\\n------------------\\n\\n[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41586-022-05255-2)### Cite this article\\n\\nXue, L., Lenz, S., Zimmermann-Kogadeeva, M. *et al.* Visualizing translation dynamics at atomic detail inside a bacterial cell.\\n *Nature* **610**, 205–211 (2022). https://doi.org/10.1038/s41586-022-05255-2\\n\\n[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41586-022-05255-2?format=refman&flavour=citation)\\n\\n* Received: 18 December 2021\\n* Accepted: 19 August 2022\\n* Published: 28 September 2022\\n* Issue Date: 06 October 2022\\n* DOI: https://doi.org/10.1038/s41586-022-05255-2\\n\\n### Share this article\\n\\nAnyone you share the following link with will be able to read this content:\\n\\nGet shareable linkSorry, a shareable link is not currently available for this article.\\n\\nCopy to clipboard\\n Provided by the Springer Nature SharedIt content-sharing initiative\\n \\n\\n\\n### Subjects\\n\\n\\n* [Bacterial structural biology](/subjects/bacterial-structural-biology)\\n* [Cryoelectron tomography](/subjects/cryoelectron-tomography)\\n* [Ribosome](/subjects/ribosome)\\n\\n\\n\\n\\n\\n\\nThis article is cited by\\n------------------------\\n\\n\\n\\n* ### [Learning structural heterogeneity from cryo-electron sub-tomograms with tomoDRGN](https://doi.org/10.1038/s41592-024-02210-z)\\n\\n\\n\\t+ Barrett M. Powell\\n\\t+ Joseph H. Davis*Nature Methods* (2024)\\n* ### [Transient disome complex formation in native polysomes during ongoing protein synthesis captured by cryo-EM](https://doi.org/10.1038/s41467-024-46092-3)\\n\\n\\n\\t+ Timo Flügel\\n\\t+ Magdalena Schacherl\\n\\t+ Christian M. T. Spahn*Nature Communications* (2024)\\n* ### [Patchy and widespread distribution of bacterial translation arrest peptides associated with the protein localization machinery](https://doi.org/10.1038/s41467-024-46993-3)\\n\\n\\n\\t+ Keigo Fujiwara\\n\\t+ Naoko Tsuji\\n\\t+ Shinobu Chiba*Nature Communications* (2024)\\n* ### [Visualization of translation and protein biogenesis at the ER membrane](https://doi.org/10.1038/s41586-022-05638-5)\\n\\n\\n\\t+ Max Gemmer\\n\\t+ Marten L. Chaillet\\n\\t+ Friedrich Förster*Nature* (2023)\\n* ### [The translating bacterial ribosome at 1.55 Å resolution generated by cryo-EM imaging services](https://doi.org/10.1038/s41467-023-36742-3)\\n\\n\\n\\t+ Simon A. Fromm\\n\\t+ Kate M. O’Connor\\n\\t+ Simone Mattei*Nature Communications* (2023)\\n\\n\\n\\n\\n\\nComments\\n--------\\n\\nBy submitting a comment you agree to abide by our [Terms](/info/tandc.html) and [Community Guidelines](/info/community-guidelines.html). If you find something abusive or that does not comply with our terms or guidelines please flag it as inappropriate.\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n[Download PDF](/articles/s41586-022-05255-2.pdf)\\n\\n\\n\\n\\nAssociated content\\n------------------\\n\\n\\n\\n\\n\\n\\n### [Atomic ribosome activity](https://www.nature.com/articles/s41594-022-00878-5)\\n\\n\\n* Jake Rogers\\n\\n\\n\\nNature Structural & Molecular Biology\\nResearch Highlight\\n09 Nov 2022\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n* Sections\\n* Figures\\n* References\\n\\n* [Abstract](#Abs1)\\n* [Main](#Sec1)\\n* [In-cell structure of the *M. pneumoniae* ribosome](#Sec2)\\n* [Structural dynamics of translation in cells](#Sec3)\\n* [Antibiotics alter translation landscapes](#Sec4)\\n* [Spatial and functional organization of translation](#Sec5)\\n* [Conclusions](#Sec6)\\n* [Methods](#Sec7)\\n* [Data availability](#data-availability)\\n* [Code availability](#code-availability)\\n* [Change history](#change-history)\\n* [References](#Bib1)\\n* [Acknowledgements](#Ack1)\\n* [Author information](#author-information)\\n* [Ethics declarations](#ethics)\\n* [Peer review](#peer-review)\\n* [Additional information](#additional-information)\\n* [Extended data figures and tables](#Sec19)\\n* [Supplementary information](#Sec20)\\n* [Rights and permissions](#rightslink)\\n* [About this article](#article-info)\\n* [This article is cited by](#further-reading)\\n* [Comments](#article-comments)\\n\\n\\n\\n\\nAdvertisement\\n\\n\\n\\n\\n[![Advertisement](//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature.com/article&sz=300x250&c=-563873083&t=pos%3Dright%26type%3Darticle%26artid%3Ds41586-022-05255-2%26doi%3D10.1038/s41586-022-05255-2%26techmeta%3D101,28%26subjmeta%3D1258,1260,1789,2536,326,337,41,45,535,574,631%26kwrd%3DBacterial+structural+biology,Cryoelectron+tomography,Ribosome)](//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature.com/article&sz=300x250&c=-563873083&t=pos%3Dright%26type%3Darticle%26artid%3Ds41586-022-05255-2%26doi%3D10.1038/s41586-022-05255-2%26techmeta%3D101,28%26subjmeta%3D1258,1260,1789,2536,326,337,41,45,535,574,631%26kwrd%3DBacterial+structural+biology,Cryoelectron+tomography,Ribosome)\\n\\n\\n\\n\\n\\n\\n* **Fig. 1: Ribosome structure in *M.* *pneumoniae* cells.**\\n\\n![figure 1]()[View in article](#Fig1)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/1)\\n* **Fig. 2: Ribosome classification reconstructs the translation elongation cycle.**\\n\\n![figure 2]()[View in article](#Fig2)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/2)\\n* **Fig. 3: Antibiotics induce distinct translation elongation landscapes in cells.**\\n\\n![figure 3]()[View in article](#Fig3)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/3)\\n* **Fig. 4: Spatial and functional organization of ribosomes in native cells.**\\n\\n![figure 4]()[View in article](#Fig4)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/4)\\n* **Extended Data Fig. 1 *M. pneumoniae* in-cell ribosome maps.**\\n\\n![extended data figure 5]()[View in article](#Fig5)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/5)\\n* **Extended Data Fig. 2 Structural features of the *M. pneumoniae* ribosome.**\\n\\n![extended data figure 6]()[View in article](#Fig6)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/6)\\n* **Extended Data Fig. 3 Ribosomal protein extensions.**\\n\\n![extended data figure 7]()[View in article](#Fig7)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/7)\\n* **Extended Data Fig. 4 Classification and refinement of ribosomes in native untreated cells.**\\n\\n![extended data figure 8]()[View in article](#Fig8)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/8)\\n* **Extended Data Fig. 5 Validation of ribosome detection and classification.**\\n\\n![extended data figure 9]()[View in article](#Fig9)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/9)\\n* **Extended Data Fig. 6 Local-resolution maps for ribosome classes in untreated cells.**\\n\\n![extended data figure 10]()[View in article](#Fig10)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/10)\\n* **Extended Data Fig. 7 Model building for ribosome classes in untreated cells.**\\n\\n![extended data figure 11]()[View in article](#Fig11)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/11)\\n* **Extended Data Fig. 8 Early-to-late translocation intermediates during elongation.**\\n\\n![extended data figure 12]()[View in article](#Fig12)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/12)\\n* **Extended Data Fig. 9 Classification, refinement and modelling of ribosomes in Cm-treated cells.**\\n\\n![extended data figure 13]()[View in article](#Fig13)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/13)\\n* **Extended Data Fig. 10 Classification, refinement and modelling of ribosomes in Spc-treated cells.**\\n\\n![extended data figure 14]()[View in article](#Fig14)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/14)\\n* **Extended Data Fig. 11 Classification, refinement and modelling of ribosomes in PUM-treated cells.**\\n\\n![extended data figure 15]()[View in article](#Fig15)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/15)\\n* **Extended Data Fig. 12 Spatial analysis of ribosomes and polysomes in native untreated cells.**\\n\\n![extended data figure 16]()[View in article](#Fig16)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/16)\\n* **Extended Data Fig. 13 Correlation of translation elongation states within polysomes in native untreated cells.**\\n\\n![extended data figure 17]()[View in article](#Fig17)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/17)\\n\\n1. Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. *Annu. Rev. Biochem.* **82**, 203–236 (2013).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-biochem-113009-092313) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFOqsrzJ) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23746255) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20the%20translational%20elongation%20cycle&journal=Annu.%20Rev.%20Biochem.&doi=10.1146%2Fannurev-biochem-113009-092313&volume=82&pages=203-236&publication_year=2013&author=Voorhees%2CRM&author=Ramakrishnan%2CV)\\n2. O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. *Science* **369**, 554–557 (2020).\\n\\n[Article](https://doi.org/10.1126%2Fscience.abb3758) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020Sci...369..554O) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFWgtr7N) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32732422) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115962) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=In-cell%20architecture%20of%20an%20actively%20transcribing%E2%80%93translating%20expressome&journal=Science&doi=10.1126%2Fscience.abb3758&volume=369&pages=554-557&publication_year=2020&author=O%27Reilly%2CFJ)\\n3. Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. *Nat. Methods* **18**, 186–193 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-020-01054-7) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjt1SmsL8%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33542511) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7611018) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-particle%20cryo-EM%20refinement%20with%20M%20visualizes%20ribosome%E2%80%93antibiotic%20complex%20at%203.5%20%C3%85%20in%20cells&journal=Nat.%20Methods&doi=10.1038%2Fs41592-020-01054-7&volume=18&pages=186-193&publication_year=2021&author=Tegunov%2CD&author=Xue%2CL&author=Dienemann%2CC&author=Cramer%2CP&author=Mahamid%2CJ)\\n4. Brandt, F. et al. The native 3D organization of bacterial polysomes. *Cell* **136**, 261–271 (2009).\\n\\n[Article](https://doi.org/10.1016%2Fj.cell.2008.11.016) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhs1Kiurk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19167328) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20native%203D%20organization%20of%20bacterial%20polysomes&journal=Cell&doi=10.1016%2Fj.cell.2008.11.016&volume=136&pages=261-271&publication_year=2009&author=Brandt%2CF)\\n5. Rodnina, M. V. Translation in prokaryotes. *Cold Spring Harb. Perspect. Biol.* **10**, a032664 (2018).\\n\\n[Article](https://doi.org/10.1101%2Fcshperspect.a032664) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29661790) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6120702) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Translation%20in%20prokaryotes&journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&doi=10.1101%2Fcshperspect.a032664&volume=10&publication_year=2018&author=Rodnina%2CMV)\\n6. Frank, J. & Agrawal, R. K. A ratchet-like inter-subunit reorganization of the ribosome during translocation. *Nature* **406**, 318–322 (2000).\\n\\n[Article](https://doi.org/10.1038%2F35018597) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2000Natur.406..318F) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXlsFKnsrk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10917535) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20ratchet-like%20inter-subunit%20reorganization%20of%20the%20ribosome%20during%20translocation&journal=Nature&doi=10.1038%2F35018597&volume=406&pages=318-322&publication_year=2000&author=Frank%2CJ&author=Agrawal%2CRK)\\n7. Villa, E. et al. Ribosome-induced changes in elongation factor Tu conformation control GTP hydrolysis. *Proc. Natl Acad. Sci. USA* **106**, 1063–1068 (2009).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.0811370106) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009PNAS..106.1063V) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhsFyjtbk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19122150) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2613361) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome-induced%20changes%20in%20elongation%20factor%20Tu%20conformation%20control%20GTP%20hydrolysis&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.0811370106&volume=106&pages=1063-1068&publication_year=2009&author=Villa%2CE)\\n8. Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. *Nature* **466**, 329–333 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature09206) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.466..329F) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXosl2rsr0%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20631791) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome%20dynamics%20and%20tRNA%20movement%20by%20time-resolved%20electron%20cryomicroscopy&journal=Nature&doi=10.1038%2Fnature09206&volume=466&pages=329-333&publication_year=2010&author=Fischer%2CN&author=Konevega%2CAL&author=Wintermeyer%2CW&author=Rodnina%2CMV&author=Stark%2CH)\\n9. Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. *Nature* **468**, 713–716 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature09547) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.468..713R) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsFSku7jF) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21124459) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3272701) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Head%20swivel%20on%20the%20ribosome%20facilitates%20translocation%20by%20means%20of%20intra-subunit%20tRNA%20hybrid%20sites&journal=Nature&doi=10.1038%2Fnature09547&volume=468&pages=713-716&publication_year=2010&author=Ratje%2CAH)\\n10. Ramrath, D. J. et al. Visualization of two transfer RNAs trapped in transit during elongation factor G-mediated translocation. *Proc. Natl Acad. Sci. USA* **110**, 20964–20969 (2013).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1320387110) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2013PNAS..11020964R) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXnsF2jug%3D%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24324168) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876219) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualization%20of%20two%20transfer%20RNAs%20trapped%20in%20transit%20during%20elongation%20factor%20G-mediated%20translocation&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1320387110&volume=110&pages=20964-20969&publication_year=2013&author=Ramrath%2CDJ)\\n11. Brilot, A. F., Korostelev, A. A., Ermolenko, D. N. & Grigorieff, N. Structure of the ribosome with elongation factor G trapped in the pretranslocation state. *Proc. Natl Acad. Sci. USA* **110**, 20994–20999 (2013).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1311423110) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2013PNAS..11020994B) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXnsF2iuw%3D%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24324137) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876248) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structure%20of%20the%20ribosome%20with%20elongation%20factor%20G%20trapped%20in%20the%20pretranslocation%20state&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1311423110&volume=110&pages=20994-20999&publication_year=2013&author=Brilot%2CAF&author=Korostelev%2CAA&author=Ermolenko%2CDN&author=Grigorieff%2CN)\\n12. Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. *Cell* **161**, 845–857 (2015).\\n\\n[Article](https://doi.org/10.1016%2Fj.cell.2015.03.052) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXotVKlur0%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25957688) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4432480) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20snapshots%20of%20actively%20translating%20human%20ribosomes&journal=Cell&doi=10.1016%2Fj.cell.2015.03.052&volume=161&pages=845-857&publication_year=2015&author=Behrmann%2CE)\\n13. Loveland, A. B., Demo, G. & Korostelev, A. A. Cryo-EM of elongating ribosome with EF-Tu\\\\*GTP elucidates tRNA proofreading. *Nature* **584**, 640–645 (2020).\\n\\n[Article](https://doi.org/10.1038%2Fs41586-020-2447-x) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020Natur.584..640L) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlShtL%2FE) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32612237) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7483604) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cryo-EM%20of%20elongating%20ribosome%20with%20EF-Tu%2AGTP%20elucidates%20tRNA%20proofreading&journal=Nature&doi=10.1038%2Fs41586-020-2447-x&volume=584&pages=640-645&publication_year=2020&author=Loveland%2CAB&author=Demo%2CG&author=Korostelev%2CAA)\\n14. Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. *Nature* **595**, 741–745 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41586-021-03713-x) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Natur.595..741R) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFSnsbnM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34234344) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8318882) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20early%20translocation%20events%20on%20the%20ribosome&journal=Nature&doi=10.1038%2Fs41586-021-03713-x&volume=595&pages=741-745&publication_year=2021&author=Rundlet%2CEJ)\\n15. Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. *Nat. Commun.* **12**, 5933 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-26133-x) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.5933P) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1aqsL7M) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34635670) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8505512) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20mechanism%20of%20GTPase-powered%20ribosome%E2%80%93tRNA%20movement&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-26133-x&volume=12&publication_year=2021&author=Petrychenko%2CV)\\n16. Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. *Nat. Commun.* **12**, 7236 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-27415-0) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.7236C) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislejtbjN) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34903725) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8668904) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Time-resolved%20cryo-EM%20visualizes%20ribosomal%20translocation%20with%20EF-G%20and%20GTP&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-27415-0&volume=12&publication_year=2021&author=Carbone%2CCE)\\n17. Brown, A. & Shao, S. Ribosomes and cryo-EM: a duet. *Curr. Opin. Struct. Biol.* **52**, 1–7 (2018).\\n\\n[Article](https://doi.org/10.1016%2Fj.sbi.2018.07.001) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlSjs77J) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30015201) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosomes%20and%20cryo-EM%3A%20a%20duet&journal=Curr.%20Opin.%20Struct.%20Biol.&doi=10.1016%2Fj.sbi.2018.07.001&volume=52&pages=1-7&publication_year=2018&author=Brown%2CA&author=Shao%2CS)\\n18. Kuhner, S. et al. Proteome organization in a genome-reduced bacterium. *Science* **326**, 1235–1240 (2009).\\n\\n[Article](https://doi.org/10.1126%2Fscience.1176343) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009Sci...326.1235K) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19965468) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Proteome%20organization%20in%20a%20genome-reduced%20bacterium&journal=Science&doi=10.1126%2Fscience.1176343&volume=326&pages=1235-1240&publication_year=2009&author=Kuhner%2CS)\\n19. Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. *Mol. Cell* **39**, 560–569 (2010).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2010.08.003) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVyrt7jF) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20797628) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20three-dimensional%20organization%20of%20polyribosomes%20in%20intact%20human%20cells&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2010.08.003&volume=39&pages=560-569&publication_year=2010&author=Brandt%2CF&author=Carlson%2CLA&author=Hartl%2CFU&author=Baumeister%2CW&author=Grunewald%2CK)\\n20. Mahamid, J. et al. Visualizing the molecular sociology at the HeLa cell nuclear periphery. *Science* **351**, 969–972 (2016).\\n\\n[Article](https://doi.org/10.1126%2Fscience.aad8857) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016Sci...351..969M) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XivFOnt7c%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26917770) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualizing%20the%20molecular%20sociology%20at%20the%20HeLa%20cell%20nuclear%20periphery&journal=Science&doi=10.1126%2Fscience.aad8857&volume=351&pages=969-972&publication_year=2016&author=Mahamid%2CJ)\\n21. Schaffer, M. et al. A cryo-FIB lift-out technique enables molecular-resolution cryo-ET within native *Caenorhabditis elegans* tissue. *Nat. Methods* **16**, 757–762 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0497-5) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVOku7zL) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31363205) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20cryo-FIB%20lift-out%20technique%20enables%20molecular-resolution%20cryo-ET%20within%20native%20Caenorhabditis%20elegans%20tissue&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0497-5&volume=16&pages=757-762&publication_year=2019&author=Schaffer%2CM)\\n22. Waltz, F. et al. How to build a ribosome from RNA fragments in *Chlamydomonas* mitochondria. *Nat. Commun.* **12**, 7176 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-27200-z) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.7176W) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislWktLrO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34887394) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8660880) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=How%20to%20build%20a%20ribosome%20from%20RNA%20fragments%20in%20Chlamydomonas%20mitochondria&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-27200-z&volume=12&publication_year=2021&author=Waltz%2CF)\\n23. Melnikov, S., Manakongtreecheep, K. & Soll, D. Revising the structural diversity of ribosomal proteins across the three domains of life. *Mol. Biol. Evol.* **35**, 1588–1598 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsy021) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtF2rur7L) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29529322) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5995209) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Revising%20the%20structural%20diversity%20of%20ribosomal%20proteins%20across%20the%20three%20domains%20of%20life&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsy021&volume=35&pages=1588-1598&publication_year=2018&author=Melnikov%2CS&author=Manakongtreecheep%2CK&author=Soll%2CD)\\n24. Miravet-Verde, S., Burgos, R., Delgado, J., Lluch-Senar, M. & Serrano, L. FASTQINS and ANUBIS: two bioinformatic tools to explore facts and artifacts in transposon sequencing and essentiality studies. *Nucleic Acids Res.* **48**, e102 (2020).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkaa679) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXkt1GntbY%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32813015) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7515713) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FASTQINS%20and%20ANUBIS%3A%20two%20bioinformatic%20tools%20to%20explore%20facts%20and%20artifacts%20in%20transposon%20sequencing%20and%20essentiality%20studies&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkaa679&volume=48&publication_year=2020&author=Miravet-Verde%2CS&author=Burgos%2CR&author=Delgado%2CJ&author=Lluch-Senar%2CM&author=Serrano%2CL)\\n25. Borg, A., Pavlov, M. & Ehrenberg, M. Complete kinetic mechanism for recycling of the bacterial ribosome. *RNA* **22**, 10–21 (2016).\\n\\n[Article](https://doi.org/10.1261%2Frna.053157.115) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVKlur3M) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26527791) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4691825) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Complete%20kinetic%20mechanism%20for%20recycling%20of%20the%20bacterial%20ribosome&journal=RNA&doi=10.1261%2Frna.053157.115&volume=22&pages=10-21&publication_year=2016&author=Borg%2CA&author=Pavlov%2CM&author=Ehrenberg%2CM)\\n26. Munro, J. B., Altman, R. B., O’Connor, N. & Blanchard, S. C. Identification of two distinct hybrid state intermediates on the ribosome. *Mol. Cell* **25**, 505–517 (2007).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2007.01.022) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXjtlWkur8%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17317624) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2244649) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20two%20distinct%20hybrid%20state%20intermediates%20on%20the%20ribosome&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2007.01.022&volume=25&pages=505-517&publication_year=2007&author=Munro%2CJB&author=Altman%2CRB&author=O%E2%80%99Connor%2CN&author=Blanchard%2CSC)\\n27. Belardinelli, R. et al. Choreography of molecular movements during ribosome progression along mRNA. *Nat. Struct. Mol. Biol.* **23**, 342–348 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.3193) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XktFeqtrk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26999556) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Choreography%20of%20molecular%20movements%20during%20ribosome%20progression%20along%20mRNA&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.3193&volume=23&pages=342-348&publication_year=2016&author=Belardinelli%2CR)\\n28. Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. *Nat. Struct. Mol. Biol.* **23**, 333–341 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.3177) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XivFKhurY%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26926435) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4821728) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiperspective%20smFRET%20reveals%20rate-determining%20late%20intermediates%20of%20ribosomal%20translocation&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.3177&volume=23&pages=333-341&publication_year=2016&author=Wasserman%2CMR&author=Alejo%2CJL&author=Altman%2CRB&author=Blanchard%2CSC)\\n29. Fei, J., Kosuri, P., MacDougall, D. D. & Gonzalez, R. L. Jr Coupling of ribosomal L1 stalk and tRNA dynamics during translation elongation. *Mol. Cell* **30**, 348–359 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2008.03.012) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXmtFaktLg%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18471980) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Coupling%20of%20ribosomal%20L1%20stalk%20and%20tRNA%20dynamics%20during%20translation%20elongation&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2008.03.012&volume=30&pages=348-359&publication_year=2008&author=Fei%2CJ&author=Kosuri%2CP&author=MacDougall%2CDD&author=Gonzalez%2CRL)\\n30. Zhou, J., Lancaster, L., Donohue, J. P. & Noller, H. F. How the ribosome hands the A-site tRNA to the P site during EF-G-catalyzed translocation. *Science* **345**, 1188–1191 (2014).\\n\\n[Article](https://doi.org/10.1126%2Fscience.1255030) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2014Sci...345.1188Z) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsVGhtb%2FO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25190797) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4242719) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=How%20the%20ribosome%20hands%20the%20A-site%20tRNA%20to%20the%20P%20site%20during%20EF-G-catalyzed%20translocation&journal=Science&doi=10.1126%2Fscience.1255030&volume=345&pages=1188-1191&publication_year=2014&author=Zhou%2CJ&author=Lancaster%2CL&author=Donohue%2CJP&author=Noller%2CHF)\\n31. Uemura, S. et al. Real-time tRNA transit on single translating ribosomes at codon resolution. *Nature* **464**, 1012–U1073 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature08925) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.464.1012U) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXkslart7c%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20393556) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4466108) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20tRNA%20transit%20on%20single%20translating%20ribosomes%20at%20codon%20resolution&journal=Nature&doi=10.1038%2Fnature08925&volume=464&pages=1012-U1073&publication_year=2010&author=Uemura%2CS)\\n32. Chen, C. et al. Single-molecule fluorescence measurements of ribosomal translocation dynamics. *Mol. Cell* **42**, 367–377 (2011).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2011.03.024) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXlvVyiu7w%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21549313) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3090999) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-molecule%20fluorescence%20measurements%20of%20ribosomal%20translocation%20dynamics&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2011.03.024&volume=42&pages=367-377&publication_year=2011&author=Chen%2CC)\\n33. Wilson, D. N. & Nierhaus, K. H. The E-site story: the importance of maintaining two tRNAs on the ribosome during protein synthesis. *Cell. Mol. Life Sci.* **63**, 2725–2737 (2006).\\n\\n[Article](https://link.springer.com/doi/10.1007/s00018-006-6125-4) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFSrt7w%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17013564) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20E-site%20story%3A%20the%20importance%20of%20maintaining%20two%20tRNAs%20on%20the%20ribosome%20during%20protein%20synthesis&journal=Cell.%20Mol.%20Life%20Sci.&doi=10.1007%2Fs00018-006-6125-4&volume=63&pages=2725-2737&publication_year=2006&author=Wilson%2CDN&author=Nierhaus%2CKH)\\n34. Munro, J. B., Sanbonmatsu, K. Y., Spahn, C. M. & Blanchard, S. C. Navigating the ribosome’s metastable energy landscape. *Trends Biochem. Sci* **34**, 390–400 (2009).\\n\\n[Article](https://doi.org/10.1016%2Fj.tibs.2009.04.004) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVSntrnM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19647434) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2914510) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Navigating%20the%20ribosome%E2%80%99s%20metastable%20energy%20landscape&journal=Trends%20Biochem.%20Sci&doi=10.1016%2Fj.tibs.2009.04.004&volume=34&pages=390-400&publication_year=2009&author=Munro%2CJB&author=Sanbonmatsu%2CKY&author=Spahn%2CCM&author=Blanchard%2CSC)\\n35. Bock, L. V. et al. Energy barriers and driving forces in tRNA translocation through the ribosome. *Nat. Struct. Mol. Biol.* **20**, 1390–1396 (2013).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.2690) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslWisbvK) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24186064) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Energy%20barriers%20and%20driving%20forces%20in%20tRNA%20translocation%20through%20the%20ribosome&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.2690&volume=20&pages=1390-1396&publication_year=2013&author=Bock%2CLV)\\n36. Julian, P. et al. Structure of ratcheted ribosomes with tRNAs in hybrid states. *Proc. Natl Acad. Sci. USA* **105**, 16924–16927 (2008).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.0809587105) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2008PNAS..10516924J) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlKqt7%2FM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18971332) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2579354) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structure%20of%20ratcheted%20ribosomes%20with%20tRNAs%20in%20hybrid%20states&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.0809587105&volume=105&pages=16924-16927&publication_year=2008&author=Julian%2CP)\\n37. Agirrezabala, X. et al. Visualization of the hybrid state of tRNA binding promoted by spontaneous ratcheting of the ribosome. *Mol. Cell* **32**, 190–197 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2008.10.001) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlehsb3L) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18951087) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614368) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualization%20of%20the%20hybrid%20state%20of%20tRNA%20binding%20promoted%20by%20spontaneous%20ratcheting%20of%20the%20ribosome&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2008.10.001&volume=32&pages=190-197&publication_year=2008&author=Agirrezabala%2CX)\\n38. Rodnina, M. V., Peske, F., Peng, B. Z., Belardinelli, R. & Wintermeyer, W. Converting GTP hydrolysis into motion: versatile translational elongation factor G. *Biol. Chem.* **401**, 131–142 (2019).\\n\\n[Article](https://doi.org/10.1515%2Fhsz-2019-0313) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31600135) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Converting%20GTP%20hydrolysis%20into%20motion%3A%20versatile%20translational%20elongation%20factor%20G&journal=Biol.%20Chem.&doi=10.1515%2Fhsz-2019-0313&volume=401&pages=131-142&publication_year=2019&author=Rodnina%2CMV&author=Peske%2CF&author=Peng%2CBZ&author=Belardinelli%2CR&author=Wintermeyer%2CW)\\n39. Lin, J., Zhou, D., Steitz, T. A., Polikanov, Y. S. & Gagnon, M. G. Ribosome-targeting antibiotics: modes of action, mechanisms of resistance, and implications for drug design. *Annu. Rev. Biochem.* **87**, 451–478 (2018).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-biochem-062917-011942) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXlslSrurw%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29570352) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9176271) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome-targeting%20antibiotics%3A%20modes%20of%20action%2C%20mechanisms%20of%20resistance%2C%20and%20implications%20for%20drug%20design&journal=Annu.%20Rev.%20Biochem.&doi=10.1146%2Fannurev-biochem-062917-011942&volume=87&pages=451-478&publication_year=2018&author=Lin%2CJ&author=Zhou%2CD&author=Steitz%2CTA&author=Polikanov%2CYS&author=Gagnon%2CMG)\\n40. Bulkley, D., Innis, C. A., Blaha, G. & Steitz, T. A. Revisiting the structures of several antibiotics bound to the bacterial ribosome. *Proc. Natl Acad. Sci. USA* **107**, 17158–17163 (2010).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1008685107) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010PNAS..10717158B) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlSjurrM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20876130) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2951403) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Revisiting%20the%20structures%20of%20several%20antibiotics%20bound%20to%20the%20bacterial%20ribosome&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1008685107&volume=107&pages=17158-17163&publication_year=2010&author=Bulkley%2CD&author=Innis%2CCA&author=Blaha%2CG&author=Steitz%2CTA)\\n41. Svetlov, M. S. et al. High-resolution crystal structures of ribosome-bound chloramphenicol and erythromycin provide the ultimate basis for their competition. *RNA* **25**, 600–606 (2019).\\n\\n[Article](https://doi.org/10.1261%2Frna.069260.118) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1Ons77I) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30733327) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6467010) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-resolution%20crystal%20structures%20of%20ribosome-bound%20chloramphenicol%20and%20erythromycin%20provide%20the%20ultimate%20basis%20for%20their%20competition&journal=RNA&doi=10.1261%2Frna.069260.118&volume=25&pages=600-606&publication_year=2019&author=Svetlov%2CMS)\\n42. Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. *ACS Chem. Biol.* **2**, 545–552 (2007).\\n\\n[Article](https://doi.org/10.1021%2Fcb700100n) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXpvVOhtrc%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17696316) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4624401) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20steric%20block%20in%20translation%20caused%20by%20the%20antibiotic%20spectinomycin&journal=ACS%20Chem.%20Biol.&doi=10.1021%2Fcb700100n&volume=2&pages=545-552&publication_year=2007&author=Borovinskaya%2CMA&author=Shoji%2CS&author=Holton%2CJM&author=Fredrick%2CK&author=Cate%2CJHD)\\n43. Polikanov, Y. S. et al. Distinct tRNA accommodation intermediates observed on the ribosome with the antibiotics hygromycin A and A201A. *Mol. Cell* **58**, 832–844 (2015).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2015.04.014) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXpt1Skurg%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26028538) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4458074) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20tRNA%20accommodation%20intermediates%20observed%20on%20the%20ribosome%20with%20the%20antibiotics%20hygromycin%20A%20and%20A201A&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2015.04.014&volume=58&pages=832-844&publication_year=2015&author=Polikanov%2CYS)\\n44. Morse, J. C. et al. Elongation factor-Tu can repetitively engage aminoacyl-tRNA within the ribosome during the proofreading stage of tRNA selection. *Proc. Natl Acad. Sci. USA* **117**, 3610–3620 (2020).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1904469117) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020PNAS..117.3610M) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjsFyhu7Y%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32024753) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7035488) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Elongation%20factor-Tu%20can%20repetitively%20engage%20aminoacyl-tRNA%20within%20the%20ribosome%20during%20the%20proofreading%20stage%20of%20tRNA%20selection&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1904469117&volume=117&pages=3610-3620&publication_year=2020&author=Morse%2CJC)\\n45. Choi, J. et al. Dynamics of the context-specific translation arrest by chloramphenicol and linezolid. *Nat. Chem. Biol.* **16**, 310–317 (2020).\\n\\n[Article](https://doi.org/10.1038%2Fs41589-019-0423-2) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVSktb%2FM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31844301) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dynamics%20of%20the%20context-specific%20translation%20arrest%20by%20chloramphenicol%20and%20linezolid&journal=Nat.%20Chem.%20Biol.&doi=10.1038%2Fs41589-019-0423-2&volume=16&pages=310-317&publication_year=2020&author=Choi%2CJ)\\n46. Marks, J. et al. Context-specific inhibition of translation by ribosomal antibiotics targeting the peptidyl transferase center. *Proc. Natl Acad. Sci. USA* **113**, 12150–12155 (2016).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1613055113) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016PNAS..11312150M) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1elur3N) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27791002) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5086994) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Context-specific%20inhibition%20of%20translation%20by%20ribosomal%20antibiotics%20targeting%20the%20peptidyl%20transferase%20center&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1613055113&volume=113&pages=12150-12155&publication_year=2016&author=Marks%2CJ)\\n47. Vazquez-Laslop, N. & Mankin, A. S. Context-specific action of ribosomal antibiotics. *Annu. Rev. Microbiol.* **72**, 185–207 (2018).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-micro-090817-062329) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFCjs77O) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29906204) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8742604) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Context-specific%20action%20of%20ribosomal%20antibiotics&journal=Annu.%20Rev.%20Microbiol.&doi=10.1146%2Fannurev-micro-090817-062329&volume=72&pages=185-207&publication_year=2018&author=Vazquez-Laslop%2CN&author=Mankin%2CAS)\\n48. Aguirre Rivera, J. et al. Real-time measurements of aminoglycoside effects on protein synthesis in live cells. *Proc. Natl Acad. Sci. USA* **118**, e2013315118 (2021).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.2013315118) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33619089) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7936356) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20measurements%20of%20aminoglycoside%20effects%20on%20protein%20synthesis%20in%20live%20cells&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.2013315118&volume=118&publication_year=2021&author=Aguirre%20Rivera%2CJ)\\n49. Baquero, F. & Levin, B. R. Proximate and ultimate causes of the bactericidal action of antibiotics. *Nat. Rev. Microbiol.* **19**, 123–132 (2021).\\n50. Myasnikov, A. G. et al. The molecular structure of the left-handed supra-molecular helix of eukaryotic polyribosomes. *Nat. Commun.* **5**, 5294 (2014).\\n\\n[Article](https://doi.org/10.1038%2Fncomms6294) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2014NatCo...5.5294M) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFantrfM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25376914) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20molecular%20structure%20of%20the%20left-handed%20supra-molecular%20helix%20of%20eukaryotic%20polyribosomes&journal=Nat.%20Commun.&doi=10.1038%2Fncomms6294&volume=5&publication_year=2014&author=Myasnikov%2CAG)\\n51. Dunkle, J. A. & Cate, J. H. D. in *Ribosomes* (eds Rodnina, M. V., Wintermeyer, W. & Green, R.) Ch. 6 (2011).\\n52. Herr, A. J., Nelson, C. C., Wills, N. M., Gesteland, R. F. & Atkins, J. F. Analysis of the roles of tRNA structure, ribosomal protein L9, and the bacteriophage T4 gene 60 bypassing signals during ribosome slippage on mRNA. *J. Mol. Biol.* **309**, 1029–1048 (2001).\\n\\n[Article](https://doi.org/10.1006%2Fjmbi.2001.4717) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXkslGgurY%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11399077) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20the%20roles%20of%20tRNA%20structure%2C%20ribosomal%20protein%20L9%2C%20and%20the%20bacteriophage%20T4%20gene%2060%20bypassing%20signals%20during%20ribosome%20slippage%20on%20mRNA&journal=J.%20Mol.%20Biol.&doi=10.1006%2Fjmbi.2001.4717&volume=309&pages=1029-1048&publication_year=2001&author=Herr%2CAJ&author=Nelson%2CCC&author=Wills%2CNM&author=Gesteland%2CRF&author=Atkins%2CJF)\\n53. Smith, A. M., Costello, M. S., Kettring, A. H., Wingo, R. J. & Moore, S. D. Ribosome collisions alter frameshifting at translational reprogramming motifs in bacterial mRNAs. *Proc. Natl Acad. Sci. USA* **116**, 21769–21779 (2019).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1910613116) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019PNAS..11621769S) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitValtbjM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31591196) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6815119) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome%20collisions%20alter%20frameshifting%20at%20translational%20reprogramming%20motifs%20in%20bacterial%20mRNAs&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1910613116&volume=116&pages=21769-21779&publication_year=2019&author=Smith%2CAM&author=Costello%2CMS&author=Kettring%2CAH&author=Wingo%2CRJ&author=Moore%2CSD)\\n54. Tegunov, D. & Cramer, P. Real-time cryo-electron microscopy data preprocessing with Warp. *Nat. Methods* **16**, 1146–1152 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0580-y) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFejs77E) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31591575) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6858868) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20cryo-electron%20microscopy%20data%20preprocessing%20with%20Warp&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0580-y&volume=16&pages=1146-1152&publication_year=2019&author=Tegunov%2CD&author=Cramer%2CP)\\n55. Hagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. *J. Struct. Biol.* **197**, 191–198 (2017).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2016.06.007) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27313000) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5287356) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Implementation%20of%20a%20cryo-electron%20tomography%20tilt-scheme%20optimized%20for%20high%20resolution%20subtomogram%20averaging&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2016.06.007&volume=197&pages=191-198&publication_year=2017&author=Hagen%2CWJH&author=Wan%2CW&author=Briggs%2CJAG)\\n56. Schorb, M., Haberbosch, I., Hagen, W. J. H., Schwab, Y. & Mastronarde, D. N. Software tools for automated transmission electron microscopy. *Nat. Methods* **16**, 471–477 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0396-9) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVSqs7zL) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31086343) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7000238) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Software%20tools%20for%20automated%20transmission%20electron%20microscopy&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0396-9&volume=16&pages=471-477&publication_year=2019&author=Schorb%2CM&author=Haberbosch%2CI&author=Hagen%2CWJH&author=Schwab%2CY&author=Mastronarde%2CDN)\\n57. Hrabe, T. et al. PyTom: a python-based toolbox for localization of macromolecules in cryo-electron tomograms and subtomogram analysis. *J. Struct. Biol.* **178**, 177–188 (2012).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2011.12.003) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xnt1ehtL4%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22193517) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PyTom%3A%20a%20python-based%20toolbox%20for%20localization%20of%20macromolecules%20in%20cryo-electron%20tomograms%20and%20subtomogram%20analysis&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2011.12.003&volume=178&pages=177-188&publication_year=2012&author=Hrabe%2CT)\\n58. Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. *Nat. Protoc.* **11**, 2054–2065 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnprot.2016.124) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsF2mtLrL) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27685097) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5215819) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Resolving%20macromolecular%20structures%20from%20electron%20cryo-tomography%20data%20using%20subtomogram%20averaging%20in%20RELION&journal=Nat.%20Protoc.&doi=10.1038%2Fnprot.2016.124&volume=11&pages=2054-2065&publication_year=2016&author=Bharat%2CTA&author=Scheres%2CSH)\\n59. Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. *eLife* **7**, e42166 (2018).\\n\\n[Article](https://doi.org/10.7554%2FeLife.42166) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30412051) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6250425) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=New%20tools%20for%20automated%20high-resolution%20cryo-EM%20structure%20determination%20in%20RELION-3&journal=eLife&doi=10.7554%2FeLife.42166&volume=7&publication_year=2018&author=Zivanov%2CJ)\\n60. Waterhouse, A. et al. SWISS-MODEL: homology modelling of protein structures and complexes. *Nucleic Acids Res.* **46**, W296–W303 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky427) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyru70%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29788355) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030848) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SWISS-MODEL%3A%20homology%20modelling%20of%20protein%20structures%20and%20complexes&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky427&volume=46&pages=W296-W303&publication_year=2018&author=Waterhouse%2CA)\\n61. Pruesse, E., Peplies, J. & Glockner, F. O. SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes. *Bioinformatics* **28**, 1823–1829 (2012).\\n\\n[Article](https://doi.org/10.1093%2Fbioinformatics%2Fbts252) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVSgsbfO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22556368) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3389763) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SINA%3A%20accurate%20high-throughput%20multiple%20sequence%20alignment%20of%20ribosomal%20RNA%20genes&journal=Bioinformatics&doi=10.1093%2Fbioinformatics%2Fbts252&volume=28&pages=1823-1829&publication_year=2012&author=Pruesse%2CE&author=Peplies%2CJ&author=Glockner%2CFO)\\n62. Rother, M., Rother, K., Puton, T. & Bujnicki, J. M. ModeRNA: a tool for comparative modeling of RNA 3D structure. *Nucleic Acids Res.* **39**, 4007–4022 (2011).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkq1320) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmvValsL4%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21300639) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3105415) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ModeRNA%3A%20a%20tool%20for%20comparative%20modeling%20of%20RNA%203D%20structure&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkq1320&volume=39&pages=4007-4022&publication_year=2011&author=Rother%2CM&author=Rother%2CK&author=Puton%2CT&author=Bujnicki%2CJM)\\n63. Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. *J. Comput. Chem.* **25**, 1605–1612 (2004).\\n\\n[Article](https://doi.org/10.1002%2Fjcc.20084) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXmvVOhsbs%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15264254) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCSF%20Chimera%E2%80%94a%20visualization%20system%20for%20exploratory%20research%20and%20analysis&journal=J.%20Comput.%20Chem.&doi=10.1002%2Fjcc.20084&volume=25&pages=1605-1612&publication_year=2004&author=Pettersen%2CEF)\\n64. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. *Acta Crystallogr D* **74**, 531–544 (2018).\\n\\n[Article](https://doi.org/10.1107%2FS2059798318006551) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2ltbzP) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-space%20refinement%20in%20PHENIX%20for%20cryo-EM%20and%20crystallography&journal=Acta%20Crystallogr%20D&doi=10.1107%2FS2059798318006551&volume=74&pages=531-544&publication_year=2018&author=Afonine%2CPV)\\n65. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. *Acta Crystallogr D* **66**, 486–501 (2010).\\n\\n[Article](https://doi.org/10.1107%2FS0907444910007493) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXksFKisb8%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20383002) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2852313) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Features%20and%20development%20of%20Coot&journal=Acta%20Crystallogr%20D&doi=10.1107%2FS0907444910007493&volume=66&pages=486-501&publication_year=2010&author=Emsley%2CP&author=Lohkamp%2CB&author=Scott%2CWG&author=Cowtan%2CK)\\n66. Williams, C. J. et al. MolProbity: more and better reference data for improved all-atom structure validation. *Protein Sci.* **27**, 293–315 (2018).\\n\\n[Article](https://doi.org/10.1002%2Fpro.3330) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvVKgsrnO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29067766) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MolProbity%3A%20more%20and%20better%20reference%20data%20for%20improved%20all-atom%20structure%20validation&journal=Protein%20Sci.&doi=10.1002%2Fpro.3330&volume=27&pages=293-315&publication_year=2018&author=Williams%2CCJ)\\n67. Chen, S. et al. High-resolution noise substitution to measure overfitting and validate resolution in 3D structure determination by single particle electron cryomicroscopy. *Ultramicroscopy* **135**, 24–35 (2013).\\n\\n[Article](https://doi.org/10.1016%2Fj.ultramic.2013.06.004) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslWhtLzO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23872039) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3834153) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-resolution%20noise%20substitution%20to%20measure%20overfitting%20and%20validate%20resolution%20in%203D%20structure%20determination%20by%20single%20particle%20electron%20cryomicroscopy&journal=Ultramicroscopy&doi=10.1016%2Fj.ultramic.2013.06.004&volume=135&pages=24-35&publication_year=2013&author=Chen%2CS)\\n68. Tatusov, R. L., Galperin, M. Y., Natale, D. A. & Koonin, E. V. The COG database: a tool for genome-scale analysis of protein functions and evolution. *Nucleic Acids Res.* **28**, 33–36 (2000).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2F28.1.33) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXhvVGqu7w%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10592175) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC102395) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20COG%20database%3A%20a%20tool%20for%20genome-scale%20analysis%20of%20protein%20functions%20and%20evolution&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2F28.1.33&volume=28&pages=33-36&publication_year=2000&author=Tatusov%2CRL&author=Galperin%2CMY&author=Natale%2CDA&author=Koonin%2CEV)\\n69. Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. *Nucleic Acids Res.* **47**, D309–D314 (2019).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky1085) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Gqtr%2FL) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30418610) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=eggNOG%205.0%3A%20a%20hierarchical%2C%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky1085&volume=47&pages=D309-D314&publication_year=2019&author=Huerta-Cepas%2CJ)\\n70. Katoh, K., Rozewicki, J. & Yamada, K. D. MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization. *Brief. Bioinform.* **20**, 1160–1166 (2019).\\n\\n[Article](https://doi.org/10.1093%2Fbib%2Fbbx108) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1Kqtb3K) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28968734) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MAFFT%20online%20service%3A%20multiple%20sequence%20alignment%2C%20interactive%20sequence%20choice%20and%20visualization&journal=Brief.%20Bioinform.&doi=10.1093%2Fbib%2Fbbx108&volume=20&pages=1160-1166&publication_year=2019&author=Katoh%2CK&author=Rozewicki%2CJ&author=Yamada%2CKD)\\n71. Letunic, I. & Bork, P. Interactive Tree Of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. *Nucleic Acids Res.* **49**, W293–W296 (2021).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkab301) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvV2isLvM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33885785) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8265157) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interactive%20Tree%20Of%20Life%20%28iTOL%29%20v5%3A%20an%20online%20tool%20for%20phylogenetic%20tree%20display%20and%20annotation&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkab301&volume=49&pages=W293-W296&publication_year=2021&author=Letunic%2CI&author=Bork%2CP)\\n72. Huerta-Cepas, J. et al. Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper. *Mol. Biol. Evol.* **34**, 2115–2122 (2017).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsx148) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFaru7vF) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28460117) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850834) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20genome-wide%20functional%20annotation%20through%20orthology%20assignment%20by%20eggNOG-Mapper&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsx148&volume=34&pages=2115-2122&publication_year=2017&author=Huerta-Cepas%2CJ)\\n73. Huerta-Cepas, J., Serra, F. & Bork, P. ETE 3: reconstruction, analysis, and visualization of phylogenomic data. *Mol. Biol. Evol.* **33**, 1635–1638 (2016).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsw046) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFeisrrN) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26921390) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4868116) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ETE%203%3A%20reconstruction%2C%20analysis%2C%20and%20visualization%20of%20phylogenomic%20data&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsw046&volume=33&pages=1635-1638&publication_year=2016&author=Huerta-Cepas%2CJ&author=Serra%2CF&author=Bork%2CP)\\n74. Meszaros, B., Erdos, G. & Dosztanyi, Z. IUPred2A: context-dependent prediction of protein disorder as a function of redox state and protein binding. *Nucleic Acids Res.* **46**, W329–W337 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky384) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyrt7o%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29860432) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030935) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=IUPred2A%3A%20context-dependent%20prediction%20of%20protein%20disorder%20as%20a%20function%20of%20redox%20state%20and%20protein%20binding&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky384&volume=46&pages=W329-W337&publication_year=2018&author=Meszaros%2CB&author=Erdos%2CG&author=Dosztanyi%2CZ)\\n75. Cole, C., Barber, J. D. & Barton, G. J. The Jpred 3 secondary structure prediction server. *Nucleic Acids Res.* **36**, W197–W201 (2008).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkn238) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXptlKkt7w%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18463136) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2447793) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Jpred%203%20secondary%20structure%20prediction%20server&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkn238&volume=36&pages=W197-W201&publication_year=2008&author=Cole%2CC&author=Barber%2CJD&author=Barton%2CGJ)\\n76. Scheres, S. H. Processing of structurally heterogeneous cryo-EM data in RELION. *Methods Enzymol.* **579**, 125–157 (2016).\\n\\n[Article](https://doi.org/10.1016%2Fbs.mie.2016.04.012) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmtlKnurk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27572726) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Processing%20of%20structurally%20heterogeneous%20cryo-EM%20data%20in%20RELION&journal=Methods%20Enzymol.&doi=10.1016%2Fbs.mie.2016.04.012&volume=579&pages=125-157&publication_year=2016&author=Scheres%2CSH)\\n77. Kidmose, R. T. et al. Namdinator—automatic molecular dynamics flexible fitting of structural models into cryo-EM and crystallography experimental maps. *IUCrJ* **6**, 526–531 (2019).\\n\\n[Article](https://doi.org/10.1107%2FS2052252519007619) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlWit7%2FO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31316797) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6608625) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Namdinator%E2%80%94automatic%20molecular%20dynamics%20flexible%20fitting%20of%20structural%20models%20into%20cryo-EM%20and%20crystallography%20experimental%20maps&journal=IUCrJ&doi=10.1107%2FS2052252519007619&volume=6&pages=526-531&publication_year=2019&author=Kidmose%2CRT)\\n78. Trabuco, L. G., Villa, E., Mitra, K., Frank, J. & Schulten, K. Flexible fitting of atomic structures into electron microscopy maps using molecular dynamics. *Structure* **16**, 673–683 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.str.2008.03.005) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXlslGrtro%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18462672) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2430731) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Flexible%20fitting%20of%20atomic%20structures%20into%20electron%20microscopy%20maps%20using%20molecular%20dynamics&journal=Structure&doi=10.1016%2Fj.str.2008.03.005&volume=16&pages=673-683&publication_year=2008&author=Trabuco%2CLG&author=Villa%2CE&author=Mitra%2CK&author=Frank%2CJ&author=Schulten%2CK)\\n79. Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. *J. Chem. Phys.* **153**, 044130 (2020).\\n\\n[Article](https://doi.org/10.1063%2F5.0014475) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFajsL3J) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32752662) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7395834) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scalable%20molecular%20dynamics%20on%20CPU%20and%20GPU%20architectures%20with%20NAMD&journal=J.%20Chem.%20Phys.&doi=10.1063%2F5.0014475&volume=153&publication_year=2020&author=Phillips%2CJC)\\n80. Nickell, S. et al. TOM software toolbox: acquisition and analysis for electron tomography. *J. Struct. Biol.* **149**, 227–234 (2005).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2004.10.006) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15721576) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TOM%20software%20toolbox%3A%20acquisition%20and%20analysis%20for%20electron%20tomography&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2004.10.006&volume=149&pages=227-234&publication_year=2005&author=Nickell%2CS)\\n81. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. *Protein Sci.* **30**, 70–82 (2021).\\n\\n[Article](https://doi.org/10.1002%2Fpro.3943) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFamtrzJ) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32881101) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCSF%20ChimeraX%3A%20structure%20visualization%20for%20researchers%2C%20educators%2C%20and%20developers&journal=Protein%20Sci.&doi=10.1002%2Fpro.3943&volume=30&pages=70-82&publication_year=2021&author=Pettersen%2CEF)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n Nature (*Nature*)\\n \\n\\nISSN 1476-4687 (online)\\n \\n\\nISSN 0028-0836 (print)\\n \\n\\n\\n\\n\\n\\n\\n\\n\\nnature.com sitemap\\n------------------\\n\\n\\n\\n\\n\\n### About Nature Portfolio\\n\\n\\n* [About us](https://www.nature.com/npg_/company_info/index.html)\\n* [Press releases](https://www.nature.com/npg_/press_room/press_releases.html)\\n* [Press office](https://press.nature.com/)\\n* [Contact us](https://support.nature.com/support/home)\\n\\n\\n\\n\\n### Discover content\\n\\n\\n* [Journals A-Z](https://www.nature.com/siteindex)\\n* [Articles by subject](https://www.nature.com/subjects)\\n* [protocols.io](https://www.protocols.io/)\\n* [Nature Index](https://www.natureindex.com/)\\n\\n\\n\\n\\n### Publishing policies\\n\\n\\n* [Nature portfolio policies](https://www.nature.com/authors/editorial_policies)\\n* [Open access](https://www.nature.com/nature-research/open-access)\\n\\n\\n\\n\\n### Author & Researcher services\\n\\n\\n* [Reprints & permissions](https://www.nature.com/reprints)\\n* [Research data](https://www.springernature.com/gp/authors/research-data)\\n* [Language editing](https://authorservices.springernature.com/language-editing/)\\n* [Scientific editing](https://authorservices.springernature.com/scientific-editing/)\\n* [Nature Masterclasses](https://masterclasses.nature.com/)\\n* [Research Solutions](https://solutions.springernature.com/)\\n\\n\\n\\n\\n### Libraries & institutions\\n\\n\\n* [Librarian service & tools](https://www.springernature.com/gp/librarians/tools-services)\\n* [Librarian portal](https://www.springernature.com/gp/librarians/manage-your-account/librarianportal)\\n* [Open research](https://www.nature.com/openresearch/about-open-access/information-for-institutions)\\n* [Recommend to library](https://www.springernature.com/gp/librarians/recommend-to-your-library)\\n\\n\\n\\n\\n### Advertising & partnerships\\n\\n\\n* [Advertising](https://partnerships.nature.com/product/digital-advertising/)\\n* [Partnerships & Services](https://partnerships.nature.com/)\\n* [Media kits](https://partnerships.nature.com/media-kits/)\\n* [Branded\\n content](https://partnerships.nature.com/product/branded-content-native-advertising/)\\n\\n\\n\\n\\n### Professional development\\n\\n\\n* [Nature Careers](https://www.nature.com/naturecareers/)\\n* [Nature \\n Conferences](https://conferences.nature.com)\\n\\n\\n\\n\\n### Regional websites\\n\\n\\n* [Nature Africa](https://www.nature.com/natafrica)\\n* [Nature China](http://www.naturechina.com)\\n* [Nature India](https://www.nature.com/nindia)\\n* [Nature Italy](https://www.nature.com/natitaly)\\n* [Nature Japan](https://www.natureasia.com/ja-jp)\\n* [Nature Middle East](https://www.nature.com/nmiddleeast)\\n\\n\\n\\n\\n\\n\\n* [Privacy\\n Policy](https://www.nature.com/info/privacy)\\n* [Use\\n of cookies](https://www.nature.com/info/cookies)\\n* Your privacy choices/Manage cookies\\n* [Legal\\n notice](https://www.nature.com/info/legal-notice)\\n* [Accessibility\\n statement](https://www.nature.com/info/accessibility-statement)\\n* [Terms & Conditions](https://www.nature.com/info/terms-and-conditions)\\n* [Your US state privacy rights](https://www.springernature.com/ccpa)\\n\\n\\n\\n\\n\\n[![Springer Nature](/static/images/logos/sn-logo-white-ea63208b81.svg)](https://www.springernature.com/)\\n© 2024 Springer Nature Limited\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nClose banner\\n\\n\\n\\n\\n\\n\\nClose\\n\\n\\n\\n\\n\\n![Nature Briefing](/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg)\\nSign up for the *Nature Briefing* newsletter — what matters in science, free to your inbox daily.\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nEmail address\\n\\n\\n\\nSign up\\n\\n\\n\\nI agree my information will be processed in accordance with the *Nature* and Springer Nature Limited [Privacy Policy](https://www.nature.com/info/privacy).\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nClose banner\\n\\n\\n\\n\\n\\n\\nClose\\n\\n\\n\\nGet the most important science stories of the day, free in your inbox.\\n[Sign up for Nature Briefing](https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner) \\n\\n\\n\\n\\n![](https://verify.nature.com/verify/nature.png)\\n\\n\\n![](/e8zlnwlx/article/s41586-022-05255-2)\\n\\n\"\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 2:prompt:ChatPromptTemplate] [4ms] Exiting Prompt run with output:\n",
      "\u001b[0m[outputs]\n",
      "\u001b[32;1m\u001b[1;3m[llm/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 3:llm:ChatOpenAI] Entering LLM run with input:\n",
      "\u001b[0m{\n",
      "  \"prompts\": [\n",
      "    \"System: \\n    You are helpful assistant, helping a scientist to extract information from scientific literature.\\n\\nHuman: \\n    Read the following Markdown from scientific publication (delimited with triple backticks).\\n\\n    Markdown Code:- ```\\nVisualizing translation dynamics at atomic detail inside a bacterial cell | Nature\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nYour privacy, your choice\\n-------------------------\\n\\n\\n\\n\\n\\nWe use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.\\n\\nBy accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.\\n\\nSee our [privacy policy](https://www.nature.com/info/privacy) for more information on the use of your personal data.\\n\\nManage preferences for further information and to change your choices.\\n\\n\\n\\n\\n\\nAccept all cookies\\n\\n\\n\\n\\n\\n\\n[Skip to main content](#content)\\n\\n\\nThank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain\\n the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in\\n Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles\\n and JavaScript.\\n\\n\\n\\n\\n\\n\\n\\n\\nAdvertisement\\n\\n\\n\\n\\n[![Advertisement](//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature.com/article&sz=728x90&c=-918719175&t=pos%3Dtop%26type%3Darticle%26artid%3Ds41586-022-05255-2%26doi%3D10.1038/s41586-022-05255-2%26techmeta%3D101,28%26subjmeta%3D1258,1260,1789,2536,326,337,41,45,535,574,631%26kwrd%3DBacterial+structural+biology,Cryoelectron+tomography,Ribosome)](//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature.com/article&sz=728x90&c=-918719175&t=pos%3Dtop%26type%3Darticle%26artid%3Ds41586-022-05255-2%26doi%3D10.1038/s41586-022-05255-2%26techmeta%3D101,28%26subjmeta%3D1258,1260,1789,2536,326,337,41,45,535,574,631%26kwrd%3DBacterial+structural+biology,Cryoelectron+tomography,Ribosome)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n[![Nature](https://media.springernature.com/full/nature-cms/uploads/product/nature/header-86f1267ea01eccd46b530284be10585e.svg)](/)\\n\\n* [View all journals](https://www.nature.com/siteindex)\\n* [Search](javascript:;)\\n\\nSearch\\n------\\n\\n\\n\\nSearch articles by subject, keyword or author\\n\\n\\n\\n\\n\\n\\nShow results from\\n\\nAll journals\\nThis journal\\n\\n\\n\\nSearch\\n\\n\\n\\n\\n\\n[Advanced search](/search/advanced) \\n\\n### Quick links\\n\\n\\n\\n\\t+ [Explore articles by subject](/subjects)\\n\\t+ [Find a job](/naturecareers)\\n\\t+ [Guide to authors](/authors/index.html)\\n\\t+ [Editorial policies](/authors/editorial_policies/)\\n* [Log in](https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41586-022-05255-2)\\n\\n\\n\\n\\n\\n\\n\\n\\n* [Explore content](javascript:;)\\n\\nExplore content\\n---------------\\n\\n\\n\\n\\t+ [Research articles](/nature/research-articles)\\n\\t+ [News](/news)\\n\\t+ [Opinion](/opinion)\\n\\t+ [Research Analysis](/research-analysis)\\n\\t+ [Careers](/careers)\\n\\t+ [Books & Culture](/books-culture)\\n\\t+ [Podcasts](/nature/podcasts)\\n\\t+ [Videos](/nature/videos)\\n\\t+ [Current issue](/nature/current-issue)\\n\\t+ [Browse issues](/nature/browse-issues)\\n\\t+ [Collections](/nature/collections)\\n\\t+ [Subjects](/nature/browse-subjects)\\n\\n\\t+ [Follow us on Facebook](https://www.facebook.com/Nature)\\n\\t+ [Follow us on Twitter](https://twitter.com/nature)\\n\\t+ [Sign up for alerts](https://www.nature.com/my-account/alerts/subscribe-journal?list-id=1)\\n\\t+ [RSS feed](https://www.nature.com/nature.rss)\\n* [About the journal](javascript:;)\\n\\nAbout the journal\\n-----------------\\n\\n\\n\\n\\t+ [Journal Staff](/nature/journal-staff)\\n\\t+ [About the Editors](/nature/editors)\\n\\t+ [Journal Information](/nature/journal-information)\\n\\t+ [Our publishing models](/nature/our-publishing-models)\\n\\t+ [Editorial Values Statement](/nature/editorial-values-statement)\\n\\t+ [Journal Metrics](/nature/journal-impact)\\n\\t+ [Awards](/nature/awards)\\n\\t+ [Contact](/nature/contact)\\n\\t+ [Editorial policies](/nature/editorial-policies)\\n\\t+ [History of Nature](/nature/history-of-nature)\\n\\t+ [Send a news tip](/nature/send-a-news-tip)\\n* [Publish with us](javascript:;)\\n\\nPublish with us\\n---------------\\n\\n\\n\\n\\t+ [For Authors](/nature/for-authors)\\n\\t+ [For Referees](/nature/for-referees)\\n\\t+ [Language editing services](https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022)\\n\\t+ [Submit manuscript](https://mts-nature.nature.com/)\\n\\n\\n* [Sign up for alerts](https://idp.nature.com/auth/personal/springernature?redirect_uri=https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D1%26journal-link%3Dhttps%253A%252F%252Fwww.nature.com%252Fnature%252F)\\n* [RSS feed](https://www.nature.com/nature.rss)\\n\\n\\n\\n\\n\\n\\n\\n\\n1. [nature](/)\\n2. [articles](/nature/articles?type=article)\\n3. article\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n Visualizing translation dynamics at atomic detail inside a bacterial cell\\n \\n\\n[Download PDF](/articles/s41586-022-05255-2.pdf)\\n\\n\\n\\n\\n\\n\\n\\n[Download PDF](/articles/s41586-022-05255-2.pdf)\\n\\n\\n\\n\\n\\n* Article\\n* [Open access](https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research)\\n* Published: 28 September 2022\\n\\n\\nVisualizing translation dynamics at atomic detail inside a bacterial cell\\n=========================================================================\\n\\n\\n* [Liang Xue](#auth-Liang-Xue-Aff1-Aff2) \\n [ORCID: orcid.org/0000-0003-4368-2526](http://orcid.org/0000-0003-4368-2526)[1](#Aff1),[2](#Aff2),\\n* [Swantje Lenz](#auth-Swantje-Lenz-Aff3) \\n [ORCID: orcid.org/0000-0002-8839-5371](http://orcid.org/0000-0002-8839-5371)[3](#Aff3),\\n* [Maria Zimmermann-Kogadeeva](#auth-Maria-Zimmermann_Kogadeeva-Aff1) \\n [ORCID: orcid.org/0000-0001-6021-1246](http://orcid.org/0000-0001-6021-1246)[1](#Aff1),\\n* [Dimitry Tegunov](#auth-Dimitry-Tegunov-Aff4)[4](#Aff4),\\n* [Patrick Cramer](#auth-Patrick-Cramer-Aff4) \\n [ORCID: orcid.org/0000-0001-5454-7755](http://orcid.org/0000-0001-5454-7755)[4](#Aff4),\\n* [Peer Bork](#auth-Peer-Bork-Aff1-Aff5-Aff6) \\n [ORCID: orcid.org/0000-0002-2627-833X](http://orcid.org/0000-0002-2627-833X)[1](#Aff1),[5](#Aff5),[6](#Aff6),\\n* [Juri Rappsilber](#auth-Juri-Rappsilber-Aff3-Aff7) \\n [ORCID: orcid.org/0000-0001-5999-1310](http://orcid.org/0000-0001-5999-1310)[3](#Aff3),[7](#Aff7) &\\n* …\\n* [Julia Mahamid](#auth-Julia-Mahamid-Aff1) \\n [ORCID: orcid.org/0000-0001-6968-041X](http://orcid.org/0000-0001-6968-041X)[1](#Aff1)\\n\\nShow authors\\n\\n[*Nature*](/)\\n**volume 610**, pages 205–211 (2022)[Cite this article](#citeas)\\n\\n\\n\\n\\n* 34k Accesses\\n* 42 Citations\\n* 268 Altmetric\\n* [Metrics details](/articles/s41586-022-05255-2/metrics)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nA [Publisher Correction](https://doi.org/10.1038/s41586-022-05455-w) to this article was published on 10 November 2022\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nThis article has been [updated](#change-history)\\n\\n\\n\\n\\n\\nAbstract\\n--------\\n\\nTranslation is the fundamental process of protein synthesis and is catalysed by the ribosome in all living cells[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"). Here we use advances in cryo-electron tomography and sub-tomogram analysis[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"),[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\") to visualize the structural dynamics of translation inside the bacterium *Mycoplasma pneumoniae*. To interpret the functional states in detail, we first obtain a high-resolution in-cell average map of all translating ribosomes and build an atomic model for the *M.* *pneumoniae* ribosome that reveals distinct extensions of ribosomal proteins. Classification then resolves 13 ribosome states that differ in their conformation and composition. These recapitulate major states that were previously resolved in vitro, and reflect intermediates during active translation. On the basis of these states, we animate translation elongation inside native cells and show how antibiotics reshape the cellular translation landscapes. During translation elongation, ribosomes often assemble in defined three-dimensional arrangements to form polysomes[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"). By mapping the intracellular organization of translating ribosomes, we show that their association into polysomes involves a local coordination mechanism that is mediated by the ribosomal protein L9. We propose that an extended conformation of L9 within polysomes mitigates collisions to facilitate translation fidelity. Our work thus demonstrates the feasibility of visualizing molecular processes at atomic detail inside cells.\\n\\n\\n\\n\\n\\n### Similar content being viewed by others\\n\\n\\n\\n\\n\\n![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-022-34997-w/MediaObjects/41467_2022_34997_Fig1_HTML.png)\\n\\n### [Structures of the eukaryotic ribosome and its translational states in situ](https://www.nature.com/articles/s41467-022-34997-w?fromPaywallRec=false)\\n\\n\\n\\nArticle\\nOpen access\\n02 December 2022\\n\\n\\n\\n\\n\\n\\n![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-019-1249-5/MediaObjects/41586_2019_1249_Fig1_HTML.png)\\n\\n### [Late steps in bacterial translation initiation visualized using time-resolved cryo-EM](https://www.nature.com/articles/s41586-019-1249-5?fromPaywallRec=false)\\n\\n\\n\\nArticle\\n20 May 2019\\n\\n\\n\\n\\n\\n\\n![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs12276-023-00933-1/MediaObjects/12276_2023_933_Fig1_HTML.png)\\n\\n### [Single-molecule visualization of mRNA circularization during translation](https://www.nature.com/articles/s12276-023-00933-1?fromPaywallRec=false)\\n\\n\\n\\nArticle\\nOpen access\\n31 January 2023\\n\\n\\n\\n\\n\\n\\n\\n\\nMain\\n----\\n\\nTranslation of genetic information through messenger RNAs (mRNAs) into proteins is performed by the ribosome, one of the primordial macromolecular machines in cells[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"). The ribosome consists of a small and a large subunit (30S and 50S in prokaryotes), which form the aminoacyl (A), peptidyl (P) and exit (E) transfer RNA (tRNA) binding sites at their interface. The translation process can be divided into four phases: initiation, elongation, termination and recycling[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"). During the elongation phase, the ribosome undergoes a fundamental cycle to add one amino acid to the nascent peptide chain, which can be subdivided into three steps: decoding, peptidyl transfer and translocation. These steps involve structural changes in the ribosome that include subunit rotations, elongation factor association and tRNA accommodation[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"). Many intermediates during the elongation cycle have been identified on the basis of structures derived by cryo-electron microscopy (cryo-EM) and computational image classification[6](#ref-CR6 \\\"Frank, J. & Agrawal, R. K. A ratchet-like inter-subunit reorganization of the ribosome during translocation. Nature 406, 318–322 (2000).\\\"),[7](#ref-CR7 \\\"Villa, E. et al. Ribosome-induced changes in elongation factor Tu conformation control GTP hydrolysis. Proc. Natl Acad. Sci. USA 106, 1063–1068 (2009).\\\"),[8](#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[9](#ref-CR9 \\\"Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. Nature 468, 713–716 (2010).\\\"),[10](#ref-CR10 \\\"Ramrath, D. J. et al. Visualization of two transfer RNAs trapped in transit during elongation factor G-mediated translocation. Proc. Natl Acad. Sci. USA 110, 20964–20969 (2013).\\\"),[11](#ref-CR11 \\\"Brilot, A. F., Korostelev, A. A., Ermolenko, D. N. & Grigorieff, N. Structure of the ribosome with elongation factor G trapped in the pretranslocation state. Proc. Natl Acad. Sci. USA 110, 20994–20999 (2013).\\\"),[12](#ref-CR12 \\\"Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. Cell 161, 845–857 (2015).\\\"),[13](#ref-CR13 \\\"Loveland, A. B., Demo, G. & Korostelev, A. A. Cryo-EM of elongating ribosome with EF-Tu*GTP elucidates tRNA proofreading. Nature 584, 640–645 (2020).\\\"),[14](#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[15](#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\"),[16](/articles/s41586-022-05255-2#ref-CR16 \\\"Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. Nat. Commun. 12, 7236 (2021).\\\"). Most available structures, however, are of ribosomes that were isolated from model bacteria such as *Escherichia coli* and *Thermus thermophilus*, and were often trapped in specific states by antibiotics, GTP analogues or mutations[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"),[17](/articles/s41586-022-05255-2#ref-CR17 \\\"Brown, A. & Shao, S. Ribosomes and cryo-EM: a duet. Curr. Opin. Struct. Biol. 52, 1–7 (2018).\\\"). Thus, a detailed structural description of the translation process within the native cellular context is lacking. Although actively translating ribosomes have been visualized inside cells by cryo-electron tomography (cryo-ET), the maps generated in previous studies were limited to resolutions on the nanometre scale[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"),[18](#ref-CR18 \\\"Kuhner, S. et al. Proteome organization in a genome-reduced bacterium. Science 326, 1235–1240 (2009).\\\"),[19](#ref-CR19 \\\"Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. Mol. Cell 39, 560–569 (2010).\\\"),[20](#ref-CR20 \\\"Mahamid, J. et al. Visualizing the molecular sociology at the HeLa cell nuclear periphery. Science 351, 969–972 (2016).\\\"),[21](#ref-CR21 \\\"Schaffer, M. et al. A cryo-FIB lift-out technique enables molecular-resolution cryo-ET within native Caenorhabditis elegans tissue. Nat. Methods 16, 757–762 (2019).\\\"),[22](/articles/s41586-022-05255-2#ref-CR22 \\\"Waltz, F. et al. How to build a ribosome from RNA fragments in Chlamydomonas mitochondria. Nat. Commun. 12, 7176 (2021).\\\"). We have recently developed image-processing algorithms for cryo-ET that make it possible to resolve stalled ribosomes to residue level inside the genome-reduced bacterium *M.* *pneumoniae*[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\"). Here, we use these technical advances to perform large-scale structure classification and spatial analysis of ribosomes, to visualize parts of the translation process in great detail inside *M.* *pneumoniae* as a prokaryotic minimal cell model[18](/articles/s41586-022-05255-2#ref-CR18 \\\"Kuhner, S. et al. Proteome organization in a genome-reduced bacterium. Science 326, 1235–1240 (2009).\\\").\\n\\nIn-cell structure of the *M. pneumoniae* ribosome\\n-------------------------------------------------\\n\\nTo investigate the structural details of the translation machinery, we first obtained 3.5-Å consensus maps by averaging all ribosomes detected in cryo-electron tomograms of intact *M.* *pneumoniae* cells (Fig. [1a,b](/articles/s41586-022-05255-2#Fig1), Extended Data Fig. [1](/articles/s41586-022-05255-2#Fig5), Supplementary Table [1](/articles/s41586-022-05255-2#MOESM1), [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1) and [Methods](/articles/s41586-022-05255-2#Sec7)). Focused refinements on the 30S and 50S subunits improved the map quality and revealed well-resolved ribosomal RNA (rRNA) bases and ribosomal protein amino acid side chains (Fig. [1b](/articles/s41586-022-05255-2#Fig1) and Extended Data Fig. [2a–c](/articles/s41586-022-05255-2#Fig6)). The high-resolution in-cell consensus maps allowed us to build de novo an atomic model for the *M.* *pneumoniae* ribosome (Extended Data Fig. [2d–g](/articles/s41586-022-05255-2#Fig6) and [Methods](/articles/s41586-022-05255-2#Sec7)). The structure shows high similarity to other bacterial ribosomes, but also reveals several new features (Fig. [1c](/articles/s41586-022-05255-2#Fig1) and Extended Data Fig. [2d–g](/articles/s41586-022-05255-2#Fig6)). Specifically, 11 of the 52 ribosomal proteins in *M.* *pneumoniae* have extended sequences compared to *E.* *coli* (Extended Data Fig. [3a](/articles/s41586-022-05255-2#Fig7) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). Most of these extensions are predicted to be disordered, but those of ribosomal proteins S6, L22 and L29 form secondary structures and were built in the model (Fig. [1c](/articles/s41586-022-05255-2#Fig1) and Extended Data Figs. [2d–e](/articles/s41586-022-05255-2#Fig6) and [3a](/articles/s41586-022-05255-2#Fig7)). We found such extensions to be common throughout the bacterial kingdom (Extended Data Fig. [3b](/articles/s41586-022-05255-2#Fig7), [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1) and Supplementary Table [2](/articles/s41586-022-05255-2#MOESM1)), possibly representing ribosome diversity in adaptation to different environments and lifestyles[23](/articles/s41586-022-05255-2#ref-CR23 \\\"Melnikov, S., Manakongtreecheep, K. & Soll, D. Revising the structural diversity of ribosomal proteins across the three domains of life. Mol. Biol. Evol. 35, 1588–1598 (2018).\\\"). Although the functions of these extensions remain largely elusive, their disruption affects cellular fitness or survival in *M.*  *pneumoniae*[24](/articles/s41586-022-05255-2#ref-CR24 \\\"Miravet-Verde, S., Burgos, R., Delgado, J., Lluch-Senar, M. & Serrano, L. FASTQINS and ANUBIS: two bioinformatic tools to explore facts and artifacts in transposon sequencing and essentiality studies. Nucleic Acids Res. 48, e102 (2020).\\\") ([Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). Thus, the high-resolution ribosome map and the atomic model derived from intact *M.* *pneumoniae* provide the basis to investigate in detail the conformational and compositional changes of ribosomes during translation inside cells.\\n\\n\\n\\n**Fig. 1: Ribosome structure in *M.* *pneumoniae* cells.**\\n\\n[![figure 1](//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_Fig1_HTML.png)](/articles/s41586-022-05255-2/figures/1)**a**, A denoised tomographic slice of a *M.* *pneumoniae* cell. AO, attachment organelle; PM, plasma membrane. Examples of ribosomes are circled. The representative tomogram (selected from 15) was acquired with a Volta phase plate for better visualization of the cellular morphology. Similar imaging conditions, excluding the use of a phase plate, were used for the acquisition of all tomograms for subsequent analysis. Scale bar, 100 nm. **b**, A 3.5-Å in-cell ribosome map (left) shows well-resolved rRNA bases and ribosomal protein side chains (right). **c**, An atomic model of the *M.* *pneumoniae* ribosome shows structural similarity to other bacterial ribosomes. Eleven ribosomal proteins (dark green) have sequence extensions (magenta).\\n\\n[Full size image](/articles/s41586-022-05255-2/figures/1)Structural dynamics of translation in cells\\n-------------------------------------------\\n\\nTo analyse the structural changes that are associated with the translation process inside cells, we performed computational classification of 101,696 ribosomes from 356 tomograms of native cells, and obtained 13 distinct ribosome classes (Extended Data Fig. [4](/articles/s41586-022-05255-2#Fig8), Supplementary Table [3](/articles/s41586-022-05255-2#MOESM1) and [Methods](/articles/s41586-022-05255-2#Sec7)). Ten classes determined at resolutions ranging from 4 to 10 Å were assigned to the translation elongation phase on the basis of elongation factor and tRNAs binding to the ribosome (Fig. [2a](/articles/s41586-022-05255-2#Fig2) and Extended Data Figs. [4](/articles/s41586-022-05255-2#Fig8)–[7](/articles/s41586-022-05255-2#Fig11)). The remaining three classes represent 70S with a single P/E-site tRNA, 50S in complex with the ribosome recycling factor, and free 50S subunits (Extended Data Figs. [4](/articles/s41586-022-05255-2#Fig8)–[7](/articles/s41586-022-05255-2#Fig11) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). The ten classes within the elongation phase account for 70% of the detected ribosomes, consistent with the expectation that most ribosomes inside living cells are engaged in the elongation phase, which lasts considerably longer than the initiation, termination and recycling phases[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"),[25](/articles/s41586-022-05255-2#ref-CR25 \\\"Borg, A., Pavlov, M. & Ehrenberg, M. Complete kinetic mechanism for recycling of the bacterial ribosome. RNA 22, 10–21 (2016).\\\").\\n\\n\\n\\n**Fig. 2: Ribosome classification reconstructs the translation elongation cycle.**\\n\\n[![figure 2](//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_Fig2_HTML.png)](/articles/s41586-022-05255-2/figures/2)**a**, Ten 70S structures determined inside *M.* *pneumoniae* cells represent translation elongation intermediate states, which are characterized by the binding of different elongation factors (EFs) and tRNAs. The frequencies of occurrence for different intermediates are calculated from the classified particle numbers (Extended Data Fig. [4](/articles/s41586-022-05255-2#Fig8)). **b**, Densities for mRNA, tRNAs and the nascent peptide chain are well-resolved in the most-populated 'A, P' state. **c**, Trajectories of the elongation factors EF-Tu and EF-G and the A-, P- and E-site tRNAs along translation elongation. **d**, Major conformational changes of the ribosome along the elongation trajectory: 30S body rotation, 30S head swivel and L1 stalk opening.\\n\\n[Full size image](/articles/s41586-022-05255-2/figures/2)The identified elongation classes can be ordered to reconstruct the translation elongation cycle[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[5](/articles/s41586-022-05255-2#ref-CR5 \\\"Rodnina, M. V. Translation in prokaryotes. Cold Spring Harb. Perspect. Biol. 10, a032664 (2018).\\\"). By flexible fitting of our *M.* *pneumoniae* ribosome atomic model into the classified maps ([Methods](/articles/s41586-022-05255-2#Sec7) and Supplementary Table [3](/articles/s41586-022-05255-2#MOESM1)), we obtained pseudo-atomic models that delineate conformational and compositional changes of the ribosome complex, including rotations of 30S body and head, coordination of elongation factors, movement of tRNAs through the A–P–E sites and dynamics of the L1 stalk (Fig. [2](/articles/s41586-022-05255-2#Fig2), Extended Data Fig. [7a–j](/articles/s41586-022-05255-2#Fig11) and Supplementary Videos [1](/articles/s41586-022-05255-2#MOESM3) and [2](/articles/s41586-022-05255-2#MOESM4)). The number of ribosomes within each of the classes provides the relative abundance of translation elongation intermediates, which together reflect a steady state distribution determined by their relative rates of formation and depletion inside the cells (Fig. [2a](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8)). The classic, non-rotated 'A, P' state was the most populated and determined at the highest resolution, showing clear density for mRNA, tRNAs and the nascent peptide (Fig. [2b](/articles/s41586-022-05255-2#Fig2)). In the following states, we observed the oscillation of A- and P-site tRNAs into the hybrid A/P and P/E state, coupled with 30S subunit rotations and L1 stalk movement (Fig. [2a,c,d](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [7a–j](/articles/s41586-022-05255-2#Fig11)), consistent with in vitro studies[6](/articles/s41586-022-05255-2#ref-CR6 \\\"Frank, J. & Agrawal, R. K. A ratchet-like inter-subunit reorganization of the ribosome during translocation. Nature 406, 318–322 (2000).\\\"),[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[9](/articles/s41586-022-05255-2#ref-CR9 \\\"Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. Nature 468, 713–716 (2010).\\\"),[12](/articles/s41586-022-05255-2#ref-CR12 \\\"Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. Cell 161, 845–857 (2015).\\\"),[26](#ref-CR26 \\\"Munro, J. B., Altman, R. B., O’Connor, N. & Blanchard, S. C. Identification of two distinct hybrid state intermediates on the ribosome. Mol. Cell 25, 505–517 (2007).\\\"),[27](#ref-CR27 \\\"Belardinelli, R. et al. Choreography of molecular movements during ribosome progression along mRNA. Nat. Struct. Mol. Biol. 23, 342–348 (2016).\\\"),[28](#ref-CR28 \\\"Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. Nat. Struct. Mol. Biol. 23, 333–341 (2016).\\\"),[29](#ref-CR29 \\\"Fei, J., Kosuri, P., MacDougall, D. D. & Gonzalez, R. L. Jr Coupling of ribosomal L1 stalk and tRNA dynamics during translation elongation. Mol. Cell 30, 348–359 (2008).\\\"),[30](/articles/s41586-022-05255-2#ref-CR30 \\\"Zhou, J., Lancaster, L., Donohue, J. P. & Noller, H. F. How the ribosome hands the A-site tRNA to the P site during EF-G-catalyzed translocation. Science 345, 1188–1191 (2014).\\\"). Along this trajectory, we also determined a partial hybrid 'A\\\\*, P/E' state with rotated 30S and deacyl-tRNA in the hybrid P/E site, but with only marginally relocated peptidyl-tRNA in the A site (Fig. [2a,d](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [8a](/articles/s41586-022-05255-2#Fig12)), similar to a previously reported pre-translocational H2\\\\* state[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[26](/articles/s41586-022-05255-2#ref-CR26 \\\"Munro, J. B., Altman, R. B., O’Connor, N. & Blanchard, S. C. Identification of two distinct hybrid state intermediates on the ribosome. Mol. Cell 25, 505–517 (2007).\\\"). When viewed in the context of this trajectory, the high enrichment of the 'A, P' state could suggest that inter-subunit rotation represents a rate-limiting step for translation elongation in *M.* *pneumoniae*. However, in view of the entirety of the elongation cycle, processive inter-subunit rotation leading to efficient translocation requires the binding of the elongation factor EF-G. Notably, EF-G was found to bind to the ribosome in either the partial hybrid or the full hybrid states (Extended Data Fig. [8b–m](/articles/s41586-022-05255-2#Fig12)). EF-G in the partial hybrid state (class 6e) is less extended and its domain IV does not overlap with the A site compared to the full hybrid state (Extended Data Fig. [8b–h](/articles/s41586-022-05255-2#Fig12)). This EF-G-bound partial hybrid state therefore resembles early translocation intermediates before phosphate release[9](/articles/s41586-022-05255-2#ref-CR9 \\\"Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. Nature 468, 713–716 (2010).\\\"),[14](#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[15](#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\"),[16](/articles/s41586-022-05255-2#ref-CR16 \\\"Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. Nat. Commun. 12, 7236 (2021).\\\"). In the following full hybrid state (class 7), domain IV of EF-G extends about 20 Å towards the A site, owing to both the rotation of entire EF-G and its inter-domain rearrangement (Fig. [2c](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [8f](/articles/s41586-022-05255-2#Fig12)). Class 8 next shows a reverse rotation of the 30S body, the largest magnitude for 30S head swivel, the fully extended EF-G and the chimeric 'ap/P, pe/E' tRNAs (Fig. [2c,d](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [8g, j](/articles/s41586-022-05255-2#Fig12)), indicating that it is a late translocation intermediate[9](/articles/s41586-022-05255-2#ref-CR9 \\\"Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. Nature 468, 713–716 (2010).\\\"),[10](/articles/s41586-022-05255-2#ref-CR10 \\\"Ramrath, D. J. et al. Visualization of two transfer RNAs trapped in transit during elongation factor G-mediated translocation. Proc. Natl Acad. Sci. USA 110, 20964–20969 (2013).\\\"),[14](#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[15](#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\"),[16](/articles/s41586-022-05255-2#ref-CR16 \\\"Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. Nat. Commun. 12, 7236 (2021).\\\"),[28](/articles/s41586-022-05255-2#ref-CR28 \\\"Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. Nat. Struct. Mol. Biol. 23, 333–341 (2016).\\\"). We thus demonstrate the existence of continual intermediates from early to late translocation stages during active translation elongation inside cells. Two EF-Tu-associated structures were also determined, with and without E-site tRNA (Fig. [2a](/articles/s41586-022-05255-2#Fig2) and Extended Data Fig. [7c,d](/articles/s41586-022-05255-2#Fig11)). This suggests that the binding of EF-Tu•tRNA to the ribosome and the disassociation of E-site tRNA are independent of each other. The 'P, E' class has a relatively low abundance, which suggests that the E-site tRNA is not stable and tends to disassociate quickly after translocation. This is in agreement with previous single-molecule fluorescence studies showing rapid release of the E-site tRNA[31](/articles/s41586-022-05255-2#ref-CR31 \\\"Uemura, S. et al. Real-time tRNA transit on single translating ribosomes at codon resolution. Nature 464, 1012–U1073 (2010).\\\"),[32](/articles/s41586-022-05255-2#ref-CR32 \\\"Chen, C. et al. Single-molecule fluorescence measurements of ribosomal translocation dynamics. Mol. Cell 42, 367–377 (2011).\\\"), and helps to resolve a long-standing controversy over its disassociation time point[1](/articles/s41586-022-05255-2#ref-CR1 \\\"Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. Annu. Rev. Biochem. 82, 203–236 (2013).\\\"),[12](/articles/s41586-022-05255-2#ref-CR12 \\\"Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. Cell 161, 845–857 (2015).\\\"),[33](/articles/s41586-022-05255-2#ref-CR33 \\\"Wilson, D. N. & Nierhaus, K. H. The E-site story: the importance of maintaining two tRNAs on the ribosome during protein synthesis. Cell. Mol. Life Sci. 63, 2725–2737 (2006).\\\"). In summary, these results recapitulate major steps in translation elongation that have been defined by controlled in vitro studies and reconstruct the structural dynamics of the elongation cycles inside cells, highlighting the possible various paths of the reaction catalysed by the ribosome complex. Furthermore, with these results, a probability map for the occurrence of intermediate states can be derived to illustrate a putative energy landscape of translation elongation in cells. Energy landscapes of translation are suggested to be rugged[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[12](/articles/s41586-022-05255-2#ref-CR12 \\\"Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. Cell 161, 845–857 (2015).\\\"),[34](/articles/s41586-022-05255-2#ref-CR34 \\\"Munro, J. B., Sanbonmatsu, K. Y., Spahn, C. M. & Blanchard, S. C. Navigating the ribosome’s metastable energy landscape. Trends Biochem. Sci 34, 390–400 (2009).\\\"),[35](/articles/s41586-022-05255-2#ref-CR35 \\\"Bock, L. V. et al. Energy barriers and driving forces in tRNA translocation through the ribosome. Nat. Struct. Mol. Biol. 20, 1390–1396 (2013).\\\"), depending on factors such as Mg2+ concentrations[36](/articles/s41586-022-05255-2#ref-CR36 \\\"Julian, P. et al. Structure of ratcheted ribosomes with tRNAs in hybrid states. Proc. Natl Acad. Sci. USA 105, 16924–16927 (2008).\\\"),[37](/articles/s41586-022-05255-2#ref-CR37 \\\"Agirrezabala, X. et al. Visualization of the hybrid state of tRNA binding promoted by spontaneous ratcheting of the ribosome. Mol. Cell 32, 190–197 (2008).\\\"), the presence of EF-G[34](/articles/s41586-022-05255-2#ref-CR34 \\\"Munro, J. B., Sanbonmatsu, K. Y., Spahn, C. M. & Blanchard, S. C. Navigating the ribosome’s metastable energy landscape. Trends Biochem. Sci 34, 390–400 (2009).\\\"),[38](/articles/s41586-022-05255-2#ref-CR38 \\\"Rodnina, M. V., Peske, F., Peng, B. Z., Belardinelli, R. & Wintermeyer, W. Converting GTP hydrolysis into motion: versatile translational elongation factor G. Biol. Chem. 401, 131–142 (2019).\\\") and temperature[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"). In *M.* *pneumoniae*, the intracellular concentrations of ribosomes, translation elongation factors and substrates can be up to ten times lower than those in *E.* *coli* ([Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). It is therefore possible that the energy landscapes of translation vary for different organisms, cell types and conditions. Applying the analysis introduced here to other cell types and conditions can deepen our understanding of the translation process within the native cellular context.\\n\\nAntibiotics alter translation landscapes\\n----------------------------------------\\n\\nThe ribosome is one of the most important targets for antibiotics, many of which are known to stall translation by stabilizing certain intermediates in the process[39](/articles/s41586-022-05255-2#ref-CR39 \\\"Lin, J., Zhou, D., Steitz, T. A., Polikanov, Y. S. & Gagnon, M. G. Ribosome-targeting antibiotics: modes of action, mechanisms of resistance, and implications for drug design. Annu. Rev. Biochem. 87, 451–478 (2018).\\\"). To investigate the effects of antibiotics on the cellular translation machinery, we analysed ribosomes in cells that were treated for a short time (15–20 min; [Methods](/articles/s41586-022-05255-2#Sec7)) with two representative ribosome-targeting antibiotics: chloramphenicol (Cm), which binds to the peptidyl transfer centre in the 50S subunit and inhibits peptide bond formation[40](/articles/s41586-022-05255-2#ref-CR40 \\\"Bulkley, D., Innis, C. A., Blaha, G. & Steitz, T. A. Revisiting the structures of several antibiotics bound to the bacterial ribosome. Proc. Natl Acad. Sci. USA 107, 17158–17163 (2010).\\\"),[41](/articles/s41586-022-05255-2#ref-CR41 \\\"Svetlov, M. S. et al. High-resolution crystal structures of ribosome-bound chloramphenicol and erythromycin provide the ultimate basis for their competition. RNA 25, 600–606 (2019).\\\"); and spectinomycin (Spc), which binds to the 30S neck and blocks translocation[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[28](/articles/s41586-022-05255-2#ref-CR28 \\\"Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. Nat. Struct. Mol. Biol. 23, 333–341 (2016).\\\"),[42](/articles/s41586-022-05255-2#ref-CR42 \\\"Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. ACS Chem. Biol. 2, 545–552 (2007).\\\"). Overall, our sub-tomogram analysis of antibiotic-treated cells results in 17 ribosome maps, and shows that translation landscapes are markedly reshaped by different antibiotics (Fig. [3a](/articles/s41586-022-05255-2#Fig3), Extended Data Figs. [9](/articles/s41586-022-05255-2#Fig13)–[11](/articles/s41586-022-05255-2#Fig15), Supplementary Tables [4](/articles/s41586-022-05255-2#MOESM1)–[6](/articles/s41586-022-05255-2#MOESM1) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)).\\n\\n\\n\\n**Fig. 3: Antibiotics induce distinct translation elongation landscapes in cells.**\\n\\n[![figure 3](//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_Fig3_HTML.png)](/articles/s41586-022-05255-2/figures/3)**a**, Distribution of translation elongation intermediates in native untreated cells, and in cells treated with three different antibiotics. Bar and whiskers indicate mean and s.d. for each class across all cells in the different treatment groups: untreated (*n* = 356 cells); +Cm, chloramphenicol-treated (*n* = 65 cells); +Spc, spectinomycin-treated (*n* = 70 cells); and +PUM, pseudouridimycin-treated (*n* = 86 cells). **b**, Ribosomes in Spc-treated cells are largely stalled in the 'EF-G, A/PSpc, P/E' state. **c**, The Spc molecule (magenta) is well-resolved and built in the 'EF-G, A/PSpc, P/E' ribosome model. It is surrounded by several 16S rRNA bases and loop 2 of ribosomal protein S5 near the 30S neck. **d**, The major state in Spc-treated cells is similar to the 'EF-G, A/PPUM, P/E' class in PUM-treated cells (light blue) and the 'EF-G, A/P, P/E' class in untreated cells (light grey), differing only in the position of the A/P-site tRNA on the 50S side. **e**, Single-cell clustering analysis on the basis of the translation elongation states of 577 individual cells under native and different antibiotic treatment conditions.\\n\\n[Full size image](/articles/s41586-022-05255-2/figures/3)In Cm-treated cells, 72% of 70S ribosomes are trapped in the 'A, P' state (class 4), owing to the inhibition of peptidyl transfer[39](/articles/s41586-022-05255-2#ref-CR39 \\\"Lin, J., Zhou, D., Steitz, T. A., Polikanov, Y. S. & Gagnon, M. G. Ribosome-targeting antibiotics: modes of action, mechanisms of resistance, and implications for drug design. Annu. Rev. Biochem. 87, 451–478 (2018).\\\") (Fig. [3a–d](/articles/s41586-022-05255-2#Fig3), Extended Data Fig. [9](/articles/s41586-022-05255-2#Fig13) and Supplementary Table [4](/articles/s41586-022-05255-2#MOESM1)). The Cm molecule was resolved in its canonical binding site[40](/articles/s41586-022-05255-2#ref-CR40 \\\"Bulkley, D., Innis, C. A., Blaha, G. & Steitz, T. A. Revisiting the structures of several antibiotics bound to the bacterial ribosome. Proc. Natl Acad. Sci. USA 107, 17158–17163 (2010).\\\"),[41](/articles/s41586-022-05255-2#ref-CR41 \\\"Svetlov, M. S. et al. High-resolution crystal structures of ribosome-bound chloramphenicol and erythromycin provide the ultimate basis for their competition. RNA 25, 600–606 (2019).\\\") (Extended Data Fig. [9d,g](/articles/s41586-022-05255-2#Fig13)). Of note, 28% of ribosomes were found in several other elongation states, either before or following the major 'A, P' state (Fig. [3a](/articles/s41586-022-05255-2#Fig3) and Extended Data Fig. [9](/articles/s41586-022-05255-2#Fig13)). The existence of an EF-Tu•tRNA-bound state (class 3) is reminiscent of the effects of other antibiotics that inhibit peptidyl transfer in a similar manner[43](#ref-CR43 \\\"Polikanov, Y. S. et al. Distinct tRNA accommodation intermediates observed on the ribosome with the antibiotics hygromycin A and A201A. Mol. Cell 58, 832–844 (2015).\\\"),[44](#ref-CR44 \\\"Morse, J. C. et al. Elongation factor-Tu can repetitively engage aminoacyl-tRNA within the ribosome during the proofreading stage of tRNA selection. Proc. Natl Acad. Sci. USA 117, 3610–3620 (2020).\\\"),[45](/articles/s41586-022-05255-2#ref-CR45 \\\"Choi, J. et al. Dynamics of the context-specific translation arrest by chloramphenicol and linezolid. Nat. Chem. Biol. 16, 310–317 (2020).\\\").\\n\\nSpc is suggested to inhibit translocation by blocking head swivel of the 30S subunit[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[28](/articles/s41586-022-05255-2#ref-CR28 \\\"Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. Nat. Struct. Mol. Biol. 23, 333–341 (2016).\\\"),[42](/articles/s41586-022-05255-2#ref-CR42 \\\"Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. ACS Chem. Biol. 2, 545–552 (2007).\\\"). We found that 67.5% of 70S ribosomes were stalled in the 'EF-G, A/PSpc, P/E' state in Spc-treated cells, a state similar to the pre-translocational 'EF-G, A/P, P/E' state (class 7) in untreated cells (Fig. [3a–d](/articles/s41586-022-05255-2#Fig3), Extended Data Fig. [10](/articles/s41586-022-05255-2#Fig14) and Supplementary Table [5](/articles/s41586-022-05255-2#MOESM1)). The Spc molecule density is clearly visible near helix 34 within the 30S neck region (Fig. [3b,c](/articles/s41586-022-05255-2#Fig3) and Extended Data Fig. [10d,g](/articles/s41586-022-05255-2#Fig14)), consistent with its reported binding site[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"),[42](/articles/s41586-022-05255-2#ref-CR42 \\\"Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. ACS Chem. Biol. 2, 545–552 (2007).\\\"). These results show that Spc inhibits translocation by impeding the dynamics of the 30S subunit. Similar to Cm, ribosomes in three additional elongation states could be detected with lower frequencies (Fig. [3a](/articles/s41586-022-05255-2#Fig3) and Extended Data Fig. [10e,f](/articles/s41586-022-05255-2#Fig14)).\\n\\nPerturbation of other molecular pathways that are functionally coupled to translation in bacterial cells also affects the translation landscape; RNA polymerase stalled by pseudouridimycin (PUM) can physically block mRNA translocation in the ribosome that collides with it during transcription–translation coupling[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). Consistently, 59.5% of 70S ribosomes in PUM-treated cells were found in the 'EF-G, A/PPUM, P/E' state (Fig. [3a](/articles/s41586-022-05255-2#Fig3), Extended Data Fig. [11](/articles/s41586-022-05255-2#Fig15) and Supplementary Table [6](/articles/s41586-022-05255-2#MOESM1)), which resembles the pre-translocational 'EF-G, A/P, P/E' state in untreated cells and the stalled 'EF-G, A/PSpc, P/E' state in Spc-treated cells (Fig. [3d](/articles/s41586-022-05255-2#Fig3)). Our finding that physically obstructing mRNA translocation by a PUM-stalled RNA polymerase and chemically impeding 30S head dynamics by Spc lead to similar structures further confirms that mRNA translocation and 30S rotations are directly coupled.\\n\\nThe observation that treatment with antibiotics resulted in minor states that are not expected from their specific binding prompted us to investigate possible cell-to-cell variability in response to the antibiotics. To this end, we performed clustering analysis on the basis of translation elongation profiles in 577 tomograms of single cells under native and antibiotic treatment conditions. The analysis resulted in four major clusters in accordance with the four treatment groups, demonstrating small cell-to-cell variability within each cluster (Fig. [3e](/articles/s41586-022-05255-2#Fig3)). Thus, our results show that the translation landscapes in cells are globally reshaped by small molecules specifically binding to ribosomes, as well as to other targets ([Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1) and Supplementary Table [8](/articles/s41586-022-05255-2#MOESM1)). The presence of minor elongation states under antibiotic treatment is reminiscent of previous studies that have shown ongoing slow translation in antibiotic-treated cells and context-dependent inhibition[46](#ref-CR46 \\\"Marks, J. et al. Context-specific inhibition of translation by ribosomal antibiotics targeting the peptidyl transferase center. Proc. Natl Acad. Sci. USA 113, 12150–12155 (2016).\\\"),[47](#ref-CR47 \\\"Vazquez-Laslop, N. & Mankin, A. S. Context-specific action of ribosomal antibiotics. Annu. Rev. Microbiol. 72, 185–207 (2018).\\\"),[48](/articles/s41586-022-05255-2#ref-CR48 \\\"Aguirre Rivera, J. et al. Real-time measurements of aminoglycoside effects on protein synthesis in live cells. Proc. Natl Acad. Sci. USA 118, e2013315118 (2021).\\\"). For example, Cm inhibition is affected by specific residues of the nascent peptide[46](/articles/s41586-022-05255-2#ref-CR46 \\\"Marks, J. et al. Context-specific inhibition of translation by ribosomal antibiotics targeting the peptidyl transferase center. Proc. Natl Acad. Sci. USA 113, 12150–12155 (2016).\\\"),[47](/articles/s41586-022-05255-2#ref-CR47 \\\"Vazquez-Laslop, N. & Mankin, A. S. Context-specific action of ribosomal antibiotics. Annu. Rev. Microbiol. 72, 185–207 (2018).\\\"). Most antibiotics, including Cm and Spc, are known to inhibit cell growth but do not immediately kill the cell[49](/articles/s41586-022-05255-2#ref-CR49 \\\"Baquero, F. & Levin, B. R. Proximate and ultimate causes of the bactericidal action of antibiotics. Nat. Rev. Microbiol. 19, 123–132 (2021).\\\"). It is possible that the reshaped translation landscapes by antibiotics lead to an imbalance in protein synthesis, which in the long run has detrimental consequences for the cell.\\n\\nSpatial and functional organization of translation\\n--------------------------------------------------\\n\\nFinally, we investigated the spatial organization of active translation in native *M.* *pneumoniae* cells (Fig. [4a](/articles/s41586-022-05255-2#Fig4) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). It is known that ribosomes translating on the same mRNA can assemble closely in space to form polysomes[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"),[19](/articles/s41586-022-05255-2#ref-CR19 \\\"Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. Mol. Cell 39, 560–569 (2010).\\\"). We defined polysomes using a distance cut-off of 7 nm from the mRNA exit site to the mRNA entry site between neighbouring ribosomes (Extended Data Fig. [12a–e](/articles/s41586-022-05255-2#Fig16), [Methods](/articles/s41586-022-05255-2#Sec7) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). The detected polysomes account for 26.2% of all 70S ribosomes, and two arrangement patterns between neighbouring ribosomes can be defined (Fig. [4b–e](/articles/s41586-022-05255-2#Fig4) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)): the so-called 'top-top' (t-t; 78.5%, mRNA exit-to-entry distance 4.2 ± 1.4 nm) and 'top-back' (t-b; 21.5%, 5.4 ± 1.5 nm) configurations[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"). We also observed various topologies for long polysomes, ranging from loose assembly to tight packing with helix-like configurations[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"),[19](/articles/s41586-022-05255-2#ref-CR19 \\\"Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. Mol. Cell 39, 560–569 (2010).\\\"),[50](/articles/s41586-022-05255-2#ref-CR50 \\\"Myasnikov, A. G. et al. The molecular structure of the left-handed supra-molecular helix of eukaryotic polyribosomes. Nat. Commun. 5, 5294 (2014).\\\") (Fig. [4c,d](/articles/s41586-022-05255-2#Fig4)).\\n\\n\\n\\n**Fig. 4: Spatial and functional organization of ribosomes in native cells.**\\n\\n[![figure 4](//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_Fig4_HTML.png)](/articles/s41586-022-05255-2/figures/4)**a**, Three-dimensional map of ribosomes in a representative native untreated cell (selected from 356). Top, *x*–*y* view; bottom, orthogonal view. The 70S intermediates within the translation elongation cycle (classes 1–8, as in Fig. [2a](/articles/s41586-022-05255-2#Fig2)), and additional classes (classes 11–13 and 50–51 as detailed in Extended Data Fig. [4](/articles/s41586-022-05255-2#Fig8)) are coloured as indicated in the colour scheme (inset). 50S: light grey. **b**, 'top-back' (t-b) and 'top-top' (t-t) assembly patterns of adjacent ribosomes in polysomes. **c**,**d**, Representative long polysomes of loose (**c**) and tight (**d**) topologies, with the corresponding putative mRNA paths and nascent chain vectors shown underneath (not drawn to scale). **e**, Distribution of polysome lengths. **f**, Distribution of elongation states in polysomes compared to all ribosomes and mono-ribosomes. Bars and whiskers are mean and s.d. across 356 tomograms of untreated cells (*n* = 356 cells). Highlighted are states for which the fraction in polysomes differs by more than 50% compared to all ribosomes or mono-ribosomes. Asterisks indicate false discovery rate (FDR)-adjusted *P* value (*P*FDR) < 0.01 (two-sided Wilcoxon rank sum test). *P*FDR values for class 5: 6.44 × 10−23 (polysome versus all) and 2.76 × 10−39 (polysome versus monosome); class 6a: 3.06 × 10−28 and 1.44 × 10−34, respectively. **g**, Occurrence frequencies of elongation state pairs of adjacent ribosomes in polysomes normalized to the theoretical probability of random pairs. States that require elongation factor binding to proceed are 1, 2a, 5 and 6a (in bold). **h**, Map of a di-ribosome within polysomes shows the intervening mRNA density (inset: blue arrowhead) and the extended L9 of the preceding ribosome (i). The C-terminal domain of extended L9 can interfere with elongation factor (EF) binding to the following ribosome (i+1).\\n\\n[Full size image](/articles/s41586-022-05255-2/figures/4)Whether translation elongation is synchronized or coordinated within polysomes is a long-standing question[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"),[19](/articles/s41586-022-05255-2#ref-CR19 \\\"Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. Mol. Cell 39, 560–569 (2010).\\\"),[50](/articles/s41586-022-05255-2#ref-CR50 \\\"Myasnikov, A. G. et al. The molecular structure of the left-handed supra-molecular helix of eukaryotic polyribosomes. Nat. Commun. 5, 5294 (2014).\\\"). To address this, we first examined whether the distribution of elongation states differs between the total ribosome population and polysomes. We found that although most states occurred equally frequently in both populations, the fractions of two states before EF-G binding (class 5, 'A\\\\*, P/E' and class 6a, 'A/P, P/E') are more frequent in polysomes (Fig. [4f](/articles/s41586-022-05255-2#Fig4)). We next calculated the frequencies of state pairs between two adjacent ribosomes (preceding versus following) and compared them to theoretical pair frequencies calculated from the bulk distribution (Extended Data Fig. [13a–c](/articles/s41586-022-05255-2#Fig17)). This comparison revealed that the occurrence of states among preceding and following ribosomes is not symmetric: following ribosomes more frequently populate states that require elongation factor binding to proceed to the next state (classes 1, 2a, 5 and 6a; Fig. [4g](/articles/s41586-022-05255-2#Fig4)). We statistically validated such differences with a permutation test, and further show that the asymmetry between preceding and following ribosomes increases as the distance threshold used to define polysomes decreases (Extended Data Fig. [13d–h](/articles/s41586-022-05255-2#Fig17), [Methods](/articles/s41586-022-05255-2#Sec7) and [Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)). This suggests that local coordination of translation elongation between adjacent ribosomes within the polysome is likely to be achieved by obstructing elongation factor binding that is required for the following ribosome to proceed to the next elongation state.\\n\\nTo investigate whether local coordination arises from structures specific to ribosomes engaged in polysomes, we performed structural classification of ribosome pairs to better resolve the ribosome–ribosome interface (Fig. [4h](/articles/s41586-022-05255-2#Fig4) and Extended Data Fig. [12j,k](/articles/s41586-022-05255-2#Fig16)). We found that within the interface of tightly packed polysomes, the ribosomal protein L9 of the preceding ribosome adopts an extended conformation and its C-terminal domain protrudes into the elongation-factor-binding site of the following ribosome (Fig. [4h](/articles/s41586-022-05255-2#Fig4) and Extended Data Fig. [12j–m](/articles/s41586-022-05255-2#Fig16)). Independent focused classification on L9 of all ribosomes showed that it mainly (68.9%) adopts a flat conformation on the ribosome surface, whereas it is extended in 20.2% of 70S ribosomes (Extended Data Fig. [12n](/articles/s41586-022-05255-2#Fig16)). Ribosomes with the extended L9 largely overlap with the ribosomes detected as polysomes, especially those with a tighter 't-t' arrangement (Extended Data Fig. [12n–p](/articles/s41586-022-05255-2#Fig16)). These results suggest that L9 tends to have a flat conformation in single ribosomes and to adopt the extended conformation within tightly assembled polysomes. This clarifies previous observations of L9 being extended in X-ray crystallography structures in which the crystal packing recapitulates configurations of compacted polysomes, but being found in the flat conformation in cryo-EM structures of isolated ribosomes[51](/articles/s41586-022-05255-2#ref-CR51 \\\"Dunkle, J. A. & Cate, J. H. D. in Ribosomes (eds Rodnina, M. V., Wintermeyer, W. & Green, R.) Ch. 6 (2011).\\\"). Although L9 has been reported to be non-essential, its mutations can cause increased frameshifting and ribosome compaction by one codon[52](/articles/s41586-022-05255-2#ref-CR52 \\\"Herr, A. J., Nelson, C. C., Wills, N. M., Gesteland, R. F. & Atkins, J. F. Analysis of the roles of tRNA structure, ribosomal protein L9, and the bacteriophage T4 gene 60 bypassing signals during ribosome slippage on mRNA. J. Mol. Biol. 309, 1029–1048 (2001).\\\"),[53](/articles/s41586-022-05255-2#ref-CR53 \\\"Smith, A. M., Costello, M. S., Kettring, A. H., Wingo, R. J. & Moore, S. D. Ribosome collisions alter frameshifting at translational reprogramming motifs in bacterial mRNAs. Proc. Natl Acad. Sci. USA 116, 21769–21779 (2019).\\\"). We therefore propose that within tightly assembled polysomes, L9 of one ribosome can adopt an extended conformation that sterically interferes with elongation factor binding to the following ribosome. This local coordination mechanism can buffer adjacent ribosomes and help to maintain translation fidelity by avoiding direct collision within polysomes during active translation elongation.\\n\\nConclusions\\n-----------\\n\\nOur study demonstrates the potential of cryo-ET in shedding light on dynamic processes that are performed by macromolecular machines in living cells at a high level of detail. The analysis captures the structural and functional diversity of actively translating ribosomes inside a genome-reduced bacterium. The translation elongation cycle retrieved from the cellular cryo-ET data recapitulates and complements current mechanistic models of translation that are derived from controlled in vitro studies. The quantitative structural profiling of the cellular translation machinery presented here improves our understanding of protein biogenesis by providing distribution probabilities of translation intermediates. It also reveals how the translation machinery that functions as an interconnected system responds to different antibiotic perturbations on the single-cell level. Our investigation of polysomes further illustrates the advantage of in-cell structural biology, which can relate functional states of a molecular machine to its molecular sociology and cellular context, leading to the discovery of an elongation coordination mechanism mediated by L9. The approaches developed here establish a framework to analyse structural dynamics of cellular processes in the future and will contribute to the construction of functional cell models at atomic detail.\\n\\nMethods\\n-------\\n\\n### Cryo-ET sample preparation and data collection\\n\\nThe *M.* *pneumoniae* cultivation, sample preparation and data collection were described previously[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). The three datasets of native untreated, Cm-treated and PUM-treated cells were re-processed in Warp and M 1.0.7 (the alpha versions that were officially released as v.1.0.9.)[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\"),[54](/articles/s41586-022-05255-2#ref-CR54 \\\"Tegunov, D. & Cramer, P. Real-time cryo-electron microscopy data preprocessing with Warp. Nat. Methods 16, 1146–1152 (2019).\\\"). A small dataset of 15 tomograms acquired with Volta phase plate was processed with the denoising network in Warp 1.0.9 for visualization purposes only (as shown in Fig. [1a](/articles/s41586-022-05255-2#Fig1)).\\n\\nA dataset of spectinomycin-treated cells was collected following the same procedure as before[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). In brief, spectinomycin (Sigma-Aldrich) at a final concentration of 0.4 mg ml−1 was added into the culture medium, 15–20 min before plunge-freezing. Tilt-series collection with the dose-symmetric scheme[55](/articles/s41586-022-05255-2#ref-CR55 \\\"Hagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. J. Struct. Biol. 197, 191–198 (2017).\\\") was performed on a Titan Krios transmission electron microscope equipped with a K3 camera (Gatan) using SerialEM 3.8 (ref. [56](/articles/s41586-022-05255-2#ref-CR56 \\\"Schorb, M., Haberbosch, I., Hagen, W. J. H., Schwab, Y. & Mastronarde, D. N. Software tools for automated transmission electron microscopy. Nat. Methods 16, 471–477 (2019).\\\")) with the following parameters: magnification 81,000×, pixel size 1.053 Å, tilt range −60° to 60° with 3° increment, total dose 120–140 e− Å−2.\\n\\nIn total, 356 native untreated, 65 Cm-treated, 86 PUM-treated and 70 Spc-treated cellular tomograms were analysed in this work. Details of cryo-EM data collection, refinement and validation statistics are provided in Supplementary Tables [1](/articles/s41586-022-05255-2#MOESM1) and [3](/articles/s41586-022-05255-2#MOESM1)–[6](/articles/s41586-022-05255-2#MOESM1).\\n\\n### Image processing, ribosome template matching and map refinement\\n\\nPre-processing (motion correction, CTF estimation, dose filtering and tilt-series sorting) was performed in Warp 1.0.9 (ref.[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\")). For the untreated, Cm-treated and PUM-treated datasets, the ribosome coordinates were adopted from previous particle picking[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). For the Spc-treated dataset, template matching was performed in PyTom[57](/articles/s41586-022-05255-2#ref-CR57 \\\"Hrabe, T. et al. PyTom: a python-based toolbox for localization of macromolecules in cryo-electron tomograms and subtomogram analysis. J. Struct. Biol. 178, 177–188 (2012).\\\"), followed by computational classification in RELION 3.0 (refs.[58](/articles/s41586-022-05255-2#ref-CR58 \\\"Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. Nat. Protoc. 11, 2054–2065 (2016).\\\"),[59](/articles/s41586-022-05255-2#ref-CR59 \\\"Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7, e42166 (2018).\\\") to exclude false positives, without manual cleaning. In total, 109,990 untreated, 21,299 Cm-treated, 23,014 PUM-treated and 13,418 Spc-treated ribosome sub-tomograms were reconstructed in Warp.\\n\\nThree-dimensional (3D) refinement and classification were performed in RELION 3.0 (refs. [58](/articles/s41586-022-05255-2#ref-CR58 \\\"Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. Nat. Protoc. 11, 2054–2065 (2016).\\\"),[59](/articles/s41586-022-05255-2#ref-CR59 \\\"Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7, e42166 (2018).\\\")). We then used the software M (v.1.0.9) to perform multi-particle refinement of the tilt-series and refine the average map[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\"). Refinement of both geometric (image and volume deformation) and CTF parameters was done for five rounds in a sequential manner. After M refinement on the 70S, we performed focused refinement on the 30S and 50S subunits separately to improve the local map quality. Fourier shell correlation (FSC) calculation between independently refined random half subsets, local resolution estimation and post-processing were done in M and RELION.\\n\\n### Atomic model building in high-resolution ribosome maps\\n\\nAtomic models for the 30S and 50S subunits were built de novo using maps from focused refinement of Cm-treated ribosomes reported in our previous work[3](/articles/s41586-022-05255-2#ref-CR3 \\\"Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. Nat. Methods 18, 186–193 (2021).\\\"), which were deposited in the Electron Microscopy Data Bank (EMDB) under the accession codes EMD-11998 and EMD-11999. Homology models for *M.* *pneumoniae* ribosomal proteins were generated using the SWISS-MODEL[60](/articles/s41586-022-05255-2#ref-CR60 \\\"Waterhouse, A. et al. SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic Acids Res. 46, W296–W303 (2018).\\\") online server (<https://swissmodel.expasy.org/>; accessed April 2020), with a *Bacillus subtilis* ribosome (Protein Data Bank (PDB) 3J9W) as the template, except for the ribosomal proteins L9 (PDB 1DIV and 4V63), L10 (PDB 1ZAV) and S21 (PDB 5MMJ). rRNA sequences (REFSEQ NC\\\\_000912) were aligned to the *E.* *coli* rRNA (PDB 4YBB) using the SINA online server[61](/articles/s41586-022-05255-2#ref-CR61 \\\"Pruesse, E., Peplies, J. & Glockner, F. O. SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes. Bioinformatics 28, 1823–1829 (2012).\\\"). ModeRNA[62](/articles/s41586-022-05255-2#ref-CR62 \\\"Rother, M., Rother, K., Puton, T. & Bujnicki, J. M. ModeRNA: a tool for comparative modeling of RNA 3D structure. Nucleic Acids Res. 39, 4007–4022 (2011).\\\") was used to build homology models for rRNAs on the basis of the alignment, with PDB 4YBB as the template. Homology models were rigid-body-fitted into the cryo-ET densities using Chimera[63](/articles/s41586-022-05255-2#ref-CR63 \\\"Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605–1612 (2004).\\\"), followed by iterative refinement using PHENIX real-space refinement[64](/articles/s41586-022-05255-2#ref-CR64 \\\"Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. Acta Crystallogr D 74, 531–544 (2018).\\\") and manual adjustment in Coot[65](/articles/s41586-022-05255-2#ref-CR65 \\\"Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Acta Crystallogr D 66, 486–501 (2010).\\\"). The ribosomal proteins L9, L10 and L11 were only fitted as rigid bodies into the map owing to the less-resolved local density. The flat conformation of L9 that resides on the ribosome surface was found to be predominant according to the focused classification (see Extended Data Fig. [12n](/articles/s41586-022-05255-2#Fig16)) and was thus built in the model. Sequence extensions for ribosomal proteins S6, L22 and L29 were built de novo. Models were validated using MolProbity[66](/articles/s41586-022-05255-2#ref-CR66 \\\"Williams, C. J. et al. MolProbity: more and better reference data for improved all-atom structure validation. Protein Sci. 27, 293–315 (2018).\\\"). FSC curves between the model and the map were also calculated for validation[67](/articles/s41586-022-05255-2#ref-CR67 \\\"Chen, S. et al. High-resolution noise substitution to measure overfitting and validate resolution in 3D structure determination by single particle electron cryomicroscopy. Ultramicroscopy 135, 24–35 (2013).\\\").\\n\\n### Bioinformatic analysis of ribosomal proteins\\n\\nBioinformatic sequence analysis of ribosomal proteins in *M.* *pneumoniae* and the comparison with *E.* *coli* homologues were performed as follows, with the different steps described in detail below: (i) protein sequences and RefSeq genome annotation for the *M.* *pneumoniae* strain M129 (ATCC 29342) were downloaded from NCBI; (ii) protein sequences were also annotated with eggNOG-mapper to obtain COG (Clusters of Orthologous Groups) IDs[8](/articles/s41586-022-05255-2#ref-CR8 \\\"Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. Nature 466, 329–333 (2010).\\\"),[68](/articles/s41586-022-05255-2#ref-CR68 \\\"Tatusov, R. L., Galperin, M. Y., Natale, D. A. & Koonin, E. V. The COG database: a tool for genome-scale analysis of protein functions and evolution. Nucleic Acids Res. 28, 33–36 (2000).\\\"); (iii) for the annotated ribosomal proteins, the corresponding COG multiple sequence alignments from representative bacterial species were downloaded from the eggNOG database[69](/articles/s41586-022-05255-2#ref-CR69 \\\"Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Nucleic Acids Res. 47, D309–D314 (2019).\\\"); (iv) as *M.* *pneumoniae* M129 is not among the representative species, its protein sequences were added to the multiple sequence alignments with MAFFT software[70](/articles/s41586-022-05255-2#ref-CR70 \\\"Katoh, K., Rozewicki, J. & Yamada, K. D. MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization. Brief. Bioinform. 20, 1160–1166 (2019).\\\"); (v) for each COG multiple sequence alignment, the number of amino acids in every representative species (including *M.* *pneumoniae* M129) present at positions before the N terminus and after the C terminus positions of *E.* *coli* K-12 substr. MG1655 were calculated; (vi) the presence of N- or C-terminus extensions longer than 20 amino acids was illustrated in iTOL using NCBI taxonomy tree as the basis[71](/articles/s41586-022-05255-2#ref-CR71 \\\"Letunic, I. & Bork, P. Interactive Tree Of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. Nucleic Acids Res. 49, W293–W296 (2021).\\\"); (vii) for all COGs corresponding to proteins with N- or C-terminus extensions in *M.* *pneumoniae* that are longer than 20 amino acids, protein disorder and secondary structure were analysed for all representative strains. The presence of cross-links or transposon insertions was also analysed.\\n\\nRefSeq genome annotation and protein sequences for the *M.* *pneumoniae* strain M129 (ATCC 29342) were downloaded from NCBI for the *M.* *pneumoniae* M129 GCF\\\\_000027345.1 assembly from <https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/027/345/GCF_000027345.1_ASM2734v1/> (accessed 21 April 2021). The file GCF\\\\_000027345.1\\\\_ASM2734v1\\\\_genomic.gff was used to identify ribosomal proteins annotated by RefSeq. The file GCF\\\\_000027345.1\\\\_ASM2734v1\\\\_protein.faa with protein sequences in FASTA format was used as input to the online tool eggNOG-mapper v2, which uses precomputed eggNOG v5.0 clusters and phylogenies for fast orthology assignment[69](/articles/s41586-022-05255-2#ref-CR69 \\\"Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Nucleic Acids Res. 47, D309–D314 (2019).\\\"),[72](/articles/s41586-022-05255-2#ref-CR72 \\\"Huerta-Cepas, J. et al. Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper. Mol. Biol. Evol. 34, 2115–2122 (2017).\\\"). The annotation tables by RefSeq and eggNOG-mapper were merged by protein ID. In total, 52 ribosomal proteins were annotated and mapped to 51 unique COGs[68](/articles/s41586-022-05255-2#ref-CR68 \\\"Tatusov, R. L., Galperin, M. Y., Natale, D. A. & Koonin, E. V. The COG database: a tool for genome-scale analysis of protein functions and evolution. Nucleic Acids Res. 28, 33–36 (2000).\\\"). Two proteins, WP\\\\_010874426.1 and WP\\\\_010874827.1, were mapped to the same COG0267 corresponding to the ribosomal protein L33. Multiple sequence alignments for the annotated proteins from representative bacterial species were downloaded from the eggNOG v5.0 database[69](/articles/s41586-022-05255-2#ref-CR69 \\\"Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Nucleic Acids Res. 47, D309–D314 (2019).\\\"). For each of the 51 COG IDs mapping to ribosomal proteins in *M.* *pneumoniae*, the trimmed alignments were downloaded from <http://eggnogapi5.embl.de/nog_data/text/trimmed_alg/COG_ID> ('COG\\\\_ID' in the url should be changed to the corresponding COG ID). As *M.* *pneumoniae* M129 is not among the imported representative species, its sequences were added to the multiple sequence alignments. The sequence of each of the proteins was saved in a separate fasta file, and the MAFFT software v.7.475 was used in the add mode for each alignment as follows: mafft --reorder --add protein.fasta --auto trimmed\\\\_alg\\\\_COG.fa > output\\\\_file (ref. [70](/articles/s41586-022-05255-2#ref-CR70 \\\"Katoh, K., Rozewicki, J. & Yamada, K. D. MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization. Brief. Bioinform. 20, 1160–1166 (2019).\\\")). For COG0627, the protein WP\\\\_010874827.1 from *M.* *pneumoniae* was used.\\n\\nFor each of the ribosomal COGs, the amino acid positions of the protein from the *E.* *coli* strain K-12 substr. MG1655 was taken as a reference, and for each protein from representative species (including *M.* *pneumoniae* M129), the number of amino acids before the N-terminus and after the C-terminus locations in *E.* *coli* was calculated. In total, 11 ribosomal proteins that had N- or C-terminus extensions of more than 20 amino acids in *M. pneumoniae* M129 were considered to have extensions (2 proteins with N-terminus extensions and 9 proteins with C-terminus extensions). Ribosomal protein S3 (COG0092), which has a 17-amino-acid extension at the C terminus, was also retained for further analysis. NCBI bacterial taxonomy file new\\\\_taxdump.zip was downloaded from <https://ftp.ncbi.nlm.nih.gov/pub/taxonomy/new_taxdump/> (version on 24 April 2021). The file names.dmp was used to map NCBI IDs to taxonomy names. Tree was reconstructed from the file taxidlineage.dmp in Python v.3.7.7 with ETE3 Toolkit v.3.1.2 (ref. [73](/articles/s41586-022-05255-2#ref-CR73 \\\"Huerta-Cepas, J., Serra, F. & Bork, P. ETE 3: reconstruction, analysis, and visualization of phylogenomic data. Mol. Biol. Evol. 33, 1635–1638 (2016).\\\")). EggNOG protein IDs containing NCBI species IDs were directly mapped to the NCBI tree nodes. The presence of ribosomal protein extensions was saved as tables and converted to iTOL format with table2itol utility (<https://github.com/mgoeker/table2itol>) in R v.3.6.1. The tree was visualized with iTOL v.6 (ref. [71](/articles/s41586-022-05255-2#ref-CR71 \\\"Letunic, I. & Bork, P. Interactive Tree Of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. Nucleic Acids Res. 49, W293–W296 (2021).\\\")).\\n\\nSequences for the 11 ribosomal proteins annotated with extensions in *M.* *pneumoniae*, and their orthologues in other species, were further analysed in terms of protein disorder and secondary structures. For disorder prediction, all protein sequences from the multiple sequence alignment files were saved in FASTA format without gaps. The IUPred2A tool in Python v.3.7.7 was run with the input parameter 'long' for each protein sequence[74](/articles/s41586-022-05255-2#ref-CR74 \\\"Meszaros, B., Erdos, G. & Dosztanyi, Z. IUPred2A: context-dependent prediction of protein disorder as a function of redox state and protein binding. Nucleic Acids Res. 46, W329–W337 (2018).\\\"). The number of disordered amino acids and disorder length in extended regions were calculated on the basis of the position of the extended region relative to *E.* *coli*, and an IUPred2A score of 0.5 was used as the disorder threshold. The JPred prediction tool[75](/articles/s41586-022-05255-2#ref-CR75 \\\"Cole, C., Barber, J. D. & Barton, G. J. The Jpred 3 secondary structure prediction server. Nucleic Acids Res. 36, W197–W201 (2008).\\\") was used to predict the secondary structure of each protein using the jpredapi utility and JPred-big-batch-submission utility for a large number of submissions (<https://github.com/fabianegli/JPred-big-batch-submission>). The number of helices in extended regions was calculated on the basis of the position of the extended region relative to *E.* *coli* protein.\\n\\nFor the 11 ribosomal proteins with extensions in *M.* *pneumoniae*, the positions of cross-links linking to the same protein or a different protein were mapped according to the amino acid cross-link positions reported previously[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"). To analyse transposon insertions in *M.* *pneumoniae* ribosomal proteins with extensions, the supplementary materials from a previous publication[24](/articles/s41586-022-05255-2#ref-CR24 \\\"Miravet-Verde, S., Burgos, R., Delgado, J., Lluch-Senar, M. & Serrano, L. FASTQINS and ANUBIS: two bioinformatic tools to explore facts and artifacts in transposon sequencing and essentiality studies. Nucleic Acids Res. 48, e102 (2020).\\\") were used. All the files in the 'SupplementaryData1\\\\_fastqins\\\\_processed.zip folder' (files with extension '.qins' but not '\\\\_fw.qins' or '\\\\_rv.qins') were concatenated to obtain a list of nucleotide coordinates of transposon insertions in the sequence of *M.* *pneumoniae* M129 NC\\\\_000912.1. The transposon insertion locations in the 11 ribosomal proteins were selected on the basis of their genomic locations as per RefSeq annotations (GCF\\\\_000027345.1\\\\_ASM2734v1\\\\_genomic.gff). Nucleotide locations were converted to amino acid locations by calculating the difference between each transposon location and the start coordinate of the corresponding gene and dividing the number of nucleotides by three. All data were mapped to the protein sequences to derive Extended Data Fig. [3](/articles/s41586-022-05255-2#Fig7).\\n\\n### Sub-tomogram classification of the translation states of ribosomes\\n\\nMaximum-likelihood 3D classification[76](/articles/s41586-022-05255-2#ref-CR76 \\\"Scheres, S. H. Processing of structurally heterogeneous cryo-EM data in RELION. Methods Enzymol. 579, 125–157 (2016).\\\") was performed in RELION 3.0 (refs. [58](/articles/s41586-022-05255-2#ref-CR58 \\\"Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. Nat. Protoc. 11, 2054–2065 (2016).\\\"),[59](/articles/s41586-022-05255-2#ref-CR59 \\\"Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7, e42166 (2018).\\\")) with the re-extracted ribosome sub-tomograms after M refinement. A hierarchical and exhaustive classification strategy ([Supplementary Discussion](/articles/s41586-022-05255-2#MOESM1)) with at least three tiers was used to handle the heterogeneity in the native untreated dataset, which is described here in detail and illustrated in Extended Data Figs. [4](/articles/s41586-022-05255-2#Fig8) and [5](/articles/s41586-022-05255-2#Fig9). A similar procedure was applied for all antibiotics-treated datasets, and is detailed in Extended Data Figs. [9](/articles/s41586-022-05255-2#Fig13)–[11](/articles/s41586-022-05255-2#Fig15).\\n\\nIn the first tier, 109,990 sub-tomograms were classified into 70S and free 50S with a global spherical mask (320-Å diameter). The 24,157 free 50S were further classified into two classes, with a local spherical mask focusing on the ribosome recycling factor binding site. Before extensive 70S classification, structural heterogeneity was evaluated by visual inspection, multibody refinement and test classification runs in RELION. Classification set-ups were extensively tested, including different masks (global 70S mask, local 30S mask, spherical tRNA path mask, solvent tRNA path mask, spherical EF mask, solvent EF mask), initial references (ribosome average, features-less sphere or none), angular search options (global, local or without alignment) and RELION optimization parameters (class numbers 2–16, T values 2–10, iterations 25–40, limit resolution E-step 5–10 Å). To avoid bias, we mostly used a sphere or other featureless shapes with soft edges as the mask. Local spherical masks covering factors (elongation factors and/or tRNAs) generally provided more consistent and stable classification results than those obtained when using larger masks for the untreated data. The class number was made higher than the number of distinct structures that could be retained in one classification job and similar resulting classes were grouped.\\n\\nIn the second tier, the 77,539 identified 70S ribosomes were further classified with a local mask (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8), mask I) focusing on the tRNA path region, roughly covering the A, P and E sites. This resulted in four major classes with different tRNA occupancies: 'P, E', 'a, P/E', 'P' and 'A, P'. In the following rounds, we could further classify the 'a, P/E' class into 'A\\\\*, P/E' and 'A/P, P/E'. In addition, a 70S class with only one hybrid 'P/E' tRNA was classified. Two of the resulting classes were not interpretable: 2,150 particles with dim 30S density that were poorly resolved and 1,484 particles with density near the P site that does not resemble a tRNA.\\n\\nIn the third tier, focused classification with a local mask (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8), mask II) around the elongation factor and A/T tRNA binding sites was carried out. For the previous class with 'P, E' tRNAs, further classification resulted in 1,803 particles without additional density, 3,324 particles with EF-G (updated as 'EF-G, ap/P, pe/E') and 4,634 particles with EF-Tu•tRNA. For the classes with partial and full hybrid tRNAs, sub-classes with EF-G were obtained. For the class with only 'P' tRNA, 12,464 particles with additional EF-Tu•tRNA were classified.\\n\\nFor each classification step, at least three parallel RELION classification jobs with the same or slightly different parameters (either 30S mask or spherical EF/tRNA mask, local angular search range, class number, T value) were carried out for comparison. The classification job with the most stable result was selected and used for sorting sub-tomograms (Extended Data Fig. [5a–g](/articles/s41586-022-05255-2#Fig9)). After sorting, subsequent validation classification runs were performed for each sorted class to test whether new structures emerge or whether particles were 'wrongly' classified. Misclassified particles were relocated to the corresponding class and the validation runs were repeated until convergence. This approach was performed for all classification steps. For each of the final classes, refinement and post-processing were done in RELION. Further classification performed with the new refinement results as inputs did not generate any new classes.\\n\\n### Model building and comparison of the ribosome translation states\\n\\nTo build models for the ribosome classes, the 30S and 50S models built as described above were used as starting models for flexible fitting. Homology models of EF-G and EF-Tu were generated by SWISS-MODEL (<https://swissmodel.expasy.org/>; accessed April 2020) with PDB structures 4V7D and 4V5L as the template, respectively. For tRNA homology models, the tRNA in an *E.* *coli* ribosome structure (PDB 4V7C) was used as the template and mutated to the sequence of *M.* *pneumoniae* Phe-tRNA (REFSEQ NC\\\\_000912). The mRNA and nascent peptide were built using PDB 3J9W as the template. The homology model of the ribosome recycling factor was built using PDB 1EH1 as the template. For each class, the starting models were first rigid-body-fitted into the density using Chimera and then flexible fitting was done using Namdinator[77](#ref-CR77 \\\"Kidmose, R. T. et al. Namdinator—automatic molecular dynamics flexible fitting of structural models into cryo-EM and crystallography experimental maps. IUCrJ 6, 526–531 (2019).\\\"),[78](#ref-CR78 \\\"Trabuco, L. G., Villa, E., Mitra, K., Frank, J. & Schulten, K. Flexible fitting of atomic structures into electron microscopy maps using molecular dynamics. Structure 16, 673–683 (2008).\\\"),[79](/articles/s41586-022-05255-2#ref-CR79 \\\"Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. J. Chem. Phys. 153, 044130 (2020).\\\"). Validation was performed as described above.\\n\\nFor measuring 30S body rotations, structures of all classes were first aligned to the 50S subunit and then the rotation angles were estimated with a pivot point near nucleotide 11 of the 16S rRNA. With the perspective from the solvent side of the 30S subunit, positive numbers equal counter-clockwise rotations and negative numbers equal clockwise rotations. For measuring 30S head rotations, all class structures were first aligned to the 30S body and then the rotation angles were determined with the axis near the 30S neck. For both rotations, the angles in the 'A, P' class were defined as 0°. To describe the L1 dynamics, the distance between the mass centre of the L1 stalk (near nucleotide 2,181 of the 23S rRNA) and a fixed point near the centre of the classical E site (determined on the basis of the 'P, E' class) was measured after aligning all classes on the 50S subunit (excluding the L1 stalk).\\n\\n### Spatial analysis of ribosomes and polysomes\\n\\nSpatial mapping of ribosomes within cellular tomograms was achieved by projecting back the ribosome structures into the tomograms, with coordinates determined by template matching and shifts and rotations determined by RELION refinement. The projection was performed using the TOM toolbox[80](/articles/s41586-022-05255-2#ref-CR80 \\\"Nickell, S. et al. TOM software toolbox: acquisition and analysis for electron tomography. J. Struct. Biol. 149, 227–234 (2005).\\\") after Euler angle format conversion, at four times binning (voxel size 6.8 Å). To calculate the ribosome concentration, we first estimated the cellular volume covered in the tomogram and then divided the total number of detected ribosomes by the volume.\\n\\nDetection of the polysomes was based on both position and orientation information (Extended Data Fig. [12c](/articles/s41586-022-05255-2#Fig16)), using a custom script in MATLAB 2016b. It is noted that the annotated polysomes only refer to those assembled closely in space and thus can be detected on the basis of their spatial proximity. The positions of mRNA entry and exit sites for all 70S ribosomes were calculated on the basis of the rotations and shifts determined during RELION refinement. The distances from the mRNA exit site of one ribosome (defined as the preceding ribosome i) to the mRNA entry sites of all neighbouring ribosomes (as potential following ribosomes i+1) were calculated. A distance threshold of 7 nm was used to define whether two ribosomes belong to the same polysome. The calculation and polysome definition were done for all 70S ribosomes. A unique identifier was assigned to each polysome, as well as the sequential number for all ribosomes within the polysome.\\n\\nThe distribution of relative positions of adjacent ribosomes within the polysome—that is, the position of the following ribosome (i+1) relative to the preceding ribosome (i)—was analysed after normalizing the relative position vectors with rotations of the preceding ribosome determined from RELION refinement. The relative rotation of the following ribosome to the preceding ribosome was represented as three Euler angles (*ψ*, *θ*, *φ* in XYZ system). Using the *k*-means clustering function in MATLAB 2016b, we determined two major arrangement configurations for adjacent ribosomes within the polysome; that is, how the following ribosome rotates relative to the preceding ribosome (as shown in Extended Data Fig. [12f,g](/articles/s41586-022-05255-2#Fig16)). These two configurations are identical to the previously reported 'top-top' (t-t) and 'top-back' (t-b) configurations[4](/articles/s41586-022-05255-2#ref-CR4 \\\"Brandt, F. et al. The native 3D organization of bacterial polysomes. Cell 136, 261–271 (2009).\\\"), and the naming was adopted.\\n\\nTo further refine the ribosome–ribosome interface in polysomes, RELION classification was performed with sub-tomograms extracted with a large box size that can accommodate two ribosomes. After refinement on the preceding ribosome (i), classification without refinement was performed with a local mask focusing on the following ribosome (i+1). Only ribosome pairs within tightly packed polysomes (5,083 pairs from 5 di-ribosome classes; all with a 't-t' arrangement and extended L9 in between) resulted in average densities with both ribosomes well resolved. Models for the resulting classes were generated with rigid-body-fitting of the above described  ribosome models and the L9 homology model with the extended conformation (PDB 4V63).\\n\\n### Statistical analysis of translation elongation states in polysomes\\n\\nTo compare the experimental elongation state frequencies in polysomes with theoretical frequencies, the distributions of frequencies of each elongation state calculated across 356 tomograms of the untreated cells were compared between polysomes and all ribosomes, and between polysomes and mono-ribosomes by calculating the fold change between distribution medians. Statistical significance was assessed with a two-sided Wilcoxon–Mann–Whitney test using the ranksum function in MATLAB 2019b. The theoretical frequency of each ribosome pair was calculated as the product of the overall frequencies of the ribosome classes for the preceding ribosome (i) and the following ribosome (i+1). Experimental polysome pair frequencies were calculated by summarizing the numbers of all ribosome pairs engaged in polysomes, and dividing these numbers by the total number of pairs. Experimental and theoretical pair frequencies were compared by calculating the fold change per pair.\\n\\nPermutation analysis was performed to test the significance of differences between occurrence frequencies of the experimental pairs compared to the theoretical pairs. The ribosome pair sequences in polysomes were represented as a matrix with single polysomes as rows and ribosome positions as columns, in which each matrix cell contains the ribosome class representing its elongation state. Columns with positions beyond each ribosome’s length were assigned to NaN (not-a-number). All polysomes across all tomograms were combined in one matrix (8,641 polysomes in total). For permutation analysis, all elements of the polysome matrix were randomly shuffled for 10,000 times with the randperm function in MATLAB 2019b. For each row, the elements were sorted so that the ribosome classes are in the front columns followed by NaNs occurring in the same row after shuffling. Rows with fewer than two ribosomes in the sequence were deleted. Shuffled ribosome pair frequencies were calculated in the same way as experimental pair frequencies. The permutation *P* value for each ribosome pair was calculated as the minimum between the number of permutations in which the pair frequency was less or equal to the experimentally observed frequency divided by the total number of permutations, and the one minus this value. Permutation *P* values were adjusted for multiple hypotheses testing with the Benjamini–Hochberg procedure using the mafdr function in MATLAB 2019b with parameters ('bhfdr', 'true').\\n\\nFor polysome distance threshold analysis, matrices were created from the full ribosome dataset by varying the distance threshold to the nearest neighbour in the range from 3 to 10 nm. The distances for polysome definition were calculated between the mRNA exit site of one ribosome and the mRNA entry site of another ribosome as described above. The fractions of elongation states in ribosome pairs were calculated as for the original ribosome set (with the threshold of 7 nm). For each distance threshold and each ribosome class, the ratio between the number of pairs in which the preceding ribosome has this class and the number of pairs in which the following ribosome has this class was calculated.\\n\\n### Single-cell clustering analysis\\n\\nThe distributions of 70S ribosome classes identified in the translation elongation phase for all four datasets (356 untreated cells, 65 Cm-treated cells, 70 Spc-treated cells and 86 PUM-treated cells) were used for clustering analysis. As each tomogram covers most of one cell, the sub-tomograms in each class could be further separated by which cell they belong to. Classes in the antibiotic-treated cells were assigned with the class identifiers according to the closest classes in the untreated dataset. The percentages of different classes within each cell were calculated and these numbers were used as inputs for clustering analysis. Hierarchical clustering analysis was done using the clustergram function in MATLAB 2016b.\\n\\nStructure visualization, preparation for figures and videos were done in Chimera[63](/articles/s41586-022-05255-2#ref-CR63 \\\"Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605–1612 (2004).\\\") and ChimeraX[81](/articles/s41586-022-05255-2#ref-CR81 \\\"Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. Protein Sci. 30, 70–82 (2021).\\\").\\n\\n### Reporting summary\\n\\nFurther information on research design is available in the [Nature Research Reporting Summary](/articles/s41586-022-05255-2#MOESM2) linked to this article.\\n\\n\\n\\n\\nData availability\\n-----------------\\n\\n\\nMaps have been deposited in the EMDB under accession codes [13234](http://www.ebi.ac.uk/pdbe/entry/EMD-13234), [13272](http://www.ebi.ac.uk/pdbe/entry/EMD-13272), [13273](http://www.ebi.ac.uk/pdbe/entry/EMD-13273), [13274](http://www.ebi.ac.uk/pdbe/entry/EMD-13274), [13275](http://www.ebi.ac.uk/pdbe/entry/EMD-13275), [13276](http://www.ebi.ac.uk/pdbe/entry/EMD-13276), [13277](http://www.ebi.ac.uk/pdbe/entry/EMD-13277), [13278](http://www.ebi.ac.uk/pdbe/entry/EMD-13278), [13279](http://www.ebi.ac.uk/pdbe/entry/EMD-13279), [13280](http://www.ebi.ac.uk/pdbe/entry/EMD-13280), [13281](http://www.ebi.ac.uk/pdbe/entry/EMD-13281), [13282](http://www.ebi.ac.uk/pdbe/entry/EMD-13282), [13283](http://www.ebi.ac.uk/pdbe/entry/EMD-13283), [13284](http://www.ebi.ac.uk/pdbe/entry/EMD-13284), [13285](http://www.ebi.ac.uk/pdbe/entry/EMD-13285), [13286](http://www.ebi.ac.uk/pdbe/entry/EMD-13286), [13410](http://www.ebi.ac.uk/pdbe/entry/EMD-13410), [13411](http://www.ebi.ac.uk/pdbe/entry/EMD-13411), [13412](http://www.ebi.ac.uk/pdbe/entry/EMD-13412), [13413](http://www.ebi.ac.uk/pdbe/entry/EMD-13413), [13414](http://www.ebi.ac.uk/pdbe/entry/EMD-13414), [13431](http://www.ebi.ac.uk/pdbe/entry/EMD-13431), [13432](http://www.ebi.ac.uk/pdbe/entry/EMD-13432), [13433](http://www.ebi.ac.uk/pdbe/entry/EMD-13433), [13434](http://www.ebi.ac.uk/pdbe/entry/EMD-13434), [13435](http://www.ebi.ac.uk/pdbe/entry/EMD-13435), [13436](http://www.ebi.ac.uk/pdbe/entry/EMD-13436), [13445](http://www.ebi.ac.uk/pdbe/entry/EMD-13445), [13446](http://www.ebi.ac.uk/pdbe/entry/EMD-13446), [13447](http://www.ebi.ac.uk/pdbe/entry/EMD-13447), [13448](http://www.ebi.ac.uk/pdbe/entry/EMD-13448), [13449](http://www.ebi.ac.uk/pdbe/entry/EMD-13449), [13450](http://www.ebi.ac.uk/pdbe/entry/EMD-13450), [13451](http://www.ebi.ac.uk/pdbe/entry/EMD-13451), [13452](http://www.ebi.ac.uk/pdbe/entry/EMD-13452), [13287](http://www.ebi.ac.uk/pdbe/entry/EMD-13287), [13288](http://www.ebi.ac.uk/pdbe/entry/EMD-13288) and [13289](http://www.ebi.ac.uk/pdbe/entry/EMD-13289). Models have been deposited in the PDB under accession codes: [7OOC](https://doi.org/10.2210/pdb7OOC/pdb), [7OOD](http://doi.org/10.2210/pdb7OOD/pdb), [7P6Z](http://doi.org/10.2210/pdb7P6Z/pdb), [7PAH](http://doi.org/10.2210/pdb7PAH/pdb), [7PAI](http://doi.org/10.2210/pdb7PAI/pdb), [7PAJ](http://doi.org/10.2210/pdb7PAJ/pdb), [7PAK](http://doi.org/10.2210/pdb7PAK/pdb), [7PAL](http://doi.org/10.2210/pdb7PAL/pdb), [7PAM](http://doi.org/10.2210/pdb7PAM/pdb), [7PAN](http://doi.org/10.2210/pdb7PAN/pdb), [7PAO](http://doi.org/10.2210/pdb7PAO/pdb), [7PAQ](http://doi.org/10.2210/pdb7PAQ/pdb), [7PAR](http://doi.org/10.2210/pdb7PAR/pdb), [7PAS](http://doi.org/10.2210/pdb7PAS/pdb), [7PAT](http://doi.org/10.2210/pdb7PAT/pdb), [7PAU](http://doi.org/10.2210/pdb7PAU/pdb), [7PH9](http://doi.org/10.2210/pdb7PH9/pdb), [7PHA](http://doi.org/10.2210/pdb7PHA/pdb), [7PHB](http://doi.org/10.2210/pdb7PHB/pdb), [7PHC](http://doi.org/10.2210/pdb7PHC/pdb), [7PI8](http://doi.org/10.2210/pdb7PI8/pdb), [7PI9](http://doi.org/10.2210/pdb7PI9/pdb), [7PIA](http://doi.org/10.2210/pdb7PIA/pdb), [7PIB](http://doi.org/10.2210/pdb7PIB/pdb), [7PIC](http://doi.org/10.2210/pdb7PIC/pdb), [7PIO](http://doi.org/10.2210/pdb7PIO/pdb), [7PIP](http://doi.org/10.2210/pdb7PIP/pdb), [7PIQ](http://doi.org/10.2210/pdb7PIQ/pdb), [7PIR](http://doi.org/10.2210/pdb7PIR/pdb), [7PIS](http://doi.org/10.2210/pdb7PIS/pdb) and [7PIT](http://doi.org/10.2210/pdb7PIT/pdb). Detailed information for all maps and models generated in this work is provided in Supplementary Tables [1](/articles/s41586-022-05255-2#MOESM1) and [3](/articles/s41586-022-05255-2#MOESM1)–[6](/articles/s41586-022-05255-2#MOESM1). Maps and atomic models used from previous studies were obtained from the EMDB ([11998](http://www.ebi.ac.uk/pdbe/entry/EMD-11998) and [11999](http://www.ebi.ac.uk/pdbe/entry/EMD-11999)) and the PDB ([3J9W](http://doi.org/10.2210/pdb3J9W/pdb), [1DIV](http://doi.org/10.2210/pdb1DIV/pdb), [4V63](https://doi.org/10.2210/pdb4V63/pdb), [1ZAV](http://doi.org/10.2210/pdb1ZAV/pdb), [5MMJ](http://doi.org/10.2210/pdb5MMJ/pdb), [4YBB](http://doi.org/10.2210/pdb4YBB/pdb), [4V7C](http://doi.org/10.2210/pdb4V7C/pdb), [4V7D](http://doi.org/10.2210/pdb4V7D/pdb), [4V5L](http://doi.org/10.2210/pdb4V5L/pdb) and [1EH1](http://doi.org/10.2210/pdb1EH1/pdb)).\\n\\n\\nCode availability\\n-----------------\\n\\n\\nThe code and associated data for bioinformatics analysis of ribosomal protein extensions (Extended Data Fig. [3](/articles/s41586-022-05255-2#Fig7)) and statistical analysis of polysome sequences (Fig. [4](/articles/s41586-022-05255-2#Fig4) and Extended Data Fig. [13](/articles/s41586-022-05255-2#Fig17)) are deposited at GitHub (<https://github.com/mszimmermann/mycoplasma_ribosome>). The MATLAB script for polysome annotation (Fig. [4](/articles/s41586-022-05255-2#Fig4) and Extended Data Fig. [12](/articles/s41586-022-05255-2#Fig16)) can be found at <https://github.com/xueliang4906/polysome_detect>.\\n\\n\\nChange history\\n--------------\\n\\n* ### 10 November 2022\\n\\nA Correction to this paper has been published: <https://doi.org/10.1038/s41586-022-05455-w>\\nReferences\\n----------\\n\\n1. Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. *Annu. Rev. Biochem.* **82**, 203–236 (2013).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-biochem-113009-092313) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFOqsrzJ) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23746255) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20the%20translational%20elongation%20cycle&journal=Annu.%20Rev.%20Biochem.&doi=10.1146%2Fannurev-biochem-113009-092313&volume=82&pages=203-236&publication_year=2013&author=Voorhees%2CRM&author=Ramakrishnan%2CV)\\n2. O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. *Science* **369**, 554–557 (2020).\\n\\n[Article](https://doi.org/10.1126%2Fscience.abb3758) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020Sci...369..554O) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFWgtr7N) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32732422) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115962) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=In-cell%20architecture%20of%20an%20actively%20transcribing%E2%80%93translating%20expressome&journal=Science&doi=10.1126%2Fscience.abb3758&volume=369&pages=554-557&publication_year=2020&author=O%27Reilly%2CFJ)\\n3. Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. *Nat. Methods* **18**, 186–193 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-020-01054-7) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXjt1SmsL8%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33542511) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7611018) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-particle%20cryo-EM%20refinement%20with%20M%20visualizes%20ribosome%E2%80%93antibiotic%20complex%20at%203.5%20%C3%85%20in%20cells&journal=Nat.%20Methods&doi=10.1038%2Fs41592-020-01054-7&volume=18&pages=186-193&publication_year=2021&author=Tegunov%2CD&author=Xue%2CL&author=Dienemann%2CC&author=Cramer%2CP&author=Mahamid%2CJ)\\n4. Brandt, F. et al. The native 3D organization of bacterial polysomes. *Cell* **136**, 261–271 (2009).\\n\\n[Article](https://doi.org/10.1016%2Fj.cell.2008.11.016) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1MXhs1Kiurk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19167328) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20native%203D%20organization%20of%20bacterial%20polysomes&journal=Cell&doi=10.1016%2Fj.cell.2008.11.016&volume=136&pages=261-271&publication_year=2009&author=Brandt%2CF)\\n5. Rodnina, M. V. Translation in prokaryotes. *Cold Spring Harb. Perspect. Biol.* **10**, a032664 (2018).\\n\\n[Article](https://doi.org/10.1101%2Fcshperspect.a032664) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29661790) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6120702) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Translation%20in%20prokaryotes&journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&doi=10.1101%2Fcshperspect.a032664&volume=10&publication_year=2018&author=Rodnina%2CMV)\\n6. Frank, J. & Agrawal, R. K. A ratchet-like inter-subunit reorganization of the ribosome during translocation. *Nature* **406**, 318–322 (2000).\\n\\n[Article](https://doi.org/10.1038%2F35018597) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2000Natur.406..318F) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD3cXlsFKnsrk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10917535) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20ratchet-like%20inter-subunit%20reorganization%20of%20the%20ribosome%20during%20translocation&journal=Nature&doi=10.1038%2F35018597&volume=406&pages=318-322&publication_year=2000&author=Frank%2CJ&author=Agrawal%2CRK)\\n7. Villa, E. et al. Ribosome-induced changes in elongation factor Tu conformation control GTP hydrolysis. *Proc. Natl Acad. Sci. USA* **106**, 1063–1068 (2009).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.0811370106) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009PNAS..106.1063V) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1MXhsFyjtbk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19122150) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2613361) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome-induced%20changes%20in%20elongation%20factor%20Tu%20conformation%20control%20GTP%20hydrolysis&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.0811370106&volume=106&pages=1063-1068&publication_year=2009&author=Villa%2CE)\\n8. Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. *Nature* **466**, 329–333 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature09206) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.466..329F) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXosl2rsr0%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20631791) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome%20dynamics%20and%20tRNA%20movement%20by%20time-resolved%20electron%20cryomicroscopy&journal=Nature&doi=10.1038%2Fnature09206&volume=466&pages=329-333&publication_year=2010&author=Fischer%2CN&author=Konevega%2CAL&author=Wintermeyer%2CW&author=Rodnina%2CMV&author=Stark%2CH)\\n9. Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. *Nature* **468**, 713–716 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature09547) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.468..713R) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsFSku7jF) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21124459) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3272701) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Head%20swivel%20on%20the%20ribosome%20facilitates%20translocation%20by%20means%20of%20intra-subunit%20tRNA%20hybrid%20sites&journal=Nature&doi=10.1038%2Fnature09547&volume=468&pages=713-716&publication_year=2010&author=Ratje%2CAH)\\n10. Ramrath, D. J. et al. Visualization of two transfer RNAs trapped in transit during elongation factor G-mediated translocation. *Proc. Natl Acad. Sci. USA* **110**, 20964–20969 (2013).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1320387110) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2013PNAS..11020964R) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXnsF2jug%3D%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24324168) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876219) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualization%20of%20two%20transfer%20RNAs%20trapped%20in%20transit%20during%20elongation%20factor%20G-mediated%20translocation&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1320387110&volume=110&pages=20964-20969&publication_year=2013&author=Ramrath%2CDJ)\\n11. Brilot, A. F., Korostelev, A. A., Ermolenko, D. N. & Grigorieff, N. Structure of the ribosome with elongation factor G trapped in the pretranslocation state. *Proc. Natl Acad. Sci. USA* **110**, 20994–20999 (2013).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1311423110) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2013PNAS..11020994B) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXnsF2iuw%3D%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24324137) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876248) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structure%20of%20the%20ribosome%20with%20elongation%20factor%20G%20trapped%20in%20the%20pretranslocation%20state&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1311423110&volume=110&pages=20994-20999&publication_year=2013&author=Brilot%2CAF&author=Korostelev%2CAA&author=Ermolenko%2CDN&author=Grigorieff%2CN)\\n12. Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. *Cell* **161**, 845–857 (2015).\\n\\n[Article](https://doi.org/10.1016%2Fj.cell.2015.03.052) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2MXotVKlur0%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25957688) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4432480) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20snapshots%20of%20actively%20translating%20human%20ribosomes&journal=Cell&doi=10.1016%2Fj.cell.2015.03.052&volume=161&pages=845-857&publication_year=2015&author=Behrmann%2CE)\\n13. Loveland, A. B., Demo, G. & Korostelev, A. A. Cryo-EM of elongating ribosome with EF-Tu\\\\*GTP elucidates tRNA proofreading. *Nature* **584**, 640–645 (2020).\\n\\n[Article](https://doi.org/10.1038%2Fs41586-020-2447-x) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020Natur.584..640L) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlShtL%2FE) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32612237) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7483604) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cryo-EM%20of%20elongating%20ribosome%20with%20EF-Tu%2AGTP%20elucidates%20tRNA%20proofreading&journal=Nature&doi=10.1038%2Fs41586-020-2447-x&volume=584&pages=640-645&publication_year=2020&author=Loveland%2CAB&author=Demo%2CG&author=Korostelev%2CAA)\\n14. Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. *Nature* **595**, 741–745 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41586-021-03713-x) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Natur.595..741R) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFSnsbnM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34234344) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8318882) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20early%20translocation%20events%20on%20the%20ribosome&journal=Nature&doi=10.1038%2Fs41586-021-03713-x&volume=595&pages=741-745&publication_year=2021&author=Rundlet%2CEJ)\\n15. Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. *Nat. Commun.* **12**, 5933 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-26133-x) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.5933P) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1aqsL7M) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34635670) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8505512) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20mechanism%20of%20GTPase-powered%20ribosome%E2%80%93tRNA%20movement&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-26133-x&volume=12&publication_year=2021&author=Petrychenko%2CV)\\n16. Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. *Nat. Commun.* **12**, 7236 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-27415-0) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.7236C) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXislejtbjN) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34903725) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8668904) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Time-resolved%20cryo-EM%20visualizes%20ribosomal%20translocation%20with%20EF-G%20and%20GTP&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-27415-0&volume=12&publication_year=2021&author=Carbone%2CCE)\\n17. Brown, A. & Shao, S. Ribosomes and cryo-EM: a duet. *Curr. Opin. Struct. Biol.* **52**, 1–7 (2018).\\n\\n[Article](https://doi.org/10.1016%2Fj.sbi.2018.07.001) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlSjs77J) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30015201) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosomes%20and%20cryo-EM%3A%20a%20duet&journal=Curr.%20Opin.%20Struct.%20Biol.&doi=10.1016%2Fj.sbi.2018.07.001&volume=52&pages=1-7&publication_year=2018&author=Brown%2CA&author=Shao%2CS)\\n18. Kuhner, S. et al. Proteome organization in a genome-reduced bacterium. *Science* **326**, 1235–1240 (2009).\\n\\n[Article](https://doi.org/10.1126%2Fscience.1176343) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009Sci...326.1235K) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19965468) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Proteome%20organization%20in%20a%20genome-reduced%20bacterium&journal=Science&doi=10.1126%2Fscience.1176343&volume=326&pages=1235-1240&publication_year=2009&author=Kuhner%2CS)\\n19. Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. *Mol. Cell* **39**, 560–569 (2010).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2010.08.003) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVyrt7jF) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20797628) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20three-dimensional%20organization%20of%20polyribosomes%20in%20intact%20human%20cells&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2010.08.003&volume=39&pages=560-569&publication_year=2010&author=Brandt%2CF&author=Carlson%2CLA&author=Hartl%2CFU&author=Baumeister%2CW&author=Grunewald%2CK)\\n20. Mahamid, J. et al. Visualizing the molecular sociology at the HeLa cell nuclear periphery. *Science* **351**, 969–972 (2016).\\n\\n[Article](https://doi.org/10.1126%2Fscience.aad8857) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016Sci...351..969M) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XivFOnt7c%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26917770) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualizing%20the%20molecular%20sociology%20at%20the%20HeLa%20cell%20nuclear%20periphery&journal=Science&doi=10.1126%2Fscience.aad8857&volume=351&pages=969-972&publication_year=2016&author=Mahamid%2CJ)\\n21. Schaffer, M. et al. A cryo-FIB lift-out technique enables molecular-resolution cryo-ET within native *Caenorhabditis elegans* tissue. *Nat. Methods* **16**, 757–762 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0497-5) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVOku7zL) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31363205) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20cryo-FIB%20lift-out%20technique%20enables%20molecular-resolution%20cryo-ET%20within%20native%20Caenorhabditis%20elegans%20tissue&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0497-5&volume=16&pages=757-762&publication_year=2019&author=Schaffer%2CM)\\n22. Waltz, F. et al. How to build a ribosome from RNA fragments in *Chlamydomonas* mitochondria. *Nat. Commun.* **12**, 7176 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-27200-z) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.7176W) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXislWktLrO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34887394) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8660880) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=How%20to%20build%20a%20ribosome%20from%20RNA%20fragments%20in%20Chlamydomonas%20mitochondria&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-27200-z&volume=12&publication_year=2021&author=Waltz%2CF)\\n23. Melnikov, S., Manakongtreecheep, K. & Soll, D. Revising the structural diversity of ribosomal proteins across the three domains of life. *Mol. Biol. Evol.* **35**, 1588–1598 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsy021) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtF2rur7L) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29529322) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5995209) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Revising%20the%20structural%20diversity%20of%20ribosomal%20proteins%20across%20the%20three%20domains%20of%20life&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsy021&volume=35&pages=1588-1598&publication_year=2018&author=Melnikov%2CS&author=Manakongtreecheep%2CK&author=Soll%2CD)\\n24. Miravet-Verde, S., Burgos, R., Delgado, J., Lluch-Senar, M. & Serrano, L. FASTQINS and ANUBIS: two bioinformatic tools to explore facts and artifacts in transposon sequencing and essentiality studies. *Nucleic Acids Res.* **48**, e102 (2020).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkaa679) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXkt1GntbY%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32813015) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7515713) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FASTQINS%20and%20ANUBIS%3A%20two%20bioinformatic%20tools%20to%20explore%20facts%20and%20artifacts%20in%20transposon%20sequencing%20and%20essentiality%20studies&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkaa679&volume=48&publication_year=2020&author=Miravet-Verde%2CS&author=Burgos%2CR&author=Delgado%2CJ&author=Lluch-Senar%2CM&author=Serrano%2CL)\\n25. Borg, A., Pavlov, M. & Ehrenberg, M. Complete kinetic mechanism for recycling of the bacterial ribosome. *RNA* **22**, 10–21 (2016).\\n\\n[Article](https://doi.org/10.1261%2Frna.053157.115) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVKlur3M) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26527791) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4691825) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Complete%20kinetic%20mechanism%20for%20recycling%20of%20the%20bacterial%20ribosome&journal=RNA&doi=10.1261%2Frna.053157.115&volume=22&pages=10-21&publication_year=2016&author=Borg%2CA&author=Pavlov%2CM&author=Ehrenberg%2CM)\\n26. Munro, J. B., Altman, R. B., O’Connor, N. & Blanchard, S. C. Identification of two distinct hybrid state intermediates on the ribosome. *Mol. Cell* **25**, 505–517 (2007).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2007.01.022) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2sXjtlWkur8%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17317624) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2244649) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20two%20distinct%20hybrid%20state%20intermediates%20on%20the%20ribosome&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2007.01.022&volume=25&pages=505-517&publication_year=2007&author=Munro%2CJB&author=Altman%2CRB&author=O%E2%80%99Connor%2CN&author=Blanchard%2CSC)\\n27. Belardinelli, R. et al. Choreography of molecular movements during ribosome progression along mRNA. *Nat. Struct. Mol. Biol.* **23**, 342–348 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.3193) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XktFeqtrk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26999556) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Choreography%20of%20molecular%20movements%20during%20ribosome%20progression%20along%20mRNA&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.3193&volume=23&pages=342-348&publication_year=2016&author=Belardinelli%2CR)\\n28. Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. *Nat. Struct. Mol. Biol.* **23**, 333–341 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.3177) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XivFKhurY%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26926435) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4821728) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiperspective%20smFRET%20reveals%20rate-determining%20late%20intermediates%20of%20ribosomal%20translocation&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.3177&volume=23&pages=333-341&publication_year=2016&author=Wasserman%2CMR&author=Alejo%2CJL&author=Altman%2CRB&author=Blanchard%2CSC)\\n29. Fei, J., Kosuri, P., MacDougall, D. D. & Gonzalez, R. L. Jr Coupling of ribosomal L1 stalk and tRNA dynamics during translation elongation. *Mol. Cell* **30**, 348–359 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2008.03.012) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXmtFaktLg%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18471980) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Coupling%20of%20ribosomal%20L1%20stalk%20and%20tRNA%20dynamics%20during%20translation%20elongation&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2008.03.012&volume=30&pages=348-359&publication_year=2008&author=Fei%2CJ&author=Kosuri%2CP&author=MacDougall%2CDD&author=Gonzalez%2CRL)\\n30. Zhou, J., Lancaster, L., Donohue, J. P. & Noller, H. F. How the ribosome hands the A-site tRNA to the P site during EF-G-catalyzed translocation. *Science* **345**, 1188–1191 (2014).\\n\\n[Article](https://doi.org/10.1126%2Fscience.1255030) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2014Sci...345.1188Z) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsVGhtb%2FO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25190797) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4242719) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=How%20the%20ribosome%20hands%20the%20A-site%20tRNA%20to%20the%20P%20site%20during%20EF-G-catalyzed%20translocation&journal=Science&doi=10.1126%2Fscience.1255030&volume=345&pages=1188-1191&publication_year=2014&author=Zhou%2CJ&author=Lancaster%2CL&author=Donohue%2CJP&author=Noller%2CHF)\\n31. Uemura, S. et al. Real-time tRNA transit on single translating ribosomes at codon resolution. *Nature* **464**, 1012–U1073 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature08925) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.464.1012U) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXkslart7c%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20393556) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4466108) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20tRNA%20transit%20on%20single%20translating%20ribosomes%20at%20codon%20resolution&journal=Nature&doi=10.1038%2Fnature08925&volume=464&pages=1012-U1073&publication_year=2010&author=Uemura%2CS)\\n32. Chen, C. et al. Single-molecule fluorescence measurements of ribosomal translocation dynamics. *Mol. Cell* **42**, 367–377 (2011).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2011.03.024) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3MXlvVyiu7w%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21549313) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3090999) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-molecule%20fluorescence%20measurements%20of%20ribosomal%20translocation%20dynamics&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2011.03.024&volume=42&pages=367-377&publication_year=2011&author=Chen%2CC)\\n33. Wilson, D. N. & Nierhaus, K. H. The E-site story: the importance of maintaining two tRNAs on the ribosome during protein synthesis. *Cell. Mol. Life Sci.* **63**, 2725–2737 (2006).\\n\\n[Article](https://link.springer.com/doi/10.1007/s00018-006-6125-4) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFSrt7w%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17013564) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20E-site%20story%3A%20the%20importance%20of%20maintaining%20two%20tRNAs%20on%20the%20ribosome%20during%20protein%20synthesis&journal=Cell.%20Mol.%20Life%20Sci.&doi=10.1007%2Fs00018-006-6125-4&volume=63&pages=2725-2737&publication_year=2006&author=Wilson%2CDN&author=Nierhaus%2CKH)\\n34. Munro, J. B., Sanbonmatsu, K. Y., Spahn, C. M. & Blanchard, S. C. Navigating the ribosome’s metastable energy landscape. *Trends Biochem. Sci* **34**, 390–400 (2009).\\n\\n[Article](https://doi.org/10.1016%2Fj.tibs.2009.04.004) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVSntrnM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19647434) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2914510) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Navigating%20the%20ribosome%E2%80%99s%20metastable%20energy%20landscape&journal=Trends%20Biochem.%20Sci&doi=10.1016%2Fj.tibs.2009.04.004&volume=34&pages=390-400&publication_year=2009&author=Munro%2CJB&author=Sanbonmatsu%2CKY&author=Spahn%2CCM&author=Blanchard%2CSC)\\n35. Bock, L. V. et al. Energy barriers and driving forces in tRNA translocation through the ribosome. *Nat. Struct. Mol. Biol.* **20**, 1390–1396 (2013).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.2690) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslWisbvK) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24186064) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Energy%20barriers%20and%20driving%20forces%20in%20tRNA%20translocation%20through%20the%20ribosome&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.2690&volume=20&pages=1390-1396&publication_year=2013&author=Bock%2CLV)\\n36. Julian, P. et al. Structure of ratcheted ribosomes with tRNAs in hybrid states. *Proc. Natl Acad. Sci. USA* **105**, 16924–16927 (2008).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.0809587105) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2008PNAS..10516924J) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlKqt7%2FM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18971332) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2579354) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structure%20of%20ratcheted%20ribosomes%20with%20tRNAs%20in%20hybrid%20states&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.0809587105&volume=105&pages=16924-16927&publication_year=2008&author=Julian%2CP)\\n37. Agirrezabala, X. et al. Visualization of the hybrid state of tRNA binding promoted by spontaneous ratcheting of the ribosome. *Mol. Cell* **32**, 190–197 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2008.10.001) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlehsb3L) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18951087) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614368) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualization%20of%20the%20hybrid%20state%20of%20tRNA%20binding%20promoted%20by%20spontaneous%20ratcheting%20of%20the%20ribosome&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2008.10.001&volume=32&pages=190-197&publication_year=2008&author=Agirrezabala%2CX)\\n38. Rodnina, M. V., Peske, F., Peng, B. Z., Belardinelli, R. & Wintermeyer, W. Converting GTP hydrolysis into motion: versatile translational elongation factor G. *Biol. Chem.* **401**, 131–142 (2019).\\n\\n[Article](https://doi.org/10.1515%2Fhsz-2019-0313) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31600135) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Converting%20GTP%20hydrolysis%20into%20motion%3A%20versatile%20translational%20elongation%20factor%20G&journal=Biol.%20Chem.&doi=10.1515%2Fhsz-2019-0313&volume=401&pages=131-142&publication_year=2019&author=Rodnina%2CMV&author=Peske%2CF&author=Peng%2CBZ&author=Belardinelli%2CR&author=Wintermeyer%2CW)\\n39. Lin, J., Zhou, D., Steitz, T. A., Polikanov, Y. S. & Gagnon, M. G. Ribosome-targeting antibiotics: modes of action, mechanisms of resistance, and implications for drug design. *Annu. Rev. Biochem.* **87**, 451–478 (2018).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-biochem-062917-011942) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXlslSrurw%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29570352) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9176271) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome-targeting%20antibiotics%3A%20modes%20of%20action%2C%20mechanisms%20of%20resistance%2C%20and%20implications%20for%20drug%20design&journal=Annu.%20Rev.%20Biochem.&doi=10.1146%2Fannurev-biochem-062917-011942&volume=87&pages=451-478&publication_year=2018&author=Lin%2CJ&author=Zhou%2CD&author=Steitz%2CTA&author=Polikanov%2CYS&author=Gagnon%2CMG)\\n40. Bulkley, D., Innis, C. A., Blaha, G. & Steitz, T. A. Revisiting the structures of several antibiotics bound to the bacterial ribosome. *Proc. Natl Acad. Sci. USA* **107**, 17158–17163 (2010).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1008685107) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010PNAS..10717158B) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlSjurrM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20876130) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2951403) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Revisiting%20the%20structures%20of%20several%20antibiotics%20bound%20to%20the%20bacterial%20ribosome&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1008685107&volume=107&pages=17158-17163&publication_year=2010&author=Bulkley%2CD&author=Innis%2CCA&author=Blaha%2CG&author=Steitz%2CTA)\\n41. Svetlov, M. S. et al. High-resolution crystal structures of ribosome-bound chloramphenicol and erythromycin provide the ultimate basis for their competition. *RNA* **25**, 600–606 (2019).\\n\\n[Article](https://doi.org/10.1261%2Frna.069260.118) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1Ons77I) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30733327) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6467010) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-resolution%20crystal%20structures%20of%20ribosome-bound%20chloramphenicol%20and%20erythromycin%20provide%20the%20ultimate%20basis%20for%20their%20competition&journal=RNA&doi=10.1261%2Frna.069260.118&volume=25&pages=600-606&publication_year=2019&author=Svetlov%2CMS)\\n42. Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. *ACS Chem. Biol.* **2**, 545–552 (2007).\\n\\n[Article](https://doi.org/10.1021%2Fcb700100n) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2sXpvVOhtrc%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17696316) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4624401) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20steric%20block%20in%20translation%20caused%20by%20the%20antibiotic%20spectinomycin&journal=ACS%20Chem.%20Biol.&doi=10.1021%2Fcb700100n&volume=2&pages=545-552&publication_year=2007&author=Borovinskaya%2CMA&author=Shoji%2CS&author=Holton%2CJM&author=Fredrick%2CK&author=Cate%2CJHD)\\n43. Polikanov, Y. S. et al. Distinct tRNA accommodation intermediates observed on the ribosome with the antibiotics hygromycin A and A201A. *Mol. Cell* **58**, 832–844 (2015).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2015.04.014) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2MXpt1Skurg%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26028538) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4458074) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20tRNA%20accommodation%20intermediates%20observed%20on%20the%20ribosome%20with%20the%20antibiotics%20hygromycin%20A%20and%20A201A&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2015.04.014&volume=58&pages=832-844&publication_year=2015&author=Polikanov%2CYS)\\n44. Morse, J. C. et al. Elongation factor-Tu can repetitively engage aminoacyl-tRNA within the ribosome during the proofreading stage of tRNA selection. *Proc. Natl Acad. Sci. USA* **117**, 3610–3620 (2020).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1904469117) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020PNAS..117.3610M) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXjsFyhu7Y%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32024753) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7035488) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Elongation%20factor-Tu%20can%20repetitively%20engage%20aminoacyl-tRNA%20within%20the%20ribosome%20during%20the%20proofreading%20stage%20of%20tRNA%20selection&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1904469117&volume=117&pages=3610-3620&publication_year=2020&author=Morse%2CJC)\\n45. Choi, J. et al. Dynamics of the context-specific translation arrest by chloramphenicol and linezolid. *Nat. Chem. Biol.* **16**, 310–317 (2020).\\n\\n[Article](https://doi.org/10.1038%2Fs41589-019-0423-2) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVSktb%2FM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31844301) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dynamics%20of%20the%20context-specific%20translation%20arrest%20by%20chloramphenicol%20and%20linezolid&journal=Nat.%20Chem.%20Biol.&doi=10.1038%2Fs41589-019-0423-2&volume=16&pages=310-317&publication_year=2020&author=Choi%2CJ)\\n46. Marks, J. et al. Context-specific inhibition of translation by ribosomal antibiotics targeting the peptidyl transferase center. *Proc. Natl Acad. Sci. USA* **113**, 12150–12155 (2016).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1613055113) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016PNAS..11312150M) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1elur3N) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27791002) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5086994) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Context-specific%20inhibition%20of%20translation%20by%20ribosomal%20antibiotics%20targeting%20the%20peptidyl%20transferase%20center&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1613055113&volume=113&pages=12150-12155&publication_year=2016&author=Marks%2CJ)\\n47. Vazquez-Laslop, N. & Mankin, A. S. Context-specific action of ribosomal antibiotics. *Annu. Rev. Microbiol.* **72**, 185–207 (2018).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-micro-090817-062329) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFCjs77O) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29906204) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8742604) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Context-specific%20action%20of%20ribosomal%20antibiotics&journal=Annu.%20Rev.%20Microbiol.&doi=10.1146%2Fannurev-micro-090817-062329&volume=72&pages=185-207&publication_year=2018&author=Vazquez-Laslop%2CN&author=Mankin%2CAS)\\n48. Aguirre Rivera, J. et al. Real-time measurements of aminoglycoside effects on protein synthesis in live cells. *Proc. Natl Acad. Sci. USA* **118**, e2013315118 (2021).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.2013315118) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33619089) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7936356) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20measurements%20of%20aminoglycoside%20effects%20on%20protein%20synthesis%20in%20live%20cells&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.2013315118&volume=118&publication_year=2021&author=Aguirre%20Rivera%2CJ)\\n49. Baquero, F. & Levin, B. R. Proximate and ultimate causes of the bactericidal action of antibiotics. *Nat. Rev. Microbiol.* **19**, 123–132 (2021).\\n50. Myasnikov, A. G. et al. The molecular structure of the left-handed supra-molecular helix of eukaryotic polyribosomes. *Nat. Commun.* **5**, 5294 (2014).\\n\\n[Article](https://doi.org/10.1038%2Fncomms6294) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2014NatCo...5.5294M) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFantrfM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25376914) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20molecular%20structure%20of%20the%20left-handed%20supra-molecular%20helix%20of%20eukaryotic%20polyribosomes&journal=Nat.%20Commun.&doi=10.1038%2Fncomms6294&volume=5&publication_year=2014&author=Myasnikov%2CAG)\\n51. Dunkle, J. A. & Cate, J. H. D. in *Ribosomes* (eds Rodnina, M. V., Wintermeyer, W. & Green, R.) Ch. 6 (2011).\\n52. Herr, A. J., Nelson, C. C., Wills, N. M., Gesteland, R. F. & Atkins, J. F. Analysis of the roles of tRNA structure, ribosomal protein L9, and the bacteriophage T4 gene 60 bypassing signals during ribosome slippage on mRNA. *J. Mol. Biol.* **309**, 1029–1048 (2001).\\n\\n[Article](https://doi.org/10.1006%2Fjmbi.2001.4717) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD3MXkslGgurY%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11399077) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20the%20roles%20of%20tRNA%20structure%2C%20ribosomal%20protein%20L9%2C%20and%20the%20bacteriophage%20T4%20gene%2060%20bypassing%20signals%20during%20ribosome%20slippage%20on%20mRNA&journal=J.%20Mol.%20Biol.&doi=10.1006%2Fjmbi.2001.4717&volume=309&pages=1029-1048&publication_year=2001&author=Herr%2CAJ&author=Nelson%2CCC&author=Wills%2CNM&author=Gesteland%2CRF&author=Atkins%2CJF)\\n53. Smith, A. M., Costello, M. S., Kettring, A. H., Wingo, R. J. & Moore, S. D. Ribosome collisions alter frameshifting at translational reprogramming motifs in bacterial mRNAs. *Proc. Natl Acad. Sci. USA* **116**, 21769–21779 (2019).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1910613116) \\n [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019PNAS..11621769S) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXitValtbjM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31591196) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6815119) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome%20collisions%20alter%20frameshifting%20at%20translational%20reprogramming%20motifs%20in%20bacterial%20mRNAs&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1910613116&volume=116&pages=21769-21779&publication_year=2019&author=Smith%2CAM&author=Costello%2CMS&author=Kettring%2CAH&author=Wingo%2CRJ&author=Moore%2CSD)\\n54. Tegunov, D. & Cramer, P. Real-time cryo-electron microscopy data preprocessing with Warp. *Nat. Methods* **16**, 1146–1152 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0580-y) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFejs77E) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31591575) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6858868) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20cryo-electron%20microscopy%20data%20preprocessing%20with%20Warp&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0580-y&volume=16&pages=1146-1152&publication_year=2019&author=Tegunov%2CD&author=Cramer%2CP)\\n55. Hagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. *J. Struct. Biol.* **197**, 191–198 (2017).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2016.06.007) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27313000) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5287356) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Implementation%20of%20a%20cryo-electron%20tomography%20tilt-scheme%20optimized%20for%20high%20resolution%20subtomogram%20averaging&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2016.06.007&volume=197&pages=191-198&publication_year=2017&author=Hagen%2CWJH&author=Wan%2CW&author=Briggs%2CJAG)\\n56. Schorb, M., Haberbosch, I., Hagen, W. J. H., Schwab, Y. & Mastronarde, D. N. Software tools for automated transmission electron microscopy. *Nat. Methods* **16**, 471–477 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0396-9) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVSqs7zL) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31086343) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7000238) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Software%20tools%20for%20automated%20transmission%20electron%20microscopy&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0396-9&volume=16&pages=471-477&publication_year=2019&author=Schorb%2CM&author=Haberbosch%2CI&author=Hagen%2CWJH&author=Schwab%2CY&author=Mastronarde%2CDN)\\n57. Hrabe, T. et al. PyTom: a python-based toolbox for localization of macromolecules in cryo-electron tomograms and subtomogram analysis. *J. Struct. Biol.* **178**, 177–188 (2012).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2011.12.003) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC38Xnt1ehtL4%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22193517) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PyTom%3A%20a%20python-based%20toolbox%20for%20localization%20of%20macromolecules%20in%20cryo-electron%20tomograms%20and%20subtomogram%20analysis&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2011.12.003&volume=178&pages=177-188&publication_year=2012&author=Hrabe%2CT)\\n58. Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. *Nat. Protoc.* **11**, 2054–2065 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnprot.2016.124) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XhsF2mtLrL) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27685097) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5215819) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Resolving%20macromolecular%20structures%20from%20electron%20cryo-tomography%20data%20using%20subtomogram%20averaging%20in%20RELION&journal=Nat.%20Protoc.&doi=10.1038%2Fnprot.2016.124&volume=11&pages=2054-2065&publication_year=2016&author=Bharat%2CTA&author=Scheres%2CSH)\\n59. Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. *eLife* **7**, e42166 (2018).\\n\\n[Article](https://doi.org/10.7554%2FeLife.42166) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30412051) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6250425) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=New%20tools%20for%20automated%20high-resolution%20cryo-EM%20structure%20determination%20in%20RELION-3&journal=eLife&doi=10.7554%2FeLife.42166&volume=7&publication_year=2018&author=Zivanov%2CJ)\\n60. Waterhouse, A. et al. SWISS-MODEL: homology modelling of protein structures and complexes. *Nucleic Acids Res.* **46**, W296–W303 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky427) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyru70%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29788355) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030848) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SWISS-MODEL%3A%20homology%20modelling%20of%20protein%20structures%20and%20complexes&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky427&volume=46&pages=W296-W303&publication_year=2018&author=Waterhouse%2CA)\\n61. Pruesse, E., Peplies, J. & Glockner, F. O. SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes. *Bioinformatics* **28**, 1823–1829 (2012).\\n\\n[Article](https://doi.org/10.1093%2Fbioinformatics%2Fbts252) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVSgsbfO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22556368) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3389763) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SINA%3A%20accurate%20high-throughput%20multiple%20sequence%20alignment%20of%20ribosomal%20RNA%20genes&journal=Bioinformatics&doi=10.1093%2Fbioinformatics%2Fbts252&volume=28&pages=1823-1829&publication_year=2012&author=Pruesse%2CE&author=Peplies%2CJ&author=Glockner%2CFO)\\n62. Rother, M., Rother, K., Puton, T. & Bujnicki, J. M. ModeRNA: a tool for comparative modeling of RNA 3D structure. *Nucleic Acids Res.* **39**, 4007–4022 (2011).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkq1320) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3MXmvValsL4%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21300639) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3105415) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ModeRNA%3A%20a%20tool%20for%20comparative%20modeling%20of%20RNA%203D%20structure&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkq1320&volume=39&pages=4007-4022&publication_year=2011&author=Rother%2CM&author=Rother%2CK&author=Puton%2CT&author=Bujnicki%2CJM)\\n63. Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. *J. Comput. Chem.* **25**, 1605–1612 (2004).\\n\\n[Article](https://doi.org/10.1002%2Fjcc.20084) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2cXmvVOhsbs%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15264254) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCSF%20Chimera%E2%80%94a%20visualization%20system%20for%20exploratory%20research%20and%20analysis&journal=J.%20Comput.%20Chem.&doi=10.1002%2Fjcc.20084&volume=25&pages=1605-1612&publication_year=2004&author=Pettersen%2CEF)\\n64. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. *Acta Crystallogr D* **74**, 531–544 (2018).\\n\\n[Article](https://doi.org/10.1107%2FS2059798318006551) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2ltbzP) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-space%20refinement%20in%20PHENIX%20for%20cryo-EM%20and%20crystallography&journal=Acta%20Crystallogr%20D&doi=10.1107%2FS2059798318006551&volume=74&pages=531-544&publication_year=2018&author=Afonine%2CPV)\\n65. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. *Acta Crystallogr D* **66**, 486–501 (2010).\\n\\n[Article](https://doi.org/10.1107%2FS0907444910007493) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXksFKisb8%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20383002) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2852313) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Features%20and%20development%20of%20Coot&journal=Acta%20Crystallogr%20D&doi=10.1107%2FS0907444910007493&volume=66&pages=486-501&publication_year=2010&author=Emsley%2CP&author=Lohkamp%2CB&author=Scott%2CWG&author=Cowtan%2CK)\\n66. Williams, C. J. et al. MolProbity: more and better reference data for improved all-atom structure validation. *Protein Sci.* **27**, 293–315 (2018).\\n\\n[Article](https://doi.org/10.1002%2Fpro.3330) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvVKgsrnO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29067766) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MolProbity%3A%20more%20and%20better%20reference%20data%20for%20improved%20all-atom%20structure%20validation&journal=Protein%20Sci.&doi=10.1002%2Fpro.3330&volume=27&pages=293-315&publication_year=2018&author=Williams%2CCJ)\\n67. Chen, S. et al. High-resolution noise substitution to measure overfitting and validate resolution in 3D structure determination by single particle electron cryomicroscopy. *Ultramicroscopy* **135**, 24–35 (2013).\\n\\n[Article](https://doi.org/10.1016%2Fj.ultramic.2013.06.004) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslWhtLzO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23872039) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3834153) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-resolution%20noise%20substitution%20to%20measure%20overfitting%20and%20validate%20resolution%20in%203D%20structure%20determination%20by%20single%20particle%20electron%20cryomicroscopy&journal=Ultramicroscopy&doi=10.1016%2Fj.ultramic.2013.06.004&volume=135&pages=24-35&publication_year=2013&author=Chen%2CS)\\n68. Tatusov, R. L., Galperin, M. Y., Natale, D. A. & Koonin, E. V. The COG database: a tool for genome-scale analysis of protein functions and evolution. *Nucleic Acids Res.* **28**, 33–36 (2000).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2F28.1.33) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD3cXhvVGqu7w%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10592175) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC102395) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20COG%20database%3A%20a%20tool%20for%20genome-scale%20analysis%20of%20protein%20functions%20and%20evolution&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2F28.1.33&volume=28&pages=33-36&publication_year=2000&author=Tatusov%2CRL&author=Galperin%2CMY&author=Natale%2CDA&author=Koonin%2CEV)\\n69. Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. *Nucleic Acids Res.* **47**, D309–D314 (2019).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky1085) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Gqtr%2FL) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30418610) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=eggNOG%205.0%3A%20a%20hierarchical%2C%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky1085&volume=47&pages=D309-D314&publication_year=2019&author=Huerta-Cepas%2CJ)\\n70. Katoh, K., Rozewicki, J. & Yamada, K. D. MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization. *Brief. Bioinform.* **20**, 1160–1166 (2019).\\n\\n[Article](https://doi.org/10.1093%2Fbib%2Fbbx108) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1Kqtb3K) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28968734) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MAFFT%20online%20service%3A%20multiple%20sequence%20alignment%2C%20interactive%20sequence%20choice%20and%20visualization&journal=Brief.%20Bioinform.&doi=10.1093%2Fbib%2Fbbx108&volume=20&pages=1160-1166&publication_year=2019&author=Katoh%2CK&author=Rozewicki%2CJ&author=Yamada%2CKD)\\n71. Letunic, I. & Bork, P. Interactive Tree Of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. *Nucleic Acids Res.* **49**, W293–W296 (2021).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkab301) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvV2isLvM) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33885785) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8265157) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interactive%20Tree%20Of%20Life%20%28iTOL%29%20v5%3A%20an%20online%20tool%20for%20phylogenetic%20tree%20display%20and%20annotation&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkab301&volume=49&pages=W293-W296&publication_year=2021&author=Letunic%2CI&author=Bork%2CP)\\n72. Huerta-Cepas, J. et al. Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper. *Mol. Biol. Evol.* **34**, 2115–2122 (2017).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsx148) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFaru7vF) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28460117) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850834) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20genome-wide%20functional%20annotation%20through%20orthology%20assignment%20by%20eggNOG-Mapper&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsx148&volume=34&pages=2115-2122&publication_year=2017&author=Huerta-Cepas%2CJ)\\n73. Huerta-Cepas, J., Serra, F. & Bork, P. ETE 3: reconstruction, analysis, and visualization of phylogenomic data. *Mol. Biol. Evol.* **33**, 1635–1638 (2016).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsw046) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFeisrrN) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26921390) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4868116) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ETE%203%3A%20reconstruction%2C%20analysis%2C%20and%20visualization%20of%20phylogenomic%20data&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsw046&volume=33&pages=1635-1638&publication_year=2016&author=Huerta-Cepas%2CJ&author=Serra%2CF&author=Bork%2CP)\\n74. Meszaros, B., Erdos, G. & Dosztanyi, Z. IUPred2A: context-dependent prediction of protein disorder as a function of redox state and protein binding. *Nucleic Acids Res.* **46**, W329–W337 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky384) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyrt7o%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29860432) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030935) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=IUPred2A%3A%20context-dependent%20prediction%20of%20protein%20disorder%20as%20a%20function%20of%20redox%20state%20and%20protein%20binding&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky384&volume=46&pages=W329-W337&publication_year=2018&author=Meszaros%2CB&author=Erdos%2CG&author=Dosztanyi%2CZ)\\n75. Cole, C., Barber, J. D. & Barton, G. J. The Jpred 3 secondary structure prediction server. *Nucleic Acids Res.* **36**, W197–W201 (2008).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkn238) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXptlKkt7w%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18463136) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2447793) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Jpred%203%20secondary%20structure%20prediction%20server&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkn238&volume=36&pages=W197-W201&publication_year=2008&author=Cole%2CC&author=Barber%2CJD&author=Barton%2CGJ)\\n76. Scheres, S. H. Processing of structurally heterogeneous cryo-EM data in RELION. *Methods Enzymol.* **579**, 125–157 (2016).\\n\\n[Article](https://doi.org/10.1016%2Fbs.mie.2016.04.012) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXmtlKnurk%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27572726) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Processing%20of%20structurally%20heterogeneous%20cryo-EM%20data%20in%20RELION&journal=Methods%20Enzymol.&doi=10.1016%2Fbs.mie.2016.04.012&volume=579&pages=125-157&publication_year=2016&author=Scheres%2CSH)\\n77. Kidmose, R. T. et al. Namdinator—automatic molecular dynamics flexible fitting of structural models into cryo-EM and crystallography experimental maps. *IUCrJ* **6**, 526–531 (2019).\\n\\n[Article](https://doi.org/10.1107%2FS2052252519007619) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlWit7%2FO) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31316797) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6608625) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Namdinator%E2%80%94automatic%20molecular%20dynamics%20flexible%20fitting%20of%20structural%20models%20into%20cryo-EM%20and%20crystallography%20experimental%20maps&journal=IUCrJ&doi=10.1107%2FS2052252519007619&volume=6&pages=526-531&publication_year=2019&author=Kidmose%2CRT)\\n78. Trabuco, L. G., Villa, E., Mitra, K., Frank, J. & Schulten, K. Flexible fitting of atomic structures into electron microscopy maps using molecular dynamics. *Structure* **16**, 673–683 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.str.2008.03.005) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1cXlslGrtro%3D) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18462672) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2430731) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Flexible%20fitting%20of%20atomic%20structures%20into%20electron%20microscopy%20maps%20using%20molecular%20dynamics&journal=Structure&doi=10.1016%2Fj.str.2008.03.005&volume=16&pages=673-683&publication_year=2008&author=Trabuco%2CLG&author=Villa%2CE&author=Mitra%2CK&author=Frank%2CJ&author=Schulten%2CK)\\n79. Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. *J. Chem. Phys.* **153**, 044130 (2020).\\n\\n[Article](https://doi.org/10.1063%2F5.0014475) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFajsL3J) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32752662) \\n [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7395834) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scalable%20molecular%20dynamics%20on%20CPU%20and%20GPU%20architectures%20with%20NAMD&journal=J.%20Chem.%20Phys.&doi=10.1063%2F5.0014475&volume=153&publication_year=2020&author=Phillips%2CJC)\\n80. Nickell, S. et al. TOM software toolbox: acquisition and analysis for electron tomography. *J. Struct. Biol.* **149**, 227–234 (2005).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2004.10.006) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15721576) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TOM%20software%20toolbox%3A%20acquisition%20and%20analysis%20for%20electron%20tomography&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2004.10.006&volume=149&pages=227-234&publication_year=2005&author=Nickell%2CS)\\n81. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. *Protein Sci.* **30**, 70–82 (2021).\\n\\n[Article](https://doi.org/10.1002%2Fpro.3943) \\n [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFamtrzJ) \\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32881101) \\n [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCSF%20ChimeraX%3A%20structure%20visualization%20for%20researchers%2C%20educators%2C%20and%20developers&journal=Protein%20Sci.&doi=10.1002%2Fpro.3943&volume=30&pages=70-82&publication_year=2021&author=Pettersen%2CEF)\\n\\n[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41586-022-05255-2?format=refman&flavour=references)\\n\\nAcknowledgements\\n----------------\\n\\nWe thank I. Zagoriy, L. Adam, F. Weis, W. Hagen, T. Hoffmann and EMBL IT for technical support; L. Serrano for providing the *M.* *pneumoniae* transposon library and proteomics data; M. Kuhn, F. Marotta and J. C. Somody for discussions on bioinformatic and statistical data analysis; and C. Spahn, A. Jakobi and S. Pfeffer for discussions and critical reading of the manuscript. J.R. acknowledges the Deutsche Forschungsgemeinschaft (project no. 426290502) and the Wellcome Trust Senior Research Fellowship (103139). J.M. acknowledges the EMBL and the European Research Council starting grant (3DCellPhase-, grant no. 760067).\\n\\nAuthor information\\n------------------\\n\\n### Authors and Affiliations\\n\\n1. Structural and Computational Biology Unit, European Molecular Biology Laboratory (EMBL), Heidelberg, Germany\\n\\nLiang Xue, Maria Zimmermann-Kogadeeva, Peer Bork & Julia Mahamid\\n2. Collaboration for joint PhD degree between EMBL and Heidelberg University, Faculty of Biosciences, Heidelberg, Germany\\n\\nLiang Xue\\n3. Chair of Bioanalytics, Technische Universität Berlin, Berlin, Germany\\n\\nSwantje Lenz & Juri Rappsilber\\n4. Department of Molecular Biology, Max-Planck-Institute for Biophysical Chemistry, Göttingen, Germany\\n\\nDimitry Tegunov & Patrick Cramer\\n5. Yonsei Frontier Lab, Yonsei University, Seoul, South Korea\\n\\nPeer Bork\\n6. Department of Bioinformatics, Biocenter, University of Würzburg, Würzburg, Germany\\n\\nPeer Bork\\n7. Wellcome Centre for Cell Biology, University of Edinburgh, Edinburgh, UK\\n\\nJuri Rappsilber\\n\\nAuthors1. Liang Xue[View author publications](/search?author=Liang%20Xue)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Liang%20Xue) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Liang%20Xue%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n2. Swantje Lenz[View author publications](/search?author=Swantje%20Lenz)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Swantje%20Lenz) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Swantje%20Lenz%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n3. Maria Zimmermann-Kogadeeva[View author publications](/search?author=Maria%20Zimmermann-Kogadeeva)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Maria%20Zimmermann-Kogadeeva) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Maria%20Zimmermann-Kogadeeva%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n4. Dimitry Tegunov[View author publications](/search?author=Dimitry%20Tegunov)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Dimitry%20Tegunov) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Dimitry%20Tegunov%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n5. Patrick Cramer[View author publications](/search?author=Patrick%20Cramer)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Patrick%20Cramer) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Patrick%20Cramer%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n6. Peer Bork[View author publications](/search?author=Peer%20Bork)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Peer%20Bork) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Peer%20Bork%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n7. Juri Rappsilber[View author publications](/search?author=Juri%20Rappsilber)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Juri%20Rappsilber) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Juri%20Rappsilber%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n8. Julia Mahamid[View author publications](/search?author=Julia%20Mahamid)You can also search for this author in\\n [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Julia%20Mahamid) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Julia%20Mahamid%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)\\n### Contributions\\n\\nL.X. and J.M. designed the study. L.X. collected and processed the cryo-ET data and performed structural analysis. L.X. and S.L. built the atomic models with support from J.R. M.Z.-K. performed the bioinformatic and statistical analysis with support from P.B. D.T. and P.C. provided software support and advice on data interpretation. L.X. and J.M. wrote the manuscript with input from all authors.\\n\\n### Corresponding author\\n\\nCorrespondence to\\n [Julia Mahamid](mailto:julia.mahamid@embl.de).\\n\\nEthics declarations\\n-------------------\\n\\n\\n### Competing interests\\n\\n\\nThe authors declare no competing interests.\\n\\n\\nPeer review\\n-----------\\n\\n\\n### Peer review information\\n\\n\\n*Nature* thanks Elizabeth Villa and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.\\n\\n\\nAdditional information\\n----------------------\\n\\n**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\\n\\nExtended data figures and tables\\n--------------------------------\\n\\n### [Extended Data Fig. 1 *M. pneumoniae* in-cell ribosome maps.](/articles/s41586-022-05255-2/figures/5)\\n\\n**a**, 70S ribosome map determined from 77,539 sub-tomograms from 356 untreated *M. pneumoniae* cells. **b**, Map coloured by local resolutions. 50S has the highest local resolutions as it dominates the overall alignment. The relatively lower local resolutions of the 30S indicate their high flexibility during active translation. **c**, Fourier shell correlation (FSC) curves for global 70S and focused 30S refinement, and the reported resolution value at FSC = 0.143. The Nyquist limit for the data is 3.4 Å. **d**, 70S ribosome map determined from 18,987 sub-tomograms from 65 Cm-treated cells. **e**, Map coloured by local resolutions. **f**, FSC curves of the Cm-treated 70S ribosome, and of focused refinements on 30S and 50S respectively. **g**-**h**, Focused refined 30S and 50S maps from the Cm-treated dataset. **i**-**j**, The corresponding local-resolution maps. Atomic models for 30S and 50S were first built based on maps in **g** and **h**. **k**, Model-to-map FSC curves for 30S and 50S, respectively.\\n\\n### [Extended Data Fig. 2 Structural features of the *M. pneumoniae* ribosome.](/articles/s41586-022-05255-2/figures/6)\\n\\n**a**,**b**, Examples of regions of the atomic model fitted into the density map for rRNAs (**a**) and proteins (**b**). **c**, Densities corresponding to ions are also observed. **d**, Extensions of ribosomal proteins S6, L22 and L29 form secondary structures (between arrowheads) that were clearly resolved in the map. **e**, The 70S model showing ribosomal proteins with sequence extensions (dark green). The extensions built in the model are highlighted in magenta. In addition to the helix-forming sequences of S6, L22 and L29, some loops can be traced, especially the long C-terminal loop of S6. **f**, Whole-cell cross-linking mass spectrometry data confirms the model built for the long loop extension of S6. **g**, The cross-linking network of S6 identified in the previous work[2](/articles/s41586-022-05255-2#ref-CR2 \\\"O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. Science 369, 554–557 (2020).\\\"), with the sequence range (1–167) resolved in the map coloured in green.\\n\\n### [Extended Data Fig. 3 Ribosomal protein extensions.](/articles/s41586-022-05255-2/figures/7)\\n\\n**a**, Eleven ribosomal proteins in *M. pneumoniae* have sequence extensions compared to *E. coli*. Along the sequences, cross-links identified by in-cell cross-linking mass spectrometry, secondary structure prediction, disorder prediction, and sequence essentiality prediction based on a transposon mutation library are displayed (quantified in Supplementary Table [2](/articles/s41586-022-05255-2#MOESM1)). **b**, Extensions in the eleven ribosomal proteins are found throughout bacteria, but are not specific to any sub-groups.\\n\\n### [Extended Data Fig. 4 Classification and refinement of ribosomes in native untreated cells.](/articles/s41586-022-05255-2/figures/8)\\n\\n**a**, A diagram of the image-processing workflow. The classification process can be grouped into three tiers: global classification, focused classification on the tRNA path region (mask I), and focused classification on elongation factor and A/T tRNA binding region (mask II). For each class, a unique number identifier and class name are assigned for tracking. The particle number and the global map resolution at FSC = 0.143 are provided for each class. **b**-**d**, FSC curves for all refined maps.\\n\\n### [Extended Data Fig. 5 Validation of ribosome detection and classification.](/articles/s41586-022-05255-2/figures/9)\\n\\n**a**, Mask I for focused classification on tRNA path region. **b**, A representative RELION classification job with mask I. Each line indicates the change in particle numbers in one class over 25 iterations. Classes that show the same structure were grouped according to the tRNA occupancy ('a, P/E', 'P, E', 'P', 'A, P'). 'a, P/E' contains heterogeneous density around the A site. **c**, Results of the classification job as shown in **b**, and of three additional parallel jobs. **d**, Results of following classification jobs that further classify the 'a, P/E' class into 'A\\\\*, P/E' and 'A/P, P/E'. **e**, Mask II for focused classification on elongation factor (EF) and A/T tRNA sites. **f**, Changes in particle numbers over iterations in a representative classification job with the mask II. **g**, Results of parallel focused classification jobs with mask II. **h**, Distribution of the ribosome classes against template matching cross-correlation scores used for ribosome localization. For each of the 356 tomograms of untreated cells, the 400 highest scoring hits were extracted and ranked. Obvious false positives were manually excluded first. Additional false positives were identified during RELION classification. 70S classes that are structurally similar were grouped in the plot for better visualization. **i**, Same as **h**, but only showing the 70S classes in the elongation phase. The proportions of different 70S classes remain stable across the top 400 hits, demonstrating that the classification results are not biased by the ribosome picking.\\n\\n### [Extended Data Fig. 6 Local-resolution maps for ribosome classes in untreated cells.](/articles/s41586-022-05255-2/figures/10)\\n\\nMaps of the 15 classes determined in the dataset of untreated cells (Extended Data Fig. [4](/articles/s41586-022-05255-2#Fig8)), coloured by local resolution calculated in RELION.\\n\\n### [Extended Data Fig. 7 Model building for ribosome classes in untreated cells.](/articles/s41586-022-05255-2/figures/11)\\n\\n**a**-**j**, Models of the ten ribosome classes in the elongation phase constructed by flexible fitting. **k**, The model for the ribosome with hybrid P/E-site tRNA. **l**, Free 50S in complex with ribosome recycling factor (RRF).\\n\\n### [Extended Data Fig. 8 Early-to-late translocation intermediates during elongation.](/articles/s41586-022-05255-2/figures/12)\\n\\n**a**, Early translocation intermediates prior to EF-G binding. Only tRNAs in the aligned class models are displayed. **b**, Early-to-late translocation intermediates in the presence of EF-G. Continuous structural changes are observed from class 6e to 7 to 8, including a roughly 20 Å movement of EF-G’s domain IV toward the A site (black arrow), an overall rotation of the entire EF-G (orange arrows) and inter-domain conformational changes. **c**-**e**, Densities and models for EF-G and tRNAs in classes 6e, 7 and 8. In class 6e, EF-G’s domain IV is less resolved in the map (red circle). It may contain a mixture of intermediate states with domain IV moving toward the A site. The fitted model represents the average position. Further classification could not address the high flexibility. **f**, From class 6e to 7, EF-G undergoes a small overall rotation (orange arrows) and an inter-domain rearrangement. Inset (rotated view, aligned on EF-G domains I-III) shows rotation of domain IV relative to other domains. Movement of EF-G’s domain IV towards the A site results from both overall rotation on the ribosome and inter-domain rearrangement. **g**, From class 7 to 8, EF-G rotates as one body without significant inter-domain rearrangement (inset, rotated view, aligned on EF-G domains I-III). **h**, Class 6e shows structural similarity to a reported early translocation state 'H1-EF-G–GDP-pi' (PDB 7PJV) determined by Petrychenko *et al*. 2021[15](/articles/s41586-022-05255-2#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\"). **i**, Class 7 resembles 'PRE–EF-G–GDP–Pi' (PDB 7SSL) reported by Carbone *et al*. 2021[16](/articles/s41586-022-05255-2#ref-CR16 \\\"Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. Nat. Commun. 12, 7236 (2021).\\\"), and 'INT1' (PDB 7N2V, not shown) by Rundlet *et al*. 2021[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\"). **j**, Class 8 resembles the late translocation intermediates reported by Rundlet *et al*. 2021[14](/articles/s41586-022-05255-2#ref-CR14 \\\"Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. Nature 595, 741–745 (2021).\\\") ('INT2', PDB 7N2C) and Petrychenko *et al*. 2021[15](/articles/s41586-022-05255-2#ref-CR15 \\\"Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. Nat. Commun. 12, 5933 (2021).\\\") ('CHI1-EF-G-GDP', PDB 7PJY). **k**-**m**, Minor inter-domain conformational difference found between EF-Gs in our models and those in the reported in vitro structures.\\n\\n### [Extended Data Fig. 9 Classification, refinement and modelling of ribosomes in Cm-treated cells.](/articles/s41586-022-05255-2/figures/13)\\n\\n**a**, Image-processing workflow for the Cm-treated dataset. The sub-tomogram classification and refinement are similar to those developed for the untreated dataset (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8)). Heterogeneity may exist in the minor classes (2a, 3, 5), but the low particle numbers hindered further classification. **b**, FSC curves for all classes calculated following RELION refinements. **c**, Local-resolution maps. **d**, Density of the Cm molecule is resolved in the major 'A, P' class, but not in the other three minor classes owing to relatively low resolutions of these maps. **e**, Models built for the four classes, fitted into their corresponding densities. **f**, Elongation factors and tRNAs in the four classes. **g**, In the predominant 'A, P' class, mRNA, A- and P-site tRNA, the nascent peptide chain, and the Cm drug are well-resolved. The nascent chain has strong continuous density linked to the P-site tRNA, resulting from the inhibition of peptidyl transfer by the Cm molecule.\\n\\n### [Extended Data Fig. 10 Classification, refinement and modelling of ribosomes in Spc-treated cells.](/articles/s41586-022-05255-2/figures/14)\\n\\n**a**, Image-processing workflow for the Spc-treated dataset. The sub-tomogram classification and refinement are similar to those developed for the untreated dataset (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8)), except for exclusion of the manual inspection step after template matching. **b**, FSC curves for all classes calculated following refinements. **c**, Local-resolution maps for Spc-treated 70S classes. **d**, Density corresponding to the Spc molecule (magenta) is clearly resolved in the major 'EF-G, A/PSpc, P/E' class. Densities from the untreated and PUM-treated data, where Spc is not present, are shown for comparison. **e**, Models built for the five Spc-treated 70S classes. **f**, Elongation factors and tRNAs in the five classes. **g**, The model of 'EF-G, A/PSpc, P/E' shows Spc binds to the 30S neck region, confirming its role in inhibiting 30S head dynamics and mRNA translocation.\\n\\n### [Extended Data Fig. 11 Classification, refinement and modelling of ribosomes in PUM-treated cells.](/articles/s41586-022-05255-2/figures/15)\\n\\n**a**, Image-processing workflow for the PUM-treated dataset. The sub-tomogram classification and refinement are similar to those developed for the untreated dataset (Extended Data Fig. [4a](/articles/s41586-022-05255-2#Fig8)). **b**, Distribution of translation states for ribosomes collided with a PUM-stalled RNA polymerase (purple; stalled expressome) and for the remaining ribosomes (green) in the PUM-treated cells. **c**, FSC curves for all classes calculated after refinements. **d**-**f**, Local-resolution maps. **g**, Models built for PUM-treated 70S classes. **h**, Elongation factors and tRNAs in the six 70S ribosome classes.\\n\\n### [Extended Data Fig. 12 Spatial analysis of ribosomes and polysomes in native untreated cells.](/articles/s41586-022-05255-2/figures/16)\\n\\n**a**, 70S ribosomes (grey) and detected polysomes (light blue) in a representative tomogram. **b**, Distribution of neighbouring ribosomes within a 50 nm centre-to-centre distance. Rotation normalized. **c**, Illustration of the polysome detection approach based on the distance from the mRNA exit site of one ribosome to the mRNA entry site of the next. **d**, Histogram of distances from the mRNA exit site of one ribosome to the mRNA entry sites of all neighbouring ribosomes. **e**, Percentages of ribosomes detected as polysomes using different distance thresholds (d). Mean and s.d. are across 356 tomograms (n = 356 cells). A 7 nm threshold was selected. **f**, In polysomes, the following ribosome (i) adopts various orientations relative to the preceding ribosome (i+1), mainly with regard to rotation around a plane perpendicular to the preceding ribosome’s mRNA exit site. **g**, Rotations of the following ribosome relative to the preceding ribosome in polysomes. The two clusters correspond to the previously defined 't-t' and 't-b' configurations. **h**-**i**, Positions of the following ribosomes (coloured according to different rotations) relative to the preceding ribosome (triangle indicates the mRNA exit site). **j**-**k**, Two representative di-ribosome pairs in polysomes with extended L9. **l**-**m**, Clash between the C-terminal domain of the extended L9 and the superposed EF-G and EF-Tu, respectively. **n**, *De novo* focused classification on the L9 region using a spherical mask (light green) of 70S ribosomes resulted in three classes: no L9 resolved (I), flat L9 (II) and extended L9 (III). The additional density connecting to the extended L9 originates from the neighbouring ribosome. **o**, Overlap between polysomes independently defined based on spatial analysis, RELION classification for polysomes and classified ribosomes with the extended L9. **p**, The extended L9 is more frequent in the 't-t' ribosome pairs, especially in compacted ones.\\n\\n### [Extended Data Fig. 13 Correlation of translation elongation states within polysomes in native untreated cells.](/articles/s41586-022-05255-2/figures/17)\\n\\n**a**, Occurrence frequencies of elongation state pairs of two adjacent ribosomes (preceding ribosome i versus following ribosome i+1) within polysomes calculated from the experimental data. **b**, Theoretical frequencies of elongation state pairs if there is no cross-influence within the polysome and ribosome state pairs form randomly. **c**, Frequencies of elongation state pairs calculated after random shuffling of the experimental data. **d**, Schematic representation of polysome shuffling analysis and permutation *p*-value calculation. Detailed procedure can be found in the Materials and Methods. **e**, Comparison of the experimental and shuffled pair fractions for all major pairs. **f**, Fold changes between the experimental and shuffled pair fractions from the shuffling experiments. Ribosomes of states that need elongation factor binding to proceed (states 1 and 2a for EF-Tu, and states 5 and 6a for EF-G) are more frequently engaged as the following ribosomes. Permutation *p*-values were adjusted for multiple hypotheses testing with the Benjamini–Hochberg procedure, and are provided in Supplementary Table [7](/articles/s41586-022-05255-2#MOESM1). **g**, Difference between experimental and theoretical pair frequencies when using different distance thresholds for polysome definition. **h**, Ratios of the number of ribosomes in each state being the preceding one against the following one in polysome pairs (across all polysomes), calculated using different distance thresholds to define polysomes. Symmetric engagement as the preceding and following ribosome results in a ratio of 1.\\n\\nSupplementary information\\n-------------------------\\n\\n### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_MOESM1_ESM.pdf)\\n\\nThis file contains Supplementary Discussion; Supplementary Tables 1–8; legends for Supplementary Videos 1-2 and Supplementary References.\\n\\n### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_MOESM2_ESM.pdf)\\n\\n### [Supplementary Video 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_MOESM3_ESM.mp4)\\n\\n### [Supplementary Video 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05255-2/MediaObjects/41586_2022_5255_MOESM4_ESM.mp4)\\n\\nRights and permissions\\n----------------------\\n\\n\\n**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <http://creativecommons.org/licenses/by/4.0/>.\\n\\n\\n[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Visualizing%20translation%20dynamics%20at%20atomic%20detail%20inside%20a%20bacterial%20cell&author=Liang%20Xue%20et%20al&contentID=10.1038%2Fs41586-022-05255-2&copyright=The%20Author%28s%29&publication=0028-0836&publicationDate=2022-09-28&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)\\n\\nAbout this article\\n------------------\\n\\n[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41586-022-05255-2)### Cite this article\\n\\nXue, L., Lenz, S., Zimmermann-Kogadeeva, M. *et al.* Visualizing translation dynamics at atomic detail inside a bacterial cell.\\n *Nature* **610**, 205–211 (2022). https://doi.org/10.1038/s41586-022-05255-2\\n\\n[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41586-022-05255-2?format=refman&flavour=citation)\\n\\n* Received: 18 December 2021\\n* Accepted: 19 August 2022\\n* Published: 28 September 2022\\n* Issue Date: 06 October 2022\\n* DOI: https://doi.org/10.1038/s41586-022-05255-2\\n\\n### Share this article\\n\\nAnyone you share the following link with will be able to read this content:\\n\\nGet shareable linkSorry, a shareable link is not currently available for this article.\\n\\nCopy to clipboard\\n Provided by the Springer Nature SharedIt content-sharing initiative\\n \\n\\n\\n### Subjects\\n\\n\\n* [Bacterial structural biology](/subjects/bacterial-structural-biology)\\n* [Cryoelectron tomography](/subjects/cryoelectron-tomography)\\n* [Ribosome](/subjects/ribosome)\\n\\n\\n\\n\\n\\n\\nThis article is cited by\\n------------------------\\n\\n\\n\\n* ### [Learning structural heterogeneity from cryo-electron sub-tomograms with tomoDRGN](https://doi.org/10.1038/s41592-024-02210-z)\\n\\n\\n\\t+ Barrett M. Powell\\n\\t+ Joseph H. Davis*Nature Methods* (2024)\\n* ### [Transient disome complex formation in native polysomes during ongoing protein synthesis captured by cryo-EM](https://doi.org/10.1038/s41467-024-46092-3)\\n\\n\\n\\t+ Timo Flügel\\n\\t+ Magdalena Schacherl\\n\\t+ Christian M. T. Spahn*Nature Communications* (2024)\\n* ### [Patchy and widespread distribution of bacterial translation arrest peptides associated with the protein localization machinery](https://doi.org/10.1038/s41467-024-46993-3)\\n\\n\\n\\t+ Keigo Fujiwara\\n\\t+ Naoko Tsuji\\n\\t+ Shinobu Chiba*Nature Communications* (2024)\\n* ### [Visualization of translation and protein biogenesis at the ER membrane](https://doi.org/10.1038/s41586-022-05638-5)\\n\\n\\n\\t+ Max Gemmer\\n\\t+ Marten L. Chaillet\\n\\t+ Friedrich Förster*Nature* (2023)\\n* ### [The translating bacterial ribosome at 1.55 Å resolution generated by cryo-EM imaging services](https://doi.org/10.1038/s41467-023-36742-3)\\n\\n\\n\\t+ Simon A. Fromm\\n\\t+ Kate M. O’Connor\\n\\t+ Simone Mattei*Nature Communications* (2023)\\n\\n\\n\\n\\n\\nComments\\n--------\\n\\nBy submitting a comment you agree to abide by our [Terms](/info/tandc.html) and [Community Guidelines](/info/community-guidelines.html). If you find something abusive or that does not comply with our terms or guidelines please flag it as inappropriate.\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n[Download PDF](/articles/s41586-022-05255-2.pdf)\\n\\n\\n\\n\\nAssociated content\\n------------------\\n\\n\\n\\n\\n\\n\\n### [Atomic ribosome activity](https://www.nature.com/articles/s41594-022-00878-5)\\n\\n\\n* Jake Rogers\\n\\n\\n\\nNature Structural & Molecular Biology\\nResearch Highlight\\n09 Nov 2022\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n* Sections\\n* Figures\\n* References\\n\\n* [Abstract](#Abs1)\\n* [Main](#Sec1)\\n* [In-cell structure of the *M. pneumoniae* ribosome](#Sec2)\\n* [Structural dynamics of translation in cells](#Sec3)\\n* [Antibiotics alter translation landscapes](#Sec4)\\n* [Spatial and functional organization of translation](#Sec5)\\n* [Conclusions](#Sec6)\\n* [Methods](#Sec7)\\n* [Data availability](#data-availability)\\n* [Code availability](#code-availability)\\n* [Change history](#change-history)\\n* [References](#Bib1)\\n* [Acknowledgements](#Ack1)\\n* [Author information](#author-information)\\n* [Ethics declarations](#ethics)\\n* [Peer review](#peer-review)\\n* [Additional information](#additional-information)\\n* [Extended data figures and tables](#Sec19)\\n* [Supplementary information](#Sec20)\\n* [Rights and permissions](#rightslink)\\n* [About this article](#article-info)\\n* [This article is cited by](#further-reading)\\n* [Comments](#article-comments)\\n\\n\\n\\n\\nAdvertisement\\n\\n\\n\\n\\n[![Advertisement](//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature.com/article&sz=300x250&c=-563873083&t=pos%3Dright%26type%3Darticle%26artid%3Ds41586-022-05255-2%26doi%3D10.1038/s41586-022-05255-2%26techmeta%3D101,28%26subjmeta%3D1258,1260,1789,2536,326,337,41,45,535,574,631%26kwrd%3DBacterial+structural+biology,Cryoelectron+tomography,Ribosome)](//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature.com/article&sz=300x250&c=-563873083&t=pos%3Dright%26type%3Darticle%26artid%3Ds41586-022-05255-2%26doi%3D10.1038/s41586-022-05255-2%26techmeta%3D101,28%26subjmeta%3D1258,1260,1789,2536,326,337,41,45,535,574,631%26kwrd%3DBacterial+structural+biology,Cryoelectron+tomography,Ribosome)\\n\\n\\n\\n\\n\\n\\n* **Fig. 1: Ribosome structure in *M.* *pneumoniae* cells.**\\n\\n![figure 1]()[View in article](#Fig1)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/1)\\n* **Fig. 2: Ribosome classification reconstructs the translation elongation cycle.**\\n\\n![figure 2]()[View in article](#Fig2)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/2)\\n* **Fig. 3: Antibiotics induce distinct translation elongation landscapes in cells.**\\n\\n![figure 3]()[View in article](#Fig3)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/3)\\n* **Fig. 4: Spatial and functional organization of ribosomes in native cells.**\\n\\n![figure 4]()[View in article](#Fig4)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/4)\\n* **Extended Data Fig. 1 *M. pneumoniae* in-cell ribosome maps.**\\n\\n![extended data figure 5]()[View in article](#Fig5)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/5)\\n* **Extended Data Fig. 2 Structural features of the *M. pneumoniae* ribosome.**\\n\\n![extended data figure 6]()[View in article](#Fig6)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/6)\\n* **Extended Data Fig. 3 Ribosomal protein extensions.**\\n\\n![extended data figure 7]()[View in article](#Fig7)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/7)\\n* **Extended Data Fig. 4 Classification and refinement of ribosomes in native untreated cells.**\\n\\n![extended data figure 8]()[View in article](#Fig8)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/8)\\n* **Extended Data Fig. 5 Validation of ribosome detection and classification.**\\n\\n![extended data figure 9]()[View in article](#Fig9)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/9)\\n* **Extended Data Fig. 6 Local-resolution maps for ribosome classes in untreated cells.**\\n\\n![extended data figure 10]()[View in article](#Fig10)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/10)\\n* **Extended Data Fig. 7 Model building for ribosome classes in untreated cells.**\\n\\n![extended data figure 11]()[View in article](#Fig11)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/11)\\n* **Extended Data Fig. 8 Early-to-late translocation intermediates during elongation.**\\n\\n![extended data figure 12]()[View in article](#Fig12)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/12)\\n* **Extended Data Fig. 9 Classification, refinement and modelling of ribosomes in Cm-treated cells.**\\n\\n![extended data figure 13]()[View in article](#Fig13)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/13)\\n* **Extended Data Fig. 10 Classification, refinement and modelling of ribosomes in Spc-treated cells.**\\n\\n![extended data figure 14]()[View in article](#Fig14)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/14)\\n* **Extended Data Fig. 11 Classification, refinement and modelling of ribosomes in PUM-treated cells.**\\n\\n![extended data figure 15]()[View in article](#Fig15)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/15)\\n* **Extended Data Fig. 12 Spatial analysis of ribosomes and polysomes in native untreated cells.**\\n\\n![extended data figure 16]()[View in article](#Fig16)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/16)\\n* **Extended Data Fig. 13 Correlation of translation elongation states within polysomes in native untreated cells.**\\n\\n![extended data figure 17]()[View in article](#Fig17)[Full size image](https://www.nature.com/articles/s41586-022-05255-2/figures/17)\\n\\n1. Voorhees, R. M. & Ramakrishnan, V. Structural basis of the translational elongation cycle. *Annu. Rev. Biochem.* **82**, 203–236 (2013).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-biochem-113009-092313) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFOqsrzJ) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23746255) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20the%20translational%20elongation%20cycle&journal=Annu.%20Rev.%20Biochem.&doi=10.1146%2Fannurev-biochem-113009-092313&volume=82&pages=203-236&publication_year=2013&author=Voorhees%2CRM&author=Ramakrishnan%2CV)\\n2. O'Reilly, F. J. et al. In-cell architecture of an actively transcribing–translating expressome. *Science* **369**, 554–557 (2020).\\n\\n[Article](https://doi.org/10.1126%2Fscience.abb3758) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020Sci...369..554O) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFWgtr7N) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32732422) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115962) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=In-cell%20architecture%20of%20an%20actively%20transcribing%E2%80%93translating%20expressome&journal=Science&doi=10.1126%2Fscience.abb3758&volume=369&pages=554-557&publication_year=2020&author=O%27Reilly%2CFJ)\\n3. Tegunov, D., Xue, L., Dienemann, C., Cramer, P. & Mahamid, J. Multi-particle cryo-EM refinement with M visualizes ribosome–antibiotic complex at 3.5 Å in cells. *Nat. Methods* **18**, 186–193 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-020-01054-7) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjt1SmsL8%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33542511) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7611018) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-particle%20cryo-EM%20refinement%20with%20M%20visualizes%20ribosome%E2%80%93antibiotic%20complex%20at%203.5%20%C3%85%20in%20cells&journal=Nat.%20Methods&doi=10.1038%2Fs41592-020-01054-7&volume=18&pages=186-193&publication_year=2021&author=Tegunov%2CD&author=Xue%2CL&author=Dienemann%2CC&author=Cramer%2CP&author=Mahamid%2CJ)\\n4. Brandt, F. et al. The native 3D organization of bacterial polysomes. *Cell* **136**, 261–271 (2009).\\n\\n[Article](https://doi.org/10.1016%2Fj.cell.2008.11.016) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhs1Kiurk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19167328) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20native%203D%20organization%20of%20bacterial%20polysomes&journal=Cell&doi=10.1016%2Fj.cell.2008.11.016&volume=136&pages=261-271&publication_year=2009&author=Brandt%2CF)\\n5. Rodnina, M. V. Translation in prokaryotes. *Cold Spring Harb. Perspect. Biol.* **10**, a032664 (2018).\\n\\n[Article](https://doi.org/10.1101%2Fcshperspect.a032664) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29661790) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6120702) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Translation%20in%20prokaryotes&journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&doi=10.1101%2Fcshperspect.a032664&volume=10&publication_year=2018&author=Rodnina%2CMV)\\n6. Frank, J. & Agrawal, R. K. A ratchet-like inter-subunit reorganization of the ribosome during translocation. *Nature* **406**, 318–322 (2000).\\n\\n[Article](https://doi.org/10.1038%2F35018597) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2000Natur.406..318F) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXlsFKnsrk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10917535) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20ratchet-like%20inter-subunit%20reorganization%20of%20the%20ribosome%20during%20translocation&journal=Nature&doi=10.1038%2F35018597&volume=406&pages=318-322&publication_year=2000&author=Frank%2CJ&author=Agrawal%2CRK)\\n7. Villa, E. et al. Ribosome-induced changes in elongation factor Tu conformation control GTP hydrolysis. *Proc. Natl Acad. Sci. USA* **106**, 1063–1068 (2009).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.0811370106) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009PNAS..106.1063V) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhsFyjtbk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19122150) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2613361) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome-induced%20changes%20in%20elongation%20factor%20Tu%20conformation%20control%20GTP%20hydrolysis&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.0811370106&volume=106&pages=1063-1068&publication_year=2009&author=Villa%2CE)\\n8. Fischer, N., Konevega, A. L., Wintermeyer, W., Rodnina, M. V. & Stark, H. Ribosome dynamics and tRNA movement by time-resolved electron cryomicroscopy. *Nature* **466**, 329–333 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature09206) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.466..329F) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXosl2rsr0%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20631791) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome%20dynamics%20and%20tRNA%20movement%20by%20time-resolved%20electron%20cryomicroscopy&journal=Nature&doi=10.1038%2Fnature09206&volume=466&pages=329-333&publication_year=2010&author=Fischer%2CN&author=Konevega%2CAL&author=Wintermeyer%2CW&author=Rodnina%2CMV&author=Stark%2CH)\\n9. Ratje, A. H. et al. Head swivel on the ribosome facilitates translocation by means of intra-subunit tRNA hybrid sites. *Nature* **468**, 713–716 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature09547) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.468..713R) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsFSku7jF) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21124459) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3272701) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Head%20swivel%20on%20the%20ribosome%20facilitates%20translocation%20by%20means%20of%20intra-subunit%20tRNA%20hybrid%20sites&journal=Nature&doi=10.1038%2Fnature09547&volume=468&pages=713-716&publication_year=2010&author=Ratje%2CAH)\\n10. Ramrath, D. J. et al. Visualization of two transfer RNAs trapped in transit during elongation factor G-mediated translocation. *Proc. Natl Acad. Sci. USA* **110**, 20964–20969 (2013).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1320387110) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2013PNAS..11020964R) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXnsF2jug%3D%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24324168) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876219) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualization%20of%20two%20transfer%20RNAs%20trapped%20in%20transit%20during%20elongation%20factor%20G-mediated%20translocation&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1320387110&volume=110&pages=20964-20969&publication_year=2013&author=Ramrath%2CDJ)\\n11. Brilot, A. F., Korostelev, A. A., Ermolenko, D. N. & Grigorieff, N. Structure of the ribosome with elongation factor G trapped in the pretranslocation state. *Proc. Natl Acad. Sci. USA* **110**, 20994–20999 (2013).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1311423110) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2013PNAS..11020994B) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXnsF2iuw%3D%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24324137) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876248) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structure%20of%20the%20ribosome%20with%20elongation%20factor%20G%20trapped%20in%20the%20pretranslocation%20state&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1311423110&volume=110&pages=20994-20999&publication_year=2013&author=Brilot%2CAF&author=Korostelev%2CAA&author=Ermolenko%2CDN&author=Grigorieff%2CN)\\n12. Behrmann, E. et al. Structural snapshots of actively translating human ribosomes. *Cell* **161**, 845–857 (2015).\\n\\n[Article](https://doi.org/10.1016%2Fj.cell.2015.03.052) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXotVKlur0%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25957688) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4432480) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20snapshots%20of%20actively%20translating%20human%20ribosomes&journal=Cell&doi=10.1016%2Fj.cell.2015.03.052&volume=161&pages=845-857&publication_year=2015&author=Behrmann%2CE)\\n13. Loveland, A. B., Demo, G. & Korostelev, A. A. Cryo-EM of elongating ribosome with EF-Tu\\\\*GTP elucidates tRNA proofreading. *Nature* **584**, 640–645 (2020).\\n\\n[Article](https://doi.org/10.1038%2Fs41586-020-2447-x) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020Natur.584..640L) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlShtL%2FE) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32612237) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7483604) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cryo-EM%20of%20elongating%20ribosome%20with%20EF-Tu%2AGTP%20elucidates%20tRNA%20proofreading&journal=Nature&doi=10.1038%2Fs41586-020-2447-x&volume=584&pages=640-645&publication_year=2020&author=Loveland%2CAB&author=Demo%2CG&author=Korostelev%2CAA)\\n14. Rundlet, E. J. et al. Structural basis of early translocation events on the ribosome. *Nature* **595**, 741–745 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41586-021-03713-x) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Natur.595..741R) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFSnsbnM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34234344) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8318882) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20early%20translocation%20events%20on%20the%20ribosome&journal=Nature&doi=10.1038%2Fs41586-021-03713-x&volume=595&pages=741-745&publication_year=2021&author=Rundlet%2CEJ)\\n15. Petrychenko, V. et al. Structural mechanism of GTPase-powered ribosome–tRNA movement. *Nat. Commun.* **12**, 5933 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-26133-x) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.5933P) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1aqsL7M) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34635670) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8505512) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20mechanism%20of%20GTPase-powered%20ribosome%E2%80%93tRNA%20movement&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-26133-x&volume=12&publication_year=2021&author=Petrychenko%2CV)\\n16. Carbone, C. E. et al. Time-resolved cryo-EM visualizes ribosomal translocation with EF-G and GTP. *Nat. Commun.* **12**, 7236 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-27415-0) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.7236C) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislejtbjN) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34903725) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8668904) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Time-resolved%20cryo-EM%20visualizes%20ribosomal%20translocation%20with%20EF-G%20and%20GTP&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-27415-0&volume=12&publication_year=2021&author=Carbone%2CCE)\\n17. Brown, A. & Shao, S. Ribosomes and cryo-EM: a duet. *Curr. Opin. Struct. Biol.* **52**, 1–7 (2018).\\n\\n[Article](https://doi.org/10.1016%2Fj.sbi.2018.07.001) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlSjs77J) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30015201) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosomes%20and%20cryo-EM%3A%20a%20duet&journal=Curr.%20Opin.%20Struct.%20Biol.&doi=10.1016%2Fj.sbi.2018.07.001&volume=52&pages=1-7&publication_year=2018&author=Brown%2CA&author=Shao%2CS)\\n18. Kuhner, S. et al. Proteome organization in a genome-reduced bacterium. *Science* **326**, 1235–1240 (2009).\\n\\n[Article](https://doi.org/10.1126%2Fscience.1176343) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009Sci...326.1235K) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19965468) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Proteome%20organization%20in%20a%20genome-reduced%20bacterium&journal=Science&doi=10.1126%2Fscience.1176343&volume=326&pages=1235-1240&publication_year=2009&author=Kuhner%2CS)\\n19. Brandt, F., Carlson, L. A., Hartl, F. U., Baumeister, W. & Grunewald, K. The three-dimensional organization of polyribosomes in intact human cells. *Mol. Cell* **39**, 560–569 (2010).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2010.08.003) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVyrt7jF) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20797628) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20three-dimensional%20organization%20of%20polyribosomes%20in%20intact%20human%20cells&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2010.08.003&volume=39&pages=560-569&publication_year=2010&author=Brandt%2CF&author=Carlson%2CLA&author=Hartl%2CFU&author=Baumeister%2CW&author=Grunewald%2CK)\\n20. Mahamid, J. et al. Visualizing the molecular sociology at the HeLa cell nuclear periphery. *Science* **351**, 969–972 (2016).\\n\\n[Article](https://doi.org/10.1126%2Fscience.aad8857) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016Sci...351..969M) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XivFOnt7c%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26917770) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualizing%20the%20molecular%20sociology%20at%20the%20HeLa%20cell%20nuclear%20periphery&journal=Science&doi=10.1126%2Fscience.aad8857&volume=351&pages=969-972&publication_year=2016&author=Mahamid%2CJ)\\n21. Schaffer, M. et al. A cryo-FIB lift-out technique enables molecular-resolution cryo-ET within native *Caenorhabditis elegans* tissue. *Nat. Methods* **16**, 757–762 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0497-5) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVOku7zL) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31363205) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20cryo-FIB%20lift-out%20technique%20enables%20molecular-resolution%20cryo-ET%20within%20native%20Caenorhabditis%20elegans%20tissue&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0497-5&volume=16&pages=757-762&publication_year=2019&author=Schaffer%2CM)\\n22. Waltz, F. et al. How to build a ribosome from RNA fragments in *Chlamydomonas* mitochondria. *Nat. Commun.* **12**, 7176 (2021).\\n\\n[Article](https://doi.org/10.1038%2Fs41467-021-27200-z) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.7176W) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislWktLrO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34887394) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8660880) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=How%20to%20build%20a%20ribosome%20from%20RNA%20fragments%20in%20Chlamydomonas%20mitochondria&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-27200-z&volume=12&publication_year=2021&author=Waltz%2CF)\\n23. Melnikov, S., Manakongtreecheep, K. & Soll, D. Revising the structural diversity of ribosomal proteins across the three domains of life. *Mol. Biol. Evol.* **35**, 1588–1598 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsy021) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtF2rur7L) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29529322) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5995209) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Revising%20the%20structural%20diversity%20of%20ribosomal%20proteins%20across%20the%20three%20domains%20of%20life&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsy021&volume=35&pages=1588-1598&publication_year=2018&author=Melnikov%2CS&author=Manakongtreecheep%2CK&author=Soll%2CD)\\n24. Miravet-Verde, S., Burgos, R., Delgado, J., Lluch-Senar, M. & Serrano, L. FASTQINS and ANUBIS: two bioinformatic tools to explore facts and artifacts in transposon sequencing and essentiality studies. *Nucleic Acids Res.* **48**, e102 (2020).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkaa679) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXkt1GntbY%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32813015) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7515713) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FASTQINS%20and%20ANUBIS%3A%20two%20bioinformatic%20tools%20to%20explore%20facts%20and%20artifacts%20in%20transposon%20sequencing%20and%20essentiality%20studies&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkaa679&volume=48&publication_year=2020&author=Miravet-Verde%2CS&author=Burgos%2CR&author=Delgado%2CJ&author=Lluch-Senar%2CM&author=Serrano%2CL)\\n25. Borg, A., Pavlov, M. & Ehrenberg, M. Complete kinetic mechanism for recycling of the bacterial ribosome. *RNA* **22**, 10–21 (2016).\\n\\n[Article](https://doi.org/10.1261%2Frna.053157.115) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVKlur3M) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26527791) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4691825) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Complete%20kinetic%20mechanism%20for%20recycling%20of%20the%20bacterial%20ribosome&journal=RNA&doi=10.1261%2Frna.053157.115&volume=22&pages=10-21&publication_year=2016&author=Borg%2CA&author=Pavlov%2CM&author=Ehrenberg%2CM)\\n26. Munro, J. B., Altman, R. B., O’Connor, N. & Blanchard, S. C. Identification of two distinct hybrid state intermediates on the ribosome. *Mol. Cell* **25**, 505–517 (2007).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2007.01.022) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXjtlWkur8%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17317624) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2244649) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20two%20distinct%20hybrid%20state%20intermediates%20on%20the%20ribosome&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2007.01.022&volume=25&pages=505-517&publication_year=2007&author=Munro%2CJB&author=Altman%2CRB&author=O%E2%80%99Connor%2CN&author=Blanchard%2CSC)\\n27. Belardinelli, R. et al. Choreography of molecular movements during ribosome progression along mRNA. *Nat. Struct. Mol. Biol.* **23**, 342–348 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.3193) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XktFeqtrk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26999556) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Choreography%20of%20molecular%20movements%20during%20ribosome%20progression%20along%20mRNA&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.3193&volume=23&pages=342-348&publication_year=2016&author=Belardinelli%2CR)\\n28. Wasserman, M. R., Alejo, J. L., Altman, R. B. & Blanchard, S. C. Multiperspective smFRET reveals rate-determining late intermediates of ribosomal translocation. *Nat. Struct. Mol. Biol.* **23**, 333–341 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.3177) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XivFKhurY%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26926435) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4821728) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiperspective%20smFRET%20reveals%20rate-determining%20late%20intermediates%20of%20ribosomal%20translocation&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.3177&volume=23&pages=333-341&publication_year=2016&author=Wasserman%2CMR&author=Alejo%2CJL&author=Altman%2CRB&author=Blanchard%2CSC)\\n29. Fei, J., Kosuri, P., MacDougall, D. D. & Gonzalez, R. L. Jr Coupling of ribosomal L1 stalk and tRNA dynamics during translation elongation. *Mol. Cell* **30**, 348–359 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2008.03.012) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXmtFaktLg%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18471980) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Coupling%20of%20ribosomal%20L1%20stalk%20and%20tRNA%20dynamics%20during%20translation%20elongation&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2008.03.012&volume=30&pages=348-359&publication_year=2008&author=Fei%2CJ&author=Kosuri%2CP&author=MacDougall%2CDD&author=Gonzalez%2CRL)\\n30. Zhou, J., Lancaster, L., Donohue, J. P. & Noller, H. F. How the ribosome hands the A-site tRNA to the P site during EF-G-catalyzed translocation. *Science* **345**, 1188–1191 (2014).\\n\\n[Article](https://doi.org/10.1126%2Fscience.1255030) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2014Sci...345.1188Z) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsVGhtb%2FO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25190797) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4242719) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=How%20the%20ribosome%20hands%20the%20A-site%20tRNA%20to%20the%20P%20site%20during%20EF-G-catalyzed%20translocation&journal=Science&doi=10.1126%2Fscience.1255030&volume=345&pages=1188-1191&publication_year=2014&author=Zhou%2CJ&author=Lancaster%2CL&author=Donohue%2CJP&author=Noller%2CHF)\\n31. Uemura, S. et al. Real-time tRNA transit on single translating ribosomes at codon resolution. *Nature* **464**, 1012–U1073 (2010).\\n\\n[Article](https://doi.org/10.1038%2Fnature08925) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010Natur.464.1012U) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXkslart7c%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20393556) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4466108) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20tRNA%20transit%20on%20single%20translating%20ribosomes%20at%20codon%20resolution&journal=Nature&doi=10.1038%2Fnature08925&volume=464&pages=1012-U1073&publication_year=2010&author=Uemura%2CS)\\n32. Chen, C. et al. Single-molecule fluorescence measurements of ribosomal translocation dynamics. *Mol. Cell* **42**, 367–377 (2011).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2011.03.024) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXlvVyiu7w%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21549313) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3090999) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-molecule%20fluorescence%20measurements%20of%20ribosomal%20translocation%20dynamics&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2011.03.024&volume=42&pages=367-377&publication_year=2011&author=Chen%2CC)\\n33. Wilson, D. N. & Nierhaus, K. H. The E-site story: the importance of maintaining two tRNAs on the ribosome during protein synthesis. *Cell. Mol. Life Sci.* **63**, 2725–2737 (2006).\\n\\n[Article](https://link.springer.com/doi/10.1007/s00018-006-6125-4) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFSrt7w%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17013564) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20E-site%20story%3A%20the%20importance%20of%20maintaining%20two%20tRNAs%20on%20the%20ribosome%20during%20protein%20synthesis&journal=Cell.%20Mol.%20Life%20Sci.&doi=10.1007%2Fs00018-006-6125-4&volume=63&pages=2725-2737&publication_year=2006&author=Wilson%2CDN&author=Nierhaus%2CKH)\\n34. Munro, J. B., Sanbonmatsu, K. Y., Spahn, C. M. & Blanchard, S. C. Navigating the ribosome’s metastable energy landscape. *Trends Biochem. Sci* **34**, 390–400 (2009).\\n\\n[Article](https://doi.org/10.1016%2Fj.tibs.2009.04.004) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVSntrnM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19647434) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2914510) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Navigating%20the%20ribosome%E2%80%99s%20metastable%20energy%20landscape&journal=Trends%20Biochem.%20Sci&doi=10.1016%2Fj.tibs.2009.04.004&volume=34&pages=390-400&publication_year=2009&author=Munro%2CJB&author=Sanbonmatsu%2CKY&author=Spahn%2CCM&author=Blanchard%2CSC)\\n35. Bock, L. V. et al. Energy barriers and driving forces in tRNA translocation through the ribosome. *Nat. Struct. Mol. Biol.* **20**, 1390–1396 (2013).\\n\\n[Article](https://doi.org/10.1038%2Fnsmb.2690) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslWisbvK) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24186064) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Energy%20barriers%20and%20driving%20forces%20in%20tRNA%20translocation%20through%20the%20ribosome&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.2690&volume=20&pages=1390-1396&publication_year=2013&author=Bock%2CLV)\\n36. Julian, P. et al. Structure of ratcheted ribosomes with tRNAs in hybrid states. *Proc. Natl Acad. Sci. USA* **105**, 16924–16927 (2008).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.0809587105) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2008PNAS..10516924J) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlKqt7%2FM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18971332) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2579354) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structure%20of%20ratcheted%20ribosomes%20with%20tRNAs%20in%20hybrid%20states&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.0809587105&volume=105&pages=16924-16927&publication_year=2008&author=Julian%2CP)\\n37. Agirrezabala, X. et al. Visualization of the hybrid state of tRNA binding promoted by spontaneous ratcheting of the ribosome. *Mol. Cell* **32**, 190–197 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2008.10.001) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlehsb3L) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18951087) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614368) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualization%20of%20the%20hybrid%20state%20of%20tRNA%20binding%20promoted%20by%20spontaneous%20ratcheting%20of%20the%20ribosome&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2008.10.001&volume=32&pages=190-197&publication_year=2008&author=Agirrezabala%2CX)\\n38. Rodnina, M. V., Peske, F., Peng, B. Z., Belardinelli, R. & Wintermeyer, W. Converting GTP hydrolysis into motion: versatile translational elongation factor G. *Biol. Chem.* **401**, 131–142 (2019).\\n\\n[Article](https://doi.org/10.1515%2Fhsz-2019-0313) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31600135) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Converting%20GTP%20hydrolysis%20into%20motion%3A%20versatile%20translational%20elongation%20factor%20G&journal=Biol.%20Chem.&doi=10.1515%2Fhsz-2019-0313&volume=401&pages=131-142&publication_year=2019&author=Rodnina%2CMV&author=Peske%2CF&author=Peng%2CBZ&author=Belardinelli%2CR&author=Wintermeyer%2CW)\\n39. Lin, J., Zhou, D., Steitz, T. A., Polikanov, Y. S. & Gagnon, M. G. Ribosome-targeting antibiotics: modes of action, mechanisms of resistance, and implications for drug design. *Annu. Rev. Biochem.* **87**, 451–478 (2018).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-biochem-062917-011942) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXlslSrurw%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29570352) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9176271) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome-targeting%20antibiotics%3A%20modes%20of%20action%2C%20mechanisms%20of%20resistance%2C%20and%20implications%20for%20drug%20design&journal=Annu.%20Rev.%20Biochem.&doi=10.1146%2Fannurev-biochem-062917-011942&volume=87&pages=451-478&publication_year=2018&author=Lin%2CJ&author=Zhou%2CD&author=Steitz%2CTA&author=Polikanov%2CYS&author=Gagnon%2CMG)\\n40. Bulkley, D., Innis, C. A., Blaha, G. & Steitz, T. A. Revisiting the structures of several antibiotics bound to the bacterial ribosome. *Proc. Natl Acad. Sci. USA* **107**, 17158–17163 (2010).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1008685107) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2010PNAS..10717158B) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlSjurrM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20876130) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2951403) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Revisiting%20the%20structures%20of%20several%20antibiotics%20bound%20to%20the%20bacterial%20ribosome&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1008685107&volume=107&pages=17158-17163&publication_year=2010&author=Bulkley%2CD&author=Innis%2CCA&author=Blaha%2CG&author=Steitz%2CTA)\\n41. Svetlov, M. S. et al. High-resolution crystal structures of ribosome-bound chloramphenicol and erythromycin provide the ultimate basis for their competition. *RNA* **25**, 600–606 (2019).\\n\\n[Article](https://doi.org/10.1261%2Frna.069260.118) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1Ons77I) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30733327) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6467010) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-resolution%20crystal%20structures%20of%20ribosome-bound%20chloramphenicol%20and%20erythromycin%20provide%20the%20ultimate%20basis%20for%20their%20competition&journal=RNA&doi=10.1261%2Frna.069260.118&volume=25&pages=600-606&publication_year=2019&author=Svetlov%2CMS)\\n42. Borovinskaya, M. A., Shoji, S., Holton, J. M., Fredrick, K. & Cate, J. H. D. A steric block in translation caused by the antibiotic spectinomycin. *ACS Chem. Biol.* **2**, 545–552 (2007).\\n\\n[Article](https://doi.org/10.1021%2Fcb700100n) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXpvVOhtrc%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17696316) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4624401) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20steric%20block%20in%20translation%20caused%20by%20the%20antibiotic%20spectinomycin&journal=ACS%20Chem.%20Biol.&doi=10.1021%2Fcb700100n&volume=2&pages=545-552&publication_year=2007&author=Borovinskaya%2CMA&author=Shoji%2CS&author=Holton%2CJM&author=Fredrick%2CK&author=Cate%2CJHD)\\n43. Polikanov, Y. S. et al. Distinct tRNA accommodation intermediates observed on the ribosome with the antibiotics hygromycin A and A201A. *Mol. Cell* **58**, 832–844 (2015).\\n\\n[Article](https://doi.org/10.1016%2Fj.molcel.2015.04.014) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXpt1Skurg%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26028538) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4458074) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20tRNA%20accommodation%20intermediates%20observed%20on%20the%20ribosome%20with%20the%20antibiotics%20hygromycin%20A%20and%20A201A&journal=Mol.%20Cell&doi=10.1016%2Fj.molcel.2015.04.014&volume=58&pages=832-844&publication_year=2015&author=Polikanov%2CYS)\\n44. Morse, J. C. et al. Elongation factor-Tu can repetitively engage aminoacyl-tRNA within the ribosome during the proofreading stage of tRNA selection. *Proc. Natl Acad. Sci. USA* **117**, 3610–3620 (2020).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1904469117) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020PNAS..117.3610M) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjsFyhu7Y%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32024753) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7035488) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Elongation%20factor-Tu%20can%20repetitively%20engage%20aminoacyl-tRNA%20within%20the%20ribosome%20during%20the%20proofreading%20stage%20of%20tRNA%20selection&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1904469117&volume=117&pages=3610-3620&publication_year=2020&author=Morse%2CJC)\\n45. Choi, J. et al. Dynamics of the context-specific translation arrest by chloramphenicol and linezolid. *Nat. Chem. Biol.* **16**, 310–317 (2020).\\n\\n[Article](https://doi.org/10.1038%2Fs41589-019-0423-2) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVSktb%2FM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31844301) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dynamics%20of%20the%20context-specific%20translation%20arrest%20by%20chloramphenicol%20and%20linezolid&journal=Nat.%20Chem.%20Biol.&doi=10.1038%2Fs41589-019-0423-2&volume=16&pages=310-317&publication_year=2020&author=Choi%2CJ)\\n46. Marks, J. et al. Context-specific inhibition of translation by ribosomal antibiotics targeting the peptidyl transferase center. *Proc. Natl Acad. Sci. USA* **113**, 12150–12155 (2016).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1613055113) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016PNAS..11312150M) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1elur3N) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27791002) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5086994) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Context-specific%20inhibition%20of%20translation%20by%20ribosomal%20antibiotics%20targeting%20the%20peptidyl%20transferase%20center&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1613055113&volume=113&pages=12150-12155&publication_year=2016&author=Marks%2CJ)\\n47. Vazquez-Laslop, N. & Mankin, A. S. Context-specific action of ribosomal antibiotics. *Annu. Rev. Microbiol.* **72**, 185–207 (2018).\\n\\n[Article](https://doi.org/10.1146%2Fannurev-micro-090817-062329) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFCjs77O) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29906204) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8742604) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Context-specific%20action%20of%20ribosomal%20antibiotics&journal=Annu.%20Rev.%20Microbiol.&doi=10.1146%2Fannurev-micro-090817-062329&volume=72&pages=185-207&publication_year=2018&author=Vazquez-Laslop%2CN&author=Mankin%2CAS)\\n48. Aguirre Rivera, J. et al. Real-time measurements of aminoglycoside effects on protein synthesis in live cells. *Proc. Natl Acad. Sci. USA* **118**, e2013315118 (2021).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.2013315118) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33619089) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7936356) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20measurements%20of%20aminoglycoside%20effects%20on%20protein%20synthesis%20in%20live%20cells&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.2013315118&volume=118&publication_year=2021&author=Aguirre%20Rivera%2CJ)\\n49. Baquero, F. & Levin, B. R. Proximate and ultimate causes of the bactericidal action of antibiotics. *Nat. Rev. Microbiol.* **19**, 123–132 (2021).\\n50. Myasnikov, A. G. et al. The molecular structure of the left-handed supra-molecular helix of eukaryotic polyribosomes. *Nat. Commun.* **5**, 5294 (2014).\\n\\n[Article](https://doi.org/10.1038%2Fncomms6294) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2014NatCo...5.5294M) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFantrfM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25376914) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20molecular%20structure%20of%20the%20left-handed%20supra-molecular%20helix%20of%20eukaryotic%20polyribosomes&journal=Nat.%20Commun.&doi=10.1038%2Fncomms6294&volume=5&publication_year=2014&author=Myasnikov%2CAG)\\n51. Dunkle, J. A. & Cate, J. H. D. in *Ribosomes* (eds Rodnina, M. V., Wintermeyer, W. & Green, R.) Ch. 6 (2011).\\n52. Herr, A. J., Nelson, C. C., Wills, N. M., Gesteland, R. F. & Atkins, J. F. Analysis of the roles of tRNA structure, ribosomal protein L9, and the bacteriophage T4 gene 60 bypassing signals during ribosome slippage on mRNA. *J. Mol. Biol.* **309**, 1029–1048 (2001).\\n\\n[Article](https://doi.org/10.1006%2Fjmbi.2001.4717) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXkslGgurY%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11399077) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20the%20roles%20of%20tRNA%20structure%2C%20ribosomal%20protein%20L9%2C%20and%20the%20bacteriophage%20T4%20gene%2060%20bypassing%20signals%20during%20ribosome%20slippage%20on%20mRNA&journal=J.%20Mol.%20Biol.&doi=10.1006%2Fjmbi.2001.4717&volume=309&pages=1029-1048&publication_year=2001&author=Herr%2CAJ&author=Nelson%2CCC&author=Wills%2CNM&author=Gesteland%2CRF&author=Atkins%2CJF)\\n53. Smith, A. M., Costello, M. S., Kettring, A. H., Wingo, R. J. & Moore, S. D. Ribosome collisions alter frameshifting at translational reprogramming motifs in bacterial mRNAs. *Proc. Natl Acad. Sci. USA* **116**, 21769–21779 (2019).\\n\\n[Article](https://doi.org/10.1073%2Fpnas.1910613116) [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019PNAS..11621769S) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitValtbjM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31591196) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6815119) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ribosome%20collisions%20alter%20frameshifting%20at%20translational%20reprogramming%20motifs%20in%20bacterial%20mRNAs&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1910613116&volume=116&pages=21769-21779&publication_year=2019&author=Smith%2CAM&author=Costello%2CMS&author=Kettring%2CAH&author=Wingo%2CRJ&author=Moore%2CSD)\\n54. Tegunov, D. & Cramer, P. Real-time cryo-electron microscopy data preprocessing with Warp. *Nat. Methods* **16**, 1146–1152 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0580-y) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFejs77E) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31591575) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6858868) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-time%20cryo-electron%20microscopy%20data%20preprocessing%20with%20Warp&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0580-y&volume=16&pages=1146-1152&publication_year=2019&author=Tegunov%2CD&author=Cramer%2CP)\\n55. Hagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. *J. Struct. Biol.* **197**, 191–198 (2017).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2016.06.007) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27313000) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5287356) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Implementation%20of%20a%20cryo-electron%20tomography%20tilt-scheme%20optimized%20for%20high%20resolution%20subtomogram%20averaging&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2016.06.007&volume=197&pages=191-198&publication_year=2017&author=Hagen%2CWJH&author=Wan%2CW&author=Briggs%2CJAG)\\n56. Schorb, M., Haberbosch, I., Hagen, W. J. H., Schwab, Y. & Mastronarde, D. N. Software tools for automated transmission electron microscopy. *Nat. Methods* **16**, 471–477 (2019).\\n\\n[Article](https://doi.org/10.1038%2Fs41592-019-0396-9) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVSqs7zL) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31086343) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7000238) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Software%20tools%20for%20automated%20transmission%20electron%20microscopy&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0396-9&volume=16&pages=471-477&publication_year=2019&author=Schorb%2CM&author=Haberbosch%2CI&author=Hagen%2CWJH&author=Schwab%2CY&author=Mastronarde%2CDN)\\n57. Hrabe, T. et al. PyTom: a python-based toolbox for localization of macromolecules in cryo-electron tomograms and subtomogram analysis. *J. Struct. Biol.* **178**, 177–188 (2012).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2011.12.003) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xnt1ehtL4%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22193517) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PyTom%3A%20a%20python-based%20toolbox%20for%20localization%20of%20macromolecules%20in%20cryo-electron%20tomograms%20and%20subtomogram%20analysis&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2011.12.003&volume=178&pages=177-188&publication_year=2012&author=Hrabe%2CT)\\n58. Bharat, T. A. & Scheres, S. H. Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION. *Nat. Protoc.* **11**, 2054–2065 (2016).\\n\\n[Article](https://doi.org/10.1038%2Fnprot.2016.124) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsF2mtLrL) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27685097) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5215819) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Resolving%20macromolecular%20structures%20from%20electron%20cryo-tomography%20data%20using%20subtomogram%20averaging%20in%20RELION&journal=Nat.%20Protoc.&doi=10.1038%2Fnprot.2016.124&volume=11&pages=2054-2065&publication_year=2016&author=Bharat%2CTA&author=Scheres%2CSH)\\n59. Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. *eLife* **7**, e42166 (2018).\\n\\n[Article](https://doi.org/10.7554%2FeLife.42166) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30412051) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6250425) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=New%20tools%20for%20automated%20high-resolution%20cryo-EM%20structure%20determination%20in%20RELION-3&journal=eLife&doi=10.7554%2FeLife.42166&volume=7&publication_year=2018&author=Zivanov%2CJ)\\n60. Waterhouse, A. et al. SWISS-MODEL: homology modelling of protein structures and complexes. *Nucleic Acids Res.* **46**, W296–W303 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky427) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyru70%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29788355) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030848) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SWISS-MODEL%3A%20homology%20modelling%20of%20protein%20structures%20and%20complexes&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky427&volume=46&pages=W296-W303&publication_year=2018&author=Waterhouse%2CA)\\n61. Pruesse, E., Peplies, J. & Glockner, F. O. SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes. *Bioinformatics* **28**, 1823–1829 (2012).\\n\\n[Article](https://doi.org/10.1093%2Fbioinformatics%2Fbts252) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVSgsbfO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22556368) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3389763) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SINA%3A%20accurate%20high-throughput%20multiple%20sequence%20alignment%20of%20ribosomal%20RNA%20genes&journal=Bioinformatics&doi=10.1093%2Fbioinformatics%2Fbts252&volume=28&pages=1823-1829&publication_year=2012&author=Pruesse%2CE&author=Peplies%2CJ&author=Glockner%2CFO)\\n62. Rother, M., Rother, K., Puton, T. & Bujnicki, J. M. ModeRNA: a tool for comparative modeling of RNA 3D structure. *Nucleic Acids Res.* **39**, 4007–4022 (2011).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkq1320) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmvValsL4%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21300639) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3105415) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ModeRNA%3A%20a%20tool%20for%20comparative%20modeling%20of%20RNA%203D%20structure&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkq1320&volume=39&pages=4007-4022&publication_year=2011&author=Rother%2CM&author=Rother%2CK&author=Puton%2CT&author=Bujnicki%2CJM)\\n63. Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. *J. Comput. Chem.* **25**, 1605–1612 (2004).\\n\\n[Article](https://doi.org/10.1002%2Fjcc.20084) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXmvVOhsbs%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15264254) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCSF%20Chimera%E2%80%94a%20visualization%20system%20for%20exploratory%20research%20and%20analysis&journal=J.%20Comput.%20Chem.&doi=10.1002%2Fjcc.20084&volume=25&pages=1605-1612&publication_year=2004&author=Pettersen%2CEF)\\n64. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. *Acta Crystallogr D* **74**, 531–544 (2018).\\n\\n[Article](https://doi.org/10.1107%2FS2059798318006551) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2ltbzP) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-space%20refinement%20in%20PHENIX%20for%20cryo-EM%20and%20crystallography&journal=Acta%20Crystallogr%20D&doi=10.1107%2FS2059798318006551&volume=74&pages=531-544&publication_year=2018&author=Afonine%2CPV)\\n65. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. *Acta Crystallogr D* **66**, 486–501 (2010).\\n\\n[Article](https://doi.org/10.1107%2FS0907444910007493) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXksFKisb8%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20383002) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2852313) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Features%20and%20development%20of%20Coot&journal=Acta%20Crystallogr%20D&doi=10.1107%2FS0907444910007493&volume=66&pages=486-501&publication_year=2010&author=Emsley%2CP&author=Lohkamp%2CB&author=Scott%2CWG&author=Cowtan%2CK)\\n66. Williams, C. J. et al. MolProbity: more and better reference data for improved all-atom structure validation. *Protein Sci.* **27**, 293–315 (2018).\\n\\n[Article](https://doi.org/10.1002%2Fpro.3330) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvVKgsrnO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29067766) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MolProbity%3A%20more%20and%20better%20reference%20data%20for%20improved%20all-atom%20structure%20validation&journal=Protein%20Sci.&doi=10.1002%2Fpro.3330&volume=27&pages=293-315&publication_year=2018&author=Williams%2CCJ)\\n67. Chen, S. et al. High-resolution noise substitution to measure overfitting and validate resolution in 3D structure determination by single particle electron cryomicroscopy. *Ultramicroscopy* **135**, 24–35 (2013).\\n\\n[Article](https://doi.org/10.1016%2Fj.ultramic.2013.06.004) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslWhtLzO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23872039) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3834153) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-resolution%20noise%20substitution%20to%20measure%20overfitting%20and%20validate%20resolution%20in%203D%20structure%20determination%20by%20single%20particle%20electron%20cryomicroscopy&journal=Ultramicroscopy&doi=10.1016%2Fj.ultramic.2013.06.004&volume=135&pages=24-35&publication_year=2013&author=Chen%2CS)\\n68. Tatusov, R. L., Galperin, M. Y., Natale, D. A. & Koonin, E. V. The COG database: a tool for genome-scale analysis of protein functions and evolution. *Nucleic Acids Res.* **28**, 33–36 (2000).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2F28.1.33) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXhvVGqu7w%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10592175) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC102395) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20COG%20database%3A%20a%20tool%20for%20genome-scale%20analysis%20of%20protein%20functions%20and%20evolution&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2F28.1.33&volume=28&pages=33-36&publication_year=2000&author=Tatusov%2CRL&author=Galperin%2CMY&author=Natale%2CDA&author=Koonin%2CEV)\\n69. Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. *Nucleic Acids Res.* **47**, D309–D314 (2019).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky1085) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Gqtr%2FL) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30418610) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=eggNOG%205.0%3A%20a%20hierarchical%2C%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky1085&volume=47&pages=D309-D314&publication_year=2019&author=Huerta-Cepas%2CJ)\\n70. Katoh, K., Rozewicki, J. & Yamada, K. D. MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization. *Brief. Bioinform.* **20**, 1160–1166 (2019).\\n\\n[Article](https://doi.org/10.1093%2Fbib%2Fbbx108) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1Kqtb3K) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28968734) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MAFFT%20online%20service%3A%20multiple%20sequence%20alignment%2C%20interactive%20sequence%20choice%20and%20visualization&journal=Brief.%20Bioinform.&doi=10.1093%2Fbib%2Fbbx108&volume=20&pages=1160-1166&publication_year=2019&author=Katoh%2CK&author=Rozewicki%2CJ&author=Yamada%2CKD)\\n71. Letunic, I. & Bork, P. Interactive Tree Of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. *Nucleic Acids Res.* **49**, W293–W296 (2021).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkab301) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvV2isLvM) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33885785) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8265157) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interactive%20Tree%20Of%20Life%20%28iTOL%29%20v5%3A%20an%20online%20tool%20for%20phylogenetic%20tree%20display%20and%20annotation&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkab301&volume=49&pages=W293-W296&publication_year=2021&author=Letunic%2CI&author=Bork%2CP)\\n72. Huerta-Cepas, J. et al. Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper. *Mol. Biol. Evol.* **34**, 2115–2122 (2017).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsx148) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFaru7vF) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28460117) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850834) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20genome-wide%20functional%20annotation%20through%20orthology%20assignment%20by%20eggNOG-Mapper&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsx148&volume=34&pages=2115-2122&publication_year=2017&author=Huerta-Cepas%2CJ)\\n73. Huerta-Cepas, J., Serra, F. & Bork, P. ETE 3: reconstruction, analysis, and visualization of phylogenomic data. *Mol. Biol. Evol.* **33**, 1635–1638 (2016).\\n\\n[Article](https://doi.org/10.1093%2Fmolbev%2Fmsw046) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFeisrrN) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26921390) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4868116) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ETE%203%3A%20reconstruction%2C%20analysis%2C%20and%20visualization%20of%20phylogenomic%20data&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsw046&volume=33&pages=1635-1638&publication_year=2016&author=Huerta-Cepas%2CJ&author=Serra%2CF&author=Bork%2CP)\\n74. Meszaros, B., Erdos, G. & Dosztanyi, Z. IUPred2A: context-dependent prediction of protein disorder as a function of redox state and protein binding. *Nucleic Acids Res.* **46**, W329–W337 (2018).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgky384) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyrt7o%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29860432) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030935) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=IUPred2A%3A%20context-dependent%20prediction%20of%20protein%20disorder%20as%20a%20function%20of%20redox%20state%20and%20protein%20binding&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky384&volume=46&pages=W329-W337&publication_year=2018&author=Meszaros%2CB&author=Erdos%2CG&author=Dosztanyi%2CZ)\\n75. Cole, C., Barber, J. D. & Barton, G. J. The Jpred 3 secondary structure prediction server. *Nucleic Acids Res.* **36**, W197–W201 (2008).\\n\\n[Article](https://doi.org/10.1093%2Fnar%2Fgkn238) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXptlKkt7w%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18463136) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2447793) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Jpred%203%20secondary%20structure%20prediction%20server&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkn238&volume=36&pages=W197-W201&publication_year=2008&author=Cole%2CC&author=Barber%2CJD&author=Barton%2CGJ)\\n76. Scheres, S. H. Processing of structurally heterogeneous cryo-EM data in RELION. *Methods Enzymol.* **579**, 125–157 (2016).\\n\\n[Article](https://doi.org/10.1016%2Fbs.mie.2016.04.012) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmtlKnurk%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27572726) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Processing%20of%20structurally%20heterogeneous%20cryo-EM%20data%20in%20RELION&journal=Methods%20Enzymol.&doi=10.1016%2Fbs.mie.2016.04.012&volume=579&pages=125-157&publication_year=2016&author=Scheres%2CSH)\\n77. Kidmose, R. T. et al. Namdinator—automatic molecular dynamics flexible fitting of structural models into cryo-EM and crystallography experimental maps. *IUCrJ* **6**, 526–531 (2019).\\n\\n[Article](https://doi.org/10.1107%2FS2052252519007619) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlWit7%2FO) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31316797) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6608625) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Namdinator%E2%80%94automatic%20molecular%20dynamics%20flexible%20fitting%20of%20structural%20models%20into%20cryo-EM%20and%20crystallography%20experimental%20maps&journal=IUCrJ&doi=10.1107%2FS2052252519007619&volume=6&pages=526-531&publication_year=2019&author=Kidmose%2CRT)\\n78. Trabuco, L. G., Villa, E., Mitra, K., Frank, J. & Schulten, K. Flexible fitting of atomic structures into electron microscopy maps using molecular dynamics. *Structure* **16**, 673–683 (2008).\\n\\n[Article](https://doi.org/10.1016%2Fj.str.2008.03.005) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXlslGrtro%3D) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18462672) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2430731) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Flexible%20fitting%20of%20atomic%20structures%20into%20electron%20microscopy%20maps%20using%20molecular%20dynamics&journal=Structure&doi=10.1016%2Fj.str.2008.03.005&volume=16&pages=673-683&publication_year=2008&author=Trabuco%2CLG&author=Villa%2CE&author=Mitra%2CK&author=Frank%2CJ&author=Schulten%2CK)\\n79. Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. *J. Chem. Phys.* **153**, 044130 (2020).\\n\\n[Article](https://doi.org/10.1063%2F5.0014475) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFajsL3J) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32752662) [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7395834) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scalable%20molecular%20dynamics%20on%20CPU%20and%20GPU%20architectures%20with%20NAMD&journal=J.%20Chem.%20Phys.&doi=10.1063%2F5.0014475&volume=153&publication_year=2020&author=Phillips%2CJC)\\n80. Nickell, S. et al. TOM software toolbox: acquisition and analysis for electron tomography. *J. Struct. Biol.* **149**, 227–234 (2005).\\n\\n[Article](https://doi.org/10.1016%2Fj.jsb.2004.10.006) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15721576) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TOM%20software%20toolbox%3A%20acquisition%20and%20analysis%20for%20electron%20tomography&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2004.10.006&volume=149&pages=227-234&publication_year=2005&author=Nickell%2CS)\\n81. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. *Protein Sci.* **30**, 70–82 (2021).\\n\\n[Article](https://doi.org/10.1002%2Fpro.3943) [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFamtrzJ) [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32881101) [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCSF%20ChimeraX%3A%20structure%20visualization%20for%20researchers%2C%20educators%2C%20and%20developers&journal=Protein%20Sci.&doi=10.1002%2Fpro.3943&volume=30&pages=70-82&publication_year=2021&author=Pettersen%2CEF)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n Nature (*Nature*)\\n \\n\\nISSN 1476-4687 (online)\\n \\n\\nISSN 0028-0836 (print)\\n \\n\\n\\n\\n\\n\\n\\n\\n\\nnature.com sitemap\\n------------------\\n\\n\\n\\n\\n\\n### About Nature Portfolio\\n\\n\\n* [About us](https://www.nature.com/npg_/company_info/index.html)\\n* [Press releases](https://www.nature.com/npg_/press_room/press_releases.html)\\n* [Press office](https://press.nature.com/)\\n* [Contact us](https://support.nature.com/support/home)\\n\\n\\n\\n\\n### Discover content\\n\\n\\n* [Journals A-Z](https://www.nature.com/siteindex)\\n* [Articles by subject](https://www.nature.com/subjects)\\n* [protocols.io](https://www.protocols.io/)\\n* [Nature Index](https://www.natureindex.com/)\\n\\n\\n\\n\\n### Publishing policies\\n\\n\\n* [Nature portfolio policies](https://www.nature.com/authors/editorial_policies)\\n* [Open access](https://www.nature.com/nature-research/open-access)\\n\\n\\n\\n\\n### Author & Researcher services\\n\\n\\n* [Reprints & permissions](https://www.nature.com/reprints)\\n* [Research data](https://www.springernature.com/gp/authors/research-data)\\n* [Language editing](https://authorservices.springernature.com/language-editing/)\\n* [Scientific editing](https://authorservices.springernature.com/scientific-editing/)\\n* [Nature Masterclasses](https://masterclasses.nature.com/)\\n* [Research Solutions](https://solutions.springernature.com/)\\n\\n\\n\\n\\n### Libraries & institutions\\n\\n\\n* [Librarian service & tools](https://www.springernature.com/gp/librarians/tools-services)\\n* [Librarian portal](https://www.springernature.com/gp/librarians/manage-your-account/librarianportal)\\n* [Open research](https://www.nature.com/openresearch/about-open-access/information-for-institutions)\\n* [Recommend to library](https://www.springernature.com/gp/librarians/recommend-to-your-library)\\n\\n\\n\\n\\n### Advertising & partnerships\\n\\n\\n* [Advertising](https://partnerships.nature.com/product/digital-advertising/)\\n* [Partnerships & Services](https://partnerships.nature.com/)\\n* [Media kits](https://partnerships.nature.com/media-kits/)\\n* [Branded\\n content](https://partnerships.nature.com/product/branded-content-native-advertising/)\\n\\n\\n\\n\\n### Professional development\\n\\n\\n* [Nature Careers](https://www.nature.com/naturecareers/)\\n* [Nature \\n Conferences](https://conferences.nature.com)\\n\\n\\n\\n\\n### Regional websites\\n\\n\\n* [Nature Africa](https://www.nature.com/natafrica)\\n* [Nature China](http://www.naturechina.com)\\n* [Nature India](https://www.nature.com/nindia)\\n* [Nature Italy](https://www.nature.com/natitaly)\\n* [Nature Japan](https://www.natureasia.com/ja-jp)\\n* [Nature Middle East](https://www.nature.com/nmiddleeast)\\n\\n\\n\\n\\n\\n\\n* [Privacy\\n Policy](https://www.nature.com/info/privacy)\\n* [Use\\n of cookies](https://www.nature.com/info/cookies)\\n* Your privacy choices/Manage cookies\\n* [Legal\\n notice](https://www.nature.com/info/legal-notice)\\n* [Accessibility\\n statement](https://www.nature.com/info/accessibility-statement)\\n* [Terms & Conditions](https://www.nature.com/info/terms-and-conditions)\\n* [Your US state privacy rights](https://www.springernature.com/ccpa)\\n\\n\\n\\n\\n\\n[![Springer Nature](/static/images/logos/sn-logo-white-ea63208b81.svg)](https://www.springernature.com/)\\n© 2024 Springer Nature Limited\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nClose banner\\n\\n\\n\\n\\n\\n\\nClose\\n\\n\\n\\n\\n\\n![Nature Briefing](/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg)\\nSign up for the *Nature Briefing* newsletter — what matters in science, free to your inbox daily.\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nEmail address\\n\\n\\n\\nSign up\\n\\n\\n\\nI agree my information will be processed in accordance with the *Nature* and Springer Nature Limited [Privacy Policy](https://www.nature.com/info/privacy).\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nClose banner\\n\\n\\n\\n\\n\\n\\nClose\\n\\n\\n\\nGet the most important science stories of the day, free in your inbox.\\n[Sign up for Nature Briefing](https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner) \\n\\n\\n\\n\\n![](https://verify.nature.com/verify/nature.png)\\n\\n\\n![](/e8zlnwlx/article/s41586-022-05255-2)\\n\\n```\\n\\n    From this code, identify any text paragraphs that refer to data sets with URLS links specified by '(URL)[TEXT]' Markdown fragments. \\n\\n    You will generate output as a JSON object with \\\"TEXT\\\", and \\\"LINKS\\\" as the keys. \\n\\n    The value of \\\"TEXT\\\" will be a list of paragraphs from the Markdown code that describe links to external biological datasets.\\n    The value of \\\"LINKS\\\" will be a list of dicts where each entry has two attributes: URL and TEXT extracted from each URL link.\\n    \\n    Do not provide additional explanation or context for the answer.\\n   \\n    Do not include any other response other than a JSON object.\"\n",
      "  ]\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[llm/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 3:llm:ChatOpenAI] [323.11s] Exiting LLM run with output:\n",
      "\u001b[0m{\n",
      "  \"generations\": [\n",
      "    [\n",
      "      {\n",
      "        \"text\": \"```json\\n{\\n  \\\"TEXT\\\": [\\n    \\\"Maps have been deposited in the EMDB under accession codes [13234](http://www.ebi.ac.uk/pdbe/entry/EMD-13234), [13272](http://www.ebi.ac.uk/pdbe/entry/EMD-13272), [13273](http://www.ebi.ac.uk/pdbe/entry/EMD-13273), [13274](http://www.ebi.ac.uk/pdbe/entry/EMD-13274), [13275](http://www.ebi.ac.uk/pdbe/entry/EMD-13275), [13276](http://www.ebi.ac.uk/pdbe/entry/EMD-13276), [13277](http://www.ebi.ac.uk/pdbe/entry/EMD-13277), [13278](http://www.ebi.ac.uk/pdbe/entry/EMD-13278), [13279](http://www.ebi.ac.uk/pdbe/entry/EMD-13279), [13280](http://www.ebi.ac.uk/pdbe/entry/EMD-13280), [13281](http://www.ebi.ac.uk/pdbe/entry/EMD-13281), [13282](http://www.ebi.ac.uk/pdbe/entry/EMD-13282), [13283](http://www.ebi.ac.uk/pdbe/entry/EMD-13283), [13284](http://www.ebi.ac.uk/pdbe/entry/EMD-13284), [13285](http://www.ebi.ac.uk/pdbe/entry/EMD-13285), [13286](http://www.ebi.ac.uk/pdbe/entry/EMD-13286), [13410](http://www.ebi.ac.uk/pdbe/entry/EMD-13410), [13411](http://www.ebi.ac.uk/pdbe/entry/EMD-13411), [13412](http://www.ebi.ac.uk/pdbe/entry/EMD-13412), [13413](http://www.ebi.ac.uk/pdbe/entry/EMD-13413), [13414](http://www.ebi.ac.uk/pdbe/entry/EMD-13414), [13431](http://www.ebi.ac.uk/pdbe/entry/EMD-13431), [13432](http://www.ebi.ac.uk/pdbe/entry/EMD-13432), [13433](http://www.ebi.ac.uk/pdbe/entry/EMD-13433), [13434](http://www.ebi.ac.uk/pdbe/entry/EMD-13434), [13435](http://www.ebi.ac.uk/pdbe/entry/EMD-13435), [13436](http://www.ebi.ac.uk/pdbe/entry/EMD-13436), [13445](http://www.ebi.ac.uk/pdbe/entry/EMD-13445), [13446](http://www.ebi.ac.uk/pdbe/entry/EMD-13446), [13447](http://www.ebi.ac.uk/pdbe/entry/EMD-13447), [13448](http://www.ebi.ac.uk/pdbe/entry/EMD-13448), [13449](http://www.ebi.ac.uk/pdbe/entry/EMD-13449), [13450](http://www.ebi.ac.uk/pdbe/entry/EMD-13450), [13451](http://www.ebi.ac.uk/pdbe/entry/EMD-13451), [13452](http://www.ebi.ac.uk/pdbe/entry/EMD-13452), [13287](http://www.ebi.ac.uk/pdbe/entry/EMD-13287), [13288](http://www.ebi.ac.uk/pdbe/entry/EMD-13288) and [13289](http://www.ebi.ac.uk/pdbe/entry/EMD-13289). Models have been deposited in the PDB under accession codes: [7OOC](https://doi.org/10.2210/pdb7OOC/pdb), [7OOD](http://doi.org/10.2210/pdb7OOD/pdb), [7P6Z](http://doi.org/10.2210/pdb7P6Z/pdb), [7PAH](http://doi.org/10.2210/pdb7PAH/pdb), [7PAI](http://doi.org/10.2210/pdb7PAI/pdb), [7PAJ](http://doi.org/10.2210/pdb7PAJ/pdb), [7PAK](http://doi.org/10.2210/pdb7PAK/pdb), [7PAL](http://doi.org/10.2210/pdb7PAL/pdb), [7PAM](http://doi.org/10.2210/pdb7PAM/pdb), [7PAN](http://doi.org/10.2210/pdb7PAN/pdb), [7PAO](http://doi.org/10.2210/pdb7PAO/pdb), [7PAQ](http://doi.org/10.2210/pdb7PAQ/pdb), [7PAR](http://doi.org/10.2210/pdb7PAR/pdb), [7PAS](http://doi.org/10.2210/pdb7PAS/pdb), [7PAT](http://doi.org/10.2210/pdb7PAT/pdb), [7PAU](http://doi.org/10.2210/pdb7PAU/pdb), [7PH9](http://doi.org/10.2210/pdb7PH9/pdb), [7PHA](http://doi.org/10.2210/pdb7PHA/pdb), [7PHB](http://doi.org/10.2210/pdb7PHB/pdb), [7PHC](http://doi.org/10.2210/pdb7PHC/pdb), [7PI8](http://doi.org/10.2210/pdb7PI8/pdb), [7PI9](http://doi.org/10.2210/pdb7PI9/pdb), [7PIA](http://doi.org/10.2210/pdb7PIA/pdb), [7PIB](http://doi.org/10.2210/pdb7PIB/pdb), [7PIC](http://doi.org/10.2210/pdb7PIC/pdb), [7PIO](http://doi.org/10.2210/pdb7PIO/pdb), [7PIP](http://doi.org/10.2210/pdb7PIP/pdb), [7PIQ](http://doi.org/10.2210/pdb7PIQ/pdb), [7PIR](http://doi.org/10.2210/pdb7PIR/pdb), [7PIS](http://doi.org/10.2210/pdb7PIS/pdb) and [7PIT](http://doi.org/10.2210/pdb7PIT/pdb).\\\"\\n  ],\\n  \\\"LINKS\\\": [\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13234\\\",\\n      \\\"TEXT\\\": \\\"13234\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13272\\\",\\n      \\\"TEXT\\\": \\\"13272\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13273\\\",\\n      \\\"TEXT\\\": \\\"13273\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13274\\\",\\n      \\\"TEXT\\\": \\\"13274\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13275\\\",\\n      \\\"TEXT\\\": \\\"13275\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13276\\\",\\n      \\\"TEXT\\\": \\\"13276\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13277\\\",\\n      \\\"TEXT\\\": \\\"13277\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13278\\\",\\n      \\\"TEXT\\\": \\\"13278\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13279\\\",\\n      \\\"TEXT\\\": \\\"13279\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13280\\\",\\n      \\\"TEXT\\\": \\\"13280\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13281\\\",\\n      \\\"TEXT\\\": \\\"13281\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13282\\\",\\n      \\\"TEXT\\\": \\\"13282\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13283\\\",\\n      \\\"TEXT\\\": \\\"13283\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13284\\\",\\n      \\\"TEXT\\\": \\\"13284\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13285\\\",\\n      \\\"TEXT\\\": \\\"13285\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13286\\\",\\n      \\\"TEXT\\\": \\\"13286\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13410\\\",\\n      \\\"TEXT\\\": \\\"13410\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13411\\\",\\n      \\\"TEXT\\\": \\\"13411\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13412\\\",\\n      \\\"TEXT\\\": \\\"13412\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13413\\\",\\n      \\\"TEXT\\\": \\\"13413\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13414\\\",\\n      \\\"TEXT\\\": \\\"13414\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13431\\\",\\n      \\\"TEXT\\\": \\\"13431\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13432\\\",\\n      \\\"TEXT\\\": \\\"13432\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13433\\\",\\n      \\\"TEXT\\\": \\\"13433\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13434\\\",\\n      \\\"TEXT\\\": \\\"13434\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13435\\\",\\n      \\\"TEXT\\\": \\\"13435\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13436\\\",\\n      \\\"TEXT\\\": \\\"13436\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13445\\\",\\n      \\\"TEXT\\\": \\\"13445\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13446\\\",\\n      \\\"TEXT\\\": \\\"13446\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13447\\\",\\n      \\\"TEXT\\\": \\\"13447\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13448\\\",\\n      \\\"TEXT\\\": \\\"13448\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13449\\\",\\n      \\\"TEXT\\\": \\\"13449\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13450\\\",\\n      \\\"TEXT\\\": \\\"13450\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13451\\\",\\n      \\\"TEXT\\\": \\\"13451\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13452\\\",\\n      \\\"TEXT\\\": \\\"13452\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13287\\\",\\n      \\\"TEXT\\\": \\\"13287\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13288\\\",\\n      \\\"TEXT\\\": \\\"13288\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13289\\\",\\n      \\\"TEXT\\\": \\\"13289\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"https://doi.org/10.2210/pdb7OOC/pdb\\\",\\n      \\\"TEXT\\\": \\\"7OOC\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7OOD/pdb\\\",\\n      \\\"TEXT\\\": \\\"7OOD\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7P6Z/pdb\\\",\\n      \\\"TEXT\\\": \\\"7P6Z\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAH/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAH\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAI/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAI\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAJ/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAJ\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAK/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAK\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAL/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAL\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAM/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAM\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAN/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAN\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAO/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAO\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAQ/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAQ\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAR/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAR\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAS/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAS\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAT/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAT\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAU/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAU\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PH9/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PH9\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PHA/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PHA\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PHB/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PHB\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PHC/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PHC\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PI8/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PI8\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PI9/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PI9\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIA/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIA\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIB/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIB\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIC/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIC\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIO/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIO\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIP/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIP\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIQ/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIQ\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIR/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIR\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIS/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIS\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIT/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIT\\\"\\n    }\\n  ]\\n}\\n```\",\n",
      "        \"generation_info\": {\n",
      "          \"finish_reason\": \"stop\",\n",
      "          \"logprobs\": null\n",
      "        },\n",
      "        \"type\": \"ChatGeneration\",\n",
      "        \"message\": {\n",
      "          \"lc\": 1,\n",
      "          \"type\": \"constructor\",\n",
      "          \"id\": [\n",
      "            \"langchain\",\n",
      "            \"schema\",\n",
      "            \"messages\",\n",
      "            \"AIMessage\"\n",
      "          ],\n",
      "          \"kwargs\": {\n",
      "            \"content\": \"```json\\n{\\n  \\\"TEXT\\\": [\\n    \\\"Maps have been deposited in the EMDB under accession codes [13234](http://www.ebi.ac.uk/pdbe/entry/EMD-13234), [13272](http://www.ebi.ac.uk/pdbe/entry/EMD-13272), [13273](http://www.ebi.ac.uk/pdbe/entry/EMD-13273), [13274](http://www.ebi.ac.uk/pdbe/entry/EMD-13274), [13275](http://www.ebi.ac.uk/pdbe/entry/EMD-13275), [13276](http://www.ebi.ac.uk/pdbe/entry/EMD-13276), [13277](http://www.ebi.ac.uk/pdbe/entry/EMD-13277), [13278](http://www.ebi.ac.uk/pdbe/entry/EMD-13278), [13279](http://www.ebi.ac.uk/pdbe/entry/EMD-13279), [13280](http://www.ebi.ac.uk/pdbe/entry/EMD-13280), [13281](http://www.ebi.ac.uk/pdbe/entry/EMD-13281), [13282](http://www.ebi.ac.uk/pdbe/entry/EMD-13282), [13283](http://www.ebi.ac.uk/pdbe/entry/EMD-13283), [13284](http://www.ebi.ac.uk/pdbe/entry/EMD-13284), [13285](http://www.ebi.ac.uk/pdbe/entry/EMD-13285), [13286](http://www.ebi.ac.uk/pdbe/entry/EMD-13286), [13410](http://www.ebi.ac.uk/pdbe/entry/EMD-13410), [13411](http://www.ebi.ac.uk/pdbe/entry/EMD-13411), [13412](http://www.ebi.ac.uk/pdbe/entry/EMD-13412), [13413](http://www.ebi.ac.uk/pdbe/entry/EMD-13413), [13414](http://www.ebi.ac.uk/pdbe/entry/EMD-13414), [13431](http://www.ebi.ac.uk/pdbe/entry/EMD-13431), [13432](http://www.ebi.ac.uk/pdbe/entry/EMD-13432), [13433](http://www.ebi.ac.uk/pdbe/entry/EMD-13433), [13434](http://www.ebi.ac.uk/pdbe/entry/EMD-13434), [13435](http://www.ebi.ac.uk/pdbe/entry/EMD-13435), [13436](http://www.ebi.ac.uk/pdbe/entry/EMD-13436), [13445](http://www.ebi.ac.uk/pdbe/entry/EMD-13445), [13446](http://www.ebi.ac.uk/pdbe/entry/EMD-13446), [13447](http://www.ebi.ac.uk/pdbe/entry/EMD-13447), [13448](http://www.ebi.ac.uk/pdbe/entry/EMD-13448), [13449](http://www.ebi.ac.uk/pdbe/entry/EMD-13449), [13450](http://www.ebi.ac.uk/pdbe/entry/EMD-13450), [13451](http://www.ebi.ac.uk/pdbe/entry/EMD-13451), [13452](http://www.ebi.ac.uk/pdbe/entry/EMD-13452), [13287](http://www.ebi.ac.uk/pdbe/entry/EMD-13287), [13288](http://www.ebi.ac.uk/pdbe/entry/EMD-13288) and [13289](http://www.ebi.ac.uk/pdbe/entry/EMD-13289). Models have been deposited in the PDB under accession codes: [7OOC](https://doi.org/10.2210/pdb7OOC/pdb), [7OOD](http://doi.org/10.2210/pdb7OOD/pdb), [7P6Z](http://doi.org/10.2210/pdb7P6Z/pdb), [7PAH](http://doi.org/10.2210/pdb7PAH/pdb), [7PAI](http://doi.org/10.2210/pdb7PAI/pdb), [7PAJ](http://doi.org/10.2210/pdb7PAJ/pdb), [7PAK](http://doi.org/10.2210/pdb7PAK/pdb), [7PAL](http://doi.org/10.2210/pdb7PAL/pdb), [7PAM](http://doi.org/10.2210/pdb7PAM/pdb), [7PAN](http://doi.org/10.2210/pdb7PAN/pdb), [7PAO](http://doi.org/10.2210/pdb7PAO/pdb), [7PAQ](http://doi.org/10.2210/pdb7PAQ/pdb), [7PAR](http://doi.org/10.2210/pdb7PAR/pdb), [7PAS](http://doi.org/10.2210/pdb7PAS/pdb), [7PAT](http://doi.org/10.2210/pdb7PAT/pdb), [7PAU](http://doi.org/10.2210/pdb7PAU/pdb), [7PH9](http://doi.org/10.2210/pdb7PH9/pdb), [7PHA](http://doi.org/10.2210/pdb7PHA/pdb), [7PHB](http://doi.org/10.2210/pdb7PHB/pdb), [7PHC](http://doi.org/10.2210/pdb7PHC/pdb), [7PI8](http://doi.org/10.2210/pdb7PI8/pdb), [7PI9](http://doi.org/10.2210/pdb7PI9/pdb), [7PIA](http://doi.org/10.2210/pdb7PIA/pdb), [7PIB](http://doi.org/10.2210/pdb7PIB/pdb), [7PIC](http://doi.org/10.2210/pdb7PIC/pdb), [7PIO](http://doi.org/10.2210/pdb7PIO/pdb), [7PIP](http://doi.org/10.2210/pdb7PIP/pdb), [7PIQ](http://doi.org/10.2210/pdb7PIQ/pdb), [7PIR](http://doi.org/10.2210/pdb7PIR/pdb), [7PIS](http://doi.org/10.2210/pdb7PIS/pdb) and [7PIT](http://doi.org/10.2210/pdb7PIT/pdb).\\\"\\n  ],\\n  \\\"LINKS\\\": [\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13234\\\",\\n      \\\"TEXT\\\": \\\"13234\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13272\\\",\\n      \\\"TEXT\\\": \\\"13272\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13273\\\",\\n      \\\"TEXT\\\": \\\"13273\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13274\\\",\\n      \\\"TEXT\\\": \\\"13274\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13275\\\",\\n      \\\"TEXT\\\": \\\"13275\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13276\\\",\\n      \\\"TEXT\\\": \\\"13276\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13277\\\",\\n      \\\"TEXT\\\": \\\"13277\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13278\\\",\\n      \\\"TEXT\\\": \\\"13278\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13279\\\",\\n      \\\"TEXT\\\": \\\"13279\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13280\\\",\\n      \\\"TEXT\\\": \\\"13280\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13281\\\",\\n      \\\"TEXT\\\": \\\"13281\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13282\\\",\\n      \\\"TEXT\\\": \\\"13282\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13283\\\",\\n      \\\"TEXT\\\": \\\"13283\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13284\\\",\\n      \\\"TEXT\\\": \\\"13284\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13285\\\",\\n      \\\"TEXT\\\": \\\"13285\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13286\\\",\\n      \\\"TEXT\\\": \\\"13286\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13410\\\",\\n      \\\"TEXT\\\": \\\"13410\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13411\\\",\\n      \\\"TEXT\\\": \\\"13411\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13412\\\",\\n      \\\"TEXT\\\": \\\"13412\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13413\\\",\\n      \\\"TEXT\\\": \\\"13413\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13414\\\",\\n      \\\"TEXT\\\": \\\"13414\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13431\\\",\\n      \\\"TEXT\\\": \\\"13431\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13432\\\",\\n      \\\"TEXT\\\": \\\"13432\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13433\\\",\\n      \\\"TEXT\\\": \\\"13433\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13434\\\",\\n      \\\"TEXT\\\": \\\"13434\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13435\\\",\\n      \\\"TEXT\\\": \\\"13435\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13436\\\",\\n      \\\"TEXT\\\": \\\"13436\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13445\\\",\\n      \\\"TEXT\\\": \\\"13445\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13446\\\",\\n      \\\"TEXT\\\": \\\"13446\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13447\\\",\\n      \\\"TEXT\\\": \\\"13447\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13448\\\",\\n      \\\"TEXT\\\": \\\"13448\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13449\\\",\\n      \\\"TEXT\\\": \\\"13449\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13450\\\",\\n      \\\"TEXT\\\": \\\"13450\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13451\\\",\\n      \\\"TEXT\\\": \\\"13451\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13452\\\",\\n      \\\"TEXT\\\": \\\"13452\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13287\\\",\\n      \\\"TEXT\\\": \\\"13287\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13288\\\",\\n      \\\"TEXT\\\": \\\"13288\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://www.ebi.ac.uk/pdbe/entry/EMD-13289\\\",\\n      \\\"TEXT\\\": \\\"13289\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"https://doi.org/10.2210/pdb7OOC/pdb\\\",\\n      \\\"TEXT\\\": \\\"7OOC\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7OOD/pdb\\\",\\n      \\\"TEXT\\\": \\\"7OOD\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7P6Z/pdb\\\",\\n      \\\"TEXT\\\": \\\"7P6Z\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAH/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAH\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAI/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAI\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAJ/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAJ\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAK/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAK\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAL/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAL\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAM/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAM\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAN/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAN\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAO/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAO\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAQ/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAQ\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAR/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAR\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAS/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAS\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAT/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAT\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PAU/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PAU\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PH9/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PH9\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PHA/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PHA\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PHB/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PHB\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PHC/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PHC\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PI8/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PI8\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PI9/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PI9\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIA/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIA\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIB/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIB\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIC/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIC\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIO/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIO\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIP/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIP\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIQ/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIQ\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIR/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIR\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIS/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIS\\\"\\n    },\\n    {\\n      \\\"URL\\\": \\\"http://doi.org/10.2210/pdb7PIT/pdb\\\",\\n      \\\"TEXT\\\": \\\"7PIT\\\"\\n    }\\n  ]\\n}\\n```\",\n",
      "            \"additional_kwargs\": {},\n",
      "            \"tool_calls\": [],\n",
      "            \"invalid_tool_calls\": []\n",
      "          }\n",
      "        }\n",
      "      }\n",
      "    ]\n",
      "  ],\n",
      "  \"llm_output\": {\n",
      "    \"token_usage\": {\n",
      "      \"completion_tokens\": 3954,\n",
      "      \"prompt_tokens\": 92510,\n",
      "      \"total_tokens\": 96464\n",
      "    },\n",
      "    \"model_name\": \"gpt-4-1106-preview\",\n",
      "    \"system_fingerprint\": \"fp_d2051b8491\"\n",
      "  },\n",
      "  \"run\": null\n",
      "}\n",
      "\u001b[32;1m\u001b[1;3m[chain/start]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 4:parser:JsonEnclosedByTextOutputParser] Entering Parser run with input:\n",
      "\u001b[0m[inputs]\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence > 4:parser:JsonEnclosedByTextOutputParser] [2ms] Exiting Parser run with output:\n",
      "\u001b[0m{\n",
      "  \"TEXT\": [\n",
      "    \"Maps have been deposited in the EMDB under accession codes [13234](http://www.ebi.ac.uk/pdbe/entry/EMD-13234), [13272](http://www.ebi.ac.uk/pdbe/entry/EMD-13272), [13273](http://www.ebi.ac.uk/pdbe/entry/EMD-13273), [13274](http://www.ebi.ac.uk/pdbe/entry/EMD-13274), [13275](http://www.ebi.ac.uk/pdbe/entry/EMD-13275), [13276](http://www.ebi.ac.uk/pdbe/entry/EMD-13276), [13277](http://www.ebi.ac.uk/pdbe/entry/EMD-13277), [13278](http://www.ebi.ac.uk/pdbe/entry/EMD-13278), [13279](http://www.ebi.ac.uk/pdbe/entry/EMD-13279), [13280](http://www.ebi.ac.uk/pdbe/entry/EMD-13280), [13281](http://www.ebi.ac.uk/pdbe/entry/EMD-13281), [13282](http://www.ebi.ac.uk/pdbe/entry/EMD-13282), [13283](http://www.ebi.ac.uk/pdbe/entry/EMD-13283), [13284](http://www.ebi.ac.uk/pdbe/entry/EMD-13284), [13285](http://www.ebi.ac.uk/pdbe/entry/EMD-13285), [13286](http://www.ebi.ac.uk/pdbe/entry/EMD-13286), [13410](http://www.ebi.ac.uk/pdbe/entry/EMD-13410), [13411](http://www.ebi.ac.uk/pdbe/entry/EMD-13411), [13412](http://www.ebi.ac.uk/pdbe/entry/EMD-13412), [13413](http://www.ebi.ac.uk/pdbe/entry/EMD-13413), [13414](http://www.ebi.ac.uk/pdbe/entry/EMD-13414), [13431](http://www.ebi.ac.uk/pdbe/entry/EMD-13431), [13432](http://www.ebi.ac.uk/pdbe/entry/EMD-13432), [13433](http://www.ebi.ac.uk/pdbe/entry/EMD-13433), [13434](http://www.ebi.ac.uk/pdbe/entry/EMD-13434), [13435](http://www.ebi.ac.uk/pdbe/entry/EMD-13435), [13436](http://www.ebi.ac.uk/pdbe/entry/EMD-13436), [13445](http://www.ebi.ac.uk/pdbe/entry/EMD-13445), [13446](http://www.ebi.ac.uk/pdbe/entry/EMD-13446), [13447](http://www.ebi.ac.uk/pdbe/entry/EMD-13447), [13448](http://www.ebi.ac.uk/pdbe/entry/EMD-13448), [13449](http://www.ebi.ac.uk/pdbe/entry/EMD-13449), [13450](http://www.ebi.ac.uk/pdbe/entry/EMD-13450), [13451](http://www.ebi.ac.uk/pdbe/entry/EMD-13451), [13452](http://www.ebi.ac.uk/pdbe/entry/EMD-13452), [13287](http://www.ebi.ac.uk/pdbe/entry/EMD-13287), [13288](http://www.ebi.ac.uk/pdbe/entry/EMD-13288) and [13289](http://www.ebi.ac.uk/pdbe/entry/EMD-13289). Models have been deposited in the PDB under accession codes: [7OOC](https://doi.org/10.2210/pdb7OOC/pdb), [7OOD](http://doi.org/10.2210/pdb7OOD/pdb), [7P6Z](http://doi.org/10.2210/pdb7P6Z/pdb), [7PAH](http://doi.org/10.2210/pdb7PAH/pdb), [7PAI](http://doi.org/10.2210/pdb7PAI/pdb), [7PAJ](http://doi.org/10.2210/pdb7PAJ/pdb), [7PAK](http://doi.org/10.2210/pdb7PAK/pdb), [7PAL](http://doi.org/10.2210/pdb7PAL/pdb), [7PAM](http://doi.org/10.2210/pdb7PAM/pdb), [7PAN](http://doi.org/10.2210/pdb7PAN/pdb), [7PAO](http://doi.org/10.2210/pdb7PAO/pdb), [7PAQ](http://doi.org/10.2210/pdb7PAQ/pdb), [7PAR](http://doi.org/10.2210/pdb7PAR/pdb), [7PAS](http://doi.org/10.2210/pdb7PAS/pdb), [7PAT](http://doi.org/10.2210/pdb7PAT/pdb), [7PAU](http://doi.org/10.2210/pdb7PAU/pdb), [7PH9](http://doi.org/10.2210/pdb7PH9/pdb), [7PHA](http://doi.org/10.2210/pdb7PHA/pdb), [7PHB](http://doi.org/10.2210/pdb7PHB/pdb), [7PHC](http://doi.org/10.2210/pdb7PHC/pdb), [7PI8](http://doi.org/10.2210/pdb7PI8/pdb), [7PI9](http://doi.org/10.2210/pdb7PI9/pdb), [7PIA](http://doi.org/10.2210/pdb7PIA/pdb), [7PIB](http://doi.org/10.2210/pdb7PIB/pdb), [7PIC](http://doi.org/10.2210/pdb7PIC/pdb), [7PIO](http://doi.org/10.2210/pdb7PIO/pdb), [7PIP](http://doi.org/10.2210/pdb7PIP/pdb), [7PIQ](http://doi.org/10.2210/pdb7PIQ/pdb), [7PIR](http://doi.org/10.2210/pdb7PIR/pdb), [7PIS](http://doi.org/10.2210/pdb7PIS/pdb) and [7PIT](http://doi.org/10.2210/pdb7PIT/pdb).\"\n",
      "  ],\n",
      "  \"LINKS\": [\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13234\",\n",
      "      \"TEXT\": \"13234\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13272\",\n",
      "      \"TEXT\": \"13272\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13273\",\n",
      "      \"TEXT\": \"13273\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13274\",\n",
      "      \"TEXT\": \"13274\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13275\",\n",
      "      \"TEXT\": \"13275\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13276\",\n",
      "      \"TEXT\": \"13276\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13277\",\n",
      "      \"TEXT\": \"13277\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13278\",\n",
      "      \"TEXT\": \"13278\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13279\",\n",
      "      \"TEXT\": \"13279\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13280\",\n",
      "      \"TEXT\": \"13280\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13281\",\n",
      "      \"TEXT\": \"13281\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13282\",\n",
      "      \"TEXT\": \"13282\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13283\",\n",
      "      \"TEXT\": \"13283\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13284\",\n",
      "      \"TEXT\": \"13284\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13285\",\n",
      "      \"TEXT\": \"13285\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13286\",\n",
      "      \"TEXT\": \"13286\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13410\",\n",
      "      \"TEXT\": \"13410\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13411\",\n",
      "      \"TEXT\": \"13411\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13412\",\n",
      "      \"TEXT\": \"13412\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13413\",\n",
      "      \"TEXT\": \"13413\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13414\",\n",
      "      \"TEXT\": \"13414\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13431\",\n",
      "      \"TEXT\": \"13431\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13432\",\n",
      "      \"TEXT\": \"13432\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13433\",\n",
      "      \"TEXT\": \"13433\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13434\",\n",
      "      \"TEXT\": \"13434\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13435\",\n",
      "      \"TEXT\": \"13435\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13436\",\n",
      "      \"TEXT\": \"13436\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13445\",\n",
      "      \"TEXT\": \"13445\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13446\",\n",
      "      \"TEXT\": \"13446\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13447\",\n",
      "      \"TEXT\": \"13447\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13448\",\n",
      "      \"TEXT\": \"13448\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13449\",\n",
      "      \"TEXT\": \"13449\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13450\",\n",
      "      \"TEXT\": \"13450\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13451\",\n",
      "      \"TEXT\": \"13451\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13452\",\n",
      "      \"TEXT\": \"13452\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13287\",\n",
      "      \"TEXT\": \"13287\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13288\",\n",
      "      \"TEXT\": \"13288\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13289\",\n",
      "      \"TEXT\": \"13289\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"https://doi.org/10.2210/pdb7OOC/pdb\",\n",
      "      \"TEXT\": \"7OOC\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7OOD/pdb\",\n",
      "      \"TEXT\": \"7OOD\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7P6Z/pdb\",\n",
      "      \"TEXT\": \"7P6Z\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAH/pdb\",\n",
      "      \"TEXT\": \"7PAH\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAI/pdb\",\n",
      "      \"TEXT\": \"7PAI\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAJ/pdb\",\n",
      "      \"TEXT\": \"7PAJ\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAK/pdb\",\n",
      "      \"TEXT\": \"7PAK\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAL/pdb\",\n",
      "      \"TEXT\": \"7PAL\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAM/pdb\",\n",
      "      \"TEXT\": \"7PAM\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAN/pdb\",\n",
      "      \"TEXT\": \"7PAN\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAO/pdb\",\n",
      "      \"TEXT\": \"7PAO\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAQ/pdb\",\n",
      "      \"TEXT\": \"7PAQ\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAR/pdb\",\n",
      "      \"TEXT\": \"7PAR\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAS/pdb\",\n",
      "      \"TEXT\": \"7PAS\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAT/pdb\",\n",
      "      \"TEXT\": \"7PAT\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAU/pdb\",\n",
      "      \"TEXT\": \"7PAU\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PH9/pdb\",\n",
      "      \"TEXT\": \"7PH9\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PHA/pdb\",\n",
      "      \"TEXT\": \"7PHA\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PHB/pdb\",\n",
      "      \"TEXT\": \"7PHB\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PHC/pdb\",\n",
      "      \"TEXT\": \"7PHC\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PI8/pdb\",\n",
      "      \"TEXT\": \"7PI8\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PI9/pdb\",\n",
      "      \"TEXT\": \"7PI9\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIA/pdb\",\n",
      "      \"TEXT\": \"7PIA\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIB/pdb\",\n",
      "      \"TEXT\": \"7PIB\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIC/pdb\",\n",
      "      \"TEXT\": \"7PIC\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIO/pdb\",\n",
      "      \"TEXT\": \"7PIO\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIP/pdb\",\n",
      "      \"TEXT\": \"7PIP\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIQ/pdb\",\n",
      "      \"TEXT\": \"7PIQ\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIR/pdb\",\n",
      "      \"TEXT\": \"7PIR\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIS/pdb\",\n",
      "      \"TEXT\": \"7PIS\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIT/pdb\",\n",
      "      \"TEXT\": \"7PIT\"\n",
      "    }\n",
      "  ]\n",
      "}\n",
      "\u001b[36;1m\u001b[1;3m[chain/end]\u001b[0m \u001b[1m[1:chain:RunnableSequence] [323.12s] Exiting Chain run with output:\n",
      "\u001b[0m{\n",
      "  \"TEXT\": [\n",
      "    \"Maps have been deposited in the EMDB under accession codes [13234](http://www.ebi.ac.uk/pdbe/entry/EMD-13234), [13272](http://www.ebi.ac.uk/pdbe/entry/EMD-13272), [13273](http://www.ebi.ac.uk/pdbe/entry/EMD-13273), [13274](http://www.ebi.ac.uk/pdbe/entry/EMD-13274), [13275](http://www.ebi.ac.uk/pdbe/entry/EMD-13275), [13276](http://www.ebi.ac.uk/pdbe/entry/EMD-13276), [13277](http://www.ebi.ac.uk/pdbe/entry/EMD-13277), [13278](http://www.ebi.ac.uk/pdbe/entry/EMD-13278), [13279](http://www.ebi.ac.uk/pdbe/entry/EMD-13279), [13280](http://www.ebi.ac.uk/pdbe/entry/EMD-13280), [13281](http://www.ebi.ac.uk/pdbe/entry/EMD-13281), [13282](http://www.ebi.ac.uk/pdbe/entry/EMD-13282), [13283](http://www.ebi.ac.uk/pdbe/entry/EMD-13283), [13284](http://www.ebi.ac.uk/pdbe/entry/EMD-13284), [13285](http://www.ebi.ac.uk/pdbe/entry/EMD-13285), [13286](http://www.ebi.ac.uk/pdbe/entry/EMD-13286), [13410](http://www.ebi.ac.uk/pdbe/entry/EMD-13410), [13411](http://www.ebi.ac.uk/pdbe/entry/EMD-13411), [13412](http://www.ebi.ac.uk/pdbe/entry/EMD-13412), [13413](http://www.ebi.ac.uk/pdbe/entry/EMD-13413), [13414](http://www.ebi.ac.uk/pdbe/entry/EMD-13414), [13431](http://www.ebi.ac.uk/pdbe/entry/EMD-13431), [13432](http://www.ebi.ac.uk/pdbe/entry/EMD-13432), [13433](http://www.ebi.ac.uk/pdbe/entry/EMD-13433), [13434](http://www.ebi.ac.uk/pdbe/entry/EMD-13434), [13435](http://www.ebi.ac.uk/pdbe/entry/EMD-13435), [13436](http://www.ebi.ac.uk/pdbe/entry/EMD-13436), [13445](http://www.ebi.ac.uk/pdbe/entry/EMD-13445), [13446](http://www.ebi.ac.uk/pdbe/entry/EMD-13446), [13447](http://www.ebi.ac.uk/pdbe/entry/EMD-13447), [13448](http://www.ebi.ac.uk/pdbe/entry/EMD-13448), [13449](http://www.ebi.ac.uk/pdbe/entry/EMD-13449), [13450](http://www.ebi.ac.uk/pdbe/entry/EMD-13450), [13451](http://www.ebi.ac.uk/pdbe/entry/EMD-13451), [13452](http://www.ebi.ac.uk/pdbe/entry/EMD-13452), [13287](http://www.ebi.ac.uk/pdbe/entry/EMD-13287), [13288](http://www.ebi.ac.uk/pdbe/entry/EMD-13288) and [13289](http://www.ebi.ac.uk/pdbe/entry/EMD-13289). Models have been deposited in the PDB under accession codes: [7OOC](https://doi.org/10.2210/pdb7OOC/pdb), [7OOD](http://doi.org/10.2210/pdb7OOD/pdb), [7P6Z](http://doi.org/10.2210/pdb7P6Z/pdb), [7PAH](http://doi.org/10.2210/pdb7PAH/pdb), [7PAI](http://doi.org/10.2210/pdb7PAI/pdb), [7PAJ](http://doi.org/10.2210/pdb7PAJ/pdb), [7PAK](http://doi.org/10.2210/pdb7PAK/pdb), [7PAL](http://doi.org/10.2210/pdb7PAL/pdb), [7PAM](http://doi.org/10.2210/pdb7PAM/pdb), [7PAN](http://doi.org/10.2210/pdb7PAN/pdb), [7PAO](http://doi.org/10.2210/pdb7PAO/pdb), [7PAQ](http://doi.org/10.2210/pdb7PAQ/pdb), [7PAR](http://doi.org/10.2210/pdb7PAR/pdb), [7PAS](http://doi.org/10.2210/pdb7PAS/pdb), [7PAT](http://doi.org/10.2210/pdb7PAT/pdb), [7PAU](http://doi.org/10.2210/pdb7PAU/pdb), [7PH9](http://doi.org/10.2210/pdb7PH9/pdb), [7PHA](http://doi.org/10.2210/pdb7PHA/pdb), [7PHB](http://doi.org/10.2210/pdb7PHB/pdb), [7PHC](http://doi.org/10.2210/pdb7PHC/pdb), [7PI8](http://doi.org/10.2210/pdb7PI8/pdb), [7PI9](http://doi.org/10.2210/pdb7PI9/pdb), [7PIA](http://doi.org/10.2210/pdb7PIA/pdb), [7PIB](http://doi.org/10.2210/pdb7PIB/pdb), [7PIC](http://doi.org/10.2210/pdb7PIC/pdb), [7PIO](http://doi.org/10.2210/pdb7PIO/pdb), [7PIP](http://doi.org/10.2210/pdb7PIP/pdb), [7PIQ](http://doi.org/10.2210/pdb7PIQ/pdb), [7PIR](http://doi.org/10.2210/pdb7PIR/pdb), [7PIS](http://doi.org/10.2210/pdb7PIS/pdb) and [7PIT](http://doi.org/10.2210/pdb7PIT/pdb).\"\n",
      "  ],\n",
      "  \"LINKS\": [\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13234\",\n",
      "      \"TEXT\": \"13234\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13272\",\n",
      "      \"TEXT\": \"13272\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13273\",\n",
      "      \"TEXT\": \"13273\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13274\",\n",
      "      \"TEXT\": \"13274\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13275\",\n",
      "      \"TEXT\": \"13275\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13276\",\n",
      "      \"TEXT\": \"13276\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13277\",\n",
      "      \"TEXT\": \"13277\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13278\",\n",
      "      \"TEXT\": \"13278\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13279\",\n",
      "      \"TEXT\": \"13279\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13280\",\n",
      "      \"TEXT\": \"13280\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13281\",\n",
      "      \"TEXT\": \"13281\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13282\",\n",
      "      \"TEXT\": \"13282\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13283\",\n",
      "      \"TEXT\": \"13283\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13284\",\n",
      "      \"TEXT\": \"13284\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13285\",\n",
      "      \"TEXT\": \"13285\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13286\",\n",
      "      \"TEXT\": \"13286\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13410\",\n",
      "      \"TEXT\": \"13410\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13411\",\n",
      "      \"TEXT\": \"13411\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13412\",\n",
      "      \"TEXT\": \"13412\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13413\",\n",
      "      \"TEXT\": \"13413\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13414\",\n",
      "      \"TEXT\": \"13414\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13431\",\n",
      "      \"TEXT\": \"13431\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13432\",\n",
      "      \"TEXT\": \"13432\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13433\",\n",
      "      \"TEXT\": \"13433\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13434\",\n",
      "      \"TEXT\": \"13434\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13435\",\n",
      "      \"TEXT\": \"13435\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13436\",\n",
      "      \"TEXT\": \"13436\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13445\",\n",
      "      \"TEXT\": \"13445\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13446\",\n",
      "      \"TEXT\": \"13446\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13447\",\n",
      "      \"TEXT\": \"13447\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13448\",\n",
      "      \"TEXT\": \"13448\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13449\",\n",
      "      \"TEXT\": \"13449\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13450\",\n",
      "      \"TEXT\": \"13450\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13451\",\n",
      "      \"TEXT\": \"13451\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13452\",\n",
      "      \"TEXT\": \"13452\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13287\",\n",
      "      \"TEXT\": \"13287\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13288\",\n",
      "      \"TEXT\": \"13288\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://www.ebi.ac.uk/pdbe/entry/EMD-13289\",\n",
      "      \"TEXT\": \"13289\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"https://doi.org/10.2210/pdb7OOC/pdb\",\n",
      "      \"TEXT\": \"7OOC\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7OOD/pdb\",\n",
      "      \"TEXT\": \"7OOD\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7P6Z/pdb\",\n",
      "      \"TEXT\": \"7P6Z\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAH/pdb\",\n",
      "      \"TEXT\": \"7PAH\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAI/pdb\",\n",
      "      \"TEXT\": \"7PAI\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAJ/pdb\",\n",
      "      \"TEXT\": \"7PAJ\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAK/pdb\",\n",
      "      \"TEXT\": \"7PAK\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAL/pdb\",\n",
      "      \"TEXT\": \"7PAL\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAM/pdb\",\n",
      "      \"TEXT\": \"7PAM\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAN/pdb\",\n",
      "      \"TEXT\": \"7PAN\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAO/pdb\",\n",
      "      \"TEXT\": \"7PAO\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAQ/pdb\",\n",
      "      \"TEXT\": \"7PAQ\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAR/pdb\",\n",
      "      \"TEXT\": \"7PAR\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAS/pdb\",\n",
      "      \"TEXT\": \"7PAS\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAT/pdb\",\n",
      "      \"TEXT\": \"7PAT\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PAU/pdb\",\n",
      "      \"TEXT\": \"7PAU\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PH9/pdb\",\n",
      "      \"TEXT\": \"7PH9\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PHA/pdb\",\n",
      "      \"TEXT\": \"7PHA\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PHB/pdb\",\n",
      "      \"TEXT\": \"7PHB\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PHC/pdb\",\n",
      "      \"TEXT\": \"7PHC\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PI8/pdb\",\n",
      "      \"TEXT\": \"7PI8\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PI9/pdb\",\n",
      "      \"TEXT\": \"7PI9\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIA/pdb\",\n",
      "      \"TEXT\": \"7PIA\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIB/pdb\",\n",
      "      \"TEXT\": \"7PIB\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIC/pdb\",\n",
      "      \"TEXT\": \"7PIC\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIO/pdb\",\n",
      "      \"TEXT\": \"7PIO\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIP/pdb\",\n",
      "      \"TEXT\": \"7PIP\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIQ/pdb\",\n",
      "      \"TEXT\": \"7PIQ\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIR/pdb\",\n",
      "      \"TEXT\": \"7PIR\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIS/pdb\",\n",
      "      \"TEXT\": \"7PIS\"\n",
      "    },\n",
      "    {\n",
      "      \"URL\": \"http://doi.org/10.2210/pdb7PIT/pdb\",\n",
      "      \"TEXT\": \"7PIT\"\n",
      "    }\n",
      "  ]\n",
      "}\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[11], line 75\u001b[0m\n\u001b[1;32m     72\u001b[0m     \u001b[38;5;28;01mfinally\u001b[39;00m:\n\u001b[1;32m     73\u001b[0m         browser\u001b[38;5;241m.\u001b[39mquit()\n\u001b[0;32m---> 75\u001b[0m \u001b[43mread_about_available_data_from_doidotorg\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43m10.1038/s41586-022-05255-2\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mllm\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mheadless\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m)\u001b[49m\n",
      "Cell \u001b[0;32mIn[11], line 68\u001b[0m, in \u001b[0;36mread_about_available_data_from_doidotorg\u001b[0;34m(doi, llm, headless)\u001b[0m\n\u001b[1;32m     65\u001b[0m     s1 \u001b[38;5;241m=\u001b[39m {\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mmarkdown_code\u001b[39m\u001b[38;5;124m'\u001b[39m: md}\n\u001b[1;32m     67\u001b[0m     output \u001b[38;5;241m=\u001b[39m lcel\u001b[38;5;241m.\u001b[39minvoke(s1, config\u001b[38;5;241m=\u001b[39m{\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mcallbacks\u001b[39m\u001b[38;5;124m'\u001b[39m: [ConsoleCallbackHandler()]})\n\u001b[0;32m---> 68\u001b[0m     \u001b[38;5;28;43mprint\u001b[39;49m(output)\n\u001b[1;32m     70\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m     71\u001b[0m     \u001b[38;5;28mprint\u001b[39m(e)\n",
      "Cell \u001b[0;32mIn[11], line 68\u001b[0m, in \u001b[0;36mread_about_available_data_from_doidotorg\u001b[0;34m(doi, llm, headless)\u001b[0m\n\u001b[1;32m     65\u001b[0m     s1 \u001b[38;5;241m=\u001b[39m {\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mmarkdown_code\u001b[39m\u001b[38;5;124m'\u001b[39m: md}\n\u001b[1;32m     67\u001b[0m     output \u001b[38;5;241m=\u001b[39m lcel\u001b[38;5;241m.\u001b[39minvoke(s1, config\u001b[38;5;241m=\u001b[39m{\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mcallbacks\u001b[39m\u001b[38;5;124m'\u001b[39m: [ConsoleCallbackHandler()]})\n\u001b[0;32m---> 68\u001b[0m     \u001b[38;5;28;43mprint\u001b[39;49m(output)\n\u001b[1;32m     70\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m     71\u001b[0m     \u001b[38;5;28mprint\u001b[39m(e)\n",
      "File \u001b[0;32m_pydevd_bundle/pydevd_cython.pyx:1457\u001b[0m, in \u001b[0;36m_pydevd_bundle.pydevd_cython.SafeCallWrapper.__call__\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m_pydevd_bundle/pydevd_cython.pyx:701\u001b[0m, in \u001b[0;36m_pydevd_bundle.pydevd_cython.PyDBFrame.trace_dispatch\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m_pydevd_bundle/pydevd_cython.pyx:1152\u001b[0m, in \u001b[0;36m_pydevd_bundle.pydevd_cython.PyDBFrame.trace_dispatch\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m_pydevd_bundle/pydevd_cython.pyx:1135\u001b[0m, in \u001b[0;36m_pydevd_bundle.pydevd_cython.PyDBFrame.trace_dispatch\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m_pydevd_bundle/pydevd_cython.pyx:312\u001b[0m, in \u001b[0;36m_pydevd_bundle.pydevd_cython.PyDBFrame.do_wait_suspend\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m~/miniconda3/envs/alhazen/lib/python3.11/site-packages/debugpy/_vendored/pydevd/pydevd.py:2070\u001b[0m, in \u001b[0;36mPyDB.do_wait_suspend\u001b[0;34m(self, thread, frame, event, arg, exception_type)\u001b[0m\n\u001b[1;32m   2067\u001b[0m             from_this_thread\u001b[38;5;241m.\u001b[39mappend(frame_custom_thread_id)\n\u001b[1;32m   2069\u001b[0m     \u001b[38;5;28;01mwith\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_threads_suspended_single_notification\u001b[38;5;241m.\u001b[39mnotify_thread_suspended(thread_id, thread, stop_reason):\n\u001b[0;32m-> 2070\u001b[0m         keep_suspended \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_do_wait_suspend\u001b[49m\u001b[43m(\u001b[49m\u001b[43mthread\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mframe\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mevent\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43marg\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43msuspend_type\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mfrom_this_thread\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mframes_tracker\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   2072\u001b[0m frames_list \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mNone\u001b[39;00m\n\u001b[1;32m   2074\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m keep_suspended:\n\u001b[1;32m   2075\u001b[0m     \u001b[38;5;66;03m# This means that we should pause again after a set next statement.\u001b[39;00m\n",
      "File \u001b[0;32m~/miniconda3/envs/alhazen/lib/python3.11/site-packages/debugpy/_vendored/pydevd/pydevd.py:2106\u001b[0m, in \u001b[0;36mPyDB._do_wait_suspend\u001b[0;34m(self, thread, frame, event, arg, suspend_type, from_this_thread, frames_tracker)\u001b[0m\n\u001b[1;32m   2103\u001b[0m         \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_call_input_hook()\n\u001b[1;32m   2105\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mprocess_internal_commands()\n\u001b[0;32m-> 2106\u001b[0m     time\u001b[38;5;241m.\u001b[39msleep(\u001b[38;5;241m0.01\u001b[39m)\n\u001b[1;32m   2108\u001b[0m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mcancel_async_evaluation(get_current_thread_id(thread), \u001b[38;5;28mstr\u001b[39m(\u001b[38;5;28mid\u001b[39m(frame)))\n\u001b[1;32m   2110\u001b[0m \u001b[38;5;66;03m# process any stepping instructions\u001b[39;00m\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "from selenium import webdriver\n",
    "from selenium.webdriver.chrome.options import Options\n",
    "from splinter import Browser\n",
    "from selenium.webdriver.chrome.service import Service\n",
    "from langchain.prompts import ChatPromptTemplate, PromptTemplate\n",
    "from alhazen.utils.output_parsers import JsonEnclosedByTextOutputParser\n",
    "from random import randint\n",
    "from bs4 import BeautifulSoup\n",
    "from html import unescape\n",
    "import markdownify\n",
    "\n",
    "def simplify_html_to_markdown(html_text):\n",
    "    soup = BeautifulSoup(html_text, 'html.parser')\n",
    "\n",
    "    # Remove script and style tags\n",
    "    for script in soup([\"script\", \"style\"]):\n",
    "        script.extract()\n",
    "\n",
    "    # Get the text content and convert HTML tags to Markdown\n",
    "    text = soup.get_text()\n",
    "    #text = re.sub(r'<h([1-6])>(.*?)</h\\1>', r'#\\1 \\2', text)\n",
    "    #text = re.sub(r'<b>(.*?)</b>', r'**\\1**', text)\n",
    "    #text = re.sub(r'<i>(.*?)</i>', r'*\\1*', text)\n",
    "    #text = re.sub(r'<ul>(.*?)</ul>', lambda m: '\\n'.join(['- ' + line.strip() for line in re.split(r'</?li>', m.group(1)) if line.strip()]), text, flags=re.DOTALL)\n",
    "    #text = re.sub(r'<ol>(.*?)</ol>', lambda m: '\\n'.join([str(i+1) + '. ' + line.strip() for i, line in enumerate(re.split(r'</?li>', m.group(1))) if line.strip()]), text, flags=re.DOTALL)\n",
    "    #text = re.sub(r'<blockquote>(.*?)</blockquote>', r'> \\1', text, flags=re.DOTALL)\n",
    "    #text = re.sub(r'<hr\\s?/?>', r'\\n---\\n', text)\n",
    "    #text = re.sub(r'<br\\s?/?>', r'\\n', text)\n",
    "    #text = re.sub(r'<p>(.*?)</p>', r'\\1\\n\\n', text)\n",
    "\n",
    "    # Convert HTML entities to their corresponding characters\n",
    "    text = unescape(text)\n",
    "    text = re.sub(r'\\(', '\\\\(', text)\n",
    "    text = re.sub(r'\\)', '\\\\)', text)\n",
    "\n",
    "    # Convert links\n",
    "    for link in soup.find_all('a'):\n",
    "        href = link.get('href')\n",
    "        text = text.replace(str(link), f'[{link.text}]({href})')\n",
    "\n",
    "    return text.strip()\n",
    "\n",
    "def read_about_available_data_from_doidotorg(doi, llm, headless=False):\n",
    "\n",
    "    if headless:\n",
    "        browser = Browser(headless=True)\n",
    "    else:\n",
    "        browser = Browser()\n",
    "    doi = doi.replace('https://doi.org/', '').replace('doi', '')\n",
    "    print ('https://doi.org/'+doi)\n",
    "    stem_list = doi.split('/')\n",
    "    stem = '/'.join(stem_list[0:len(stem_list)-1])\n",
    "    hrefs = set()\n",
    "    pt = ChatPromptTemplate.from_messages([\n",
    "        (\"system\", system_prompt),\n",
    "        (\"human\", prompt)])\n",
    "    lcel = pt | llm | JsonEnclosedByTextOutputParser()\n",
    "\n",
    "    try:\n",
    "        # visit doi.org page\n",
    "        browser.visit('https://doi.org/'+doi)\n",
    "\n",
    "        html = browser.html\n",
    "        md = markdownify.markdownify(html)\n",
    "        s1 = {'markdown_code': md}\n",
    "        \n",
    "        output = lcel.invoke(s1, config={'callbacks': [ConsoleCallbackHandler()]})\n",
    "        print(output)\n",
    "            \n",
    "    except Exception as e:\n",
    "        print(e)\n",
    "    finally:\n",
    "        browser.quit()\n",
    "\n",
    "read_about_available_data_from_doidotorg('10.1038/s41586-022-05255-2', llm, headless=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "self = ldb\n",
    "collection_id = '2'\n",
    "\n",
    "q1 = self.session.query(SKE, SKI) \\\n",
    "        .filter(SKC_HM.ScientificKnowledgeCollection_id == collection_id) \\\n",
    "        .filter(SKC_HM.has_members_id == SKE.id) \\\n",
    "        .filter(SKE.id == SKE_HR.ScientificKnowledgeExpression_id) \\\n",
    "        .filter(SKE_HR.has_representation_id == SKI.id) \\\n",
    "        .filter(SKI.id == SKI_HP.ScientificKnowledgeItem_id) \\\n",
    "        .filter(SKI_HP.has_part_id == SKF.id) \\\n",
    "        .filter(SKI.type == 'CitationRecord') \\\n",
    "        .filter(or_(SKE.type == 'ScientificPrimaryResearchArticle', SKE.type == 'ScientificPrimaryResearchPreprint')) \n",
    "\n",
    "for ske, ski in tqdm(q1.all()):\n",
    "    b = ''\n",
    "    om = ''\n",
    "    rc = ''  \n",
    "\n",
    "    fragments = []\n",
    "    for f in ski.has_part:\n",
    "      if f.type in ['title', 'abstract']:\n",
    "        fragments.append(f)\n",
    "\n",
    "    # USE AN LLM HERE INSTEAD OF A DEEP LEARNING CLASSIFER\n",
    "\n",
    "\n",
    "    for skf in sorted(fragments, key=lambda f: f.offset):\n",
    "        for s in self.sent_detector.tokenize(skf.content):\n",
    "            m = classifier(skf.content)\n",
    "            l = lookup.get(m[0].get('label'))\n",
    "            if l == 'BACKGROUND':\n",
    "                if len(b) > 0:\n",
    "                    b += '\\n'\n",
    "                b += s\n",
    "            elif l == 'OBJECTIVE' or l == 'METHODS':\n",
    "                if len(om) > 0:\n",
    "                    om += '\\n'\n",
    "                om += s\n",
    "            else: \n",
    "                if len(rc) > 0:\n",
    "                    rc += '\\n'\n",
    "                rc += s\n",
    "    skf_stem = ske.id+'.'+ski.type+'.'\n",
    "    if len(b) > 0:\n",
    "        f_b = ScientificKnowledgeFragment(id=str(uuid.uuid4().hex)[:10], \n",
    "                type='background_sentences', offset=-1, length=len(b),\n",
    "                name=skf_stem+'background', content=b)\n",
    "        self.session.add(f_b)\n",
    "        ski.has_part.append(f_b)\n",
    "        f_b.part_of = ski.id    \n",
    "    if len(om) > 0:\n",
    "        f_om = ScientificKnowledgeFragment(id=str(uuid.uuid4().hex)[:10], \n",
    "                type='objective_methods_sentences', offset=-1, length=len(om),\n",
    "                name=skf_stem+'objective_methods', content=om)\n",
    "        self.session.add(f_om)\n",
    "        ski.has_part.append(f_om)\n",
    "        f_om.part_of = ski.id\n",
    "    if len(rc) > 0:\n",
    "        f_rc = ScientificKnowledgeFragment(id=str(uuid.uuid4().hex)[:10], \n",
    "                type='results_conclusions_sentences', offset=-1, length=len(rc),\n",
    "                name=skf_stem+'results_conclusions', content=rc)\n",
    "        self.session.add(f_rc)\n",
    "        ski.has_part.append(f_rc)\n",
    "        f_rc.part_of = ski.id\n",
    "    self.session.flush()\n",
    "self.session.commit()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "self = ldb\n",
    "collection_id = '2'\n",
    "#self.session.rollback()\n",
    "q2 = self.session.query(SKF) \\\n",
    "        .filter(SKC_HM.ScientificKnowledgeCollection_id == collection_id) \\\n",
    "        .filter(SKC_HM.has_members_id == SKE.id) \\\n",
    "        .filter(SKE.id == SKE_HR.ScientificKnowledgeExpression_id) \\\n",
    "        .filter(SKE_HR.has_representation_id == SKI.id) \\\n",
    "        .filter(SKI.id == SKI_HP.ScientificKnowledgeItem_id) \\\n",
    "        .filter(SKI_HP.has_part_id == SKF.id) \\\n",
    "        .filter(SKI.type == 'CitationRecord') \\\n",
    "        .filter(or_(SKF.type == 'results_conclusions_sentences', \\\n",
    "                SKF.type == 'objective_methods_sentences', \\\n",
    "                SKF.type == 'background_sentences'))\n",
    "for skf in tqdm(q2.all()):\n",
    "    self.delete_fragment(skf.id)\n",
    "   \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "self = ldb\n",
    "collection_id = '2'\n",
    "#self.session.rollback()\n",
    "q2 = self.session.query(SKE, SKF) \\\n",
    "        .filter(SKC_HM.ScientificKnowledgeCollection_id == collection_id) \\\n",
    "        .filter(SKC_HM.has_members_id == SKE.id) \\\n",
    "        .filter(SKE.id == SKE_HR.ScientificKnowledgeExpression_id) \\\n",
    "        .filter(SKE_HR.has_representation_id == SKI.id) \\\n",
    "        .filter(SKI.id == SKI_HP.ScientificKnowledgeItem_id) \\\n",
    "        .filter(SKI_HP.has_part_id == SKF.id) \\\n",
    "        .filter(SKI.type == 'CitationRecord') \\\n",
    "        .filter(SKF.type == 'objective_methods_sentences') \\\n",
    "        .order_by(desc(SKE.publication_date)) \\\n",
    "        .order_by(SKF.name)\n",
    "\n",
    "for ske, skf in tqdm(q2.all()):\n",
    "    print(skf)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Get full text copies of all the papers about CryoET\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cb.agent_executor.invoke({'input':'Get full text copies of all papers in the collection with id=\"2\".'})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ldb.create_new_collection_from_sample('5', 'EMPIAR CryoET Papers Tests', '4', 20, ['ScientificPrimaryResearchArticle', 'ScientificPrimaryResearchPreprint'])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Analyze Collections"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "q = ldb.session.query(SKC.id, SKC.name, SKE.id, SKI.type) \\\n",
    "        .filter(SKC.id==SKC_HM.ScientificKnowledgeCollection_id) \\\n",
    "        .filter(SKC_HM.has_members_id==SKE.id) \\\n",
    "        .filter(SKE.id==SKE_HR.ScientificKnowledgeExpression_id) \\\n",
    "        .filter(SKE_HR.has_representation_id==SKI.id) \n",
    "df = pd.DataFrame(q.all(), columns=['id', 'collection name', 'doi', 'item type'])\n",
    "df.pivot_table(index=['id', 'collection name'], columns='item type', values='doi', aggfunc=lambda x: len(x.unique())).fillna(0)   "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Survey + Run Classifications over Papers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# USE WITH CAUTION - this will delete all extracted metadata notes in the database\n",
    "# clear all notes across papers listed in `dois` list\n",
    "l = []\n",
    "q = ldb.session.query(N, SKE) \\\n",
    "        .filter(N.id == NIA.Note_id) \\\n",
    "        .filter(NIA.is_about_id == SKE.id) \\\n",
    "        .filter(N.type == 'TiAbClassificationNote__cryoet_study_types') \\\n",
    "\n",
    "output = []        \n",
    "print(len(q.all()))\n",
    "for n, ske in q.all():\n",
    "    ldb.delete_note(n.id)    \n",
    "print(len(q.all()))\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t = [t for t in cb.tk.get_tools() if isinstance(t, TitleAbstractClassifier_OneDocAtATime_Tool)][0]\n",
    "t.run({'collection_id': '5', 'classification_type':'cryoet_study_types', 'repeat_run':True})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t = [t for t in cb.tk.get_tools() if isinstance(t, TitleAbstractClassifier_OneDocAtATime_Tool)][0]\n",
    "t.run({'collection_id': '2', 'classification_type':'cryoet_study_types'})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "l = []\n",
    "ldb.session.rollback()\n",
    "q = ldb.session.query(N, SKE) \\\n",
    "        .join(NIA, NIA.Note_id == N.id) \\\n",
    "        .join(SKE, SKE.id == NIA.is_about_id) \\\n",
    "        .join(SKC_HM, SKE.id == SKC_HM.has_members_id) \\\n",
    "        .filter(N.type == 'TiAbClassificationNote__cryoet_study_types') \\\n",
    "        .filter(SKC_HM.ScientificKnowledgeCollection_id == '5') \\\n",
    "        .order_by(SKE.id, N.provenance)\n",
    "\n",
    "output = []        \n",
    "for n, ske in q.all():\n",
    "        tup = json.loads(n.content)\n",
    "        tup['doi'] = 'http://doi.org/'+re.sub('doi:', '', ske.id)\n",
    "        tup['year'] = ske.publication_date.year\n",
    "        tup['month'] = ske.publication_date.month\n",
    "        tup['ref'] = ske.content\n",
    "        output.append(tup)\n",
    "df = pd.DataFrame(output).sort_values(['year', 'month'], ascending=[False, False])\n",
    "df.to_csv(loc+'/'+db_name+'/cryoet_study_types.tsv', sep='\\t')\n",
    "df\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "study_type_lookup = {'A': 'Viral Pathogens', \n",
    "                     'B': \"Mutated protein structure\", \n",
    "                     'C': 'Bacterial pathogens', \n",
    "                     'D': 'Plant cells', \n",
    "                     'E': 'Material science', \n",
    "                     'F': 'Intracellular Transport Structure', \n",
    "                     'G': 'Synapses or Vesicle Release', \n",
    "                     'H': 'Other Intracellular Structure', \n",
    "                     'I': 'Cellular Processes',\n",
    "                     'J': 'Dynamics of molecular interactions',    \n",
    "                     'K': 'New CryoET imaging methods', \n",
    "                     'L': 'New data analysis methods'}\n",
    "\n",
    "addEMPCCollection_tool = [t for t in cb.tk.get_tools() if isinstance(t, AddCollectionFromEPMCTool)][0]\n",
    "step = 20\n",
    "\n",
    "for k in study_type_lookup.keys():\n",
    "    df1 = df[df['cryoet_study_type_code'] == k]\n",
    "    dois_to_add = [re.sub('http://doi.org/', 'doi:', r.doi) for i, r in df1.iterrows()]\n",
    "\n",
    "    c_id = '2.'+k\n",
    "    c_name = 'CryoET - ' + study_type_lookup[k]\n",
    "\n",
    "    corpus = None\n",
    "    all_existing_query = ldb.session.query(SKC).filter(SKC.id==c_id)\n",
    "    for c in all_existing_query.all():\n",
    "      corpus = c\n",
    "    if corpus is None:      \n",
    "      corpus = ScientificKnowledgeCollection(id=c_id,\n",
    "                                           type='skem:ScientificKnowledgeCollection',\n",
    "                                           name=c_name,\n",
    "                                           has_members=[])\n",
    "    ldb.session.add(corpus)\n",
    "    ldb.session.flush()\n",
    "\n",
    "    for doi in tqdm(dois_to_add):\n",
    "        p = ldb.session.query(SKE) \\\n",
    "            .filter(SKE.id==doi).first()\n",
    "        if p is None:\n",
    "          continue\n",
    "        ldb.session.add(p)\n",
    "        corpus.has_members.append(p)\n",
    "        p.member_of.append(corpus)\n",
    "        ldb.session.flush()\n",
    "ldb.session.commit()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "delete_collection_tool = [t for t in cb.tk.get_tools() if isinstance(t, DeleteCollectionTool)][0]\n",
    " \n",
    "for k in study_type_lookup.keys():\n",
    "    print(k)\n",
    "    delete_collection_tool.run({'collection_id': '2.'+k})\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Survey + Run Extractions over Papers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t = [t for t in cb.tk.get_tools() if isinstance(t, TitleAbstractExtraction_OneDocAtATime_Tool)][0]\n",
    "t.run({'collection_id': '5', 'extraction_type':'cryoet'})"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Tests + Checks \n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Agent tool selection + execution + interpretation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cb.agent_executor.invoke({'input':'Hi who are you and what can you do?'})\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Run MetaData Extraction Chain over listed papers\n",
    "\n",
    "Here, we run various versions of the metadata extraction tool to examine performance over the cryoet dataset. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['doi:10.1002/1873-3468.13916',\n",
       " 'doi:10.1007/s10974-017-9477-5',\n",
       " 'doi:10.1016/j.celrep.2023.112107',\n",
       " 'doi:10.1016/j.str.2020.07.018',\n",
       " 'doi:10.1038/s41467-020-18777-y',\n",
       " 'doi:10.1038/s41467-022-29322-4',\n",
       " 'doi:10.1038/s41477-020-00811-y',\n",
       " 'doi:10.1038/s41586-023-05904-0',\n",
       " 'doi:10.1038/s41592-019-0630-5',\n",
       " 'doi:10.1038/s41594-018-0027-7',\n",
       " 'doi:10.1073/pnas.2209823119',\n",
       " 'doi:10.1101/2020.06.26.173476',\n",
       " 'doi:10.1101/2022.03.08.482579',\n",
       " 'doi:10.1101/2022.04.07.487557',\n",
       " 'doi:10.1101/2023.08.18.553799',\n",
       " 'doi:10.1101/230276',\n",
       " 'doi:10.1126/sciadv.ade2727',\n",
       " 'doi:10.1126/science.abd5223',\n",
       " 'doi:10.1371/journal.pone.0266035',\n",
       " 'doi:10.1371/journal.ppat.1008883']"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q = ldb.session.query(SKE.id) \\\n",
    "        .filter(SKC.id==SKC_HM.ScientificKnowledgeCollection_id) \\\n",
    "        .filter(SKC_HM.has_members_id==SKE.id) \\\n",
    "        .filter(SKC.id=='5')  \n",
    "dois = [e.id for e in q.all()]\n",
    "dois\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11\n"
     ]
    }
   ],
   "source": [
    "# need to count tokens submitted to the server as a way of tracking usage. \n",
    "\n",
    "import transformers\n",
    "import torch\n",
    "\n",
    "from transformers import AutoTokenizer, AutoModelForCausalLM\n",
    "tokenizer = AutoTokenizer.from_pretrained(\"meta-llama/Meta-Llama-3-8B-Instruct\", device='mps', token=os.environ['HF_API_KEY'])\n",
    "prompt = \"The methods section of the paper is as follows:\"\n",
    "tokenized = tokenizer(prompt, return_tensors=\"pt\")\n",
    "print(len(tokenized[\"input_ids\"][0]))\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 2284/2284 [00:50<00:00, 45.39it/s]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>doi</th>\n",
       "      <th>doc_length</th>\n",
       "      <th>method_length</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>doi:10.1101/2022.04.12.488077</td>\n",
       "      <td>27084</td>\n",
       "      <td>6420</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>doi:10.1083/jcb.202204093</td>\n",
       "      <td>17912</td>\n",
       "      <td>3472</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>doi:10.1038/s41594-022-00861-0</td>\n",
       "      <td>16528</td>\n",
       "      <td>3378</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>doi:10.1038/s41586-022-05255-2</td>\n",
       "      <td>16484</td>\n",
       "      <td>6224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>doi:10.1101/2023.04.28.538734</td>\n",
       "      <td>13304</td>\n",
       "      <td>5424</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1581</th>\n",
       "      <td>doi:10.1038/ncomms5131</td>\n",
       "      <td>12159</td>\n",
       "      <td>4646</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1582</th>\n",
       "      <td>doi:10.1038/ncomms13916</td>\n",
       "      <td>12686</td>\n",
       "      <td>7662</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1583</th>\n",
       "      <td>doi:10.1074/jbc.m702025200</td>\n",
       "      <td>5892</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1584</th>\n",
       "      <td>doi:10.1038/nature04719</td>\n",
       "      <td>2633</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1585</th>\n",
       "      <td>doi:10.1083/jcb.200908045</td>\n",
       "      <td>17093</td>\n",
       "      <td>5108</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1586 rows × 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                 doi  doc_length  method_length\n",
       "0      doi:10.1101/2022.04.12.488077       27084           6420\n",
       "1          doi:10.1083/jcb.202204093       17912           3472\n",
       "2     doi:10.1038/s41594-022-00861-0       16528           3378\n",
       "3     doi:10.1038/s41586-022-05255-2       16484           6224\n",
       "4      doi:10.1101/2023.04.28.538734       13304           5424\n",
       "...                              ...         ...            ...\n",
       "1581          doi:10.1038/ncomms5131       12159           4646\n",
       "1582         doi:10.1038/ncomms13916       12686           7662\n",
       "1583      doi:10.1074/jbc.m702025200        5892              1\n",
       "1584         doi:10.1038/nature04719        2633              1\n",
       "1585       doi:10.1083/jcb.200908045       17093           5108\n",
       "\n",
       "[1586 rows x 3 columns]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import tiktoken\n",
    "\n",
    "# How long are methods sections in the CryoET papers?\n",
    "ldb.session.rollback()\n",
    "q = ldb.session.query(SKE.id) \\\n",
    "        .filter(SKC.id==SKC_HM.ScientificKnowledgeCollection_id) \\\n",
    "        .filter(SKC_HM.has_members_id==SKE.id) \\\n",
    "        .filter(SKC.id=='2') \\\n",
    "        .filter(or_(SKE.type=='ScientificPrimaryResearchArticle', SKE.type=='ScientificPrimaryResearchPreprint'))\n",
    "\n",
    "encoding = tiktoken.encoding_for_model('gpt-3.5-turbo')\n",
    "\n",
    "tups = []\n",
    "for e in tqdm(q.all()):\n",
    "    item_types = set()\n",
    "    item_type = None\n",
    "    for i in ldb.list_items_for_expression(e.id):\n",
    "        item_types.add(i.type)\n",
    "    for i_type in item_types:\n",
    "        if i_type == 'CitationRecord':\n",
    "            continue\n",
    "        item_type = i_type\n",
    "        break\n",
    "    if item_type is None:\n",
    "        continue\n",
    "\n",
    "    fragments = [f.content for f in ldb.list_fragments_for_paper(e.id, item_type, fragment_types=['section'])]\n",
    "    on_off = False\n",
    "    text = ''\n",
    "    all_text = ''\n",
    "    for t in fragments:\n",
    "        all_text += t\n",
    "        l1 = t.split('\\n')[0].lower()\n",
    "        if 'method' in l1:\n",
    "            on_off = True\n",
    "        elif 'results' in l1 or 'discussion' in l1 or 'conclusion' in l1 or 'acknowledgements' in l1 \\\n",
    "                or 'references' in l1 or 'supplementary' in l1 or 'appendix' in l1 or 'introduction' in l1 or 'abstract' in l1 or 'cited' in l1:\n",
    "            on_off = False\n",
    "        if on_off:\n",
    "            if len(text) > 0:\n",
    "                text += '\\n\\n'\n",
    "            text += t\n",
    "\n",
    "    all_text_length = len(tokenizer(all_text, return_tensors=\"pt\")['input_ids'][0])\n",
    "    text_length = len(tokenizer(text, return_tensors=\"pt\")['input_ids'][0])\n",
    "    tups.append({'doi':e.id, 'doc_length': all_text_length, 'method_length': text_length})\n",
    "df_length = pd.DataFrame(tups)\n",
    "df_length "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "print(len(df_length[df_length['method_length']>8000]))\n",
    "print(len(df_length[df_length['method_length']<8000]))\n",
    "\n",
    "# Plot the distribution of the lengths of the methods sections \n",
    "import seaborn as sns  \n",
    "import matplotlib.pyplot as plt\n",
    "def plot_length_distribution(df_length):\n",
    "    plt.hist(df_length, bins=10)\n",
    "    plt.xlabel('Length')\n",
    "    plt.ylabel('Frequency')\n",
    "    plt.title('Distribution of Lengths')\n",
    "    plt.show()\n",
    "\n",
    "plot_length_distribution(df_length['method_length'])\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "t2 = [t for t in test_tk.get_tools() if isinstance(t, MetadataExtraction_MethodsSectionOnly_Tool)][0]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "for i, r in tqdm(df_length.iterrows()):\n",
    "    if len(df[df['doi']==r['doi']]) > 0:\n",
    "        continue\n",
    "    # Run the metadata extraction tool on the doi\n",
    "    try: \n",
    "        t2.run(tool_input={'paper_id': r['doi'], 'extraction_type': 'cryoet', 'run_label': 'test_llama3'})\n",
    "    except Exception as e:\n",
    "        print(e)\n",
    "        continue\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'df_length' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[6], line 4\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[38;5;66;03m# Create a dataframe to store previously extracted metadata\u001b[39;00m\n\u001b[1;32m      2\u001b[0m \u001b[38;5;66;03m#for d in [d for d_id in dois_to_include for d in dois_to_include[d_id]]:\u001b[39;00m\n\u001b[1;32m      3\u001b[0m df2 \u001b[38;5;241m=\u001b[39m pd\u001b[38;5;241m.\u001b[39mDataFrame()\n\u001b[0;32m----> 4\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m i, r \u001b[38;5;129;01min\u001b[39;00m tqdm(\u001b[43mdf_length\u001b[49m\u001b[38;5;241m.\u001b[39miterrows()):\n\u001b[1;32m      5\u001b[0m         item_types \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mset\u001b[39m()\n\u001b[1;32m      6\u001b[0m         l \u001b[38;5;241m=\u001b[39m t2\u001b[38;5;241m.\u001b[39mread_metadata_extraction_notes(r[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mdoi\u001b[39m\u001b[38;5;124m'\u001b[39m], \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mcryoet\u001b[39m\u001b[38;5;124m'\u001b[39m, \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mtest\u001b[39m\u001b[38;5;124m'\u001b[39m)\n",
      "\u001b[0;31mNameError\u001b[0m: name 'df_length' is not defined"
     ]
    }
   ],
   "source": [
    "# Create a dataframe to store previously extracted metadata\n",
    "#for d in [d for d_id in dois_to_include for d in dois_to_include[d_id]]:\n",
    "df2 = pd.DataFrame()\n",
    "for i, r in tqdm(df_length.iterrows()):\n",
    "        item_types = set()\n",
    "        l = t2.read_metadata_extraction_notes(r['doi'], 'cryoet', 'test')\n",
    "        if(len(l) == 0):\n",
    "            continue\n",
    "        df2 = pd.concat([df2, pd.DataFrame(l)]) \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "1586it [00:07, 203.18it/s]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>sample_type</th>\n",
       "      <th>sample_preparation_type</th>\n",
       "      <th>sample_preparation_buffer_ph</th>\n",
       "      <th>grid_vitrification_cryogen</th>\n",
       "      <th>sample_preparation_cryo_protectant</th>\n",
       "      <th>grid_model</th>\n",
       "      <th>grid_material</th>\n",
       "      <th>grid_mesh</th>\n",
       "      <th>grid_support_topology</th>\n",
       "      <th>grid_vit_ctemp</th>\n",
       "      <th>grid_vit_chumid</th>\n",
       "      <th>organism_name</th>\n",
       "      <th>tissue</th>\n",
       "      <th>cell_type</th>\n",
       "      <th>cell_strain</th>\n",
       "      <th>cell_component</th>\n",
       "      <th>doi</th>\n",
       "      <th>extraction_type</th>\n",
       "      <th>run_label</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>cell</td>\n",
       "      <td>tomography</td>\n",
       "      <td>not present</td>\n",
       "      <td>[ETHANE]</td>\n",
       "      <td>[BSA]</td>\n",
       "      <td>Quantifoil R1/2</td>\n",
       "      <td>COPPER</td>\n",
       "      <td>200</td>\n",
       "      <td>HOLEY</td>\n",
       "      <td>liquid nitrogen temperature</td>\n",
       "      <td>99%</td>\n",
       "      <td>S. pombe</td>\n",
       "      <td>None</td>\n",
       "      <td>S. pombe</td>\n",
       "      <td>K972 Sp h- wt haploid</td>\n",
       "      <td>[ribosomes, FAS, NPC, mitochondria, vesicle, t...</td>\n",
       "      <td>doi:10.1101/2022.04.12.488077</td>\n",
       "      <td>cryoet</td>\n",
       "      <td>test_llama3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>cell</td>\n",
       "      <td>tomography</td>\n",
       "      <td>not present</td>\n",
       "      <td>[ethane, propane]</td>\n",
       "      <td>DMSO</td>\n",
       "      <td>R (1/4) Carbon 200-mesh gold EM grids</td>\n",
       "      <td>GOLD</td>\n",
       "      <td>200</td>\n",
       "      <td>HOLEY</td>\n",
       "      <td>37 degreesC</td>\n",
       "      <td>100%</td>\n",
       "      <td>mouse</td>\n",
       "      <td>None</td>\n",
       "      <td>mouse embryonic fibroblasts</td>\n",
       "      <td>MEFmtGFP</td>\n",
       "      <td>mitochondria, ER</td>\n",
       "      <td>doi:10.1083/jcb.202204093</td>\n",
       "      <td>cryoet</td>\n",
       "      <td>test_llama3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>[micro-organism]</td>\n",
       "      <td>tomography</td>\n",
       "      <td>not present</td>\n",
       "      <td>[ETHANE]</td>\n",
       "      <td>None</td>\n",
       "      <td>Quantifoil R 2/2 Au 200 mesh</td>\n",
       "      <td>GOLD</td>\n",
       "      <td>200</td>\n",
       "      <td>HOLEY</td>\n",
       "      <td>not present</td>\n",
       "      <td>95%</td>\n",
       "      <td>Mus musculus</td>\n",
       "      <td>not present</td>\n",
       "      <td>sperm</td>\n",
       "      <td>C57Bl/6J</td>\n",
       "      <td>[axoneme, central pair complex, microtubule do...</td>\n",
       "      <td>doi:10.1038/s41594-022-00861-0</td>\n",
       "      <td>cryoet</td>\n",
       "      <td>test_llama3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>cell</td>\n",
       "      <td>tomography</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>Mycoplasma pneumoniae</td>\n",
       "      <td>None</td>\n",
       "      <td>not present</td>\n",
       "      <td>M129</td>\n",
       "      <td>ribosomes</td>\n",
       "      <td>doi:10.1038/s41586-022-05255-2</td>\n",
       "      <td>cryoet</td>\n",
       "      <td>test_llama3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>[micro-organism]</td>\n",
       "      <td>tomography</td>\n",
       "      <td>7.3</td>\n",
       "      <td>[NITROGEN]</td>\n",
       "      <td>Ficoll 400</td>\n",
       "      <td>not present</td>\n",
       "      <td>COPPER</td>\n",
       "      <td>[75, 50, 100/400]</td>\n",
       "      <td>HOLEY ARRAY</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>[Caenorhabditis elegans, Drosophila melanogaster]</td>\n",
       "      <td>None</td>\n",
       "      <td>L1 larvae</td>\n",
       "      <td>[AM140, NK2476]</td>\n",
       "      <td>[80S ribosome, tRNAs, elongation factor]</td>\n",
       "      <td>doi:10.1101/2023.04.28.538734</td>\n",
       "      <td>cryoet</td>\n",
       "      <td>test_llama3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>not present</td>\n",
       "      <td>tomography</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>doi:10.1093/bioinformatics/bty620</td>\n",
       "      <td>cryoet</td>\n",
       "      <td>test_llama3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>[cell, tissue]</td>\n",
       "      <td>tomography</td>\n",
       "      <td>7.25</td>\n",
       "      <td>[ETHANE, NITROGEN]</td>\n",
       "      <td>dextran</td>\n",
       "      <td>Quantifoil-R5/20</td>\n",
       "      <td>GOLD</td>\n",
       "      <td>200</td>\n",
       "      <td>HOLEY</td>\n",
       "      <td>31-36.5 degreesC</td>\n",
       "      <td>not present</td>\n",
       "      <td>Rattus norvegicus</td>\n",
       "      <td>hippocampal</td>\n",
       "      <td>[neuronal, P19, HeLa, PtK2, rat HTC]</td>\n",
       "      <td>not present</td>\n",
       "      <td>[microtubules, luminal particles]</td>\n",
       "      <td>doi:10.1083/jcb.200606074</td>\n",
       "      <td>cryoet</td>\n",
       "      <td>test_llama3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>micro-organism</td>\n",
       "      <td>helical</td>\n",
       "      <td>7.4</td>\n",
       "      <td>[ETHANE]</td>\n",
       "      <td>None</td>\n",
       "      <td>C-Flat</td>\n",
       "      <td>COPPER</td>\n",
       "      <td>not present</td>\n",
       "      <td>HOLEY</td>\n",
       "      <td>liquid nitrogen temperature</td>\n",
       "      <td>not present</td>\n",
       "      <td>Saccharomyces cerevisiae</td>\n",
       "      <td>None</td>\n",
       "      <td>yeast</td>\n",
       "      <td>W303 or S288C</td>\n",
       "      <td>plasma membrane, eisosomes</td>\n",
       "      <td>doi:10.1083/jcb.201104040</td>\n",
       "      <td>cryoet</td>\n",
       "      <td>test_llama3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>micro-organism</td>\n",
       "      <td>subtomogram averaging</td>\n",
       "      <td>8.0</td>\n",
       "      <td>[ETHANE]</td>\n",
       "      <td>None</td>\n",
       "      <td>Holey carbon Quantifoil copper grids (R2/2, 20...</td>\n",
       "      <td>COPPER</td>\n",
       "      <td>200</td>\n",
       "      <td>HOLEY</td>\n",
       "      <td>-180 degreesC</td>\n",
       "      <td>not present</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>not present</td>\n",
       "      <td>None</td>\n",
       "      <td>doi:10.1038/ncomms5131</td>\n",
       "      <td>cryoet</td>\n",
       "      <td>test_llama3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>micro-organism</td>\n",
       "      <td>tomography</td>\n",
       "      <td>7.4</td>\n",
       "      <td>[ETHANE]</td>\n",
       "      <td>None</td>\n",
       "      <td>Quantifoil</td>\n",
       "      <td>COPPER</td>\n",
       "      <td>200</td>\n",
       "      <td>HOLEY ARRAY</td>\n",
       "      <td>-180°C</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>not present</td>\n",
       "      <td>doi:10.1083/jcb.200908045</td>\n",
       "      <td>cryoet</td>\n",
       "      <td>test_llama3</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1098 rows × 19 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "         sample_type sample_preparation_type sample_preparation_buffer_ph  \\\n",
       "0               cell              tomography                  not present   \n",
       "0               cell              tomography                  not present   \n",
       "0   [micro-organism]              tomography                  not present   \n",
       "0               cell              tomography                  not present   \n",
       "0   [micro-organism]              tomography                          7.3   \n",
       "..               ...                     ...                          ...   \n",
       "0        not present              tomography                  not present   \n",
       "0     [cell, tissue]              tomography                         7.25   \n",
       "0     micro-organism                 helical                          7.4   \n",
       "0     micro-organism   subtomogram averaging                          8.0   \n",
       "0     micro-organism              tomography                          7.4   \n",
       "\n",
       "   grid_vitrification_cryogen sample_preparation_cryo_protectant  \\\n",
       "0                    [ETHANE]                              [BSA]   \n",
       "0           [ethane, propane]                               DMSO   \n",
       "0                    [ETHANE]                               None   \n",
       "0                 not present                        not present   \n",
       "0                  [NITROGEN]                         Ficoll 400   \n",
       "..                        ...                                ...   \n",
       "0                 not present                        not present   \n",
       "0          [ETHANE, NITROGEN]                            dextran   \n",
       "0                    [ETHANE]                               None   \n",
       "0                    [ETHANE]                               None   \n",
       "0                    [ETHANE]                               None   \n",
       "\n",
       "                                           grid_model grid_material  \\\n",
       "0                                     Quantifoil R1/2        COPPER   \n",
       "0               R (1/4) Carbon 200-mesh gold EM grids          GOLD   \n",
       "0                        Quantifoil R 2/2 Au 200 mesh          GOLD   \n",
       "0                                         not present   not present   \n",
       "0                                         not present        COPPER   \n",
       "..                                                ...           ...   \n",
       "0                                         not present   not present   \n",
       "0                                    Quantifoil-R5/20          GOLD   \n",
       "0                                              C-Flat        COPPER   \n",
       "0   Holey carbon Quantifoil copper grids (R2/2, 20...        COPPER   \n",
       "0                                          Quantifoil        COPPER   \n",
       "\n",
       "            grid_mesh grid_support_topology               grid_vit_ctemp  \\\n",
       "0                 200                 HOLEY  liquid nitrogen temperature   \n",
       "0                 200                 HOLEY                  37 degreesC   \n",
       "0                 200                 HOLEY                  not present   \n",
       "0         not present           not present                  not present   \n",
       "0   [75, 50, 100/400]           HOLEY ARRAY                  not present   \n",
       "..                ...                   ...                          ...   \n",
       "0         not present           not present                  not present   \n",
       "0                 200                 HOLEY             31-36.5 degreesC   \n",
       "0         not present                 HOLEY  liquid nitrogen temperature   \n",
       "0                 200                 HOLEY                -180 degreesC   \n",
       "0                 200           HOLEY ARRAY                       -180°C   \n",
       "\n",
       "   grid_vit_chumid                                      organism_name  \\\n",
       "0              99%                                           S. pombe   \n",
       "0             100%                                              mouse   \n",
       "0              95%                                       Mus musculus   \n",
       "0      not present                              Mycoplasma pneumoniae   \n",
       "0      not present  [Caenorhabditis elegans, Drosophila melanogaster]   \n",
       "..             ...                                                ...   \n",
       "0      not present                                        not present   \n",
       "0      not present                                  Rattus norvegicus   \n",
       "0      not present                           Saccharomyces cerevisiae   \n",
       "0      not present                                               None   \n",
       "0      not present                                        not present   \n",
       "\n",
       "         tissue                             cell_type            cell_strain  \\\n",
       "0          None                              S. pombe  K972 Sp h- wt haploid   \n",
       "0          None           mouse embryonic fibroblasts               MEFmtGFP   \n",
       "0   not present                                 sperm               C57Bl/6J   \n",
       "0          None                           not present                   M129   \n",
       "0          None                             L1 larvae        [AM140, NK2476]   \n",
       "..          ...                                   ...                    ...   \n",
       "0   not present                           not present            not present   \n",
       "0   hippocampal  [neuronal, P19, HeLa, PtK2, rat HTC]            not present   \n",
       "0          None                                 yeast          W303 or S288C   \n",
       "0          None                                  None            not present   \n",
       "0   not present                           not present            not present   \n",
       "\n",
       "                                       cell_component  \\\n",
       "0   [ribosomes, FAS, NPC, mitochondria, vesicle, t...   \n",
       "0                                    mitochondria, ER   \n",
       "0   [axoneme, central pair complex, microtubule do...   \n",
       "0                                           ribosomes   \n",
       "0            [80S ribosome, tRNAs, elongation factor]   \n",
       "..                                                ...   \n",
       "0                                         not present   \n",
       "0                   [microtubules, luminal particles]   \n",
       "0                          plasma membrane, eisosomes   \n",
       "0                                                None   \n",
       "0                                         not present   \n",
       "\n",
       "                                  doi extraction_type    run_label  \n",
       "0       doi:10.1101/2022.04.12.488077          cryoet  test_llama3  \n",
       "0           doi:10.1083/jcb.202204093          cryoet  test_llama3  \n",
       "0      doi:10.1038/s41594-022-00861-0          cryoet  test_llama3  \n",
       "0      doi:10.1038/s41586-022-05255-2          cryoet  test_llama3  \n",
       "0       doi:10.1101/2023.04.28.538734          cryoet  test_llama3  \n",
       "..                                ...             ...          ...  \n",
       "0   doi:10.1093/bioinformatics/bty620          cryoet  test_llama3  \n",
       "0           doi:10.1083/jcb.200606074          cryoet  test_llama3  \n",
       "0           doi:10.1083/jcb.201104040          cryoet  test_llama3  \n",
       "0              doi:10.1038/ncomms5131          cryoet  test_llama3  \n",
       "0           doi:10.1083/jcb.200908045          cryoet  test_llama3  \n",
       "\n",
       "[1098 rows x 19 columns]"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Create a dataframe to store previously extracted metadata\n",
    "#for d in [d for d_id in dois_to_include for d in dois_to_include[d_id]]:\n",
    "df = pd.DataFrame()\n",
    "for i, r in tqdm(df_length.iterrows()):\n",
    "    if r['method_length'] < 8000:\n",
    "        item_types = set()\n",
    "        l = t2.read_metadata_extraction_notes(r['doi'], 'cryoet', 'test_llama3')\n",
    "        if(len(l) == 0):\n",
    "            continue\n",
    "        df = pd.concat([df, pd.DataFrame(l)]) \n",
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df['doi']=='doi:10.1101/2022.04.12.488077']\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df2[df2['doi']=='doi:10.1101/2022.04.12.488077']\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t2 = [t for t in test_tk.get_tools() if isinstance(t, MetadataExtraction_MethodsSectionOnly_Tool)][0]\n",
    "metadata_dir = '/Users/gully.burns/alhazen/em_tech/empiar/'\n",
    "t2.compile_answers('cryoet', metadata_dir)\n",
    "t2.write_answers_as_notes('cryoet', metadata_dir)\n",
    "#sorted(list(set([doi for q in t2.examples for doi in t2.examples[q]])))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import local_resources.data_files.cryoet_portal_metadata as cryoet_portal_metadata\n",
    "\n",
    "# Get the metadata extraction tool\n",
    "t2 = [t for t in test_tk.get_tools() if isinstance(t, MetadataExtraction_MethodsSectionOnly_Tool)][0]\n",
    "\n",
    "# Hack to get the path to the metadata directory as a string\n",
    "#metadata_dir = str(files(cryoet_portal_metadata).joinpath('temp'))[0:-4]\n",
    "metadata_dir = '/Users/gully.burns/alhazen/em_tech/empiar/'\n",
    "\n",
    "# Compile the answers from the metadata directory\n",
    "t2.compile_answers('cryoet', metadata_dir)\n",
    "\n",
    "# Create a dataframe to store previously extracted metadata\n",
    "#for d in [d for d_id in dois_to_include for d in dois_to_include[d_id]]:\n",
    "df = pd.DataFrame()\n",
    "for d in [d for d in dois]:\n",
    "    item_types = set()\n",
    "    l = t2.read_metadata_extraction_notes(d, 'cryoet', 'test')\n",
    "    df = pd.concat([df, pd.DataFrame(l)]) \n",
    "     \n",
    "# Iterate over papers to run the metadata extraction tool\n",
    "#for d in [d for d_id in dois_to_include for d in dois_to_include[d_id]]:\n",
    "for d in [d for d in dois]:\n",
    "    item_types = set()\n",
    "\n",
    "    # Skip if the doi is already in the database\n",
    "    if len(df)>0 and d in df.doi.unique():\n",
    "        continue\n",
    "\n",
    "    # Run the metadata extraction tool on the doi\n",
    "    t2.run(tool_input={'paper_id': d, 'extraction_type': 'cryoet', 'run_label': 'test_llama3'})\n",
    "\n",
    "    # Add the results to the dataframe    \n",
    "    l2 = t2.read_metadata_extraction_notes(d, 'cryoet', 'test')\n",
    "    df = pd.concat([df, pd.DataFrame(l2)]) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "# Create a dataframe to store previously extracted metadata\n",
    "#for d in [d for d_id in dois_to_include for d in dois_to_include[d_id]]:\n",
    "t2 = [t for t in test_tk.get_tools() if isinstance(t, MetadataExtraction_MethodsSectionOnly_Tool)][0]\n",
    "df_final = pd.DataFrame()\n",
    "for d in [d for d in dois]:\n",
    "    item_types = set()\n",
    "    l = t2.read_metadata_extraction_notes(d, 'cryoet', 'test')\n",
    "    df_final = pd.concat([df_final, pd.DataFrame(l)]) \n",
    "df_final"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import local_resources.data_files.cryoet_portal_metadata as cryoet_portal_metadata\n",
    "from rapidfuzz import fuzz\n",
    "# Get the metadata extraction tool\n",
    "t2 = [t for t in test_tk.get_tools() if isinstance(t, MetadataExtraction_MethodsSectionOnly_Tool)][0]\n",
    "\n",
    "#for d in [d for d_id in dois_to_include for d in dois_to_include[d_id]]:\n",
    "l = []\n",
    "for d in [d for d in dois]:\n",
    "    item_types = set()\n",
    "    pred1 = t2.read_metadata_extraction_notes(d, 'cryoet', 'test')\n",
    "    pred2 = t2.read_metadata_extraction_notes(d, 'cryoet', 'test_dbrx')\n",
    "    gold = t2.read_metadata_extraction_notes(d, 'cryoet', 'gold') \n",
    "    if pred1 is None or pred2 is None or gold is None or \\\n",
    "            len(pred1)==0 or len(pred2)==0 or len(gold)!=1:\n",
    "        continue\n",
    "    for k in gold[0]:\n",
    "        g_case = gold[0][k]\n",
    "        if g_case=='' or g_case is None:\n",
    "            continue    \n",
    "        for j, p_case in enumerate(pred1):\n",
    "            sim = fuzz.ratio(str(g_case), str(p_case.get(k,''))) / 100.0\n",
    "            print(k, str(g_case), str(p_case.get(k,'')), sim)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import local_resources.data_files.cryoet_portal_metadata as cryoet_portal_metadata\n",
    "from rapidfuzz import fuzz\n",
    "# Get the metadata extraction tool\n",
    "t2 = [t for t in test_tk.get_tools() if isinstance(t, MetadataExtraction_MethodsSectionOnly_Tool)][0]\n",
    "\n",
    "df = t2.report_metadata_extraction_for_collection('5', 'cryoet', 'test').set_index('doi')\n",
    "df.to_csv(loc+'/'+db_name+'/reports/cryoet_metadata_gpt4.tsv', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ldb.create_zip_archive_of_full_text_files('5', loc+'/'+db_name+'/full_text_files.zip')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "q3 = ldb.session.query(SKE.id, N.name, N.provenance, N.content) \\\n",
    "        .filter(N.id == NIA.Note_id) \\\n",
    "        .filter(NIA.is_about_id == SKE.id) \\\n",
    "        .filter(N.type == 'MetadataExtractionNote') \n",
    "l = []\n",
    "for row in q3.all():\n",
    "    paper = row[0]\n",
    "    name = row[1]\n",
    "#    provenance = json.loads(row[2])\n",
    "    result = json.loads(row[3])\n",
    "    kv = {k:result[k] for k in result}\n",
    "    kv['DOI'] = paper\n",
    "    kv['run'] = name\n",
    "    l.append(kv)\n",
    "# create a dataframe from the list of dictionaries with DOI as the index column\n",
    "if len(l)>0:\n",
    "    df = pd.DataFrame(l).set_index(['DOI', 'run'])\n",
    "else: \n",
    "    df = pd.DataFrame()\n",
    "df "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# USE WITH CAUTION - this will delete all extracted metadata notes in the database\n",
    "# clear all notes across papers listed in `dois` list\n",
    "for row in q3.all():\n",
    "    d_id = row[0]\n",
    "    e = ldb.session.query(SKE).filter(SKE.id==d_id).first()\n",
    "    notes_to_delete = []\n",
    "    for n in ldb.read_notes_about_x(e):\n",
    "        notes_to_delete.append(n.id)\n",
    "    for n in notes_to_delete:\n",
    "        ldb.delete_note(n)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Protocol Modeling + Extraction"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ldb = Ceifns_LiteratureDb(loc=loc, name=db_name)\n",
    "slm = ChatOllama(model='stablelm-zephyr') \n",
    "llm = ChatOllama(model='mixtral:instruct') \n",
    "llm2 = ChatOpenAI(model='gpt-4-1106-preview') \n",
    "llm3 = ChatOpenAI(model='gpt-3.5-turbo') \n",
    "d = (\"This tool attempts to draw a protocol design from the description of a scientific paper.\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t1 = ProcotolEntitiesExtractionTool(db=ldb, llm=llm3, description=d)\n",
    "entities = t1.run(tool_input={'paper_id': 'doi:10.1101/2022.04.12.488077'})\n",
    "entities"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t2 = ProcotolProcessesExtractionTool(db=ldb, llm=llm3, description=d)\n",
    "processes = t2.run(tool_input={'paper_id': 'doi:10.1101/2022.04.12.488077'})\n",
    "processes.get('data')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "alhazen",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
