{
 "cells": [
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# RNAquarium  \n",
    "\n",
    "> Using LLMs to extract information from RNA studies in Zebrafish"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Basics"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Python Imports\n",
    "\n",
    "Setting python imports, environment variables, and other crucial set up parameters here.  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from alhazen.core import get_langchain_chatmodel, MODEL_TYPE\n",
    "from alhazen.agent import AlhazenAgent\n",
    "from alhazen.schema_sqla import *\n",
    "from alhazen.core import get_langchain_chatmodel, MODEL_TYPE\n",
    "from alhazen.tools.basic import AddCollectionFromEPMCTool, DeleteCollectionTool\n",
    "from alhazen.tools.paperqa_emulation_tool import PaperQAEmulationTool\n",
    "from alhazen.tools.metadata_extraction_tool import * \n",
    "from alhazen.tools.protocol_extraction_tool import *\n",
    "from alhazen.toolkit import *\n",
    "from alhazen.utils.jats_text_extractor import NxmlDoc\n",
    "\n",
    "from alhazen.utils.jats_text_extractor import NxmlDoc\n",
    "from alhazen.utils.ceifns_db import Ceifns_LiteratureDb, create_ceifns_database, drop_ceifns_database, list_databases\n",
    "from alhazen.utils.searchEngineUtils import *\n",
    "\n",
    "\n",
    "from langchain.callbacks.tracers import ConsoleCallbackHandler\n",
    "from langchain.docstore.document import Document\n",
    "from langchain.text_splitter import CharacterTextSplitter\n",
    "from langchain.vectorstores.pgvector import PGVector\n",
    "from langchain_community.chat_models.ollama import ChatOllama\n",
    "from langchain_google_vertexai import ChatVertexAI\n",
    "from langchain_openai import ChatOpenAI\n",
    "\n",
    "from bs4 import BeautifulSoup,Tag,Comment,NavigableString\n",
    "from databricks import sql\n",
    "from datetime import datetime\n",
    "from importlib_resources import files\n",
    "import os\n",
    "import pandas as pd\n",
    "from pathlib import Path\n",
    "import re\n",
    "import requests\n",
    "\n",
    "from sqlalchemy import create_engine, exists, func, or_, and_, not_, desc, asc\n",
    "from sqlalchemy.orm import sessionmaker, aliased\n",
    "\n",
    "from time import time,sleep\n",
    "from tqdm import tqdm\n",
    "from urllib.request import urlopen\n",
    "from urllib.parse import quote_plus, quote, unquote\n",
    "from urllib.error import URLError, HTTPError\n",
    "import yaml"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Using Aliases like this massively simplifies the use of SQLAlchemy\n",
    "IR = aliased(InformationResource)\n",
    "\n",
    "SKC = aliased(ScientificKnowledgeCollection)\n",
    "SKC_HM = aliased(ScientificKnowledgeCollectionHasMembers)\n",
    "SKE = aliased(ScientificKnowledgeExpression)\n",
    "SKE_XREF = aliased(ScientificKnowledgeExpressionXref)\n",
    "SKE_IRI = aliased(ScientificKnowledgeExpressionIri)\n",
    "SKE_HR = aliased(ScientificKnowledgeExpressionHasRepresentation)\n",
    "SKE_MO = aliased(ScientificKnowledgeExpressionMemberOf)\n",
    "SKI = aliased(ScientificKnowledgeItem)\n",
    "SKI_HP = aliased(ScientificKnowledgeItemHasPart)\n",
    "SKF = aliased(ScientificKnowledgeFragment)\n",
    "\n",
    "N = aliased(Note)\n",
    "NIA = aliased(NoteIsAbout)\n",
    "SKC_HN = aliased(ScientificKnowledgeCollectionHasNotes)\n",
    "SKE_HN = aliased(ScientificKnowledgeExpressionHasNotes)\n",
    "SKI_HN = aliased(ScientificKnowledgeItemHasNotes)\n",
    "SKF_HN = aliased(ScientificKnowledgeFragmentHasNotes)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Environment Variables\n",
    "\n",
    "Remember to set environmental variables for this code:\n",
    "\n",
    "* `ALHAZEN_DB_NAME` - the name of the PostGresQL database you are storing information into\n",
    "* `LOCAL_FILE_PATH` - the location on disk where you save temporary files, downloaded models or other data.   "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "os.environ['ALHAZEN_DB_NAME'] = 'rnaquarium'\n",
    "os.environ['LOCAL_FILE_PATH'] = '/users/gully.burns/alhazen/'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "if os.path.exists(os.environ['LOCAL_FILE_PATH']) is False:\n",
    "    os.makedirs(os.environ['LOCAL_FILE_PATH'])\n",
    "    \n",
    "if os.environ.get('ALHAZEN_DB_NAME') is None: \n",
    "    raise Exception('Which database do you want to use for this application?')\n",
    "db_name = os.environ['ALHAZEN_DB_NAME']\n",
    "\n",
    "if os.environ.get('LOCAL_FILE_PATH') is None: \n",
    "    raise Exception('Where are you storing your local literature database?')\n",
    "loc = os.environ['LOCAL_FILE_PATH']"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Setup utils, agents, and tools "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AGENT TOOLS\n",
      "\tAddCollectionFromEPMCTool\n",
      "\tAddAuthorsToCollectionTool\n",
      "\tDescribeCollectionCompositionTool\n",
      "\tDeleteCollectionTool\n",
      "\tRetrieveFullTextTool\n",
      "\tRetrieveFullTextToolForACollection\n",
      "\tMetadataExtraction_EverythingEverywhere_Tool\n",
      "\tSimpleExtractionWithRAGTool\n",
      "\tPaperQAEmulationTool\n",
      "\tProcotolExtractionTool\n",
      "\tCheckExpressionTool\n",
      "\n",
      "TESTING TOOLS\n",
      "\tMetadataExtraction_EverythingEverywhere_Tool\n",
      "\tMetadataExtraction_RAGOnSections_Tool\n",
      "\tSimpleExtractionWithRAGTool\n"
     ]
    }
   ],
   "source": [
    "ldb = Ceifns_LiteratureDb(loc=loc, name=db_name)\n",
    "llm = ChatOllama(model='mixtral:instruct') \n",
    "llm2 = ChatOpenAI(model='gpt-4-1106-preview') \n",
    "llm2 = ChatOpenAI(model='gpt-4-1106-preview') \n",
    "#llm3 = ChatVertexAI(model_name=\"gemini-pro\", convert_system_message_to_human=True)\n",
    "\n",
    "cb = AlhazenAgent(llm, llm)\n",
    "print('AGENT TOOLS')\n",
    "for t in cb.tk.get_tools():\n",
    "    print('\\t'+type(t).__name__)\n",
    "\n",
    "test_tk = MetadataExtractionToolkit(db=ldb, llm=llm2)\n",
    "print('\\nTESTING TOOLS')\n",
    "for t in test_tk.get_tools():\n",
    "    print('\\t'+type(t).__name__)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Building the database\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Scripts to Build / Delete the database\n",
    "\n",
    "If you need to restore a deleted database from backup, use the following shell commands:\n",
    "\n",
    "```\n",
    "$ createdb em_tech\n",
    "$ psql -d em_tech -f /local/file/path/em_tech/backup<date_time>.sql\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "drop_ceifns_database(os.environ['ALHAZEN_DB_NAME'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "create_ceifns_database(os.environ['ALHAZEN_DB_NAME'])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Build CEIFNS database from 900 dois in database"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Key</th>\n",
       "      <th>Item Type</th>\n",
       "      <th>Publication Year</th>\n",
       "      <th>Author</th>\n",
       "      <th>Title</th>\n",
       "      <th>Publication Title</th>\n",
       "      <th>ISBN</th>\n",
       "      <th>ISSN</th>\n",
       "      <th>DOI</th>\n",
       "      <th>Url</th>\n",
       "      <th>...</th>\n",
       "      <th>Unnamed: 163</th>\n",
       "      <th>Unnamed: 164</th>\n",
       "      <th>Unnamed: 165</th>\n",
       "      <th>Unnamed: 166</th>\n",
       "      <th>Unnamed: 167</th>\n",
       "      <th>Unnamed: 168</th>\n",
       "      <th>Unnamed: 169</th>\n",
       "      <th>Unnamed: 170</th>\n",
       "      <th>Unnamed: 171</th>\n",
       "      <th>Unnamed: 172</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>2FYDLCGI</td>\n",
       "      <td>journalArticle</td>\n",
       "      <td>2014</td>\n",
       "      <td>Han, Peidong; Zhou, Xiao-Hai; Chang, Nannan; X...</td>\n",
       "      <td>Hydrogen peroxide primes heart regeneration wi...</td>\n",
       "      <td>Cell Research</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1748-7838</td>\n",
       "      <td>10.1038/cr.2014.108</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>8RZKQRRU</td>\n",
       "      <td>journalArticle</td>\n",
       "      <td>2014</td>\n",
       "      <td>Marín-Juez, Rubén; Jong-Raadsen, Susanne; Yang...</td>\n",
       "      <td>Hyperinsulinemia induces insulin resistance an...</td>\n",
       "      <td>The Journal of Endocrinology</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1479-6805</td>\n",
       "      <td>10.1530/JOE-14-0178</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>CPWU52VI</td>\n",
       "      <td>journalArticle</td>\n",
       "      <td>2014</td>\n",
       "      <td>Benard, Erica L.; Roobol, Stefan J.; Spaink, H...</td>\n",
       "      <td>Phagocytosis of mycobacteria by zebrafish macr...</td>\n",
       "      <td>Developmental and Comparative Immunology</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1879-0089</td>\n",
       "      <td>10.1016/j.dci.2014.07.022</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>E2XPRJPW</td>\n",
       "      <td>journalArticle</td>\n",
       "      <td>2014</td>\n",
       "      <td>Howarth, Deanna L.; Lindtner, Claudia; Vacaru,...</td>\n",
       "      <td>Activating transcription factor 6 is necessary...</td>\n",
       "      <td>PLoS genetics</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1553-7404</td>\n",
       "      <td>10.1371/journal.pgen.1004335</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>RYI6EAFJ</td>\n",
       "      <td>journalArticle</td>\n",
       "      <td>2014</td>\n",
       "      <td>Desvignes, Thomas; Beam, Michael J.; Batzel, P...</td>\n",
       "      <td>Expanding the annotation of zebrafish microRNA...</td>\n",
       "      <td>Gene</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1879-0038</td>\n",
       "      <td>10.1016/j.gene.2014.05.036</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>906</th>\n",
       "      <td>MNQQTVH8</td>\n",
       "      <td>journalArticle</td>\n",
       "      <td>2022</td>\n",
       "      <td>Zhang, Zhicong; Ji, Fengyu; Jiang, Shouwen; Wu...</td>\n",
       "      <td>Scale Development-Related Genes Identified by ...</td>\n",
       "      <td>Fishes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2410-3888</td>\n",
       "      <td>10.3390/fishes7020064</td>\n",
       "      <td>https://www.mdpi.com/2410-3888/7/2/64</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>907</th>\n",
       "      <td>E9E2BJC8</td>\n",
       "      <td>journalArticle</td>\n",
       "      <td>2022</td>\n",
       "      <td>Hu, Ming-Liang; Wu, Bao-Sheng; Xu, Wen-Jie; Zh...</td>\n",
       "      <td>Gene expression responses in zebrafish to shor...</td>\n",
       "      <td>Zoological Research</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2095-8137</td>\n",
       "      <td>10.24272/j.issn.2095-8137.2021.352</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>908</th>\n",
       "      <td>ME3REM8F</td>\n",
       "      <td>journalArticle</td>\n",
       "      <td>2022</td>\n",
       "      <td>Herrera-Rivero, Marisol; Gandhi, Shrey; Witten...</td>\n",
       "      <td>Cardiac chamber-specific genetic alterations s...</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8887543</td>\n",
       "      <td>10.1016/j.ygeno.2022.110320</td>\n",
       "      <td>https://linkinghub.elsevier.com/retrieve/pii/S...</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>909</th>\n",
       "      <td>ICZFTJFQ</td>\n",
       "      <td>journalArticle</td>\n",
       "      <td>2021</td>\n",
       "      <td>Gandhi, Shrey; Witten, Anika; De Majo, Federic...</td>\n",
       "      <td>Evolutionarily conserved transcriptional lands...</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8887543</td>\n",
       "      <td>10.1016/j.ygeno.2021.09.002</td>\n",
       "      <td>https://linkinghub.elsevier.com/retrieve/pii/S...</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>910</th>\n",
       "      <td>TF38GRRU</td>\n",
       "      <td>journalArticle</td>\n",
       "      <td>2020</td>\n",
       "      <td>Spreafico, Marco; Mangano, Eleonora; Mazzola, ...</td>\n",
       "      <td>The Genome-Wide Impact of Nipblb Loss-of-Funct...</td>\n",
       "      <td>International Journal of Molecular Sciences</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1422-0067</td>\n",
       "      <td>10.3390/ijms21249719</td>\n",
       "      <td>https://www.mdpi.com/1422-0067/21/24/9719</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>911 rows × 173 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "          Key       Item Type  Publication Year  \\\n",
       "0    2FYDLCGI  journalArticle              2014   \n",
       "1    8RZKQRRU  journalArticle              2014   \n",
       "2    CPWU52VI  journalArticle              2014   \n",
       "3    E2XPRJPW  journalArticle              2014   \n",
       "4    RYI6EAFJ  journalArticle              2014   \n",
       "..        ...             ...               ...   \n",
       "906  MNQQTVH8  journalArticle              2022   \n",
       "907  E9E2BJC8  journalArticle              2022   \n",
       "908  ME3REM8F  journalArticle              2022   \n",
       "909  ICZFTJFQ  journalArticle              2021   \n",
       "910  TF38GRRU  journalArticle              2020   \n",
       "\n",
       "                                                Author  \\\n",
       "0    Han, Peidong; Zhou, Xiao-Hai; Chang, Nannan; X...   \n",
       "1    Marín-Juez, Rubén; Jong-Raadsen, Susanne; Yang...   \n",
       "2    Benard, Erica L.; Roobol, Stefan J.; Spaink, H...   \n",
       "3    Howarth, Deanna L.; Lindtner, Claudia; Vacaru,...   \n",
       "4    Desvignes, Thomas; Beam, Michael J.; Batzel, P...   \n",
       "..                                                 ...   \n",
       "906  Zhang, Zhicong; Ji, Fengyu; Jiang, Shouwen; Wu...   \n",
       "907  Hu, Ming-Liang; Wu, Bao-Sheng; Xu, Wen-Jie; Zh...   \n",
       "908  Herrera-Rivero, Marisol; Gandhi, Shrey; Witten...   \n",
       "909  Gandhi, Shrey; Witten, Anika; De Majo, Federic...   \n",
       "910  Spreafico, Marco; Mangano, Eleonora; Mazzola, ...   \n",
       "\n",
       "                                                 Title  \\\n",
       "0    Hydrogen peroxide primes heart regeneration wi...   \n",
       "1    Hyperinsulinemia induces insulin resistance an...   \n",
       "2    Phagocytosis of mycobacteria by zebrafish macr...   \n",
       "3    Activating transcription factor 6 is necessary...   \n",
       "4    Expanding the annotation of zebrafish microRNA...   \n",
       "..                                                 ...   \n",
       "906  Scale Development-Related Genes Identified by ...   \n",
       "907  Gene expression responses in zebrafish to shor...   \n",
       "908  Cardiac chamber-specific genetic alterations s...   \n",
       "909  Evolutionarily conserved transcriptional lands...   \n",
       "910  The Genome-Wide Impact of Nipblb Loss-of-Funct...   \n",
       "\n",
       "                               Publication Title  ISBN       ISSN  \\\n",
       "0                                  Cell Research   NaN  1748-7838   \n",
       "1                   The Journal of Endocrinology   NaN  1479-6805   \n",
       "2       Developmental and Comparative Immunology   NaN  1879-0089   \n",
       "3                                  PLoS genetics   NaN  1553-7404   \n",
       "4                                           Gene   NaN  1879-0038   \n",
       "..                                           ...   ...        ...   \n",
       "906                                       Fishes   NaN  2410-3888   \n",
       "907                          Zoological Research   NaN  2095-8137   \n",
       "908                                     Genomics   NaN    8887543   \n",
       "909                                     Genomics   NaN    8887543   \n",
       "910  International Journal of Molecular Sciences   NaN  1422-0067   \n",
       "\n",
       "                                    DOI  \\\n",
       "0                   10.1038/cr.2014.108   \n",
       "1                   10.1530/JOE-14-0178   \n",
       "2             10.1016/j.dci.2014.07.022   \n",
       "3          10.1371/journal.pgen.1004335   \n",
       "4            10.1016/j.gene.2014.05.036   \n",
       "..                                  ...   \n",
       "906               10.3390/fishes7020064   \n",
       "907  10.24272/j.issn.2095-8137.2021.352   \n",
       "908         10.1016/j.ygeno.2022.110320   \n",
       "909         10.1016/j.ygeno.2021.09.002   \n",
       "910                10.3390/ijms21249719   \n",
       "\n",
       "                                                   Url  ... Unnamed: 163  \\\n",
       "0                                                  NaN  ...          NaN   \n",
       "1                                                  NaN  ...          NaN   \n",
       "2                                                  NaN  ...          NaN   \n",
       "3                                                  NaN  ...          NaN   \n",
       "4                                                  NaN  ...          NaN   \n",
       "..                                                 ...  ...          ...   \n",
       "906              https://www.mdpi.com/2410-3888/7/2/64  ...          NaN   \n",
       "907  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...  ...          NaN   \n",
       "908  https://linkinghub.elsevier.com/retrieve/pii/S...  ...          NaN   \n",
       "909  https://linkinghub.elsevier.com/retrieve/pii/S...  ...          NaN   \n",
       "910          https://www.mdpi.com/1422-0067/21/24/9719  ...          NaN   \n",
       "\n",
       "    Unnamed: 164 Unnamed: 165 Unnamed: 166 Unnamed: 167 Unnamed: 168  \\\n",
       "0            NaN          NaN          NaN          NaN          NaN   \n",
       "1            NaN          NaN          NaN          NaN          NaN   \n",
       "2            NaN          NaN          NaN          NaN          NaN   \n",
       "3            NaN          NaN          NaN          NaN          NaN   \n",
       "4            NaN          NaN          NaN          NaN          NaN   \n",
       "..           ...          ...          ...          ...          ...   \n",
       "906          NaN          NaN          NaN          NaN          NaN   \n",
       "907          NaN          NaN          NaN          NaN          NaN   \n",
       "908          NaN          NaN          NaN          NaN          NaN   \n",
       "909          NaN          NaN          NaN          NaN          NaN   \n",
       "910          NaN          NaN          NaN          NaN          NaN   \n",
       "\n",
       "     Unnamed: 169 Unnamed: 170 Unnamed: 171  Unnamed: 172  \n",
       "0             NaN          NaN          NaN           NaN  \n",
       "1             NaN          NaN          NaN           NaN  \n",
       "2             NaN          NaN          NaN           NaN  \n",
       "3             NaN          NaN          NaN           NaN  \n",
       "4             NaN          NaN          NaN           NaN  \n",
       "..            ...          ...          ...           ...  \n",
       "906           NaN          NaN          NaN           NaN  \n",
       "907           NaN          NaN          NaN           NaN  \n",
       "908           NaN          NaN          NaN           NaN  \n",
       "909           NaN          NaN          NaN           NaN  \n",
       "910           NaN          NaN          NaN           NaN  \n",
       "\n",
       "[911 rows x 173 columns]"
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import local_resources.data_files.rnaquarium as rnaquarium\n",
    "from alhazen.utils.queryTranslator import QueryTranslator, QueryType\n",
    "\n",
    "df = pd.read_csv(files(rnaquarium).joinpath('RNAquarium_paper_list.tsv'), sep='\\t')\n",
    "dois = df['DOI'].to_list()\n",
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "addEMPCCollection_tool = [t for t in cb.tk.get_tools() if isinstance(t, AddCollectionFromEPMCTool)][0]\n",
    "step = 40\n",
    "for start_i in range(800, len(dois), step):\n",
    "    query = ' OR '.join(['doi:\\\"'+dois[i]+'\\\"' for i in range(start_i, start_i+step) if i < len(dois)])\n",
    "    addEMPCCollection_tool.run({'id': '0', 'name':'RNAquarium Papers', 'query':query, 'full_text':True})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0 Missing DOIs\n"
     ]
    }
   ],
   "source": [
    "# Compare contents of database to the list of dois\n",
    "missing_list = []\n",
    "titles = []\n",
    "for doi in dois:\n",
    "    row = df[df['DOI']==doi]\n",
    "    doi_in_db = ldb.session.query(SKE).filter(SKE.id=='doi:'+doi.lower()).all()\n",
    "    if len(doi_in_db) == 0:\n",
    "        print('DOI: '+doi)\n",
    "        print('\\t%s (%d) %s %s'%(row['Author'].iloc[0],row['Publication Year'].iloc[0],row['Title'].iloc[0],row['Journal Abbreviation'].iloc[0]))\n",
    "        missing_list.append(doi)\n",
    "        titles.append(row['Title'].iloc[0])\n",
    "print('%d Missing DOIs'%(len(missing_list)))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Use OpenAlex as filler to add papers that were missed on EPMC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "945\n"
     ]
    },
    {
     "ename": "NameError",
     "evalue": "name 'missing_list' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[7], line 14\u001b[0m\n\u001b[1;32m     11\u001b[0m \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;28mlen\u001b[39m(corpus\u001b[38;5;241m.\u001b[39mhas_members))\n\u001b[1;32m     13\u001b[0m papers_to_index \u001b[38;5;241m=\u001b[39m []\n\u001b[0;32m---> 14\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m i, doi \u001b[38;5;129;01min\u001b[39;00m \u001b[38;5;28menumerate\u001b[39m(\u001b[43mmissing_list\u001b[49m):\n\u001b[1;32m     15\u001b[0m     p \u001b[38;5;241m=\u001b[39m load_paper_from_openalex(doi)\n\u001b[1;32m     16\u001b[0m     ldb\u001b[38;5;241m.\u001b[39msession\u001b[38;5;241m.\u001b[39madd(p)\n",
      "\u001b[0;31mNameError\u001b[0m: name 'missing_list' is not defined"
     ]
    }
   ],
   "source": [
    "\n",
    "from alhazen.utils.searchEngineUtils import load_paper_from_openalex, read_references_from_openalex \n",
    "from pyalex import config, Works, Work\n",
    "config.email = \"gully.burns@chanzuckerberg.com\"\n",
    "\n",
    "import requests\n",
    "import os\n",
    "ldb.session.rollback()\n",
    "corpus = ldb.session.query(SKC).filter(SKC.id=='0').first()\n",
    "count = 0\n",
    "print(len(corpus.has_members))\n",
    "\n",
    "papers_to_index = []\n",
    "for i, doi in enumerate(missing_list):\n",
    "    p = load_paper_from_openalex(doi)\n",
    "    ldb.session.add(p)\n",
    "    corpus.has_members.append(p)\n",
    "    p.member_of.append(corpus)\n",
    "    for item in p.has_representation:\n",
    "        for f in item.has_part:\n",
    "            #f.content = '\\n'.join(self.sent_detector.tokenize(f.content))\n",
    "            f.part_of = item.id\n",
    "            ldb.session.add(f)\n",
    "        item.represented_by = p.id\n",
    "        ldb.session.add(item)\n",
    "    papers_to_index.append(p)\n",
    "    ldb.session.flush()\n",
    "\n",
    "ldb.embed_expression_list(papers_to_index)\n",
    "\n",
    "ldb.session.commit()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Get full text copies of all the papers about CryoET\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "\u001b[1m> Entering new AgentExecutor chain...\u001b[0m\n",
      "\u001b[32;1m\u001b[1;3m{\n",
      "  \"action\": \"retrieve_full_text_for_paper_id\",\n",
      "  \"action_input\": {\n",
      "    \"paper_id\": \"doi:10.3389/fcell.2021.699796\"\n",
      "  }\n",
      "}\u001b[0mlist index out of range\n",
      "\u001b[33;1m\u001b[1;3m{'response': 'The full text paper with doi:`doi:10.3389/fcell.2021.699796` is available in the database.'}\u001b[0m\n",
      "\u001b[32;1m\u001b[1;3m\u001b[0m\n",
      "\n",
      "\u001b[1m> Finished chain.\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'input': 'Retrieve full text for the paper with doi=\"doi:10.3389/fcell.2021.699796\".',\n",
       " 'output': {'response': 'The full text paper with doi:`doi:10.3389/fcell.2021.699796` is available in the database.'},\n",
       " 'intermediate_steps': [(AgentAction(tool='retrieve_full_text_for_paper_id', tool_input={'paper_id': 'doi:10.3389/fcell.2021.699796'}, log='{\\n  \"action\": \"retrieve_full_text_for_paper_id\",\\n  \"action_input\": {\\n    \"paper_id\": \"doi:10.3389/fcell.2021.699796\"\\n  }\\n}'),\n",
       "   {'response': 'The full text paper with doi:`doi:10.3389/fcell.2021.699796` is available in the database.'})]}"
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cb.db.session.rollback()\n",
    "error_doi = 'doi:10.3389/fcell.2021.699796'\n",
    "cb.agent_executor.invoke({'input':'Retrieve full text for the paper with doi=\"'+error_doi+'\".'})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cb.db.session.rollback()\n",
    "cb.agent_executor.invoke({'input':'Retrieve full text for the collection with id=\"0\".'})"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Analyze Collections"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>item type</th>\n",
       "      <th>CitationRecord</th>\n",
       "      <th>HTMLFullText</th>\n",
       "      <th>JATSFullText</th>\n",
       "      <th>PDFFullText</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>id</th>\n",
       "      <th>collection name</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <th>RNAquarium Papers</th>\n",
       "      <td>945</td>\n",
       "      <td>1</td>\n",
       "      <td>610</td>\n",
       "      <td>666</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "item type             CitationRecord  HTMLFullText  JATSFullText  PDFFullText\n",
       "id collection name                                                           \n",
       "0  RNAquarium Papers             945             1           610          666"
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q = ldb.session.query(SKC.id, SKC.name, SKE.id, SKI.type) \\\n",
    "        .filter(SKC.id==SKC_HM.ScientificKnowledgeCollection_id) \\\n",
    "        .filter(SKC_HM.has_members_id==SKE.id) \\\n",
    "        .filter(SKE.id==SKE_HR.ScientificKnowledgeExpression_id) \\\n",
    "        .filter(SKE_HR.has_representation_id==SKI.id) \n",
    "df = pd.DataFrame(q.all(), columns=['id', 'collection name', 'doi', 'item type'])    \n",
    "df.pivot_table(index=['id', 'collection name'], columns='item type', values='doi', aggfunc=lambda x: len(x.unique()))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "68\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>item type</th>\n",
       "      <th>doi</th>\n",
       "      <th>CitationRecord</th>\n",
       "      <th>HTMLFullText</th>\n",
       "      <th>JATSFullText</th>\n",
       "      <th>PDFFullText</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>doi:10.1002/dvdy.24188</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>doi:10.1002/dvdy.24600</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>doi:10.1002/glia.24075</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>doi:10.1007/s00424-017-2009-8</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>doi:10.1007/s11427-020-1878-8</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>doi:10.1007/s11427-021-2223-4</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>doi:10.1016/bs.adgen.2016.04.004</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>doi:10.1016/j.aquatox.2013.10.006</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>doi:10.1016/j.aquatox.2019.105290</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>doi:10.1016/j.bbrc.2015.03.128</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>doi:10.1016/j.bbrc.2017.04.035</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>doi:10.1016/j.bbrc.2021.02.122</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>106</th>\n",
       "      <td>doi:10.1016/j.chemosphere.2017.12.029</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107</th>\n",
       "      <td>doi:10.1016/j.chemosphere.2020.126537</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>108</th>\n",
       "      <td>doi:10.1016/j.chemosphere.2021.133133</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>109</th>\n",
       "      <td>doi:10.1016/j.chemosphere.2022.137380</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>110</th>\n",
       "      <td>doi:10.1016/j.chemosphere.2023.138195</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117</th>\n",
       "      <td>doi:10.1016/j.dci.2014.07.022</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>119</th>\n",
       "      <td>doi:10.1016/j.dci.2020.103922</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>148</th>\n",
       "      <td>doi:10.1016/j.devcel.2023.06.005</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157</th>\n",
       "      <td>doi:10.1016/j.envpol.2017.12.023</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>161</th>\n",
       "      <td>doi:10.1016/j.fct.2023.113788</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>162</th>\n",
       "      <td>doi:10.1016/j.fsi.2010.08.022</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>180</th>\n",
       "      <td>doi:10.1016/j.jgg.2022.04.005</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>182</th>\n",
       "      <td>doi:10.1016/j.jhazmat.2022.129382</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>183</th>\n",
       "      <td>doi:10.1016/j.jhazmat.2023.131714</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>184</th>\n",
       "      <td>doi:10.1016/j.jhazmat.2023.131722</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>185</th>\n",
       "      <td>doi:10.1016/j.matbio.2018.07.005</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>195</th>\n",
       "      <td>doi:10.1016/j.molimm.2010.08.011</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>196</th>\n",
       "      <td>doi:10.1016/j.neurobiolaging.2020.04.017</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>203</th>\n",
       "      <td>doi:10.1016/j.scib.2017.01.008</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>207</th>\n",
       "      <td>doi:10.1016/j.scitotenv.2022.159069</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>209</th>\n",
       "      <td>doi:10.1016/j.scitotenv.2023.162901</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>224</th>\n",
       "      <td>doi:10.1016/j.ydbio.2017.07.003</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>226</th>\n",
       "      <td>doi:10.1016/j.ydbio.2019.06.005</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>240</th>\n",
       "      <td>doi:10.1016/s1055-7903(02)00331-7</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>241</th>\n",
       "      <td>doi:10.1021/acs.est.9b06386</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>247</th>\n",
       "      <td>doi:10.1038/leu.2013.132</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>251</th>\n",
       "      <td>doi:10.1038/nature14425</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>254</th>\n",
       "      <td>doi:10.1038/nature18614</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>256</th>\n",
       "      <td>doi:10.1038/nature23883</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>257</th>\n",
       "      <td>doi:10.1038/nature25969</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>273</th>\n",
       "      <td>doi:10.1038/nmeth.4437</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>274</th>\n",
       "      <td>doi:10.1038/nn.4558</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>280</th>\n",
       "      <td>doi:10.1038/s41388-019-1103-0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>288</th>\n",
       "      <td>doi:10.1038/s41422-023-00797-0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>341</th>\n",
       "      <td>doi:10.1038/s41565-022-01096-2</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>357</th>\n",
       "      <td>doi:10.1038/s41588-022-01197-7</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>359</th>\n",
       "      <td>doi:10.1038/s41593-022-01199-y</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>433</th>\n",
       "      <td>doi:10.1080/03601234.2022.2115780</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>435</th>\n",
       "      <td>doi:10.1080/17435390.2020.1735552</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>446</th>\n",
       "      <td>doi:10.1089/zeb.2015.1115</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>448</th>\n",
       "      <td>doi:10.1093/biolre/iox160</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>468</th>\n",
       "      <td>doi:10.1093/toxsci/kfaa063</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>473</th>\n",
       "      <td>doi:10.1093/toxsci/kfad031</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>474</th>\n",
       "      <td>doi:10.1093/toxsci/kfad050</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>518</th>\n",
       "      <td>doi:10.1101/527341</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>519</th>\n",
       "      <td>doi:10.1101/710491</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>554</th>\n",
       "      <td>doi:10.1126/science.aad2197</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>555</th>\n",
       "      <td>doi:10.1126/science.aaf2679</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>563</th>\n",
       "      <td>doi:10.1126/science.aba9304</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>707</th>\n",
       "      <td>doi:10.1242/dev.202315</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>708</th>\n",
       "      <td>doi:10.1242/dev.202338</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>736</th>\n",
       "      <td>doi:10.1261/rna.029090.111</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>792</th>\n",
       "      <td>doi:10.15252/embj.2022112590</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>801</th>\n",
       "      <td>doi:10.1534/genetics.119.302365</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>822</th>\n",
       "      <td>doi:10.3389/fcell.2021.699796</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>840</th>\n",
       "      <td>doi:10.3389/fimmu.2019.02720</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "item type                                       doi  CitationRecord  \\\n",
       "6                            doi:10.1002/dvdy.24188             1.0   \n",
       "7                            doi:10.1002/dvdy.24600             1.0   \n",
       "13                           doi:10.1002/glia.24075             1.0   \n",
       "26                    doi:10.1007/s00424-017-2009-8             1.0   \n",
       "31                    doi:10.1007/s11427-020-1878-8             1.0   \n",
       "32                    doi:10.1007/s11427-021-2223-4             1.0   \n",
       "35                 doi:10.1016/bs.adgen.2016.04.004             1.0   \n",
       "39                doi:10.1016/j.aquatox.2013.10.006             1.0   \n",
       "42                doi:10.1016/j.aquatox.2019.105290             1.0   \n",
       "45                   doi:10.1016/j.bbrc.2015.03.128             1.0   \n",
       "46                   doi:10.1016/j.bbrc.2017.04.035             1.0   \n",
       "47                   doi:10.1016/j.bbrc.2021.02.122             1.0   \n",
       "106           doi:10.1016/j.chemosphere.2017.12.029             1.0   \n",
       "107           doi:10.1016/j.chemosphere.2020.126537             1.0   \n",
       "108           doi:10.1016/j.chemosphere.2021.133133             1.0   \n",
       "109           doi:10.1016/j.chemosphere.2022.137380             1.0   \n",
       "110           doi:10.1016/j.chemosphere.2023.138195             1.0   \n",
       "117                   doi:10.1016/j.dci.2014.07.022             1.0   \n",
       "119                   doi:10.1016/j.dci.2020.103922             1.0   \n",
       "148                doi:10.1016/j.devcel.2023.06.005             1.0   \n",
       "157                doi:10.1016/j.envpol.2017.12.023             1.0   \n",
       "161                   doi:10.1016/j.fct.2023.113788             1.0   \n",
       "162                   doi:10.1016/j.fsi.2010.08.022             1.0   \n",
       "180                   doi:10.1016/j.jgg.2022.04.005             1.0   \n",
       "182               doi:10.1016/j.jhazmat.2022.129382             1.0   \n",
       "183               doi:10.1016/j.jhazmat.2023.131714             1.0   \n",
       "184               doi:10.1016/j.jhazmat.2023.131722             1.0   \n",
       "185                doi:10.1016/j.matbio.2018.07.005             1.0   \n",
       "195                doi:10.1016/j.molimm.2010.08.011             1.0   \n",
       "196        doi:10.1016/j.neurobiolaging.2020.04.017             1.0   \n",
       "203                  doi:10.1016/j.scib.2017.01.008             1.0   \n",
       "207             doi:10.1016/j.scitotenv.2022.159069             1.0   \n",
       "209             doi:10.1016/j.scitotenv.2023.162901             1.0   \n",
       "224                 doi:10.1016/j.ydbio.2017.07.003             1.0   \n",
       "226                 doi:10.1016/j.ydbio.2019.06.005             1.0   \n",
       "240               doi:10.1016/s1055-7903(02)00331-7             1.0   \n",
       "241                     doi:10.1021/acs.est.9b06386             1.0   \n",
       "247                        doi:10.1038/leu.2013.132             1.0   \n",
       "251                         doi:10.1038/nature14425             1.0   \n",
       "254                         doi:10.1038/nature18614             1.0   \n",
       "256                         doi:10.1038/nature23883             1.0   \n",
       "257                         doi:10.1038/nature25969             1.0   \n",
       "273                          doi:10.1038/nmeth.4437             1.0   \n",
       "274                             doi:10.1038/nn.4558             1.0   \n",
       "280                   doi:10.1038/s41388-019-1103-0             1.0   \n",
       "288                  doi:10.1038/s41422-023-00797-0             1.0   \n",
       "341                  doi:10.1038/s41565-022-01096-2             1.0   \n",
       "357                  doi:10.1038/s41588-022-01197-7             1.0   \n",
       "359                  doi:10.1038/s41593-022-01199-y             1.0   \n",
       "433               doi:10.1080/03601234.2022.2115780             1.0   \n",
       "435               doi:10.1080/17435390.2020.1735552             1.0   \n",
       "446                       doi:10.1089/zeb.2015.1115             1.0   \n",
       "448                       doi:10.1093/biolre/iox160             1.0   \n",
       "468                      doi:10.1093/toxsci/kfaa063             1.0   \n",
       "473                      doi:10.1093/toxsci/kfad031             1.0   \n",
       "474                      doi:10.1093/toxsci/kfad050             1.0   \n",
       "518                              doi:10.1101/527341             1.0   \n",
       "519                              doi:10.1101/710491             1.0   \n",
       "554                     doi:10.1126/science.aad2197             1.0   \n",
       "555                     doi:10.1126/science.aaf2679             1.0   \n",
       "563                     doi:10.1126/science.aba9304             1.0   \n",
       "707                          doi:10.1242/dev.202315             1.0   \n",
       "708                          doi:10.1242/dev.202338             1.0   \n",
       "736                      doi:10.1261/rna.029090.111             1.0   \n",
       "792                    doi:10.15252/embj.2022112590             1.0   \n",
       "801                 doi:10.1534/genetics.119.302365             1.0   \n",
       "822                   doi:10.3389/fcell.2021.699796             1.0   \n",
       "840                    doi:10.3389/fimmu.2019.02720             1.0   \n",
       "\n",
       "item type  HTMLFullText  JATSFullText  PDFFullText  \n",
       "6                   0.0           0.0          0.0  \n",
       "7                   0.0           0.0          0.0  \n",
       "13                  0.0           0.0          0.0  \n",
       "26                  0.0           0.0          0.0  \n",
       "31                  0.0           0.0          0.0  \n",
       "32                  0.0           0.0          0.0  \n",
       "35                  0.0           0.0          0.0  \n",
       "39                  0.0           0.0          0.0  \n",
       "42                  0.0           0.0          0.0  \n",
       "45                  0.0           0.0          0.0  \n",
       "46                  0.0           0.0          0.0  \n",
       "47                  0.0           0.0          0.0  \n",
       "106                 0.0           0.0          0.0  \n",
       "107                 0.0           0.0          0.0  \n",
       "108                 0.0           0.0          0.0  \n",
       "109                 0.0           0.0          0.0  \n",
       "110                 0.0           0.0          0.0  \n",
       "117                 0.0           0.0          0.0  \n",
       "119                 0.0           0.0          0.0  \n",
       "148                 0.0           0.0          0.0  \n",
       "157                 0.0           0.0          0.0  \n",
       "161                 0.0           0.0          0.0  \n",
       "162                 0.0           0.0          0.0  \n",
       "180                 0.0           0.0          0.0  \n",
       "182                 0.0           0.0          0.0  \n",
       "183                 0.0           0.0          0.0  \n",
       "184                 0.0           0.0          0.0  \n",
       "185                 0.0           0.0          0.0  \n",
       "195                 0.0           0.0          0.0  \n",
       "196                 0.0           0.0          0.0  \n",
       "203                 0.0           0.0          0.0  \n",
       "207                 0.0           0.0          0.0  \n",
       "209                 0.0           0.0          0.0  \n",
       "224                 0.0           0.0          0.0  \n",
       "226                 0.0           0.0          0.0  \n",
       "240                 0.0           0.0          0.0  \n",
       "241                 0.0           0.0          0.0  \n",
       "247                 0.0           0.0          0.0  \n",
       "251                 0.0           0.0          0.0  \n",
       "254                 0.0           0.0          0.0  \n",
       "256                 0.0           0.0          0.0  \n",
       "257                 0.0           0.0          0.0  \n",
       "273                 0.0           0.0          0.0  \n",
       "274                 0.0           0.0          0.0  \n",
       "280                 0.0           0.0          0.0  \n",
       "288                 0.0           0.0          0.0  \n",
       "341                 0.0           0.0          0.0  \n",
       "357                 0.0           0.0          0.0  \n",
       "359                 0.0           0.0          0.0  \n",
       "433                 0.0           0.0          0.0  \n",
       "435                 0.0           0.0          0.0  \n",
       "446                 0.0           0.0          0.0  \n",
       "448                 0.0           0.0          0.0  \n",
       "468                 0.0           0.0          0.0  \n",
       "473                 0.0           0.0          0.0  \n",
       "474                 0.0           0.0          0.0  \n",
       "518                 0.0           0.0          0.0  \n",
       "519                 0.0           0.0          0.0  \n",
       "554                 0.0           0.0          0.0  \n",
       "555                 0.0           0.0          0.0  \n",
       "563                 0.0           0.0          0.0  \n",
       "707                 0.0           0.0          0.0  \n",
       "708                 0.0           0.0          0.0  \n",
       "736                 0.0           0.0          0.0  \n",
       "792                 0.0           0.0          0.0  \n",
       "801                 0.0           0.0          0.0  \n",
       "822                 0.0           0.0          0.0  \n",
       "840                 0.0           0.0          0.0  "
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# show all rows in dataframe\n",
    "pd.set_option('display.max_rows', None)\n",
    "\n",
    "piv_df = df.pivot_table(index=['doi'], \n",
    "                        columns='item type', values='collection name', \n",
    "                        aggfunc=lambda x: len(x.unique()))\n",
    "piv_df.fillna(0, inplace=True)\n",
    "piv_df = piv_df.reset_index(drop=False).sort_values(by='doi')\n",
    "print(len(piv_df[(piv_df['JATSFullText']==0.0) & (piv_df['PDFFullText']==0.0)]))\n",
    "piv_df[(piv_df['JATSFullText']==0.0) & (piv_df['PDFFullText']==0.0)]\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Tests + Checks \n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Agent tool selection + execution + interpretation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cb.agent_executor.invoke({'input':'Hi who are you and what can you do?'})\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Run MetaData Extraction Chain over listed papers\n",
    "\n",
    "Here, we run various versions of the metadata extraction tool to examine performance over the cryoet dataset. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import local_resources.data_files.cryoet_portal_metadata as cryoet_portal_metadata\n",
    "str(files(cryoet_portal_metadata).joinpath('temp'))[0:-4]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import local_resources.data_files.cryoet_portal_metadata as cryoet_portal_metadata\n",
    "\n",
    "# Get the metadata extraction tool\n",
    "t2 = [t for t in test_tk.get_tools() if isinstance(t, MetadataExtraction_EverythingEverywhere_Tool)][0]\n",
    "\n",
    "# Hack to get the path to the metadata directory as a string\n",
    "metadata_dir = str(files(cryoet_portal_metadata).joinpath('temp'))[0:-4]\n",
    "\n",
    "# Compile the answers from the metadata directory\n",
    "t2.compile_answers('cryoet', metadata_dir)\n",
    "\n",
    "# Create a dataframe to store previously extracted metadata\n",
    "df = pd.DataFrame()\n",
    "for d in [d for d_id in dois for d in dois[d_id]]:\n",
    "    item_types = set()\n",
    "    d_id = 'doi:'+d\n",
    "    df = pd.concat([df, t2.build_report(d_id, 'cryoet')]) \n",
    "     \n",
    "# Iterate over papers to run the metadata extraction tool\n",
    "for d in [d for d_id in dois for d in dois[d_id]]:\n",
    "    item_types = set()\n",
    "    d_id = 'doi:'+d\n",
    "\n",
    "    # Skip if the doi is already in the database\n",
    "    if d_id in df.doi.unique():\n",
    "        continue\n",
    "\n",
    "    # Run the metadata extraction tool on the doi\n",
    "    t2.run(tool_input={'paper_id': d_id, 'extraction_type': 'cryoet'})\n",
    "\n",
    "    # Add the results to the dataframe\n",
    "    df = pd.concat([df, t2.build_report(d_id, 'cryoet')]) \n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t2 = [t for t in test_tk.get_tools() if isinstance(t, MetadataExtraction_EverythingEverywhere_Tool)][0]\n",
    "for i, row in empiar_gs_df.iterrows():\n",
    "    d_id = 'doi:'+row.doi\n",
    "    if d_id in df.doi.unique():\n",
    "        continue\n",
    "    t2.run(tool_input={'paper_id': d_id, 'extraction_type': 'cryoet'})\n",
    "    df = pd.concat([df, t2.build_report(d_id, 'cryoet')])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# USE WITH CAUTION - this will delete all extracted metadata notes in the database\n",
    "# clear all notes across papers listed in `dois` list\n",
    "for d in [d for d_id in dois for d in dois[d_id]]:\n",
    "    d_id = 'doi:'+d\n",
    "    e = ldb.session.query(SKE).filter(SKE.id==d_id).first()\n",
    "    notes_to_delete = []\n",
    "    for n in ldb.read_notes_about_x(e):\n",
    "        notes_to_delete.append(n.id)\n",
    "    for n in notes_to_delete:\n",
    "        ldb.delete_note(n)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Protocol Modeling + Extraction"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ldb = Ceifns_LiteratureDb(loc=loc, name=db_name)\n",
    "slm = ChatOllama(model='stablelm-zephyr') \n",
    "llm = ChatOllama(model='mixtral:instruct') \n",
    "llm2 = ChatOpenAI(model='gpt-4-1106-preview') \n",
    "d = (\"This tool attempts to draw a protocol design from the description of a scientific paper.\")\n",
    "t = ProcotolExtractionTool(db=ldb, llm=llm2, description=d)\n",
    "t.run(tool_input={'paper_id': 'doi:10.1101/2022.04.12.488077', 'extraction_type': 'cryoet'})"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "python3",
   "language": "python",
   "name": "python3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
